function ip(e,t){for(var i=0;i<t.length;i++){const o=t[i];if(typeof o!="string"&&!Array.isArray(o)){for(const n in o)if(n!=="default"&&!(n in e)){const a=Object.getOwnPropertyDescriptor(o,n);a&&Object.defineProperty(e,n,a.get?a:{enumerable:!0,get:()=>o[n]})}}}return Object.freeze(Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}))}(function(){const t=document.createElement("link").relList;if(t&&t.supports&&t.supports("modulepreload"))return;for(const n of document.querySelectorAll('link[rel="modulepreload"]'))o(n);new MutationObserver(n=>{for(const a of n)if(a.type==="childList")for(const s of a.addedNodes)s.tagName==="LINK"&&s.rel==="modulepreload"&&o(s)}).observe(document,{childList:!0,subtree:!0});function i(n){const a={};return n.integrity&&(a.integrity=n.integrity),n.referrerPolicy&&(a.referrerPolicy=n.referrerPolicy),n.crossOrigin==="use-credentials"?a.credentials="include":n.crossOrigin==="anonymous"?a.credentials="omit":a.credentials="same-origin",a}function o(n){if(n.ep)return;n.ep=!0;const a=i(n);fetch(n.href,a)}})();function op(e){return e&&e.__esModule&&Object.prototype.hasOwnProperty.call(e,"default")?e.default:e}var Al={exports:{}},fn={},Cl={exports:{}},N={};/**
 * @license React
 * react.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */var lo=Symbol.for("react.element"),np=Symbol.for("react.portal"),ap=Symbol.for("react.fragment"),rp=Symbol.for("react.strict_mode"),sp=Symbol.for("react.profiler"),lp=Symbol.for("react.provider"),cp=Symbol.for("react.context"),dp=Symbol.for("react.forward_ref"),pp=Symbol.for("react.suspense"),mp=Symbol.for("react.memo"),up=Symbol.for("react.lazy"),ls=Symbol.iterator;function hp(e){return e===null||typeof e!="object"?null:(e=ls&&e[ls]||e["@@iterator"],typeof e=="function"?e:null)}var Il={isMounted:function(){return!1},enqueueForceUpdate:function(){},enqueueReplaceState:function(){},enqueueSetState:function(){}},Ml=Object.assign,Gl={};function ui(e,t,i){this.props=e,this.context=t,this.refs=Gl,this.updater=i||Il}ui.prototype.isReactComponent={};ui.prototype.setState=function(e,t){if(typeof e!="object"&&typeof e!="function"&&e!=null)throw Error("setState(...): takes an object of state variables to update or a function which returns an object of state variables.");this.updater.enqueueSetState(this,e,t,"setState")};ui.prototype.forceUpdate=function(e){this.updater.enqueueForceUpdate(this,e,"forceUpdate")};function Jl(){}Jl.prototype=ui.prototype;function sr(e,t,i){this.props=e,this.context=t,this.refs=Gl,this.updater=i||Il}var lr=sr.prototype=new Jl;lr.constructor=sr;Ml(lr,ui.prototype);lr.isPureReactComponent=!0;var cs=Array.isArray,Nl=Object.prototype.hasOwnProperty,cr={current:null},El={key:!0,ref:!0,__self:!0,__source:!0};function jl(e,t,i){var o,n={},a=null,s=null;if(t!=null)for(o in t.ref!==void 0&&(s=t.ref),t.key!==void 0&&(a=""+t.key),t)Nl.call(t,o)&&!El.hasOwnProperty(o)&&(n[o]=t[o]);var l=arguments.length-2;if(l===1)n.children=i;else if(1<l){for(var c=Array(l),p=0;p<l;p++)c[p]=arguments[p+2];n.children=c}if(e&&e.defaultProps)for(o in l=e.defaultProps,l)n[o]===void 0&&(n[o]=l[o]);return{$$typeof:lo,type:e,key:a,ref:s,props:n,_owner:cr.current}}function yp(e,t){return{$$typeof:lo,type:e.type,key:t,ref:e.ref,props:e.props,_owner:e._owner}}function dr(e){return typeof e=="object"&&e!==null&&e.$$typeof===lo}function gp(e){var t={"=":"=0",":":"=2"};return"$"+e.replace(/[=:]/g,function(i){return t[i]})}var ds=/\/+/g;function Ln(e,t){return typeof e=="object"&&e!==null&&e.key!=null?gp(""+e.key):t.toString(36)}function Eo(e,t,i,o,n){var a=typeof e;(a==="undefined"||a==="boolean")&&(e=null);var s=!1;if(e===null)s=!0;else switch(a){case"string":case"number":s=!0;break;case"object":switch(e.$$typeof){case lo:case np:s=!0}}if(s)return s=e,n=n(s),e=o===""?"."+Ln(s,0):o,cs(n)?(i="",e!=null&&(i=e.replace(ds,"$&/")+"/"),Eo(n,t,i,"",function(p){return p})):n!=null&&(dr(n)&&(n=yp(n,i+(!n.key||s&&s.key===n.key?"":(""+n.key).replace(ds,"$&/")+"/")+e)),t.push(n)),1;if(s=0,o=o===""?".":o+":",cs(e))for(var l=0;l<e.length;l++){a=e[l];var c=o+Ln(a,l);s+=Eo(a,t,i,c,n)}else if(c=hp(e),typeof c=="function")for(e=c.call(e),l=0;!(a=e.next()).done;)a=a.value,c=o+Ln(a,l++),s+=Eo(a,t,i,c,n);else if(a==="object")throw t=String(e),Error("Objects are not valid as a React child (found: "+(t==="[object Object]"?"object with keys {"+Object.keys(e).join(", ")+"}":t)+"). If you meant to render a collection of children, use an array instead.");return s}function go(e,t,i){if(e==null)return e;var o=[],n=0;return Eo(e,o,"","",function(a){return t.call(i,a,n++)}),o}function fp(e){if(e._status===-1){var t=e._result;t=t(),t.then(function(i){(e._status===0||e._status===-1)&&(e._status=1,e._result=i)},function(i){(e._status===0||e._status===-1)&&(e._status=2,e._result=i)}),e._status===-1&&(e._status=0,e._result=t)}if(e._status===1)return e._result.default;throw e._result}var de={current:null},jo={transition:null},vp={ReactCurrentDispatcher:de,ReactCurrentBatchConfig:jo,ReactCurrentOwner:cr};function zl(){throw Error("act(...) is not supported in production builds of React.")}N.Children={map:go,forEach:function(e,t,i){go(e,function(){t.apply(this,arguments)},i)},count:function(e){var t=0;return go(e,function(){t++}),t},toArray:function(e){return go(e,function(t){return t})||[]},only:function(e){if(!dr(e))throw Error("React.Children.only expected to receive a single React element child.");return e}};N.Component=ui;N.Fragment=ap;N.Profiler=sp;N.PureComponent=sr;N.StrictMode=rp;N.Suspense=pp;N.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED=vp;N.act=zl;N.cloneElement=function(e,t,i){if(e==null)throw Error("React.cloneElement(...): The argument must be a React element, but you passed "+e+".");var o=Ml({},e.props),n=e.key,a=e.ref,s=e._owner;if(t!=null){if(t.ref!==void 0&&(a=t.ref,s=cr.current),t.key!==void 0&&(n=""+t.key),e.type&&e.type.defaultProps)var l=e.type.defaultProps;for(c in t)Nl.call(t,c)&&!El.hasOwnProperty(c)&&(o[c]=t[c]===void 0&&l!==void 0?l[c]:t[c])}var c=arguments.length-2;if(c===1)o.children=i;else if(1<c){l=Array(c);for(var p=0;p<c;p++)l[p]=arguments[p+2];o.children=l}return{$$typeof:lo,type:e.type,key:n,ref:a,props:o,_owner:s}};N.createContext=function(e){return e={$$typeof:cp,_currentValue:e,_currentValue2:e,_threadCount:0,Provider:null,Consumer:null,_defaultValue:null,_globalName:null},e.Provider={$$typeof:lp,_context:e},e.Consumer=e};N.createElement=jl;N.createFactory=function(e){var t=jl.bind(null,e);return t.type=e,t};N.createRef=function(){return{current:null}};N.forwardRef=function(e){return{$$typeof:dp,render:e}};N.isValidElement=dr;N.lazy=function(e){return{$$typeof:up,_payload:{_status:-1,_result:e},_init:fp}};N.memo=function(e,t){return{$$typeof:mp,type:e,compare:t===void 0?null:t}};N.startTransition=function(e){var t=jo.transition;jo.transition={};try{e()}finally{jo.transition=t}};N.unstable_act=zl;N.useCallback=function(e,t){return de.current.useCallback(e,t)};N.useContext=function(e){return de.current.useContext(e)};N.useDebugValue=function(){};N.useDeferredValue=function(e){return de.current.useDeferredValue(e)};N.useEffect=function(e,t){return de.current.useEffect(e,t)};N.useId=function(){return de.current.useId()};N.useImperativeHandle=function(e,t,i){return de.current.useImperativeHandle(e,t,i)};N.useInsertionEffect=function(e,t){return de.current.useInsertionEffect(e,t)};N.useLayoutEffect=function(e,t){return de.current.useLayoutEffect(e,t)};N.useMemo=function(e,t){return de.current.useMemo(e,t)};N.useReducer=function(e,t,i){return de.current.useReducer(e,t,i)};N.useRef=function(e){return de.current.useRef(e)};N.useState=function(e){return de.current.useState(e)};N.useSyncExternalStore=function(e,t,i){return de.current.useSyncExternalStore(e,t,i)};N.useTransition=function(){return de.current.useTransition()};N.version="18.3.1";Cl.exports=N;var T=Cl.exports;const bp=op(T),wp=ip({__proto__:null,default:bp},[T]);/**
 * @license React
 * react-jsx-runtime.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */var Tp=T,Sp=Symbol.for("react.element"),Pp=Symbol.for("react.fragment"),xp=Object.prototype.hasOwnProperty,kp=Tp.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.ReactCurrentOwner,Ap={key:!0,ref:!0,__self:!0,__source:!0};function Bl(e,t,i){var o,n={},a=null,s=null;i!==void 0&&(a=""+i),t.key!==void 0&&(a=""+t.key),t.ref!==void 0&&(s=t.ref);for(o in t)xp.call(t,o)&&!Ap.hasOwnProperty(o)&&(n[o]=t[o]);if(e&&e.defaultProps)for(o in t=e.defaultProps,t)n[o]===void 0&&(n[o]=t[o]);return{$$typeof:Sp,type:e,key:a,ref:s,props:n,_owner:kp.current}}fn.Fragment=Pp;fn.jsx=Bl;fn.jsxs=Bl;Al.exports=fn;var r=Al.exports,Dl={exports:{}},Te={},Ll={exports:{}},Hl={};/**
 * @license React
 * scheduler.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */(function(e){function t(I,G){var J=I.length;I.push(G);e:for(;0<J;){var F=J-1>>>1,X=I[F];if(0<n(X,G))I[F]=G,I[J]=X,J=F;else break e}}function i(I){return I.length===0?null:I[0]}function o(I){if(I.length===0)return null;var G=I[0],J=I.pop();if(J!==G){I[0]=J;e:for(var F=0,X=I.length,ho=X>>>1;F<ho;){var xt=2*(F+1)-1,Dn=I[xt],kt=xt+1,yo=I[kt];if(0>n(Dn,J))kt<X&&0>n(yo,Dn)?(I[F]=yo,I[kt]=J,F=kt):(I[F]=Dn,I[xt]=J,F=xt);else if(kt<X&&0>n(yo,J))I[F]=yo,I[kt]=J,F=kt;else break e}}return G}function n(I,G){var J=I.sortIndex-G.sortIndex;return J!==0?J:I.id-G.id}if(typeof performance=="object"&&typeof performance.now=="function"){var a=performance;e.unstable_now=function(){return a.now()}}else{var s=Date,l=s.now();e.unstable_now=function(){return s.now()-l}}var c=[],p=[],y=1,u=null,g=3,b=!1,w=!1,v=!1,x=typeof setTimeout=="function"?setTimeout:null,h=typeof clearTimeout=="function"?clearTimeout:null,d=typeof setImmediate<"u"?setImmediate:null;typeof navigator<"u"&&navigator.scheduling!==void 0&&navigator.scheduling.isInputPending!==void 0&&navigator.scheduling.isInputPending.bind(navigator.scheduling);function m(I){for(var G=i(p);G!==null;){if(G.callback===null)o(p);else if(G.startTime<=I)o(p),G.sortIndex=G.expirationTime,t(c,G);else break;G=i(p)}}function f(I){if(v=!1,m(I),!w)if(i(c)!==null)w=!0,zn(S);else{var G=i(p);G!==null&&Bn(f,G.startTime-I)}}function S(I,G){w=!1,v&&(v=!1,h(M),M=-1),b=!0;var J=g;try{for(m(G),u=i(c);u!==null&&(!(u.expirationTime>G)||I&&!Me());){var F=u.callback;if(typeof F=="function"){u.callback=null,g=u.priorityLevel;var X=F(u.expirationTime<=G);G=e.unstable_now(),typeof X=="function"?u.callback=X:u===i(c)&&o(c),m(G)}else o(c);u=i(c)}if(u!==null)var ho=!0;else{var xt=i(p);xt!==null&&Bn(f,xt.startTime-G),ho=!1}return ho}finally{u=null,g=J,b=!1}}var k=!1,A=null,M=-1,O=5,E=-1;function Me(){return!(e.unstable_now()-E<O)}function wi(){if(A!==null){var I=e.unstable_now();E=I;var G=!0;try{G=A(!0,I)}finally{G?Ti():(k=!1,A=null)}}else k=!1}var Ti;if(typeof d=="function")Ti=function(){d(wi)};else if(typeof MessageChannel<"u"){var ss=new MessageChannel,tp=ss.port2;ss.port1.onmessage=wi,Ti=function(){tp.postMessage(null)}}else Ti=function(){x(wi,0)};function zn(I){A=I,k||(k=!0,Ti())}function Bn(I,G){M=x(function(){I(e.unstable_now())},G)}e.unstable_IdlePriority=5,e.unstable_ImmediatePriority=1,e.unstable_LowPriority=4,e.unstable_NormalPriority=3,e.unstable_Profiling=null,e.unstable_UserBlockingPriority=2,e.unstable_cancelCallback=function(I){I.callback=null},e.unstable_continueExecution=function(){w||b||(w=!0,zn(S))},e.unstable_forceFrameRate=function(I){0>I||125<I?console.error("forceFrameRate takes a positive int between 0 and 125, forcing frame rates higher than 125 fps is not supported"):O=0<I?Math.floor(1e3/I):5},e.unstable_getCurrentPriorityLevel=function(){return g},e.unstable_getFirstCallbackNode=function(){return i(c)},e.unstable_next=function(I){switch(g){case 1:case 2:case 3:var G=3;break;default:G=g}var J=g;g=G;try{return I()}finally{g=J}},e.unstable_pauseExecution=function(){},e.unstable_requestPaint=function(){},e.unstable_runWithPriority=function(I,G){switch(I){case 1:case 2:case 3:case 4:case 5:break;default:I=3}var J=g;g=I;try{return G()}finally{g=J}},e.unstable_scheduleCallback=function(I,G,J){var F=e.unstable_now();switch(typeof J=="object"&&J!==null?(J=J.delay,J=typeof J=="number"&&0<J?F+J:F):J=F,I){case 1:var X=-1;break;case 2:X=250;break;case 5:X=1073741823;break;case 4:X=1e4;break;default:X=5e3}return X=J+X,I={id:y++,callback:G,priorityLevel:I,startTime:J,expirationTime:X,sortIndex:-1},J>F?(I.sortIndex=J,t(p,I),i(c)===null&&I===i(p)&&(v?(h(M),M=-1):v=!0,Bn(f,J-F))):(I.sortIndex=X,t(c,I),w||b||(w=!0,zn(S))),I},e.unstable_shouldYield=Me,e.unstable_wrapCallback=function(I){var G=g;return function(){var J=g;g=G;try{return I.apply(this,arguments)}finally{g=J}}}})(Hl);Ll.exports=Hl;var Cp=Ll.exports;/**
 * @license React
 * react-dom.production.min.js
 *
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */var Ip=T,we=Cp;function P(e){for(var t="https://reactjs.org/docs/error-decoder.html?invariant="+e,i=1;i<arguments.length;i++)t+="&args[]="+encodeURIComponent(arguments[i]);return"Minified React error #"+e+"; visit "+t+" for the full message or use the non-minified dev environment for full errors and additional helpful warnings."}var Rl=new Set,qi={};function Lt(e,t){ri(e,t),ri(e+"Capture",t)}function ri(e,t){for(qi[e]=t,e=0;e<t.length;e++)Rl.add(t[e])}var _e=!(typeof window>"u"||typeof window.document>"u"||typeof window.document.createElement>"u"),ma=Object.prototype.hasOwnProperty,Mp=/^[:A-Z_a-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD][:A-Z_a-z\u00C0-\u00D6\u00D8-\u00F6\u00F8-\u02FF\u0370-\u037D\u037F-\u1FFF\u200C-\u200D\u2070-\u218F\u2C00-\u2FEF\u3001-\uD7FF\uF900-\uFDCF\uFDF0-\uFFFD\-.0-9\u00B7\u0300-\u036F\u203F-\u2040]*$/,ps={},ms={};function Gp(e){return ma.call(ms,e)?!0:ma.call(ps,e)?!1:Mp.test(e)?ms[e]=!0:(ps[e]=!0,!1)}function Jp(e,t,i,o){if(i!==null&&i.type===0)return!1;switch(typeof t){case"function":case"symbol":return!0;case"boolean":return o?!1:i!==null?!i.acceptsBooleans:(e=e.toLowerCase().slice(0,5),e!=="data-"&&e!=="aria-");default:return!1}}function Np(e,t,i,o){if(t===null||typeof t>"u"||Jp(e,t,i,o))return!0;if(o)return!1;if(i!==null)switch(i.type){case 3:return!t;case 4:return t===!1;case 5:return isNaN(t);case 6:return isNaN(t)||1>t}return!1}function pe(e,t,i,o,n,a,s){this.acceptsBooleans=t===2||t===3||t===4,this.attributeName=o,this.attributeNamespace=n,this.mustUseProperty=i,this.propertyName=e,this.type=t,this.sanitizeURL=a,this.removeEmptyString=s}var oe={};"children dangerouslySetInnerHTML defaultValue defaultChecked innerHTML suppressContentEditableWarning suppressHydrationWarning style".split(" ").forEach(function(e){oe[e]=new pe(e,0,!1,e,null,!1,!1)});[["acceptCharset","accept-charset"],["className","class"],["htmlFor","for"],["httpEquiv","http-equiv"]].forEach(function(e){var t=e[0];oe[t]=new pe(t,1,!1,e[1],null,!1,!1)});["contentEditable","draggable","spellCheck","value"].forEach(function(e){oe[e]=new pe(e,2,!1,e.toLowerCase(),null,!1,!1)});["autoReverse","externalResourcesRequired","focusable","preserveAlpha"].forEach(function(e){oe[e]=new pe(e,2,!1,e,null,!1,!1)});"allowFullScreen async autoFocus autoPlay controls default defer disabled disablePictureInPicture disableRemotePlayback formNoValidate hidden loop noModule noValidate open playsInline readOnly required reversed scoped seamless itemScope".split(" ").forEach(function(e){oe[e]=new pe(e,3,!1,e.toLowerCase(),null,!1,!1)});["checked","multiple","muted","selected"].forEach(function(e){oe[e]=new pe(e,3,!0,e,null,!1,!1)});["capture","download"].forEach(function(e){oe[e]=new pe(e,4,!1,e,null,!1,!1)});["cols","rows","size","span"].forEach(function(e){oe[e]=new pe(e,6,!1,e,null,!1,!1)});["rowSpan","start"].forEach(function(e){oe[e]=new pe(e,5,!1,e.toLowerCase(),null,!1,!1)});var pr=/[\-:]([a-z])/g;function mr(e){return e[1].toUpperCase()}"accent-height alignment-baseline arabic-form baseline-shift cap-height clip-path clip-rule color-interpolation color-interpolation-filters color-profile color-rendering dominant-baseline enable-background fill-opacity fill-rule flood-color flood-opacity font-family font-size font-size-adjust font-stretch font-style font-variant font-weight glyph-name glyph-orientation-horizontal glyph-orientation-vertical horiz-adv-x horiz-origin-x image-rendering letter-spacing lighting-color marker-end marker-mid marker-start overline-position overline-thickness paint-order panose-1 pointer-events rendering-intent shape-rendering stop-color stop-opacity strikethrough-position strikethrough-thickness stroke-dasharray stroke-dashoffset stroke-linecap stroke-linejoin stroke-miterlimit stroke-opacity stroke-width text-anchor text-decoration text-rendering underline-position underline-thickness unicode-bidi unicode-range units-per-em v-alphabetic v-hanging v-ideographic v-mathematical vector-effect vert-adv-y vert-origin-x vert-origin-y word-spacing writing-mode xmlns:xlink x-height".split(" ").forEach(function(e){var t=e.replace(pr,mr);oe[t]=new pe(t,1,!1,e,null,!1,!1)});"xlink:actuate xlink:arcrole xlink:role xlink:show xlink:title xlink:type".split(" ").forEach(function(e){var t=e.replace(pr,mr);oe[t]=new pe(t,1,!1,e,"http://www.w3.org/1999/xlink",!1,!1)});["xml:base","xml:lang","xml:space"].forEach(function(e){var t=e.replace(pr,mr);oe[t]=new pe(t,1,!1,e,"http://www.w3.org/XML/1998/namespace",!1,!1)});["tabIndex","crossOrigin"].forEach(function(e){oe[e]=new pe(e,1,!1,e.toLowerCase(),null,!1,!1)});oe.xlinkHref=new pe("xlinkHref",1,!1,"xlink:href","http://www.w3.org/1999/xlink",!0,!1);["src","href","action","formAction"].forEach(function(e){oe[e]=new pe(e,1,!1,e.toLowerCase(),null,!0,!0)});function ur(e,t,i,o){var n=oe.hasOwnProperty(t)?oe[t]:null;(n!==null?n.type!==0:o||!(2<t.length)||t[0]!=="o"&&t[0]!=="O"||t[1]!=="n"&&t[1]!=="N")&&(Np(t,i,n,o)&&(i=null),o||n===null?Gp(t)&&(i===null?e.removeAttribute(t):e.setAttribute(t,""+i)):n.mustUseProperty?e[n.propertyName]=i===null?n.type===3?!1:"":i:(t=n.attributeName,o=n.attributeNamespace,i===null?e.removeAttribute(t):(n=n.type,i=n===3||n===4&&i===!0?"":""+i,o?e.setAttributeNS(o,t,i):e.setAttribute(t,i))))}var Qe=Ip.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED,fo=Symbol.for("react.element"),Ut=Symbol.for("react.portal"),Wt=Symbol.for("react.fragment"),hr=Symbol.for("react.strict_mode"),ua=Symbol.for("react.profiler"),Ul=Symbol.for("react.provider"),Wl=Symbol.for("react.context"),yr=Symbol.for("react.forward_ref"),ha=Symbol.for("react.suspense"),ya=Symbol.for("react.suspense_list"),gr=Symbol.for("react.memo"),tt=Symbol.for("react.lazy"),ql=Symbol.for("react.offscreen"),us=Symbol.iterator;function Si(e){return e===null||typeof e!="object"?null:(e=us&&e[us]||e["@@iterator"],typeof e=="function"?e:null)}var W=Object.assign,Hn;function Gi(e){if(Hn===void 0)try{throw Error()}catch(i){var t=i.stack.trim().match(/\n( *(at )?)/);Hn=t&&t[1]||""}return`
`+Hn+e}var Rn=!1;function Un(e,t){if(!e||Rn)return"";Rn=!0;var i=Error.prepareStackTrace;Error.prepareStackTrace=void 0;try{if(t)if(t=function(){throw Error()},Object.defineProperty(t.prototype,"props",{set:function(){throw Error()}}),typeof Reflect=="object"&&Reflect.construct){try{Reflect.construct(t,[])}catch(p){var o=p}Reflect.construct(e,[],t)}else{try{t.call()}catch(p){o=p}e.call(t.prototype)}else{try{throw Error()}catch(p){o=p}e()}}catch(p){if(p&&o&&typeof p.stack=="string"){for(var n=p.stack.split(`
`),a=o.stack.split(`
`),s=n.length-1,l=a.length-1;1<=s&&0<=l&&n[s]!==a[l];)l--;for(;1<=s&&0<=l;s--,l--)if(n[s]!==a[l]){if(s!==1||l!==1)do if(s--,l--,0>l||n[s]!==a[l]){var c=`
`+n[s].replace(" at new "," at ");return e.displayName&&c.includes("<anonymous>")&&(c=c.replace("<anonymous>",e.displayName)),c}while(1<=s&&0<=l);break}}}finally{Rn=!1,Error.prepareStackTrace=i}return(e=e?e.displayName||e.name:"")?Gi(e):""}function Ep(e){switch(e.tag){case 5:return Gi(e.type);case 16:return Gi("Lazy");case 13:return Gi("Suspense");case 19:return Gi("SuspenseList");case 0:case 2:case 15:return e=Un(e.type,!1),e;case 11:return e=Un(e.type.render,!1),e;case 1:return e=Un(e.type,!0),e;default:return""}}function ga(e){if(e==null)return null;if(typeof e=="function")return e.displayName||e.name||null;if(typeof e=="string")return e;switch(e){case Wt:return"Fragment";case Ut:return"Portal";case ua:return"Profiler";case hr:return"StrictMode";case ha:return"Suspense";case ya:return"SuspenseList"}if(typeof e=="object")switch(e.$$typeof){case Wl:return(e.displayName||"Context")+".Consumer";case Ul:return(e._context.displayName||"Context")+".Provider";case yr:var t=e.render;return e=e.displayName,e||(e=t.displayName||t.name||"",e=e!==""?"ForwardRef("+e+")":"ForwardRef"),e;case gr:return t=e.displayName||null,t!==null?t:ga(e.type)||"Memo";case tt:t=e._payload,e=e._init;try{return ga(e(t))}catch{}}return null}function jp(e){var t=e.type;switch(e.tag){case 24:return"Cache";case 9:return(t.displayName||"Context")+".Consumer";case 10:return(t._context.displayName||"Context")+".Provider";case 18:return"DehydratedFragment";case 11:return e=t.render,e=e.displayName||e.name||"",t.displayName||(e!==""?"ForwardRef("+e+")":"ForwardRef");case 7:return"Fragment";case 5:return t;case 4:return"Portal";case 3:return"Root";case 6:return"Text";case 16:return ga(t);case 8:return t===hr?"StrictMode":"Mode";case 22:return"Offscreen";case 12:return"Profiler";case 21:return"Scope";case 13:return"Suspense";case 19:return"SuspenseList";case 25:return"TracingMarker";case 1:case 0:case 17:case 2:case 14:case 15:if(typeof t=="function")return t.displayName||t.name||null;if(typeof t=="string")return t}return null}function vt(e){switch(typeof e){case"boolean":case"number":case"string":case"undefined":return e;case"object":return e;default:return""}}function Ol(e){var t=e.type;return(e=e.nodeName)&&e.toLowerCase()==="input"&&(t==="checkbox"||t==="radio")}function zp(e){var t=Ol(e)?"checked":"value",i=Object.getOwnPropertyDescriptor(e.constructor.prototype,t),o=""+e[t];if(!e.hasOwnProperty(t)&&typeof i<"u"&&typeof i.get=="function"&&typeof i.set=="function"){var n=i.get,a=i.set;return Object.defineProperty(e,t,{configurable:!0,get:function(){return n.call(this)},set:function(s){o=""+s,a.call(this,s)}}),Object.defineProperty(e,t,{enumerable:i.enumerable}),{getValue:function(){return o},setValue:function(s){o=""+s},stopTracking:function(){e._valueTracker=null,delete e[t]}}}}function vo(e){e._valueTracker||(e._valueTracker=zp(e))}function Fl(e){if(!e)return!1;var t=e._valueTracker;if(!t)return!0;var i=t.getValue(),o="";return e&&(o=Ol(e)?e.checked?"true":"false":e.value),e=o,e!==i?(t.setValue(e),!0):!1}function Fo(e){if(e=e||(typeof document<"u"?document:void 0),typeof e>"u")return null;try{return e.activeElement||e.body}catch{return e.body}}function fa(e,t){var i=t.checked;return W({},t,{defaultChecked:void 0,defaultValue:void 0,value:void 0,checked:i??e._wrapperState.initialChecked})}function hs(e,t){var i=t.defaultValue==null?"":t.defaultValue,o=t.checked!=null?t.checked:t.defaultChecked;i=vt(t.value!=null?t.value:i),e._wrapperState={initialChecked:o,initialValue:i,controlled:t.type==="checkbox"||t.type==="radio"?t.checked!=null:t.value!=null}}function Zl(e,t){t=t.checked,t!=null&&ur(e,"checked",t,!1)}function va(e,t){Zl(e,t);var i=vt(t.value),o=t.type;if(i!=null)o==="number"?(i===0&&e.value===""||e.value!=i)&&(e.value=""+i):e.value!==""+i&&(e.value=""+i);else if(o==="submit"||o==="reset"){e.removeAttribute("value");return}t.hasOwnProperty("value")?ba(e,t.type,i):t.hasOwnProperty("defaultValue")&&ba(e,t.type,vt(t.defaultValue)),t.checked==null&&t.defaultChecked!=null&&(e.defaultChecked=!!t.defaultChecked)}function ys(e,t,i){if(t.hasOwnProperty("value")||t.hasOwnProperty("defaultValue")){var o=t.type;if(!(o!=="submit"&&o!=="reset"||t.value!==void 0&&t.value!==null))return;t=""+e._wrapperState.initialValue,i||t===e.value||(e.value=t),e.defaultValue=t}i=e.name,i!==""&&(e.name=""),e.defaultChecked=!!e._wrapperState.initialChecked,i!==""&&(e.name=i)}function ba(e,t,i){(t!=="number"||Fo(e.ownerDocument)!==e)&&(i==null?e.defaultValue=""+e._wrapperState.initialValue:e.defaultValue!==""+i&&(e.defaultValue=""+i))}var Ji=Array.isArray;function Yt(e,t,i,o){if(e=e.options,t){t={};for(var n=0;n<i.length;n++)t["$"+i[n]]=!0;for(i=0;i<e.length;i++)n=t.hasOwnProperty("$"+e[i].value),e[i].selected!==n&&(e[i].selected=n),n&&o&&(e[i].defaultSelected=!0)}else{for(i=""+vt(i),t=null,n=0;n<e.length;n++){if(e[n].value===i){e[n].selected=!0,o&&(e[n].defaultSelected=!0);return}t!==null||e[n].disabled||(t=e[n])}t!==null&&(t.selected=!0)}}function wa(e,t){if(t.dangerouslySetInnerHTML!=null)throw Error(P(91));return W({},t,{value:void 0,defaultValue:void 0,children:""+e._wrapperState.initialValue})}function gs(e,t){var i=t.value;if(i==null){if(i=t.children,t=t.defaultValue,i!=null){if(t!=null)throw Error(P(92));if(Ji(i)){if(1<i.length)throw Error(P(93));i=i[0]}t=i}t==null&&(t=""),i=t}e._wrapperState={initialValue:vt(i)}}function _l(e,t){var i=vt(t.value),o=vt(t.defaultValue);i!=null&&(i=""+i,i!==e.value&&(e.value=i),t.defaultValue==null&&e.defaultValue!==i&&(e.defaultValue=i)),o!=null&&(e.defaultValue=""+o)}function fs(e){var t=e.textContent;t===e._wrapperState.initialValue&&t!==""&&t!==null&&(e.value=t)}function Vl(e){switch(e){case"svg":return"http://www.w3.org/2000/svg";case"math":return"http://www.w3.org/1998/Math/MathML";default:return"http://www.w3.org/1999/xhtml"}}function Ta(e,t){return e==null||e==="http://www.w3.org/1999/xhtml"?Vl(t):e==="http://www.w3.org/2000/svg"&&t==="foreignObject"?"http://www.w3.org/1999/xhtml":e}var bo,$l=function(e){return typeof MSApp<"u"&&MSApp.execUnsafeLocalFunction?function(t,i,o,n){MSApp.execUnsafeLocalFunction(function(){return e(t,i,o,n)})}:e}(function(e,t){if(e.namespaceURI!=="http://www.w3.org/2000/svg"||"innerHTML"in e)e.innerHTML=t;else{for(bo=bo||document.createElement("div"),bo.innerHTML="<svg>"+t.valueOf().toString()+"</svg>",t=bo.firstChild;e.firstChild;)e.removeChild(e.firstChild);for(;t.firstChild;)e.appendChild(t.firstChild)}});function Oi(e,t){if(t){var i=e.firstChild;if(i&&i===e.lastChild&&i.nodeType===3){i.nodeValue=t;return}}e.textContent=t}var ji={animationIterationCount:!0,aspectRatio:!0,borderImageOutset:!0,borderImageSlice:!0,borderImageWidth:!0,boxFlex:!0,boxFlexGroup:!0,boxOrdinalGroup:!0,columnCount:!0,columns:!0,flex:!0,flexGrow:!0,flexPositive:!0,flexShrink:!0,flexNegative:!0,flexOrder:!0,gridArea:!0,gridRow:!0,gridRowEnd:!0,gridRowSpan:!0,gridRowStart:!0,gridColumn:!0,gridColumnEnd:!0,gridColumnSpan:!0,gridColumnStart:!0,fontWeight:!0,lineClamp:!0,lineHeight:!0,opacity:!0,order:!0,orphans:!0,tabSize:!0,widows:!0,zIndex:!0,zoom:!0,fillOpacity:!0,floodOpacity:!0,stopOpacity:!0,strokeDasharray:!0,strokeDashoffset:!0,strokeMiterlimit:!0,strokeOpacity:!0,strokeWidth:!0},Bp=["Webkit","ms","Moz","O"];Object.keys(ji).forEach(function(e){Bp.forEach(function(t){t=t+e.charAt(0).toUpperCase()+e.substring(1),ji[t]=ji[e]})});function Kl(e,t,i){return t==null||typeof t=="boolean"||t===""?"":i||typeof t!="number"||t===0||ji.hasOwnProperty(e)&&ji[e]?(""+t).trim():t+"px"}function Ql(e,t){e=e.style;for(var i in t)if(t.hasOwnProperty(i)){var o=i.indexOf("--")===0,n=Kl(i,t[i],o);i==="float"&&(i="cssFloat"),o?e.setProperty(i,n):e[i]=n}}var Dp=W({menuitem:!0},{area:!0,base:!0,br:!0,col:!0,embed:!0,hr:!0,img:!0,input:!0,keygen:!0,link:!0,meta:!0,param:!0,source:!0,track:!0,wbr:!0});function Sa(e,t){if(t){if(Dp[e]&&(t.children!=null||t.dangerouslySetInnerHTML!=null))throw Error(P(137,e));if(t.dangerouslySetInnerHTML!=null){if(t.children!=null)throw Error(P(60));if(typeof t.dangerouslySetInnerHTML!="object"||!("__html"in t.dangerouslySetInnerHTML))throw Error(P(61))}if(t.style!=null&&typeof t.style!="object")throw Error(P(62))}}function Pa(e,t){if(e.indexOf("-")===-1)return typeof t.is=="string";switch(e){case"annotation-xml":case"color-profile":case"font-face":case"font-face-src":case"font-face-uri":case"font-face-format":case"font-face-name":case"missing-glyph":return!1;default:return!0}}var xa=null;function fr(e){return e=e.target||e.srcElement||window,e.correspondingUseElement&&(e=e.correspondingUseElement),e.nodeType===3?e.parentNode:e}var ka=null,ei=null,ti=null;function vs(e){if(e=mo(e)){if(typeof ka!="function")throw Error(P(280));var t=e.stateNode;t&&(t=Sn(t),ka(e.stateNode,e.type,t))}}function Xl(e){ei?ti?ti.push(e):ti=[e]:ei=e}function Yl(){if(ei){var e=ei,t=ti;if(ti=ei=null,vs(e),t)for(e=0;e<t.length;e++)vs(t[e])}}function ec(e,t){return e(t)}function tc(){}var Wn=!1;function ic(e,t,i){if(Wn)return e(t,i);Wn=!0;try{return ec(e,t,i)}finally{Wn=!1,(ei!==null||ti!==null)&&(tc(),Yl())}}function Fi(e,t){var i=e.stateNode;if(i===null)return null;var o=Sn(i);if(o===null)return null;i=o[t];e:switch(t){case"onClick":case"onClickCapture":case"onDoubleClick":case"onDoubleClickCapture":case"onMouseDown":case"onMouseDownCapture":case"onMouseMove":case"onMouseMoveCapture":case"onMouseUp":case"onMouseUpCapture":case"onMouseEnter":(o=!o.disabled)||(e=e.type,o=!(e==="button"||e==="input"||e==="select"||e==="textarea")),e=!o;break e;default:e=!1}if(e)return null;if(i&&typeof i!="function")throw Error(P(231,t,typeof i));return i}var Aa=!1;if(_e)try{var Pi={};Object.defineProperty(Pi,"passive",{get:function(){Aa=!0}}),window.addEventListener("test",Pi,Pi),window.removeEventListener("test",Pi,Pi)}catch{Aa=!1}function Lp(e,t,i,o,n,a,s,l,c){var p=Array.prototype.slice.call(arguments,3);try{t.apply(i,p)}catch(y){this.onError(y)}}var zi=!1,Zo=null,_o=!1,Ca=null,Hp={onError:function(e){zi=!0,Zo=e}};function Rp(e,t,i,o,n,a,s,l,c){zi=!1,Zo=null,Lp.apply(Hp,arguments)}function Up(e,t,i,o,n,a,s,l,c){if(Rp.apply(this,arguments),zi){if(zi){var p=Zo;zi=!1,Zo=null}else throw Error(P(198));_o||(_o=!0,Ca=p)}}function Ht(e){var t=e,i=e;if(e.alternate)for(;t.return;)t=t.return;else{e=t;do t=e,t.flags&4098&&(i=t.return),e=t.return;while(e)}return t.tag===3?i:null}function oc(e){if(e.tag===13){var t=e.memoizedState;if(t===null&&(e=e.alternate,e!==null&&(t=e.memoizedState)),t!==null)return t.dehydrated}return null}function bs(e){if(Ht(e)!==e)throw Error(P(188))}function Wp(e){var t=e.alternate;if(!t){if(t=Ht(e),t===null)throw Error(P(188));return t!==e?null:e}for(var i=e,o=t;;){var n=i.return;if(n===null)break;var a=n.alternate;if(a===null){if(o=n.return,o!==null){i=o;continue}break}if(n.child===a.child){for(a=n.child;a;){if(a===i)return bs(n),e;if(a===o)return bs(n),t;a=a.sibling}throw Error(P(188))}if(i.return!==o.return)i=n,o=a;else{for(var s=!1,l=n.child;l;){if(l===i){s=!0,i=n,o=a;break}if(l===o){s=!0,o=n,i=a;break}l=l.sibling}if(!s){for(l=a.child;l;){if(l===i){s=!0,i=a,o=n;break}if(l===o){s=!0,o=a,i=n;break}l=l.sibling}if(!s)throw Error(P(189))}}if(i.alternate!==o)throw Error(P(190))}if(i.tag!==3)throw Error(P(188));return i.stateNode.current===i?e:t}function nc(e){return e=Wp(e),e!==null?ac(e):null}function ac(e){if(e.tag===5||e.tag===6)return e;for(e=e.child;e!==null;){var t=ac(e);if(t!==null)return t;e=e.sibling}return null}var rc=we.unstable_scheduleCallback,ws=we.unstable_cancelCallback,qp=we.unstable_shouldYield,Op=we.unstable_requestPaint,_=we.unstable_now,Fp=we.unstable_getCurrentPriorityLevel,vr=we.unstable_ImmediatePriority,sc=we.unstable_UserBlockingPriority,Vo=we.unstable_NormalPriority,Zp=we.unstable_LowPriority,lc=we.unstable_IdlePriority,vn=null,Re=null;function _p(e){if(Re&&typeof Re.onCommitFiberRoot=="function")try{Re.onCommitFiberRoot(vn,e,void 0,(e.current.flags&128)===128)}catch{}}var je=Math.clz32?Math.clz32:Kp,Vp=Math.log,$p=Math.LN2;function Kp(e){return e>>>=0,e===0?32:31-(Vp(e)/$p|0)|0}var wo=64,To=4194304;function Ni(e){switch(e&-e){case 1:return 1;case 2:return 2;case 4:return 4;case 8:return 8;case 16:return 16;case 32:return 32;case 64:case 128:case 256:case 512:case 1024:case 2048:case 4096:case 8192:case 16384:case 32768:case 65536:case 131072:case 262144:case 524288:case 1048576:case 2097152:return e&4194240;case 4194304:case 8388608:case 16777216:case 33554432:case 67108864:return e&130023424;case 134217728:return 134217728;case 268435456:return 268435456;case 536870912:return 536870912;case 1073741824:return 1073741824;default:return e}}function $o(e,t){var i=e.pendingLanes;if(i===0)return 0;var o=0,n=e.suspendedLanes,a=e.pingedLanes,s=i&268435455;if(s!==0){var l=s&~n;l!==0?o=Ni(l):(a&=s,a!==0&&(o=Ni(a)))}else s=i&~n,s!==0?o=Ni(s):a!==0&&(o=Ni(a));if(o===0)return 0;if(t!==0&&t!==o&&!(t&n)&&(n=o&-o,a=t&-t,n>=a||n===16&&(a&4194240)!==0))return t;if(o&4&&(o|=i&16),t=e.entangledLanes,t!==0)for(e=e.entanglements,t&=o;0<t;)i=31-je(t),n=1<<i,o|=e[i],t&=~n;return o}function Qp(e,t){switch(e){case 1:case 2:case 4:return t+250;case 8:case 16:case 32:case 64:case 128:case 256:case 512:case 1024:case 2048:case 4096:case 8192:case 16384:case 32768:case 65536:case 131072:case 262144:case 524288:case 1048576:case 2097152:return t+5e3;case 4194304:case 8388608:case 16777216:case 33554432:case 67108864:return-1;case 134217728:case 268435456:case 536870912:case 1073741824:return-1;default:return-1}}function Xp(e,t){for(var i=e.suspendedLanes,o=e.pingedLanes,n=e.expirationTimes,a=e.pendingLanes;0<a;){var s=31-je(a),l=1<<s,c=n[s];c===-1?(!(l&i)||l&o)&&(n[s]=Qp(l,t)):c<=t&&(e.expiredLanes|=l),a&=~l}}function Ia(e){return e=e.pendingLanes&-1073741825,e!==0?e:e&1073741824?1073741824:0}function cc(){var e=wo;return wo<<=1,!(wo&4194240)&&(wo=64),e}function qn(e){for(var t=[],i=0;31>i;i++)t.push(e);return t}function co(e,t,i){e.pendingLanes|=t,t!==536870912&&(e.suspendedLanes=0,e.pingedLanes=0),e=e.eventTimes,t=31-je(t),e[t]=i}function Yp(e,t){var i=e.pendingLanes&~t;e.pendingLanes=t,e.suspendedLanes=0,e.pingedLanes=0,e.expiredLanes&=t,e.mutableReadLanes&=t,e.entangledLanes&=t,t=e.entanglements;var o=e.eventTimes;for(e=e.expirationTimes;0<i;){var n=31-je(i),a=1<<n;t[n]=0,o[n]=-1,e[n]=-1,i&=~a}}function br(e,t){var i=e.entangledLanes|=t;for(e=e.entanglements;i;){var o=31-je(i),n=1<<o;n&t|e[o]&t&&(e[o]|=t),i&=~n}}var z=0;function dc(e){return e&=-e,1<e?4<e?e&268435455?16:536870912:4:1}var pc,wr,mc,uc,hc,Ma=!1,So=[],ct=null,dt=null,pt=null,Zi=new Map,_i=new Map,ot=[],em="mousedown mouseup touchcancel touchend touchstart auxclick dblclick pointercancel pointerdown pointerup dragend dragstart drop compositionend compositionstart keydown keypress keyup input textInput copy cut paste click change contextmenu reset submit".split(" ");function Ts(e,t){switch(e){case"focusin":case"focusout":ct=null;break;case"dragenter":case"dragleave":dt=null;break;case"mouseover":case"mouseout":pt=null;break;case"pointerover":case"pointerout":Zi.delete(t.pointerId);break;case"gotpointercapture":case"lostpointercapture":_i.delete(t.pointerId)}}function xi(e,t,i,o,n,a){return e===null||e.nativeEvent!==a?(e={blockedOn:t,domEventName:i,eventSystemFlags:o,nativeEvent:a,targetContainers:[n]},t!==null&&(t=mo(t),t!==null&&wr(t)),e):(e.eventSystemFlags|=o,t=e.targetContainers,n!==null&&t.indexOf(n)===-1&&t.push(n),e)}function tm(e,t,i,o,n){switch(t){case"focusin":return ct=xi(ct,e,t,i,o,n),!0;case"dragenter":return dt=xi(dt,e,t,i,o,n),!0;case"mouseover":return pt=xi(pt,e,t,i,o,n),!0;case"pointerover":var a=n.pointerId;return Zi.set(a,xi(Zi.get(a)||null,e,t,i,o,n)),!0;case"gotpointercapture":return a=n.pointerId,_i.set(a,xi(_i.get(a)||null,e,t,i,o,n)),!0}return!1}function yc(e){var t=It(e.target);if(t!==null){var i=Ht(t);if(i!==null){if(t=i.tag,t===13){if(t=oc(i),t!==null){e.blockedOn=t,hc(e.priority,function(){mc(i)});return}}else if(t===3&&i.stateNode.current.memoizedState.isDehydrated){e.blockedOn=i.tag===3?i.stateNode.containerInfo:null;return}}}e.blockedOn=null}function zo(e){if(e.blockedOn!==null)return!1;for(var t=e.targetContainers;0<t.length;){var i=Ga(e.domEventName,e.eventSystemFlags,t[0],e.nativeEvent);if(i===null){i=e.nativeEvent;var o=new i.constructor(i.type,i);xa=o,i.target.dispatchEvent(o),xa=null}else return t=mo(i),t!==null&&wr(t),e.blockedOn=i,!1;t.shift()}return!0}function Ss(e,t,i){zo(e)&&i.delete(t)}function im(){Ma=!1,ct!==null&&zo(ct)&&(ct=null),dt!==null&&zo(dt)&&(dt=null),pt!==null&&zo(pt)&&(pt=null),Zi.forEach(Ss),_i.forEach(Ss)}function ki(e,t){e.blockedOn===t&&(e.blockedOn=null,Ma||(Ma=!0,we.unstable_scheduleCallback(we.unstable_NormalPriority,im)))}function Vi(e){function t(n){return ki(n,e)}if(0<So.length){ki(So[0],e);for(var i=1;i<So.length;i++){var o=So[i];o.blockedOn===e&&(o.blockedOn=null)}}for(ct!==null&&ki(ct,e),dt!==null&&ki(dt,e),pt!==null&&ki(pt,e),Zi.forEach(t),_i.forEach(t),i=0;i<ot.length;i++)o=ot[i],o.blockedOn===e&&(o.blockedOn=null);for(;0<ot.length&&(i=ot[0],i.blockedOn===null);)yc(i),i.blockedOn===null&&ot.shift()}var ii=Qe.ReactCurrentBatchConfig,Ko=!0;function om(e,t,i,o){var n=z,a=ii.transition;ii.transition=null;try{z=1,Tr(e,t,i,o)}finally{z=n,ii.transition=a}}function nm(e,t,i,o){var n=z,a=ii.transition;ii.transition=null;try{z=4,Tr(e,t,i,o)}finally{z=n,ii.transition=a}}function Tr(e,t,i,o){if(Ko){var n=Ga(e,t,i,o);if(n===null)Yn(e,t,o,Qo,i),Ts(e,o);else if(tm(n,e,t,i,o))o.stopPropagation();else if(Ts(e,o),t&4&&-1<em.indexOf(e)){for(;n!==null;){var a=mo(n);if(a!==null&&pc(a),a=Ga(e,t,i,o),a===null&&Yn(e,t,o,Qo,i),a===n)break;n=a}n!==null&&o.stopPropagation()}else Yn(e,t,o,null,i)}}var Qo=null;function Ga(e,t,i,o){if(Qo=null,e=fr(o),e=It(e),e!==null)if(t=Ht(e),t===null)e=null;else if(i=t.tag,i===13){if(e=oc(t),e!==null)return e;e=null}else if(i===3){if(t.stateNode.current.memoizedState.isDehydrated)return t.tag===3?t.stateNode.containerInfo:null;e=null}else t!==e&&(e=null);return Qo=e,null}function gc(e){switch(e){case"cancel":case"click":case"close":case"contextmenu":case"copy":case"cut":case"auxclick":case"dblclick":case"dragend":case"dragstart":case"drop":case"focusin":case"focusout":case"input":case"invalid":case"keydown":case"keypress":case"keyup":case"mousedown":case"mouseup":case"paste":case"pause":case"play":case"pointercancel":case"pointerdown":case"pointerup":case"ratechange":case"reset":case"resize":case"seeked":case"submit":case"touchcancel":case"touchend":case"touchstart":case"volumechange":case"change":case"selectionchange":case"textInput":case"compositionstart":case"compositionend":case"compositionupdate":case"beforeblur":case"afterblur":case"beforeinput":case"blur":case"fullscreenchange":case"focus":case"hashchange":case"popstate":case"select":case"selectstart":return 1;case"drag":case"dragenter":case"dragexit":case"dragleave":case"dragover":case"mousemove":case"mouseout":case"mouseover":case"pointermove":case"pointerout":case"pointerover":case"scroll":case"toggle":case"touchmove":case"wheel":case"mouseenter":case"mouseleave":case"pointerenter":case"pointerleave":return 4;case"message":switch(Fp()){case vr:return 1;case sc:return 4;case Vo:case Zp:return 16;case lc:return 536870912;default:return 16}default:return 16}}var at=null,Sr=null,Bo=null;function fc(){if(Bo)return Bo;var e,t=Sr,i=t.length,o,n="value"in at?at.value:at.textContent,a=n.length;for(e=0;e<i&&t[e]===n[e];e++);var s=i-e;for(o=1;o<=s&&t[i-o]===n[a-o];o++);return Bo=n.slice(e,1<o?1-o:void 0)}function Do(e){var t=e.keyCode;return"charCode"in e?(e=e.charCode,e===0&&t===13&&(e=13)):e=t,e===10&&(e=13),32<=e||e===13?e:0}function Po(){return!0}function Ps(){return!1}function Se(e){function t(i,o,n,a,s){this._reactName=i,this._targetInst=n,this.type=o,this.nativeEvent=a,this.target=s,this.currentTarget=null;for(var l in e)e.hasOwnProperty(l)&&(i=e[l],this[l]=i?i(a):a[l]);return this.isDefaultPrevented=(a.defaultPrevented!=null?a.defaultPrevented:a.returnValue===!1)?Po:Ps,this.isPropagationStopped=Ps,this}return W(t.prototype,{preventDefault:function(){this.defaultPrevented=!0;var i=this.nativeEvent;i&&(i.preventDefault?i.preventDefault():typeof i.returnValue!="unknown"&&(i.returnValue=!1),this.isDefaultPrevented=Po)},stopPropagation:function(){var i=this.nativeEvent;i&&(i.stopPropagation?i.stopPropagation():typeof i.cancelBubble!="unknown"&&(i.cancelBubble=!0),this.isPropagationStopped=Po)},persist:function(){},isPersistent:Po}),t}var hi={eventPhase:0,bubbles:0,cancelable:0,timeStamp:function(e){return e.timeStamp||Date.now()},defaultPrevented:0,isTrusted:0},Pr=Se(hi),po=W({},hi,{view:0,detail:0}),am=Se(po),On,Fn,Ai,bn=W({},po,{screenX:0,screenY:0,clientX:0,clientY:0,pageX:0,pageY:0,ctrlKey:0,shiftKey:0,altKey:0,metaKey:0,getModifierState:xr,button:0,buttons:0,relatedTarget:function(e){return e.relatedTarget===void 0?e.fromElement===e.srcElement?e.toElement:e.fromElement:e.relatedTarget},movementX:function(e){return"movementX"in e?e.movementX:(e!==Ai&&(Ai&&e.type==="mousemove"?(On=e.screenX-Ai.screenX,Fn=e.screenY-Ai.screenY):Fn=On=0,Ai=e),On)},movementY:function(e){return"movementY"in e?e.movementY:Fn}}),xs=Se(bn),rm=W({},bn,{dataTransfer:0}),sm=Se(rm),lm=W({},po,{relatedTarget:0}),Zn=Se(lm),cm=W({},hi,{animationName:0,elapsedTime:0,pseudoElement:0}),dm=Se(cm),pm=W({},hi,{clipboardData:function(e){return"clipboardData"in e?e.clipboardData:window.clipboardData}}),mm=Se(pm),um=W({},hi,{data:0}),ks=Se(um),hm={Esc:"Escape",Spacebar:" ",Left:"ArrowLeft",Up:"ArrowUp",Right:"ArrowRight",Down:"ArrowDown",Del:"Delete",Win:"OS",Menu:"ContextMenu",Apps:"ContextMenu",Scroll:"ScrollLock",MozPrintableKey:"Unidentified"},ym={8:"Backspace",9:"Tab",12:"Clear",13:"Enter",16:"Shift",17:"Control",18:"Alt",19:"Pause",20:"CapsLock",27:"Escape",32:" ",33:"PageUp",34:"PageDown",35:"End",36:"Home",37:"ArrowLeft",38:"ArrowUp",39:"ArrowRight",40:"ArrowDown",45:"Insert",46:"Delete",112:"F1",113:"F2",114:"F3",115:"F4",116:"F5",117:"F6",118:"F7",119:"F8",120:"F9",121:"F10",122:"F11",123:"F12",144:"NumLock",145:"ScrollLock",224:"Meta"},gm={Alt:"altKey",Control:"ctrlKey",Meta:"metaKey",Shift:"shiftKey"};function fm(e){var t=this.nativeEvent;return t.getModifierState?t.getModifierState(e):(e=gm[e])?!!t[e]:!1}function xr(){return fm}var vm=W({},po,{key:function(e){if(e.key){var t=hm[e.key]||e.key;if(t!=="Unidentified")return t}return e.type==="keypress"?(e=Do(e),e===13?"Enter":String.fromCharCode(e)):e.type==="keydown"||e.type==="keyup"?ym[e.keyCode]||"Unidentified":""},code:0,location:0,ctrlKey:0,shiftKey:0,altKey:0,metaKey:0,repeat:0,locale:0,getModifierState:xr,charCode:function(e){return e.type==="keypress"?Do(e):0},keyCode:function(e){return e.type==="keydown"||e.type==="keyup"?e.keyCode:0},which:function(e){return e.type==="keypress"?Do(e):e.type==="keydown"||e.type==="keyup"?e.keyCode:0}}),bm=Se(vm),wm=W({},bn,{pointerId:0,width:0,height:0,pressure:0,tangentialPressure:0,tiltX:0,tiltY:0,twist:0,pointerType:0,isPrimary:0}),As=Se(wm),Tm=W({},po,{touches:0,targetTouches:0,changedTouches:0,altKey:0,metaKey:0,ctrlKey:0,shiftKey:0,getModifierState:xr}),Sm=Se(Tm),Pm=W({},hi,{propertyName:0,elapsedTime:0,pseudoElement:0}),xm=Se(Pm),km=W({},bn,{deltaX:function(e){return"deltaX"in e?e.deltaX:"wheelDeltaX"in e?-e.wheelDeltaX:0},deltaY:function(e){return"deltaY"in e?e.deltaY:"wheelDeltaY"in e?-e.wheelDeltaY:"wheelDelta"in e?-e.wheelDelta:0},deltaZ:0,deltaMode:0}),Am=Se(km),Cm=[9,13,27,32],kr=_e&&"CompositionEvent"in window,Bi=null;_e&&"documentMode"in document&&(Bi=document.documentMode);var Im=_e&&"TextEvent"in window&&!Bi,vc=_e&&(!kr||Bi&&8<Bi&&11>=Bi),Cs=" ",Is=!1;function bc(e,t){switch(e){case"keyup":return Cm.indexOf(t.keyCode)!==-1;case"keydown":return t.keyCode!==229;case"keypress":case"mousedown":case"focusout":return!0;default:return!1}}function wc(e){return e=e.detail,typeof e=="object"&&"data"in e?e.data:null}var qt=!1;function Mm(e,t){switch(e){case"compositionend":return wc(t);case"keypress":return t.which!==32?null:(Is=!0,Cs);case"textInput":return e=t.data,e===Cs&&Is?null:e;default:return null}}function Gm(e,t){if(qt)return e==="compositionend"||!kr&&bc(e,t)?(e=fc(),Bo=Sr=at=null,qt=!1,e):null;switch(e){case"paste":return null;case"keypress":if(!(t.ctrlKey||t.altKey||t.metaKey)||t.ctrlKey&&t.altKey){if(t.char&&1<t.char.length)return t.char;if(t.which)return String.fromCharCode(t.which)}return null;case"compositionend":return vc&&t.locale!=="ko"?null:t.data;default:return null}}var Jm={color:!0,date:!0,datetime:!0,"datetime-local":!0,email:!0,month:!0,number:!0,password:!0,range:!0,search:!0,tel:!0,text:!0,time:!0,url:!0,week:!0};function Ms(e){var t=e&&e.nodeName&&e.nodeName.toLowerCase();return t==="input"?!!Jm[e.type]:t==="textarea"}function Tc(e,t,i,o){Xl(o),t=Xo(t,"onChange"),0<t.length&&(i=new Pr("onChange","change",null,i,o),e.push({event:i,listeners:t}))}var Di=null,$i=null;function Nm(e){Nc(e,0)}function wn(e){var t=Zt(e);if(Fl(t))return e}function Em(e,t){if(e==="change")return t}var Sc=!1;if(_e){var _n;if(_e){var Vn="oninput"in document;if(!Vn){var Gs=document.createElement("div");Gs.setAttribute("oninput","return;"),Vn=typeof Gs.oninput=="function"}_n=Vn}else _n=!1;Sc=_n&&(!document.documentMode||9<document.documentMode)}function Js(){Di&&(Di.detachEvent("onpropertychange",Pc),$i=Di=null)}function Pc(e){if(e.propertyName==="value"&&wn($i)){var t=[];Tc(t,$i,e,fr(e)),ic(Nm,t)}}function jm(e,t,i){e==="focusin"?(Js(),Di=t,$i=i,Di.attachEvent("onpropertychange",Pc)):e==="focusout"&&Js()}function zm(e){if(e==="selectionchange"||e==="keyup"||e==="keydown")return wn($i)}function Bm(e,t){if(e==="click")return wn(t)}function Dm(e,t){if(e==="input"||e==="change")return wn(t)}function Lm(e,t){return e===t&&(e!==0||1/e===1/t)||e!==e&&t!==t}var Be=typeof Object.is=="function"?Object.is:Lm;function Ki(e,t){if(Be(e,t))return!0;if(typeof e!="object"||e===null||typeof t!="object"||t===null)return!1;var i=Object.keys(e),o=Object.keys(t);if(i.length!==o.length)return!1;for(o=0;o<i.length;o++){var n=i[o];if(!ma.call(t,n)||!Be(e[n],t[n]))return!1}return!0}function Ns(e){for(;e&&e.firstChild;)e=e.firstChild;return e}function Es(e,t){var i=Ns(e);e=0;for(var o;i;){if(i.nodeType===3){if(o=e+i.textContent.length,e<=t&&o>=t)return{node:i,offset:t-e};e=o}e:{for(;i;){if(i.nextSibling){i=i.nextSibling;break e}i=i.parentNode}i=void 0}i=Ns(i)}}function xc(e,t){return e&&t?e===t?!0:e&&e.nodeType===3?!1:t&&t.nodeType===3?xc(e,t.parentNode):"contains"in e?e.contains(t):e.compareDocumentPosition?!!(e.compareDocumentPosition(t)&16):!1:!1}function kc(){for(var e=window,t=Fo();t instanceof e.HTMLIFrameElement;){try{var i=typeof t.contentWindow.location.href=="string"}catch{i=!1}if(i)e=t.contentWindow;else break;t=Fo(e.document)}return t}function Ar(e){var t=e&&e.nodeName&&e.nodeName.toLowerCase();return t&&(t==="input"&&(e.type==="text"||e.type==="search"||e.type==="tel"||e.type==="url"||e.type==="password")||t==="textarea"||e.contentEditable==="true")}function Hm(e){var t=kc(),i=e.focusedElem,o=e.selectionRange;if(t!==i&&i&&i.ownerDocument&&xc(i.ownerDocument.documentElement,i)){if(o!==null&&Ar(i)){if(t=o.start,e=o.end,e===void 0&&(e=t),"selectionStart"in i)i.selectionStart=t,i.selectionEnd=Math.min(e,i.value.length);else if(e=(t=i.ownerDocument||document)&&t.defaultView||window,e.getSelection){e=e.getSelection();var n=i.textContent.length,a=Math.min(o.start,n);o=o.end===void 0?a:Math.min(o.end,n),!e.extend&&a>o&&(n=o,o=a,a=n),n=Es(i,a);var s=Es(i,o);n&&s&&(e.rangeCount!==1||e.anchorNode!==n.node||e.anchorOffset!==n.offset||e.focusNode!==s.node||e.focusOffset!==s.offset)&&(t=t.createRange(),t.setStart(n.node,n.offset),e.removeAllRanges(),a>o?(e.addRange(t),e.extend(s.node,s.offset)):(t.setEnd(s.node,s.offset),e.addRange(t)))}}for(t=[],e=i;e=e.parentNode;)e.nodeType===1&&t.push({element:e,left:e.scrollLeft,top:e.scrollTop});for(typeof i.focus=="function"&&i.focus(),i=0;i<t.length;i++)e=t[i],e.element.scrollLeft=e.left,e.element.scrollTop=e.top}}var Rm=_e&&"documentMode"in document&&11>=document.documentMode,Ot=null,Ja=null,Li=null,Na=!1;function js(e,t,i){var o=i.window===i?i.document:i.nodeType===9?i:i.ownerDocument;Na||Ot==null||Ot!==Fo(o)||(o=Ot,"selectionStart"in o&&Ar(o)?o={start:o.selectionStart,end:o.selectionEnd}:(o=(o.ownerDocument&&o.ownerDocument.defaultView||window).getSelection(),o={anchorNode:o.anchorNode,anchorOffset:o.anchorOffset,focusNode:o.focusNode,focusOffset:o.focusOffset}),Li&&Ki(Li,o)||(Li=o,o=Xo(Ja,"onSelect"),0<o.length&&(t=new Pr("onSelect","select",null,t,i),e.push({event:t,listeners:o}),t.target=Ot)))}function xo(e,t){var i={};return i[e.toLowerCase()]=t.toLowerCase(),i["Webkit"+e]="webkit"+t,i["Moz"+e]="moz"+t,i}var Ft={animationend:xo("Animation","AnimationEnd"),animationiteration:xo("Animation","AnimationIteration"),animationstart:xo("Animation","AnimationStart"),transitionend:xo("Transition","TransitionEnd")},$n={},Ac={};_e&&(Ac=document.createElement("div").style,"AnimationEvent"in window||(delete Ft.animationend.animation,delete Ft.animationiteration.animation,delete Ft.animationstart.animation),"TransitionEvent"in window||delete Ft.transitionend.transition);function Tn(e){if($n[e])return $n[e];if(!Ft[e])return e;var t=Ft[e],i;for(i in t)if(t.hasOwnProperty(i)&&i in Ac)return $n[e]=t[i];return e}var Cc=Tn("animationend"),Ic=Tn("animationiteration"),Mc=Tn("animationstart"),Gc=Tn("transitionend"),Jc=new Map,zs="abort auxClick cancel canPlay canPlayThrough click close contextMenu copy cut drag dragEnd dragEnter dragExit dragLeave dragOver dragStart drop durationChange emptied encrypted ended error gotPointerCapture input invalid keyDown keyPress keyUp load loadedData loadedMetadata loadStart lostPointerCapture mouseDown mouseMove mouseOut mouseOver mouseUp paste pause play playing pointerCancel pointerDown pointerMove pointerOut pointerOver pointerUp progress rateChange reset resize seeked seeking stalled submit suspend timeUpdate touchCancel touchEnd touchStart volumeChange scroll toggle touchMove waiting wheel".split(" ");function wt(e,t){Jc.set(e,t),Lt(t,[e])}for(var Kn=0;Kn<zs.length;Kn++){var Qn=zs[Kn],Um=Qn.toLowerCase(),Wm=Qn[0].toUpperCase()+Qn.slice(1);wt(Um,"on"+Wm)}wt(Cc,"onAnimationEnd");wt(Ic,"onAnimationIteration");wt(Mc,"onAnimationStart");wt("dblclick","onDoubleClick");wt("focusin","onFocus");wt("focusout","onBlur");wt(Gc,"onTransitionEnd");ri("onMouseEnter",["mouseout","mouseover"]);ri("onMouseLeave",["mouseout","mouseover"]);ri("onPointerEnter",["pointerout","pointerover"]);ri("onPointerLeave",["pointerout","pointerover"]);Lt("onChange","change click focusin focusout input keydown keyup selectionchange".split(" "));Lt("onSelect","focusout contextmenu dragend focusin keydown keyup mousedown mouseup selectionchange".split(" "));Lt("onBeforeInput",["compositionend","keypress","textInput","paste"]);Lt("onCompositionEnd","compositionend focusout keydown keypress keyup mousedown".split(" "));Lt("onCompositionStart","compositionstart focusout keydown keypress keyup mousedown".split(" "));Lt("onCompositionUpdate","compositionupdate focusout keydown keypress keyup mousedown".split(" "));var Ei="abort canplay canplaythrough durationchange emptied encrypted ended error loadeddata loadedmetadata loadstart pause play playing progress ratechange resize seeked seeking stalled suspend timeupdate volumechange waiting".split(" "),qm=new Set("cancel close invalid load scroll toggle".split(" ").concat(Ei));function Bs(e,t,i){var o=e.type||"unknown-event";e.currentTarget=i,Up(o,t,void 0,e),e.currentTarget=null}function Nc(e,t){t=(t&4)!==0;for(var i=0;i<e.length;i++){var o=e[i],n=o.event;o=o.listeners;e:{var a=void 0;if(t)for(var s=o.length-1;0<=s;s--){var l=o[s],c=l.instance,p=l.currentTarget;if(l=l.listener,c!==a&&n.isPropagationStopped())break e;Bs(n,l,p),a=c}else for(s=0;s<o.length;s++){if(l=o[s],c=l.instance,p=l.currentTarget,l=l.listener,c!==a&&n.isPropagationStopped())break e;Bs(n,l,p),a=c}}}if(_o)throw e=Ca,_o=!1,Ca=null,e}function D(e,t){var i=t[Da];i===void 0&&(i=t[Da]=new Set);var o=e+"__bubble";i.has(o)||(Ec(t,e,2,!1),i.add(o))}function Xn(e,t,i){var o=0;t&&(o|=4),Ec(i,e,o,t)}var ko="_reactListening"+Math.random().toString(36).slice(2);function Qi(e){if(!e[ko]){e[ko]=!0,Rl.forEach(function(i){i!=="selectionchange"&&(qm.has(i)||Xn(i,!1,e),Xn(i,!0,e))});var t=e.nodeType===9?e:e.ownerDocument;t===null||t[ko]||(t[ko]=!0,Xn("selectionchange",!1,t))}}function Ec(e,t,i,o){switch(gc(t)){case 1:var n=om;break;case 4:n=nm;break;default:n=Tr}i=n.bind(null,t,i,e),n=void 0,!Aa||t!=="touchstart"&&t!=="touchmove"&&t!=="wheel"||(n=!0),o?n!==void 0?e.addEventListener(t,i,{capture:!0,passive:n}):e.addEventListener(t,i,!0):n!==void 0?e.addEventListener(t,i,{passive:n}):e.addEventListener(t,i,!1)}function Yn(e,t,i,o,n){var a=o;if(!(t&1)&&!(t&2)&&o!==null)e:for(;;){if(o===null)return;var s=o.tag;if(s===3||s===4){var l=o.stateNode.containerInfo;if(l===n||l.nodeType===8&&l.parentNode===n)break;if(s===4)for(s=o.return;s!==null;){var c=s.tag;if((c===3||c===4)&&(c=s.stateNode.containerInfo,c===n||c.nodeType===8&&c.parentNode===n))return;s=s.return}for(;l!==null;){if(s=It(l),s===null)return;if(c=s.tag,c===5||c===6){o=a=s;continue e}l=l.parentNode}}o=o.return}ic(function(){var p=a,y=fr(i),u=[];e:{var g=Jc.get(e);if(g!==void 0){var b=Pr,w=e;switch(e){case"keypress":if(Do(i)===0)break e;case"keydown":case"keyup":b=bm;break;case"focusin":w="focus",b=Zn;break;case"focusout":w="blur",b=Zn;break;case"beforeblur":case"afterblur":b=Zn;break;case"click":if(i.button===2)break e;case"auxclick":case"dblclick":case"mousedown":case"mousemove":case"mouseup":case"mouseout":case"mouseover":case"contextmenu":b=xs;break;case"drag":case"dragend":case"dragenter":case"dragexit":case"dragleave":case"dragover":case"dragstart":case"drop":b=sm;break;case"touchcancel":case"touchend":case"touchmove":case"touchstart":b=Sm;break;case Cc:case Ic:case Mc:b=dm;break;case Gc:b=xm;break;case"scroll":b=am;break;case"wheel":b=Am;break;case"copy":case"cut":case"paste":b=mm;break;case"gotpointercapture":case"lostpointercapture":case"pointercancel":case"pointerdown":case"pointermove":case"pointerout":case"pointerover":case"pointerup":b=As}var v=(t&4)!==0,x=!v&&e==="scroll",h=v?g!==null?g+"Capture":null:g;v=[];for(var d=p,m;d!==null;){m=d;var f=m.stateNode;if(m.tag===5&&f!==null&&(m=f,h!==null&&(f=Fi(d,h),f!=null&&v.push(Xi(d,f,m)))),x)break;d=d.return}0<v.length&&(g=new b(g,w,null,i,y),u.push({event:g,listeners:v}))}}if(!(t&7)){e:{if(g=e==="mouseover"||e==="pointerover",b=e==="mouseout"||e==="pointerout",g&&i!==xa&&(w=i.relatedTarget||i.fromElement)&&(It(w)||w[Ve]))break e;if((b||g)&&(g=y.window===y?y:(g=y.ownerDocument)?g.defaultView||g.parentWindow:window,b?(w=i.relatedTarget||i.toElement,b=p,w=w?It(w):null,w!==null&&(x=Ht(w),w!==x||w.tag!==5&&w.tag!==6)&&(w=null)):(b=null,w=p),b!==w)){if(v=xs,f="onMouseLeave",h="onMouseEnter",d="mouse",(e==="pointerout"||e==="pointerover")&&(v=As,f="onPointerLeave",h="onPointerEnter",d="pointer"),x=b==null?g:Zt(b),m=w==null?g:Zt(w),g=new v(f,d+"leave",b,i,y),g.target=x,g.relatedTarget=m,f=null,It(y)===p&&(v=new v(h,d+"enter",w,i,y),v.target=m,v.relatedTarget=x,f=v),x=f,b&&w)t:{for(v=b,h=w,d=0,m=v;m;m=Rt(m))d++;for(m=0,f=h;f;f=Rt(f))m++;for(;0<d-m;)v=Rt(v),d--;for(;0<m-d;)h=Rt(h),m--;for(;d--;){if(v===h||h!==null&&v===h.alternate)break t;v=Rt(v),h=Rt(h)}v=null}else v=null;b!==null&&Ds(u,g,b,v,!1),w!==null&&x!==null&&Ds(u,x,w,v,!0)}}e:{if(g=p?Zt(p):window,b=g.nodeName&&g.nodeName.toLowerCase(),b==="select"||b==="input"&&g.type==="file")var S=Em;else if(Ms(g))if(Sc)S=Dm;else{S=zm;var k=jm}else(b=g.nodeName)&&b.toLowerCase()==="input"&&(g.type==="checkbox"||g.type==="radio")&&(S=Bm);if(S&&(S=S(e,p))){Tc(u,S,i,y);break e}k&&k(e,g,p),e==="focusout"&&(k=g._wrapperState)&&k.controlled&&g.type==="number"&&ba(g,"number",g.value)}switch(k=p?Zt(p):window,e){case"focusin":(Ms(k)||k.contentEditable==="true")&&(Ot=k,Ja=p,Li=null);break;case"focusout":Li=Ja=Ot=null;break;case"mousedown":Na=!0;break;case"contextmenu":case"mouseup":case"dragend":Na=!1,js(u,i,y);break;case"selectionchange":if(Rm)break;case"keydown":case"keyup":js(u,i,y)}var A;if(kr)e:{switch(e){case"compositionstart":var M="onCompositionStart";break e;case"compositionend":M="onCompositionEnd";break e;case"compositionupdate":M="onCompositionUpdate";break e}M=void 0}else qt?bc(e,i)&&(M="onCompositionEnd"):e==="keydown"&&i.keyCode===229&&(M="onCompositionStart");M&&(vc&&i.locale!=="ko"&&(qt||M!=="onCompositionStart"?M==="onCompositionEnd"&&qt&&(A=fc()):(at=y,Sr="value"in at?at.value:at.textContent,qt=!0)),k=Xo(p,M),0<k.length&&(M=new ks(M,e,null,i,y),u.push({event:M,listeners:k}),A?M.data=A:(A=wc(i),A!==null&&(M.data=A)))),(A=Im?Mm(e,i):Gm(e,i))&&(p=Xo(p,"onBeforeInput"),0<p.length&&(y=new ks("onBeforeInput","beforeinput",null,i,y),u.push({event:y,listeners:p}),y.data=A))}Nc(u,t)})}function Xi(e,t,i){return{instance:e,listener:t,currentTarget:i}}function Xo(e,t){for(var i=t+"Capture",o=[];e!==null;){var n=e,a=n.stateNode;n.tag===5&&a!==null&&(n=a,a=Fi(e,i),a!=null&&o.unshift(Xi(e,a,n)),a=Fi(e,t),a!=null&&o.push(Xi(e,a,n))),e=e.return}return o}function Rt(e){if(e===null)return null;do e=e.return;while(e&&e.tag!==5);return e||null}function Ds(e,t,i,o,n){for(var a=t._reactName,s=[];i!==null&&i!==o;){var l=i,c=l.alternate,p=l.stateNode;if(c!==null&&c===o)break;l.tag===5&&p!==null&&(l=p,n?(c=Fi(i,a),c!=null&&s.unshift(Xi(i,c,l))):n||(c=Fi(i,a),c!=null&&s.push(Xi(i,c,l)))),i=i.return}s.length!==0&&e.push({event:t,listeners:s})}var Om=/\r\n?/g,Fm=/\u0000|\uFFFD/g;function Ls(e){return(typeof e=="string"?e:""+e).replace(Om,`
`).replace(Fm,"")}function Ao(e,t,i){if(t=Ls(t),Ls(e)!==t&&i)throw Error(P(425))}function Yo(){}var Ea=null,ja=null;function za(e,t){return e==="textarea"||e==="noscript"||typeof t.children=="string"||typeof t.children=="number"||typeof t.dangerouslySetInnerHTML=="object"&&t.dangerouslySetInnerHTML!==null&&t.dangerouslySetInnerHTML.__html!=null}var Ba=typeof setTimeout=="function"?setTimeout:void 0,Zm=typeof clearTimeout=="function"?clearTimeout:void 0,Hs=typeof Promise=="function"?Promise:void 0,_m=typeof queueMicrotask=="function"?queueMicrotask:typeof Hs<"u"?function(e){return Hs.resolve(null).then(e).catch(Vm)}:Ba;function Vm(e){setTimeout(function(){throw e})}function ea(e,t){var i=t,o=0;do{var n=i.nextSibling;if(e.removeChild(i),n&&n.nodeType===8)if(i=n.data,i==="/$"){if(o===0){e.removeChild(n),Vi(t);return}o--}else i!=="$"&&i!=="$?"&&i!=="$!"||o++;i=n}while(i);Vi(t)}function mt(e){for(;e!=null;e=e.nextSibling){var t=e.nodeType;if(t===1||t===3)break;if(t===8){if(t=e.data,t==="$"||t==="$!"||t==="$?")break;if(t==="/$")return null}}return e}function Rs(e){e=e.previousSibling;for(var t=0;e;){if(e.nodeType===8){var i=e.data;if(i==="$"||i==="$!"||i==="$?"){if(t===0)return e;t--}else i==="/$"&&t++}e=e.previousSibling}return null}var yi=Math.random().toString(36).slice(2),He="__reactFiber$"+yi,Yi="__reactProps$"+yi,Ve="__reactContainer$"+yi,Da="__reactEvents$"+yi,$m="__reactListeners$"+yi,Km="__reactHandles$"+yi;function It(e){var t=e[He];if(t)return t;for(var i=e.parentNode;i;){if(t=i[Ve]||i[He]){if(i=t.alternate,t.child!==null||i!==null&&i.child!==null)for(e=Rs(e);e!==null;){if(i=e[He])return i;e=Rs(e)}return t}e=i,i=e.parentNode}return null}function mo(e){return e=e[He]||e[Ve],!e||e.tag!==5&&e.tag!==6&&e.tag!==13&&e.tag!==3?null:e}function Zt(e){if(e.tag===5||e.tag===6)return e.stateNode;throw Error(P(33))}function Sn(e){return e[Yi]||null}var La=[],_t=-1;function Tt(e){return{current:e}}function L(e){0>_t||(e.current=La[_t],La[_t]=null,_t--)}function B(e,t){_t++,La[_t]=e.current,e.current=t}var bt={},se=Tt(bt),he=Tt(!1),Et=bt;function si(e,t){var i=e.type.contextTypes;if(!i)return bt;var o=e.stateNode;if(o&&o.__reactInternalMemoizedUnmaskedChildContext===t)return o.__reactInternalMemoizedMaskedChildContext;var n={},a;for(a in i)n[a]=t[a];return o&&(e=e.stateNode,e.__reactInternalMemoizedUnmaskedChildContext=t,e.__reactInternalMemoizedMaskedChildContext=n),n}function ye(e){return e=e.childContextTypes,e!=null}function en(){L(he),L(se)}function Us(e,t,i){if(se.current!==bt)throw Error(P(168));B(se,t),B(he,i)}function jc(e,t,i){var o=e.stateNode;if(t=t.childContextTypes,typeof o.getChildContext!="function")return i;o=o.getChildContext();for(var n in o)if(!(n in t))throw Error(P(108,jp(e)||"Unknown",n));return W({},i,o)}function tn(e){return e=(e=e.stateNode)&&e.__reactInternalMemoizedMergedChildContext||bt,Et=se.current,B(se,e),B(he,he.current),!0}function Ws(e,t,i){var o=e.stateNode;if(!o)throw Error(P(169));i?(e=jc(e,t,Et),o.__reactInternalMemoizedMergedChildContext=e,L(he),L(se),B(se,e)):L(he),B(he,i)}var qe=null,Pn=!1,ta=!1;function zc(e){qe===null?qe=[e]:qe.push(e)}function Qm(e){Pn=!0,zc(e)}function St(){if(!ta&&qe!==null){ta=!0;var e=0,t=z;try{var i=qe;for(z=1;e<i.length;e++){var o=i[e];do o=o(!0);while(o!==null)}qe=null,Pn=!1}catch(n){throw qe!==null&&(qe=qe.slice(e+1)),rc(vr,St),n}finally{z=t,ta=!1}}return null}var Vt=[],$t=0,on=null,nn=0,Pe=[],xe=0,jt=null,Oe=1,Fe="";function At(e,t){Vt[$t++]=nn,Vt[$t++]=on,on=e,nn=t}function Bc(e,t,i){Pe[xe++]=Oe,Pe[xe++]=Fe,Pe[xe++]=jt,jt=e;var o=Oe;e=Fe;var n=32-je(o)-1;o&=~(1<<n),i+=1;var a=32-je(t)+n;if(30<a){var s=n-n%5;a=(o&(1<<s)-1).toString(32),o>>=s,n-=s,Oe=1<<32-je(t)+n|i<<n|o,Fe=a+e}else Oe=1<<a|i<<n|o,Fe=e}function Cr(e){e.return!==null&&(At(e,1),Bc(e,1,0))}function Ir(e){for(;e===on;)on=Vt[--$t],Vt[$t]=null,nn=Vt[--$t],Vt[$t]=null;for(;e===jt;)jt=Pe[--xe],Pe[xe]=null,Fe=Pe[--xe],Pe[xe]=null,Oe=Pe[--xe],Pe[xe]=null}var be=null,ve=null,H=!1,Ee=null;function Dc(e,t){var i=ke(5,null,null,0);i.elementType="DELETED",i.stateNode=t,i.return=e,t=e.deletions,t===null?(e.deletions=[i],e.flags|=16):t.push(i)}function qs(e,t){switch(e.tag){case 5:var i=e.type;return t=t.nodeType!==1||i.toLowerCase()!==t.nodeName.toLowerCase()?null:t,t!==null?(e.stateNode=t,be=e,ve=mt(t.firstChild),!0):!1;case 6:return t=e.pendingProps===""||t.nodeType!==3?null:t,t!==null?(e.stateNode=t,be=e,ve=null,!0):!1;case 13:return t=t.nodeType!==8?null:t,t!==null?(i=jt!==null?{id:Oe,overflow:Fe}:null,e.memoizedState={dehydrated:t,treeContext:i,retryLane:1073741824},i=ke(18,null,null,0),i.stateNode=t,i.return=e,e.child=i,be=e,ve=null,!0):!1;default:return!1}}function Ha(e){return(e.mode&1)!==0&&(e.flags&128)===0}function Ra(e){if(H){var t=ve;if(t){var i=t;if(!qs(e,t)){if(Ha(e))throw Error(P(418));t=mt(i.nextSibling);var o=be;t&&qs(e,t)?Dc(o,i):(e.flags=e.flags&-4097|2,H=!1,be=e)}}else{if(Ha(e))throw Error(P(418));e.flags=e.flags&-4097|2,H=!1,be=e}}}function Os(e){for(e=e.return;e!==null&&e.tag!==5&&e.tag!==3&&e.tag!==13;)e=e.return;be=e}function Co(e){if(e!==be)return!1;if(!H)return Os(e),H=!0,!1;var t;if((t=e.tag!==3)&&!(t=e.tag!==5)&&(t=e.type,t=t!=="head"&&t!=="body"&&!za(e.type,e.memoizedProps)),t&&(t=ve)){if(Ha(e))throw Lc(),Error(P(418));for(;t;)Dc(e,t),t=mt(t.nextSibling)}if(Os(e),e.tag===13){if(e=e.memoizedState,e=e!==null?e.dehydrated:null,!e)throw Error(P(317));e:{for(e=e.nextSibling,t=0;e;){if(e.nodeType===8){var i=e.data;if(i==="/$"){if(t===0){ve=mt(e.nextSibling);break e}t--}else i!=="$"&&i!=="$!"&&i!=="$?"||t++}e=e.nextSibling}ve=null}}else ve=be?mt(e.stateNode.nextSibling):null;return!0}function Lc(){for(var e=ve;e;)e=mt(e.nextSibling)}function li(){ve=be=null,H=!1}function Mr(e){Ee===null?Ee=[e]:Ee.push(e)}var Xm=Qe.ReactCurrentBatchConfig;function Ci(e,t,i){if(e=i.ref,e!==null&&typeof e!="function"&&typeof e!="object"){if(i._owner){if(i=i._owner,i){if(i.tag!==1)throw Error(P(309));var o=i.stateNode}if(!o)throw Error(P(147,e));var n=o,a=""+e;return t!==null&&t.ref!==null&&typeof t.ref=="function"&&t.ref._stringRef===a?t.ref:(t=function(s){var l=n.refs;s===null?delete l[a]:l[a]=s},t._stringRef=a,t)}if(typeof e!="string")throw Error(P(284));if(!i._owner)throw Error(P(290,e))}return e}function Io(e,t){throw e=Object.prototype.toString.call(t),Error(P(31,e==="[object Object]"?"object with keys {"+Object.keys(t).join(", ")+"}":e))}function Fs(e){var t=e._init;return t(e._payload)}function Hc(e){function t(h,d){if(e){var m=h.deletions;m===null?(h.deletions=[d],h.flags|=16):m.push(d)}}function i(h,d){if(!e)return null;for(;d!==null;)t(h,d),d=d.sibling;return null}function o(h,d){for(h=new Map;d!==null;)d.key!==null?h.set(d.key,d):h.set(d.index,d),d=d.sibling;return h}function n(h,d){return h=gt(h,d),h.index=0,h.sibling=null,h}function a(h,d,m){return h.index=m,e?(m=h.alternate,m!==null?(m=m.index,m<d?(h.flags|=2,d):m):(h.flags|=2,d)):(h.flags|=1048576,d)}function s(h){return e&&h.alternate===null&&(h.flags|=2),h}function l(h,d,m,f){return d===null||d.tag!==6?(d=la(m,h.mode,f),d.return=h,d):(d=n(d,m),d.return=h,d)}function c(h,d,m,f){var S=m.type;return S===Wt?y(h,d,m.props.children,f,m.key):d!==null&&(d.elementType===S||typeof S=="object"&&S!==null&&S.$$typeof===tt&&Fs(S)===d.type)?(f=n(d,m.props),f.ref=Ci(h,d,m),f.return=h,f):(f=Oo(m.type,m.key,m.props,null,h.mode,f),f.ref=Ci(h,d,m),f.return=h,f)}function p(h,d,m,f){return d===null||d.tag!==4||d.stateNode.containerInfo!==m.containerInfo||d.stateNode.implementation!==m.implementation?(d=ca(m,h.mode,f),d.return=h,d):(d=n(d,m.children||[]),d.return=h,d)}function y(h,d,m,f,S){return d===null||d.tag!==7?(d=Nt(m,h.mode,f,S),d.return=h,d):(d=n(d,m),d.return=h,d)}function u(h,d,m){if(typeof d=="string"&&d!==""||typeof d=="number")return d=la(""+d,h.mode,m),d.return=h,d;if(typeof d=="object"&&d!==null){switch(d.$$typeof){case fo:return m=Oo(d.type,d.key,d.props,null,h.mode,m),m.ref=Ci(h,null,d),m.return=h,m;case Ut:return d=ca(d,h.mode,m),d.return=h,d;case tt:var f=d._init;return u(h,f(d._payload),m)}if(Ji(d)||Si(d))return d=Nt(d,h.mode,m,null),d.return=h,d;Io(h,d)}return null}function g(h,d,m,f){var S=d!==null?d.key:null;if(typeof m=="string"&&m!==""||typeof m=="number")return S!==null?null:l(h,d,""+m,f);if(typeof m=="object"&&m!==null){switch(m.$$typeof){case fo:return m.key===S?c(h,d,m,f):null;case Ut:return m.key===S?p(h,d,m,f):null;case tt:return S=m._init,g(h,d,S(m._payload),f)}if(Ji(m)||Si(m))return S!==null?null:y(h,d,m,f,null);Io(h,m)}return null}function b(h,d,m,f,S){if(typeof f=="string"&&f!==""||typeof f=="number")return h=h.get(m)||null,l(d,h,""+f,S);if(typeof f=="object"&&f!==null){switch(f.$$typeof){case fo:return h=h.get(f.key===null?m:f.key)||null,c(d,h,f,S);case Ut:return h=h.get(f.key===null?m:f.key)||null,p(d,h,f,S);case tt:var k=f._init;return b(h,d,m,k(f._payload),S)}if(Ji(f)||Si(f))return h=h.get(m)||null,y(d,h,f,S,null);Io(d,f)}return null}function w(h,d,m,f){for(var S=null,k=null,A=d,M=d=0,O=null;A!==null&&M<m.length;M++){A.index>M?(O=A,A=null):O=A.sibling;var E=g(h,A,m[M],f);if(E===null){A===null&&(A=O);break}e&&A&&E.alternate===null&&t(h,A),d=a(E,d,M),k===null?S=E:k.sibling=E,k=E,A=O}if(M===m.length)return i(h,A),H&&At(h,M),S;if(A===null){for(;M<m.length;M++)A=u(h,m[M],f),A!==null&&(d=a(A,d,M),k===null?S=A:k.sibling=A,k=A);return H&&At(h,M),S}for(A=o(h,A);M<m.length;M++)O=b(A,h,M,m[M],f),O!==null&&(e&&O.alternate!==null&&A.delete(O.key===null?M:O.key),d=a(O,d,M),k===null?S=O:k.sibling=O,k=O);return e&&A.forEach(function(Me){return t(h,Me)}),H&&At(h,M),S}function v(h,d,m,f){var S=Si(m);if(typeof S!="function")throw Error(P(150));if(m=S.call(m),m==null)throw Error(P(151));for(var k=S=null,A=d,M=d=0,O=null,E=m.next();A!==null&&!E.done;M++,E=m.next()){A.index>M?(O=A,A=null):O=A.sibling;var Me=g(h,A,E.value,f);if(Me===null){A===null&&(A=O);break}e&&A&&Me.alternate===null&&t(h,A),d=a(Me,d,M),k===null?S=Me:k.sibling=Me,k=Me,A=O}if(E.done)return i(h,A),H&&At(h,M),S;if(A===null){for(;!E.done;M++,E=m.next())E=u(h,E.value,f),E!==null&&(d=a(E,d,M),k===null?S=E:k.sibling=E,k=E);return H&&At(h,M),S}for(A=o(h,A);!E.done;M++,E=m.next())E=b(A,h,M,E.value,f),E!==null&&(e&&E.alternate!==null&&A.delete(E.key===null?M:E.key),d=a(E,d,M),k===null?S=E:k.sibling=E,k=E);return e&&A.forEach(function(wi){return t(h,wi)}),H&&At(h,M),S}function x(h,d,m,f){if(typeof m=="object"&&m!==null&&m.type===Wt&&m.key===null&&(m=m.props.children),typeof m=="object"&&m!==null){switch(m.$$typeof){case fo:e:{for(var S=m.key,k=d;k!==null;){if(k.key===S){if(S=m.type,S===Wt){if(k.tag===7){i(h,k.sibling),d=n(k,m.props.children),d.return=h,h=d;break e}}else if(k.elementType===S||typeof S=="object"&&S!==null&&S.$$typeof===tt&&Fs(S)===k.type){i(h,k.sibling),d=n(k,m.props),d.ref=Ci(h,k,m),d.return=h,h=d;break e}i(h,k);break}else t(h,k);k=k.sibling}m.type===Wt?(d=Nt(m.props.children,h.mode,f,m.key),d.return=h,h=d):(f=Oo(m.type,m.key,m.props,null,h.mode,f),f.ref=Ci(h,d,m),f.return=h,h=f)}return s(h);case Ut:e:{for(k=m.key;d!==null;){if(d.key===k)if(d.tag===4&&d.stateNode.containerInfo===m.containerInfo&&d.stateNode.implementation===m.implementation){i(h,d.sibling),d=n(d,m.children||[]),d.return=h,h=d;break e}else{i(h,d);break}else t(h,d);d=d.sibling}d=ca(m,h.mode,f),d.return=h,h=d}return s(h);case tt:return k=m._init,x(h,d,k(m._payload),f)}if(Ji(m))return w(h,d,m,f);if(Si(m))return v(h,d,m,f);Io(h,m)}return typeof m=="string"&&m!==""||typeof m=="number"?(m=""+m,d!==null&&d.tag===6?(i(h,d.sibling),d=n(d,m),d.return=h,h=d):(i(h,d),d=la(m,h.mode,f),d.return=h,h=d),s(h)):i(h,d)}return x}var ci=Hc(!0),Rc=Hc(!1),an=Tt(null),rn=null,Kt=null,Gr=null;function Jr(){Gr=Kt=rn=null}function Nr(e){var t=an.current;L(an),e._currentValue=t}function Ua(e,t,i){for(;e!==null;){var o=e.alternate;if((e.childLanes&t)!==t?(e.childLanes|=t,o!==null&&(o.childLanes|=t)):o!==null&&(o.childLanes&t)!==t&&(o.childLanes|=t),e===i)break;e=e.return}}function oi(e,t){rn=e,Gr=Kt=null,e=e.dependencies,e!==null&&e.firstContext!==null&&(e.lanes&t&&(ue=!0),e.firstContext=null)}function Ce(e){var t=e._currentValue;if(Gr!==e)if(e={context:e,memoizedValue:t,next:null},Kt===null){if(rn===null)throw Error(P(308));Kt=e,rn.dependencies={lanes:0,firstContext:e}}else Kt=Kt.next=e;return t}var Mt=null;function Er(e){Mt===null?Mt=[e]:Mt.push(e)}function Uc(e,t,i,o){var n=t.interleaved;return n===null?(i.next=i,Er(t)):(i.next=n.next,n.next=i),t.interleaved=i,$e(e,o)}function $e(e,t){e.lanes|=t;var i=e.alternate;for(i!==null&&(i.lanes|=t),i=e,e=e.return;e!==null;)e.childLanes|=t,i=e.alternate,i!==null&&(i.childLanes|=t),i=e,e=e.return;return i.tag===3?i.stateNode:null}var it=!1;function jr(e){e.updateQueue={baseState:e.memoizedState,firstBaseUpdate:null,lastBaseUpdate:null,shared:{pending:null,interleaved:null,lanes:0},effects:null}}function Wc(e,t){e=e.updateQueue,t.updateQueue===e&&(t.updateQueue={baseState:e.baseState,firstBaseUpdate:e.firstBaseUpdate,lastBaseUpdate:e.lastBaseUpdate,shared:e.shared,effects:e.effects})}function Ze(e,t){return{eventTime:e,lane:t,tag:0,payload:null,callback:null,next:null}}function ut(e,t,i){var o=e.updateQueue;if(o===null)return null;if(o=o.shared,j&2){var n=o.pending;return n===null?t.next=t:(t.next=n.next,n.next=t),o.pending=t,$e(e,i)}return n=o.interleaved,n===null?(t.next=t,Er(o)):(t.next=n.next,n.next=t),o.interleaved=t,$e(e,i)}function Lo(e,t,i){if(t=t.updateQueue,t!==null&&(t=t.shared,(i&4194240)!==0)){var o=t.lanes;o&=e.pendingLanes,i|=o,t.lanes=i,br(e,i)}}function Zs(e,t){var i=e.updateQueue,o=e.alternate;if(o!==null&&(o=o.updateQueue,i===o)){var n=null,a=null;if(i=i.firstBaseUpdate,i!==null){do{var s={eventTime:i.eventTime,lane:i.lane,tag:i.tag,payload:i.payload,callback:i.callback,next:null};a===null?n=a=s:a=a.next=s,i=i.next}while(i!==null);a===null?n=a=t:a=a.next=t}else n=a=t;i={baseState:o.baseState,firstBaseUpdate:n,lastBaseUpdate:a,shared:o.shared,effects:o.effects},e.updateQueue=i;return}e=i.lastBaseUpdate,e===null?i.firstBaseUpdate=t:e.next=t,i.lastBaseUpdate=t}function sn(e,t,i,o){var n=e.updateQueue;it=!1;var a=n.firstBaseUpdate,s=n.lastBaseUpdate,l=n.shared.pending;if(l!==null){n.shared.pending=null;var c=l,p=c.next;c.next=null,s===null?a=p:s.next=p,s=c;var y=e.alternate;y!==null&&(y=y.updateQueue,l=y.lastBaseUpdate,l!==s&&(l===null?y.firstBaseUpdate=p:l.next=p,y.lastBaseUpdate=c))}if(a!==null){var u=n.baseState;s=0,y=p=c=null,l=a;do{var g=l.lane,b=l.eventTime;if((o&g)===g){y!==null&&(y=y.next={eventTime:b,lane:0,tag:l.tag,payload:l.payload,callback:l.callback,next:null});e:{var w=e,v=l;switch(g=t,b=i,v.tag){case 1:if(w=v.payload,typeof w=="function"){u=w.call(b,u,g);break e}u=w;break e;case 3:w.flags=w.flags&-65537|128;case 0:if(w=v.payload,g=typeof w=="function"?w.call(b,u,g):w,g==null)break e;u=W({},u,g);break e;case 2:it=!0}}l.callback!==null&&l.lane!==0&&(e.flags|=64,g=n.effects,g===null?n.effects=[l]:g.push(l))}else b={eventTime:b,lane:g,tag:l.tag,payload:l.payload,callback:l.callback,next:null},y===null?(p=y=b,c=u):y=y.next=b,s|=g;if(l=l.next,l===null){if(l=n.shared.pending,l===null)break;g=l,l=g.next,g.next=null,n.lastBaseUpdate=g,n.shared.pending=null}}while(!0);if(y===null&&(c=u),n.baseState=c,n.firstBaseUpdate=p,n.lastBaseUpdate=y,t=n.shared.interleaved,t!==null){n=t;do s|=n.lane,n=n.next;while(n!==t)}else a===null&&(n.shared.lanes=0);Bt|=s,e.lanes=s,e.memoizedState=u}}function _s(e,t,i){if(e=t.effects,t.effects=null,e!==null)for(t=0;t<e.length;t++){var o=e[t],n=o.callback;if(n!==null){if(o.callback=null,o=i,typeof n!="function")throw Error(P(191,n));n.call(o)}}}var uo={},Ue=Tt(uo),eo=Tt(uo),to=Tt(uo);function Gt(e){if(e===uo)throw Error(P(174));return e}function zr(e,t){switch(B(to,t),B(eo,e),B(Ue,uo),e=t.nodeType,e){case 9:case 11:t=(t=t.documentElement)?t.namespaceURI:Ta(null,"");break;default:e=e===8?t.parentNode:t,t=e.namespaceURI||null,e=e.tagName,t=Ta(t,e)}L(Ue),B(Ue,t)}function di(){L(Ue),L(eo),L(to)}function qc(e){Gt(to.current);var t=Gt(Ue.current),i=Ta(t,e.type);t!==i&&(B(eo,e),B(Ue,i))}function Br(e){eo.current===e&&(L(Ue),L(eo))}var R=Tt(0);function ln(e){for(var t=e;t!==null;){if(t.tag===13){var i=t.memoizedState;if(i!==null&&(i=i.dehydrated,i===null||i.data==="$?"||i.data==="$!"))return t}else if(t.tag===19&&t.memoizedProps.revealOrder!==void 0){if(t.flags&128)return t}else if(t.child!==null){t.child.return=t,t=t.child;continue}if(t===e)break;for(;t.sibling===null;){if(t.return===null||t.return===e)return null;t=t.return}t.sibling.return=t.return,t=t.sibling}return null}var ia=[];function Dr(){for(var e=0;e<ia.length;e++)ia[e]._workInProgressVersionPrimary=null;ia.length=0}var Ho=Qe.ReactCurrentDispatcher,oa=Qe.ReactCurrentBatchConfig,zt=0,U=null,K=null,Y=null,cn=!1,Hi=!1,io=0,Ym=0;function ne(){throw Error(P(321))}function Lr(e,t){if(t===null)return!1;for(var i=0;i<t.length&&i<e.length;i++)if(!Be(e[i],t[i]))return!1;return!0}function Hr(e,t,i,o,n,a){if(zt=a,U=t,t.memoizedState=null,t.updateQueue=null,t.lanes=0,Ho.current=e===null||e.memoizedState===null?ou:nu,e=i(o,n),Hi){a=0;do{if(Hi=!1,io=0,25<=a)throw Error(P(301));a+=1,Y=K=null,t.updateQueue=null,Ho.current=au,e=i(o,n)}while(Hi)}if(Ho.current=dn,t=K!==null&&K.next!==null,zt=0,Y=K=U=null,cn=!1,t)throw Error(P(300));return e}function Rr(){var e=io!==0;return io=0,e}function Le(){var e={memoizedState:null,baseState:null,baseQueue:null,queue:null,next:null};return Y===null?U.memoizedState=Y=e:Y=Y.next=e,Y}function Ie(){if(K===null){var e=U.alternate;e=e!==null?e.memoizedState:null}else e=K.next;var t=Y===null?U.memoizedState:Y.next;if(t!==null)Y=t,K=e;else{if(e===null)throw Error(P(310));K=e,e={memoizedState:K.memoizedState,baseState:K.baseState,baseQueue:K.baseQueue,queue:K.queue,next:null},Y===null?U.memoizedState=Y=e:Y=Y.next=e}return Y}function oo(e,t){return typeof t=="function"?t(e):t}function na(e){var t=Ie(),i=t.queue;if(i===null)throw Error(P(311));i.lastRenderedReducer=e;var o=K,n=o.baseQueue,a=i.pending;if(a!==null){if(n!==null){var s=n.next;n.next=a.next,a.next=s}o.baseQueue=n=a,i.pending=null}if(n!==null){a=n.next,o=o.baseState;var l=s=null,c=null,p=a;do{var y=p.lane;if((zt&y)===y)c!==null&&(c=c.next={lane:0,action:p.action,hasEagerState:p.hasEagerState,eagerState:p.eagerState,next:null}),o=p.hasEagerState?p.eagerState:e(o,p.action);else{var u={lane:y,action:p.action,hasEagerState:p.hasEagerState,eagerState:p.eagerState,next:null};c===null?(l=c=u,s=o):c=c.next=u,U.lanes|=y,Bt|=y}p=p.next}while(p!==null&&p!==a);c===null?s=o:c.next=l,Be(o,t.memoizedState)||(ue=!0),t.memoizedState=o,t.baseState=s,t.baseQueue=c,i.lastRenderedState=o}if(e=i.interleaved,e!==null){n=e;do a=n.lane,U.lanes|=a,Bt|=a,n=n.next;while(n!==e)}else n===null&&(i.lanes=0);return[t.memoizedState,i.dispatch]}function aa(e){var t=Ie(),i=t.queue;if(i===null)throw Error(P(311));i.lastRenderedReducer=e;var o=i.dispatch,n=i.pending,a=t.memoizedState;if(n!==null){i.pending=null;var s=n=n.next;do a=e(a,s.action),s=s.next;while(s!==n);Be(a,t.memoizedState)||(ue=!0),t.memoizedState=a,t.baseQueue===null&&(t.baseState=a),i.lastRenderedState=a}return[a,o]}function Oc(){}function Fc(e,t){var i=U,o=Ie(),n=t(),a=!Be(o.memoizedState,n);if(a&&(o.memoizedState=n,ue=!0),o=o.queue,Ur(Vc.bind(null,i,o,e),[e]),o.getSnapshot!==t||a||Y!==null&&Y.memoizedState.tag&1){if(i.flags|=2048,no(9,_c.bind(null,i,o,n,t),void 0,null),ee===null)throw Error(P(349));zt&30||Zc(i,t,n)}return n}function Zc(e,t,i){e.flags|=16384,e={getSnapshot:t,value:i},t=U.updateQueue,t===null?(t={lastEffect:null,stores:null},U.updateQueue=t,t.stores=[e]):(i=t.stores,i===null?t.stores=[e]:i.push(e))}function _c(e,t,i,o){t.value=i,t.getSnapshot=o,$c(t)&&Kc(e)}function Vc(e,t,i){return i(function(){$c(t)&&Kc(e)})}function $c(e){var t=e.getSnapshot;e=e.value;try{var i=t();return!Be(e,i)}catch{return!0}}function Kc(e){var t=$e(e,1);t!==null&&ze(t,e,1,-1)}function Vs(e){var t=Le();return typeof e=="function"&&(e=e()),t.memoizedState=t.baseState=e,e={pending:null,interleaved:null,lanes:0,dispatch:null,lastRenderedReducer:oo,lastRenderedState:e},t.queue=e,e=e.dispatch=iu.bind(null,U,e),[t.memoizedState,e]}function no(e,t,i,o){return e={tag:e,create:t,destroy:i,deps:o,next:null},t=U.updateQueue,t===null?(t={lastEffect:null,stores:null},U.updateQueue=t,t.lastEffect=e.next=e):(i=t.lastEffect,i===null?t.lastEffect=e.next=e:(o=i.next,i.next=e,e.next=o,t.lastEffect=e)),e}function Qc(){return Ie().memoizedState}function Ro(e,t,i,o){var n=Le();U.flags|=e,n.memoizedState=no(1|t,i,void 0,o===void 0?null:o)}function xn(e,t,i,o){var n=Ie();o=o===void 0?null:o;var a=void 0;if(K!==null){var s=K.memoizedState;if(a=s.destroy,o!==null&&Lr(o,s.deps)){n.memoizedState=no(t,i,a,o);return}}U.flags|=e,n.memoizedState=no(1|t,i,a,o)}function $s(e,t){return Ro(8390656,8,e,t)}function Ur(e,t){return xn(2048,8,e,t)}function Xc(e,t){return xn(4,2,e,t)}function Yc(e,t){return xn(4,4,e,t)}function ed(e,t){if(typeof t=="function")return e=e(),t(e),function(){t(null)};if(t!=null)return e=e(),t.current=e,function(){t.current=null}}function td(e,t,i){return i=i!=null?i.concat([e]):null,xn(4,4,ed.bind(null,t,e),i)}function Wr(){}function id(e,t){var i=Ie();t=t===void 0?null:t;var o=i.memoizedState;return o!==null&&t!==null&&Lr(t,o[1])?o[0]:(i.memoizedState=[e,t],e)}function od(e,t){var i=Ie();t=t===void 0?null:t;var o=i.memoizedState;return o!==null&&t!==null&&Lr(t,o[1])?o[0]:(e=e(),i.memoizedState=[e,t],e)}function nd(e,t,i){return zt&21?(Be(i,t)||(i=cc(),U.lanes|=i,Bt|=i,e.baseState=!0),t):(e.baseState&&(e.baseState=!1,ue=!0),e.memoizedState=i)}function eu(e,t){var i=z;z=i!==0&&4>i?i:4,e(!0);var o=oa.transition;oa.transition={};try{e(!1),t()}finally{z=i,oa.transition=o}}function ad(){return Ie().memoizedState}function tu(e,t,i){var o=yt(e);if(i={lane:o,action:i,hasEagerState:!1,eagerState:null,next:null},rd(e))sd(t,i);else if(i=Uc(e,t,i,o),i!==null){var n=ce();ze(i,e,o,n),ld(i,t,o)}}function iu(e,t,i){var o=yt(e),n={lane:o,action:i,hasEagerState:!1,eagerState:null,next:null};if(rd(e))sd(t,n);else{var a=e.alternate;if(e.lanes===0&&(a===null||a.lanes===0)&&(a=t.lastRenderedReducer,a!==null))try{var s=t.lastRenderedState,l=a(s,i);if(n.hasEagerState=!0,n.eagerState=l,Be(l,s)){var c=t.interleaved;c===null?(n.next=n,Er(t)):(n.next=c.next,c.next=n),t.interleaved=n;return}}catch{}finally{}i=Uc(e,t,n,o),i!==null&&(n=ce(),ze(i,e,o,n),ld(i,t,o))}}function rd(e){var t=e.alternate;return e===U||t!==null&&t===U}function sd(e,t){Hi=cn=!0;var i=e.pending;i===null?t.next=t:(t.next=i.next,i.next=t),e.pending=t}function ld(e,t,i){if(i&4194240){var o=t.lanes;o&=e.pendingLanes,i|=o,t.lanes=i,br(e,i)}}var dn={readContext:Ce,useCallback:ne,useContext:ne,useEffect:ne,useImperativeHandle:ne,useInsertionEffect:ne,useLayoutEffect:ne,useMemo:ne,useReducer:ne,useRef:ne,useState:ne,useDebugValue:ne,useDeferredValue:ne,useTransition:ne,useMutableSource:ne,useSyncExternalStore:ne,useId:ne,unstable_isNewReconciler:!1},ou={readContext:Ce,useCallback:function(e,t){return Le().memoizedState=[e,t===void 0?null:t],e},useContext:Ce,useEffect:$s,useImperativeHandle:function(e,t,i){return i=i!=null?i.concat([e]):null,Ro(4194308,4,ed.bind(null,t,e),i)},useLayoutEffect:function(e,t){return Ro(4194308,4,e,t)},useInsertionEffect:function(e,t){return Ro(4,2,e,t)},useMemo:function(e,t){var i=Le();return t=t===void 0?null:t,e=e(),i.memoizedState=[e,t],e},useReducer:function(e,t,i){var o=Le();return t=i!==void 0?i(t):t,o.memoizedState=o.baseState=t,e={pending:null,interleaved:null,lanes:0,dispatch:null,lastRenderedReducer:e,lastRenderedState:t},o.queue=e,e=e.dispatch=tu.bind(null,U,e),[o.memoizedState,e]},useRef:function(e){var t=Le();return e={current:e},t.memoizedState=e},useState:Vs,useDebugValue:Wr,useDeferredValue:function(e){return Le().memoizedState=e},useTransition:function(){var e=Vs(!1),t=e[0];return e=eu.bind(null,e[1]),Le().memoizedState=e,[t,e]},useMutableSource:function(){},useSyncExternalStore:function(e,t,i){var o=U,n=Le();if(H){if(i===void 0)throw Error(P(407));i=i()}else{if(i=t(),ee===null)throw Error(P(349));zt&30||Zc(o,t,i)}n.memoizedState=i;var a={value:i,getSnapshot:t};return n.queue=a,$s(Vc.bind(null,o,a,e),[e]),o.flags|=2048,no(9,_c.bind(null,o,a,i,t),void 0,null),i},useId:function(){var e=Le(),t=ee.identifierPrefix;if(H){var i=Fe,o=Oe;i=(o&~(1<<32-je(o)-1)).toString(32)+i,t=":"+t+"R"+i,i=io++,0<i&&(t+="H"+i.toString(32)),t+=":"}else i=Ym++,t=":"+t+"r"+i.toString(32)+":";return e.memoizedState=t},unstable_isNewReconciler:!1},nu={readContext:Ce,useCallback:id,useContext:Ce,useEffect:Ur,useImperativeHandle:td,useInsertionEffect:Xc,useLayoutEffect:Yc,useMemo:od,useReducer:na,useRef:Qc,useState:function(){return na(oo)},useDebugValue:Wr,useDeferredValue:function(e){var t=Ie();return nd(t,K.memoizedState,e)},useTransition:function(){var e=na(oo)[0],t=Ie().memoizedState;return[e,t]},useMutableSource:Oc,useSyncExternalStore:Fc,useId:ad,unstable_isNewReconciler:!1},au={readContext:Ce,useCallback:id,useContext:Ce,useEffect:Ur,useImperativeHandle:td,useInsertionEffect:Xc,useLayoutEffect:Yc,useMemo:od,useReducer:aa,useRef:Qc,useState:function(){return aa(oo)},useDebugValue:Wr,useDeferredValue:function(e){var t=Ie();return K===null?t.memoizedState=e:nd(t,K.memoizedState,e)},useTransition:function(){var e=aa(oo)[0],t=Ie().memoizedState;return[e,t]},useMutableSource:Oc,useSyncExternalStore:Fc,useId:ad,unstable_isNewReconciler:!1};function Je(e,t){if(e&&e.defaultProps){t=W({},t),e=e.defaultProps;for(var i in e)t[i]===void 0&&(t[i]=e[i]);return t}return t}function Wa(e,t,i,o){t=e.memoizedState,i=i(o,t),i=i==null?t:W({},t,i),e.memoizedState=i,e.lanes===0&&(e.updateQueue.baseState=i)}var kn={isMounted:function(e){return(e=e._reactInternals)?Ht(e)===e:!1},enqueueSetState:function(e,t,i){e=e._reactInternals;var o=ce(),n=yt(e),a=Ze(o,n);a.payload=t,i!=null&&(a.callback=i),t=ut(e,a,n),t!==null&&(ze(t,e,n,o),Lo(t,e,n))},enqueueReplaceState:function(e,t,i){e=e._reactInternals;var o=ce(),n=yt(e),a=Ze(o,n);a.tag=1,a.payload=t,i!=null&&(a.callback=i),t=ut(e,a,n),t!==null&&(ze(t,e,n,o),Lo(t,e,n))},enqueueForceUpdate:function(e,t){e=e._reactInternals;var i=ce(),o=yt(e),n=Ze(i,o);n.tag=2,t!=null&&(n.callback=t),t=ut(e,n,o),t!==null&&(ze(t,e,o,i),Lo(t,e,o))}};function Ks(e,t,i,o,n,a,s){return e=e.stateNode,typeof e.shouldComponentUpdate=="function"?e.shouldComponentUpdate(o,a,s):t.prototype&&t.prototype.isPureReactComponent?!Ki(i,o)||!Ki(n,a):!0}function cd(e,t,i){var o=!1,n=bt,a=t.contextType;return typeof a=="object"&&a!==null?a=Ce(a):(n=ye(t)?Et:se.current,o=t.contextTypes,a=(o=o!=null)?si(e,n):bt),t=new t(i,a),e.memoizedState=t.state!==null&&t.state!==void 0?t.state:null,t.updater=kn,e.stateNode=t,t._reactInternals=e,o&&(e=e.stateNode,e.__reactInternalMemoizedUnmaskedChildContext=n,e.__reactInternalMemoizedMaskedChildContext=a),t}function Qs(e,t,i,o){e=t.state,typeof t.componentWillReceiveProps=="function"&&t.componentWillReceiveProps(i,o),typeof t.UNSAFE_componentWillReceiveProps=="function"&&t.UNSAFE_componentWillReceiveProps(i,o),t.state!==e&&kn.enqueueReplaceState(t,t.state,null)}function qa(e,t,i,o){var n=e.stateNode;n.props=i,n.state=e.memoizedState,n.refs={},jr(e);var a=t.contextType;typeof a=="object"&&a!==null?n.context=Ce(a):(a=ye(t)?Et:se.current,n.context=si(e,a)),n.state=e.memoizedState,a=t.getDerivedStateFromProps,typeof a=="function"&&(Wa(e,t,a,i),n.state=e.memoizedState),typeof t.getDerivedStateFromProps=="function"||typeof n.getSnapshotBeforeUpdate=="function"||typeof n.UNSAFE_componentWillMount!="function"&&typeof n.componentWillMount!="function"||(t=n.state,typeof n.componentWillMount=="function"&&n.componentWillMount(),typeof n.UNSAFE_componentWillMount=="function"&&n.UNSAFE_componentWillMount(),t!==n.state&&kn.enqueueReplaceState(n,n.state,null),sn(e,i,n,o),n.state=e.memoizedState),typeof n.componentDidMount=="function"&&(e.flags|=4194308)}function pi(e,t){try{var i="",o=t;do i+=Ep(o),o=o.return;while(o);var n=i}catch(a){n=`
Error generating stack: `+a.message+`
`+a.stack}return{value:e,source:t,stack:n,digest:null}}function ra(e,t,i){return{value:e,source:null,stack:i??null,digest:t??null}}function Oa(e,t){try{console.error(t.value)}catch(i){setTimeout(function(){throw i})}}var ru=typeof WeakMap=="function"?WeakMap:Map;function dd(e,t,i){i=Ze(-1,i),i.tag=3,i.payload={element:null};var o=t.value;return i.callback=function(){mn||(mn=!0,er=o),Oa(e,t)},i}function pd(e,t,i){i=Ze(-1,i),i.tag=3;var o=e.type.getDerivedStateFromError;if(typeof o=="function"){var n=t.value;i.payload=function(){return o(n)},i.callback=function(){Oa(e,t)}}var a=e.stateNode;return a!==null&&typeof a.componentDidCatch=="function"&&(i.callback=function(){Oa(e,t),typeof o!="function"&&(ht===null?ht=new Set([this]):ht.add(this));var s=t.stack;this.componentDidCatch(t.value,{componentStack:s!==null?s:""})}),i}function Xs(e,t,i){var o=e.pingCache;if(o===null){o=e.pingCache=new ru;var n=new Set;o.set(t,n)}else n=o.get(t),n===void 0&&(n=new Set,o.set(t,n));n.has(i)||(n.add(i),e=wu.bind(null,e,t,i),t.then(e,e))}function Ys(e){do{var t;if((t=e.tag===13)&&(t=e.memoizedState,t=t!==null?t.dehydrated!==null:!0),t)return e;e=e.return}while(e!==null);return null}function el(e,t,i,o,n){return e.mode&1?(e.flags|=65536,e.lanes=n,e):(e===t?e.flags|=65536:(e.flags|=128,i.flags|=131072,i.flags&=-52805,i.tag===1&&(i.alternate===null?i.tag=17:(t=Ze(-1,1),t.tag=2,ut(i,t,1))),i.lanes|=1),e)}var su=Qe.ReactCurrentOwner,ue=!1;function le(e,t,i,o){t.child=e===null?Rc(t,null,i,o):ci(t,e.child,i,o)}function tl(e,t,i,o,n){i=i.render;var a=t.ref;return oi(t,n),o=Hr(e,t,i,o,a,n),i=Rr(),e!==null&&!ue?(t.updateQueue=e.updateQueue,t.flags&=-2053,e.lanes&=~n,Ke(e,t,n)):(H&&i&&Cr(t),t.flags|=1,le(e,t,o,n),t.child)}function il(e,t,i,o,n){if(e===null){var a=i.type;return typeof a=="function"&&!Kr(a)&&a.defaultProps===void 0&&i.compare===null&&i.defaultProps===void 0?(t.tag=15,t.type=a,md(e,t,a,o,n)):(e=Oo(i.type,null,o,t,t.mode,n),e.ref=t.ref,e.return=t,t.child=e)}if(a=e.child,!(e.lanes&n)){var s=a.memoizedProps;if(i=i.compare,i=i!==null?i:Ki,i(s,o)&&e.ref===t.ref)return Ke(e,t,n)}return t.flags|=1,e=gt(a,o),e.ref=t.ref,e.return=t,t.child=e}function md(e,t,i,o,n){if(e!==null){var a=e.memoizedProps;if(Ki(a,o)&&e.ref===t.ref)if(ue=!1,t.pendingProps=o=a,(e.lanes&n)!==0)e.flags&131072&&(ue=!0);else return t.lanes=e.lanes,Ke(e,t,n)}return Fa(e,t,i,o,n)}function ud(e,t,i){var o=t.pendingProps,n=o.children,a=e!==null?e.memoizedState:null;if(o.mode==="hidden")if(!(t.mode&1))t.memoizedState={baseLanes:0,cachePool:null,transitions:null},B(Xt,fe),fe|=i;else{if(!(i&1073741824))return e=a!==null?a.baseLanes|i:i,t.lanes=t.childLanes=1073741824,t.memoizedState={baseLanes:e,cachePool:null,transitions:null},t.updateQueue=null,B(Xt,fe),fe|=e,null;t.memoizedState={baseLanes:0,cachePool:null,transitions:null},o=a!==null?a.baseLanes:i,B(Xt,fe),fe|=o}else a!==null?(o=a.baseLanes|i,t.memoizedState=null):o=i,B(Xt,fe),fe|=o;return le(e,t,n,i),t.child}function hd(e,t){var i=t.ref;(e===null&&i!==null||e!==null&&e.ref!==i)&&(t.flags|=512,t.flags|=2097152)}function Fa(e,t,i,o,n){var a=ye(i)?Et:se.current;return a=si(t,a),oi(t,n),i=Hr(e,t,i,o,a,n),o=Rr(),e!==null&&!ue?(t.updateQueue=e.updateQueue,t.flags&=-2053,e.lanes&=~n,Ke(e,t,n)):(H&&o&&Cr(t),t.flags|=1,le(e,t,i,n),t.child)}function ol(e,t,i,o,n){if(ye(i)){var a=!0;tn(t)}else a=!1;if(oi(t,n),t.stateNode===null)Uo(e,t),cd(t,i,o),qa(t,i,o,n),o=!0;else if(e===null){var s=t.stateNode,l=t.memoizedProps;s.props=l;var c=s.context,p=i.contextType;typeof p=="object"&&p!==null?p=Ce(p):(p=ye(i)?Et:se.current,p=si(t,p));var y=i.getDerivedStateFromProps,u=typeof y=="function"||typeof s.getSnapshotBeforeUpdate=="function";u||typeof s.UNSAFE_componentWillReceiveProps!="function"&&typeof s.componentWillReceiveProps!="function"||(l!==o||c!==p)&&Qs(t,s,o,p),it=!1;var g=t.memoizedState;s.state=g,sn(t,o,s,n),c=t.memoizedState,l!==o||g!==c||he.current||it?(typeof y=="function"&&(Wa(t,i,y,o),c=t.memoizedState),(l=it||Ks(t,i,l,o,g,c,p))?(u||typeof s.UNSAFE_componentWillMount!="function"&&typeof s.componentWillMount!="function"||(typeof s.componentWillMount=="function"&&s.componentWillMount(),typeof s.UNSAFE_componentWillMount=="function"&&s.UNSAFE_componentWillMount()),typeof s.componentDidMount=="function"&&(t.flags|=4194308)):(typeof s.componentDidMount=="function"&&(t.flags|=4194308),t.memoizedProps=o,t.memoizedState=c),s.props=o,s.state=c,s.context=p,o=l):(typeof s.componentDidMount=="function"&&(t.flags|=4194308),o=!1)}else{s=t.stateNode,Wc(e,t),l=t.memoizedProps,p=t.type===t.elementType?l:Je(t.type,l),s.props=p,u=t.pendingProps,g=s.context,c=i.contextType,typeof c=="object"&&c!==null?c=Ce(c):(c=ye(i)?Et:se.current,c=si(t,c));var b=i.getDerivedStateFromProps;(y=typeof b=="function"||typeof s.getSnapshotBeforeUpdate=="function")||typeof s.UNSAFE_componentWillReceiveProps!="function"&&typeof s.componentWillReceiveProps!="function"||(l!==u||g!==c)&&Qs(t,s,o,c),it=!1,g=t.memoizedState,s.state=g,sn(t,o,s,n);var w=t.memoizedState;l!==u||g!==w||he.current||it?(typeof b=="function"&&(Wa(t,i,b,o),w=t.memoizedState),(p=it||Ks(t,i,p,o,g,w,c)||!1)?(y||typeof s.UNSAFE_componentWillUpdate!="function"&&typeof s.componentWillUpdate!="function"||(typeof s.componentWillUpdate=="function"&&s.componentWillUpdate(o,w,c),typeof s.UNSAFE_componentWillUpdate=="function"&&s.UNSAFE_componentWillUpdate(o,w,c)),typeof s.componentDidUpdate=="function"&&(t.flags|=4),typeof s.getSnapshotBeforeUpdate=="function"&&(t.flags|=1024)):(typeof s.componentDidUpdate!="function"||l===e.memoizedProps&&g===e.memoizedState||(t.flags|=4),typeof s.getSnapshotBeforeUpdate!="function"||l===e.memoizedProps&&g===e.memoizedState||(t.flags|=1024),t.memoizedProps=o,t.memoizedState=w),s.props=o,s.state=w,s.context=c,o=p):(typeof s.componentDidUpdate!="function"||l===e.memoizedProps&&g===e.memoizedState||(t.flags|=4),typeof s.getSnapshotBeforeUpdate!="function"||l===e.memoizedProps&&g===e.memoizedState||(t.flags|=1024),o=!1)}return Za(e,t,i,o,a,n)}function Za(e,t,i,o,n,a){hd(e,t);var s=(t.flags&128)!==0;if(!o&&!s)return n&&Ws(t,i,!1),Ke(e,t,a);o=t.stateNode,su.current=t;var l=s&&typeof i.getDerivedStateFromError!="function"?null:o.render();return t.flags|=1,e!==null&&s?(t.child=ci(t,e.child,null,a),t.child=ci(t,null,l,a)):le(e,t,l,a),t.memoizedState=o.state,n&&Ws(t,i,!0),t.child}function yd(e){var t=e.stateNode;t.pendingContext?Us(e,t.pendingContext,t.pendingContext!==t.context):t.context&&Us(e,t.context,!1),zr(e,t.containerInfo)}function nl(e,t,i,o,n){return li(),Mr(n),t.flags|=256,le(e,t,i,o),t.child}var _a={dehydrated:null,treeContext:null,retryLane:0};function Va(e){return{baseLanes:e,cachePool:null,transitions:null}}function gd(e,t,i){var o=t.pendingProps,n=R.current,a=!1,s=(t.flags&128)!==0,l;if((l=s)||(l=e!==null&&e.memoizedState===null?!1:(n&2)!==0),l?(a=!0,t.flags&=-129):(e===null||e.memoizedState!==null)&&(n|=1),B(R,n&1),e===null)return Ra(t),e=t.memoizedState,e!==null&&(e=e.dehydrated,e!==null)?(t.mode&1?e.data==="$!"?t.lanes=8:t.lanes=1073741824:t.lanes=1,null):(s=o.children,e=o.fallback,a?(o=t.mode,a=t.child,s={mode:"hidden",children:s},!(o&1)&&a!==null?(a.childLanes=0,a.pendingProps=s):a=In(s,o,0,null),e=Nt(e,o,i,null),a.return=t,e.return=t,a.sibling=e,t.child=a,t.child.memoizedState=Va(i),t.memoizedState=_a,e):qr(t,s));if(n=e.memoizedState,n!==null&&(l=n.dehydrated,l!==null))return lu(e,t,s,o,l,n,i);if(a){a=o.fallback,s=t.mode,n=e.child,l=n.sibling;var c={mode:"hidden",children:o.children};return!(s&1)&&t.child!==n?(o=t.child,o.childLanes=0,o.pendingProps=c,t.deletions=null):(o=gt(n,c),o.subtreeFlags=n.subtreeFlags&14680064),l!==null?a=gt(l,a):(a=Nt(a,s,i,null),a.flags|=2),a.return=t,o.return=t,o.sibling=a,t.child=o,o=a,a=t.child,s=e.child.memoizedState,s=s===null?Va(i):{baseLanes:s.baseLanes|i,cachePool:null,transitions:s.transitions},a.memoizedState=s,a.childLanes=e.childLanes&~i,t.memoizedState=_a,o}return a=e.child,e=a.sibling,o=gt(a,{mode:"visible",children:o.children}),!(t.mode&1)&&(o.lanes=i),o.return=t,o.sibling=null,e!==null&&(i=t.deletions,i===null?(t.deletions=[e],t.flags|=16):i.push(e)),t.child=o,t.memoizedState=null,o}function qr(e,t){return t=In({mode:"visible",children:t},e.mode,0,null),t.return=e,e.child=t}function Mo(e,t,i,o){return o!==null&&Mr(o),ci(t,e.child,null,i),e=qr(t,t.pendingProps.children),e.flags|=2,t.memoizedState=null,e}function lu(e,t,i,o,n,a,s){if(i)return t.flags&256?(t.flags&=-257,o=ra(Error(P(422))),Mo(e,t,s,o)):t.memoizedState!==null?(t.child=e.child,t.flags|=128,null):(a=o.fallback,n=t.mode,o=In({mode:"visible",children:o.children},n,0,null),a=Nt(a,n,s,null),a.flags|=2,o.return=t,a.return=t,o.sibling=a,t.child=o,t.mode&1&&ci(t,e.child,null,s),t.child.memoizedState=Va(s),t.memoizedState=_a,a);if(!(t.mode&1))return Mo(e,t,s,null);if(n.data==="$!"){if(o=n.nextSibling&&n.nextSibling.dataset,o)var l=o.dgst;return o=l,a=Error(P(419)),o=ra(a,o,void 0),Mo(e,t,s,o)}if(l=(s&e.childLanes)!==0,ue||l){if(o=ee,o!==null){switch(s&-s){case 4:n=2;break;case 16:n=8;break;case 64:case 128:case 256:case 512:case 1024:case 2048:case 4096:case 8192:case 16384:case 32768:case 65536:case 131072:case 262144:case 524288:case 1048576:case 2097152:case 4194304:case 8388608:case 16777216:case 33554432:case 67108864:n=32;break;case 536870912:n=268435456;break;default:n=0}n=n&(o.suspendedLanes|s)?0:n,n!==0&&n!==a.retryLane&&(a.retryLane=n,$e(e,n),ze(o,e,n,-1))}return $r(),o=ra(Error(P(421))),Mo(e,t,s,o)}return n.data==="$?"?(t.flags|=128,t.child=e.child,t=Tu.bind(null,e),n._reactRetry=t,null):(e=a.treeContext,ve=mt(n.nextSibling),be=t,H=!0,Ee=null,e!==null&&(Pe[xe++]=Oe,Pe[xe++]=Fe,Pe[xe++]=jt,Oe=e.id,Fe=e.overflow,jt=t),t=qr(t,o.children),t.flags|=4096,t)}function al(e,t,i){e.lanes|=t;var o=e.alternate;o!==null&&(o.lanes|=t),Ua(e.return,t,i)}function sa(e,t,i,o,n){var a=e.memoizedState;a===null?e.memoizedState={isBackwards:t,rendering:null,renderingStartTime:0,last:o,tail:i,tailMode:n}:(a.isBackwards=t,a.rendering=null,a.renderingStartTime=0,a.last=o,a.tail=i,a.tailMode=n)}function fd(e,t,i){var o=t.pendingProps,n=o.revealOrder,a=o.tail;if(le(e,t,o.children,i),o=R.current,o&2)o=o&1|2,t.flags|=128;else{if(e!==null&&e.flags&128)e:for(e=t.child;e!==null;){if(e.tag===13)e.memoizedState!==null&&al(e,i,t);else if(e.tag===19)al(e,i,t);else if(e.child!==null){e.child.return=e,e=e.child;continue}if(e===t)break e;for(;e.sibling===null;){if(e.return===null||e.return===t)break e;e=e.return}e.sibling.return=e.return,e=e.sibling}o&=1}if(B(R,o),!(t.mode&1))t.memoizedState=null;else switch(n){case"forwards":for(i=t.child,n=null;i!==null;)e=i.alternate,e!==null&&ln(e)===null&&(n=i),i=i.sibling;i=n,i===null?(n=t.child,t.child=null):(n=i.sibling,i.sibling=null),sa(t,!1,n,i,a);break;case"backwards":for(i=null,n=t.child,t.child=null;n!==null;){if(e=n.alternate,e!==null&&ln(e)===null){t.child=n;break}e=n.sibling,n.sibling=i,i=n,n=e}sa(t,!0,i,null,a);break;case"together":sa(t,!1,null,null,void 0);break;default:t.memoizedState=null}return t.child}function Uo(e,t){!(t.mode&1)&&e!==null&&(e.alternate=null,t.alternate=null,t.flags|=2)}function Ke(e,t,i){if(e!==null&&(t.dependencies=e.dependencies),Bt|=t.lanes,!(i&t.childLanes))return null;if(e!==null&&t.child!==e.child)throw Error(P(153));if(t.child!==null){for(e=t.child,i=gt(e,e.pendingProps),t.child=i,i.return=t;e.sibling!==null;)e=e.sibling,i=i.sibling=gt(e,e.pendingProps),i.return=t;i.sibling=null}return t.child}function cu(e,t,i){switch(t.tag){case 3:yd(t),li();break;case 5:qc(t);break;case 1:ye(t.type)&&tn(t);break;case 4:zr(t,t.stateNode.containerInfo);break;case 10:var o=t.type._context,n=t.memoizedProps.value;B(an,o._currentValue),o._currentValue=n;break;case 13:if(o=t.memoizedState,o!==null)return o.dehydrated!==null?(B(R,R.current&1),t.flags|=128,null):i&t.child.childLanes?gd(e,t,i):(B(R,R.current&1),e=Ke(e,t,i),e!==null?e.sibling:null);B(R,R.current&1);break;case 19:if(o=(i&t.childLanes)!==0,e.flags&128){if(o)return fd(e,t,i);t.flags|=128}if(n=t.memoizedState,n!==null&&(n.rendering=null,n.tail=null,n.lastEffect=null),B(R,R.current),o)break;return null;case 22:case 23:return t.lanes=0,ud(e,t,i)}return Ke(e,t,i)}var vd,$a,bd,wd;vd=function(e,t){for(var i=t.child;i!==null;){if(i.tag===5||i.tag===6)e.appendChild(i.stateNode);else if(i.tag!==4&&i.child!==null){i.child.return=i,i=i.child;continue}if(i===t)break;for(;i.sibling===null;){if(i.return===null||i.return===t)return;i=i.return}i.sibling.return=i.return,i=i.sibling}};$a=function(){};bd=function(e,t,i,o){var n=e.memoizedProps;if(n!==o){e=t.stateNode,Gt(Ue.current);var a=null;switch(i){case"input":n=fa(e,n),o=fa(e,o),a=[];break;case"select":n=W({},n,{value:void 0}),o=W({},o,{value:void 0}),a=[];break;case"textarea":n=wa(e,n),o=wa(e,o),a=[];break;default:typeof n.onClick!="function"&&typeof o.onClick=="function"&&(e.onclick=Yo)}Sa(i,o);var s;i=null;for(p in n)if(!o.hasOwnProperty(p)&&n.hasOwnProperty(p)&&n[p]!=null)if(p==="style"){var l=n[p];for(s in l)l.hasOwnProperty(s)&&(i||(i={}),i[s]="")}else p!=="dangerouslySetInnerHTML"&&p!=="children"&&p!=="suppressContentEditableWarning"&&p!=="suppressHydrationWarning"&&p!=="autoFocus"&&(qi.hasOwnProperty(p)?a||(a=[]):(a=a||[]).push(p,null));for(p in o){var c=o[p];if(l=n!=null?n[p]:void 0,o.hasOwnProperty(p)&&c!==l&&(c!=null||l!=null))if(p==="style")if(l){for(s in l)!l.hasOwnProperty(s)||c&&c.hasOwnProperty(s)||(i||(i={}),i[s]="");for(s in c)c.hasOwnProperty(s)&&l[s]!==c[s]&&(i||(i={}),i[s]=c[s])}else i||(a||(a=[]),a.push(p,i)),i=c;else p==="dangerouslySetInnerHTML"?(c=c?c.__html:void 0,l=l?l.__html:void 0,c!=null&&l!==c&&(a=a||[]).push(p,c)):p==="children"?typeof c!="string"&&typeof c!="number"||(a=a||[]).push(p,""+c):p!=="suppressContentEditableWarning"&&p!=="suppressHydrationWarning"&&(qi.hasOwnProperty(p)?(c!=null&&p==="onScroll"&&D("scroll",e),a||l===c||(a=[])):(a=a||[]).push(p,c))}i&&(a=a||[]).push("style",i);var p=a;(t.updateQueue=p)&&(t.flags|=4)}};wd=function(e,t,i,o){i!==o&&(t.flags|=4)};function Ii(e,t){if(!H)switch(e.tailMode){case"hidden":t=e.tail;for(var i=null;t!==null;)t.alternate!==null&&(i=t),t=t.sibling;i===null?e.tail=null:i.sibling=null;break;case"collapsed":i=e.tail;for(var o=null;i!==null;)i.alternate!==null&&(o=i),i=i.sibling;o===null?t||e.tail===null?e.tail=null:e.tail.sibling=null:o.sibling=null}}function ae(e){var t=e.alternate!==null&&e.alternate.child===e.child,i=0,o=0;if(t)for(var n=e.child;n!==null;)i|=n.lanes|n.childLanes,o|=n.subtreeFlags&14680064,o|=n.flags&14680064,n.return=e,n=n.sibling;else for(n=e.child;n!==null;)i|=n.lanes|n.childLanes,o|=n.subtreeFlags,o|=n.flags,n.return=e,n=n.sibling;return e.subtreeFlags|=o,e.childLanes=i,t}function du(e,t,i){var o=t.pendingProps;switch(Ir(t),t.tag){case 2:case 16:case 15:case 0:case 11:case 7:case 8:case 12:case 9:case 14:return ae(t),null;case 1:return ye(t.type)&&en(),ae(t),null;case 3:return o=t.stateNode,di(),L(he),L(se),Dr(),o.pendingContext&&(o.context=o.pendingContext,o.pendingContext=null),(e===null||e.child===null)&&(Co(t)?t.flags|=4:e===null||e.memoizedState.isDehydrated&&!(t.flags&256)||(t.flags|=1024,Ee!==null&&(or(Ee),Ee=null))),$a(e,t),ae(t),null;case 5:Br(t);var n=Gt(to.current);if(i=t.type,e!==null&&t.stateNode!=null)bd(e,t,i,o,n),e.ref!==t.ref&&(t.flags|=512,t.flags|=2097152);else{if(!o){if(t.stateNode===null)throw Error(P(166));return ae(t),null}if(e=Gt(Ue.current),Co(t)){o=t.stateNode,i=t.type;var a=t.memoizedProps;switch(o[He]=t,o[Yi]=a,e=(t.mode&1)!==0,i){case"dialog":D("cancel",o),D("close",o);break;case"iframe":case"object":case"embed":D("load",o);break;case"video":case"audio":for(n=0;n<Ei.length;n++)D(Ei[n],o);break;case"source":D("error",o);break;case"img":case"image":case"link":D("error",o),D("load",o);break;case"details":D("toggle",o);break;case"input":hs(o,a),D("invalid",o);break;case"select":o._wrapperState={wasMultiple:!!a.multiple},D("invalid",o);break;case"textarea":gs(o,a),D("invalid",o)}Sa(i,a),n=null;for(var s in a)if(a.hasOwnProperty(s)){var l=a[s];s==="children"?typeof l=="string"?o.textContent!==l&&(a.suppressHydrationWarning!==!0&&Ao(o.textContent,l,e),n=["children",l]):typeof l=="number"&&o.textContent!==""+l&&(a.suppressHydrationWarning!==!0&&Ao(o.textContent,l,e),n=["children",""+l]):qi.hasOwnProperty(s)&&l!=null&&s==="onScroll"&&D("scroll",o)}switch(i){case"input":vo(o),ys(o,a,!0);break;case"textarea":vo(o),fs(o);break;case"select":case"option":break;default:typeof a.onClick=="function"&&(o.onclick=Yo)}o=n,t.updateQueue=o,o!==null&&(t.flags|=4)}else{s=n.nodeType===9?n:n.ownerDocument,e==="http://www.w3.org/1999/xhtml"&&(e=Vl(i)),e==="http://www.w3.org/1999/xhtml"?i==="script"?(e=s.createElement("div"),e.innerHTML="<script><\/script>",e=e.removeChild(e.firstChild)):typeof o.is=="string"?e=s.createElement(i,{is:o.is}):(e=s.createElement(i),i==="select"&&(s=e,o.multiple?s.multiple=!0:o.size&&(s.size=o.size))):e=s.createElementNS(e,i),e[He]=t,e[Yi]=o,vd(e,t,!1,!1),t.stateNode=e;e:{switch(s=Pa(i,o),i){case"dialog":D("cancel",e),D("close",e),n=o;break;case"iframe":case"object":case"embed":D("load",e),n=o;break;case"video":case"audio":for(n=0;n<Ei.length;n++)D(Ei[n],e);n=o;break;case"source":D("error",e),n=o;break;case"img":case"image":case"link":D("error",e),D("load",e),n=o;break;case"details":D("toggle",e),n=o;break;case"input":hs(e,o),n=fa(e,o),D("invalid",e);break;case"option":n=o;break;case"select":e._wrapperState={wasMultiple:!!o.multiple},n=W({},o,{value:void 0}),D("invalid",e);break;case"textarea":gs(e,o),n=wa(e,o),D("invalid",e);break;default:n=o}Sa(i,n),l=n;for(a in l)if(l.hasOwnProperty(a)){var c=l[a];a==="style"?Ql(e,c):a==="dangerouslySetInnerHTML"?(c=c?c.__html:void 0,c!=null&&$l(e,c)):a==="children"?typeof c=="string"?(i!=="textarea"||c!=="")&&Oi(e,c):typeof c=="number"&&Oi(e,""+c):a!=="suppressContentEditableWarning"&&a!=="suppressHydrationWarning"&&a!=="autoFocus"&&(qi.hasOwnProperty(a)?c!=null&&a==="onScroll"&&D("scroll",e):c!=null&&ur(e,a,c,s))}switch(i){case"input":vo(e),ys(e,o,!1);break;case"textarea":vo(e),fs(e);break;case"option":o.value!=null&&e.setAttribute("value",""+vt(o.value));break;case"select":e.multiple=!!o.multiple,a=o.value,a!=null?Yt(e,!!o.multiple,a,!1):o.defaultValue!=null&&Yt(e,!!o.multiple,o.defaultValue,!0);break;default:typeof n.onClick=="function"&&(e.onclick=Yo)}switch(i){case"button":case"input":case"select":case"textarea":o=!!o.autoFocus;break e;case"img":o=!0;break e;default:o=!1}}o&&(t.flags|=4)}t.ref!==null&&(t.flags|=512,t.flags|=2097152)}return ae(t),null;case 6:if(e&&t.stateNode!=null)wd(e,t,e.memoizedProps,o);else{if(typeof o!="string"&&t.stateNode===null)throw Error(P(166));if(i=Gt(to.current),Gt(Ue.current),Co(t)){if(o=t.stateNode,i=t.memoizedProps,o[He]=t,(a=o.nodeValue!==i)&&(e=be,e!==null))switch(e.tag){case 3:Ao(o.nodeValue,i,(e.mode&1)!==0);break;case 5:e.memoizedProps.suppressHydrationWarning!==!0&&Ao(o.nodeValue,i,(e.mode&1)!==0)}a&&(t.flags|=4)}else o=(i.nodeType===9?i:i.ownerDocument).createTextNode(o),o[He]=t,t.stateNode=o}return ae(t),null;case 13:if(L(R),o=t.memoizedState,e===null||e.memoizedState!==null&&e.memoizedState.dehydrated!==null){if(H&&ve!==null&&t.mode&1&&!(t.flags&128))Lc(),li(),t.flags|=98560,a=!1;else if(a=Co(t),o!==null&&o.dehydrated!==null){if(e===null){if(!a)throw Error(P(318));if(a=t.memoizedState,a=a!==null?a.dehydrated:null,!a)throw Error(P(317));a[He]=t}else li(),!(t.flags&128)&&(t.memoizedState=null),t.flags|=4;ae(t),a=!1}else Ee!==null&&(or(Ee),Ee=null),a=!0;if(!a)return t.flags&65536?t:null}return t.flags&128?(t.lanes=i,t):(o=o!==null,o!==(e!==null&&e.memoizedState!==null)&&o&&(t.child.flags|=8192,t.mode&1&&(e===null||R.current&1?Q===0&&(Q=3):$r())),t.updateQueue!==null&&(t.flags|=4),ae(t),null);case 4:return di(),$a(e,t),e===null&&Qi(t.stateNode.containerInfo),ae(t),null;case 10:return Nr(t.type._context),ae(t),null;case 17:return ye(t.type)&&en(),ae(t),null;case 19:if(L(R),a=t.memoizedState,a===null)return ae(t),null;if(o=(t.flags&128)!==0,s=a.rendering,s===null)if(o)Ii(a,!1);else{if(Q!==0||e!==null&&e.flags&128)for(e=t.child;e!==null;){if(s=ln(e),s!==null){for(t.flags|=128,Ii(a,!1),o=s.updateQueue,o!==null&&(t.updateQueue=o,t.flags|=4),t.subtreeFlags=0,o=i,i=t.child;i!==null;)a=i,e=o,a.flags&=14680066,s=a.alternate,s===null?(a.childLanes=0,a.lanes=e,a.child=null,a.subtreeFlags=0,a.memoizedProps=null,a.memoizedState=null,a.updateQueue=null,a.dependencies=null,a.stateNode=null):(a.childLanes=s.childLanes,a.lanes=s.lanes,a.child=s.child,a.subtreeFlags=0,a.deletions=null,a.memoizedProps=s.memoizedProps,a.memoizedState=s.memoizedState,a.updateQueue=s.updateQueue,a.type=s.type,e=s.dependencies,a.dependencies=e===null?null:{lanes:e.lanes,firstContext:e.firstContext}),i=i.sibling;return B(R,R.current&1|2),t.child}e=e.sibling}a.tail!==null&&_()>mi&&(t.flags|=128,o=!0,Ii(a,!1),t.lanes=4194304)}else{if(!o)if(e=ln(s),e!==null){if(t.flags|=128,o=!0,i=e.updateQueue,i!==null&&(t.updateQueue=i,t.flags|=4),Ii(a,!0),a.tail===null&&a.tailMode==="hidden"&&!s.alternate&&!H)return ae(t),null}else 2*_()-a.renderingStartTime>mi&&i!==1073741824&&(t.flags|=128,o=!0,Ii(a,!1),t.lanes=4194304);a.isBackwards?(s.sibling=t.child,t.child=s):(i=a.last,i!==null?i.sibling=s:t.child=s,a.last=s)}return a.tail!==null?(t=a.tail,a.rendering=t,a.tail=t.sibling,a.renderingStartTime=_(),t.sibling=null,i=R.current,B(R,o?i&1|2:i&1),t):(ae(t),null);case 22:case 23:return Vr(),o=t.memoizedState!==null,e!==null&&e.memoizedState!==null!==o&&(t.flags|=8192),o&&t.mode&1?fe&1073741824&&(ae(t),t.subtreeFlags&6&&(t.flags|=8192)):ae(t),null;case 24:return null;case 25:return null}throw Error(P(156,t.tag))}function pu(e,t){switch(Ir(t),t.tag){case 1:return ye(t.type)&&en(),e=t.flags,e&65536?(t.flags=e&-65537|128,t):null;case 3:return di(),L(he),L(se),Dr(),e=t.flags,e&65536&&!(e&128)?(t.flags=e&-65537|128,t):null;case 5:return Br(t),null;case 13:if(L(R),e=t.memoizedState,e!==null&&e.dehydrated!==null){if(t.alternate===null)throw Error(P(340));li()}return e=t.flags,e&65536?(t.flags=e&-65537|128,t):null;case 19:return L(R),null;case 4:return di(),null;case 10:return Nr(t.type._context),null;case 22:case 23:return Vr(),null;case 24:return null;default:return null}}var Go=!1,re=!1,mu=typeof WeakSet=="function"?WeakSet:Set,C=null;function Qt(e,t){var i=e.ref;if(i!==null)if(typeof i=="function")try{i(null)}catch(o){q(e,t,o)}else i.current=null}function Ka(e,t,i){try{i()}catch(o){q(e,t,o)}}var rl=!1;function uu(e,t){if(Ea=Ko,e=kc(),Ar(e)){if("selectionStart"in e)var i={start:e.selectionStart,end:e.selectionEnd};else e:{i=(i=e.ownerDocument)&&i.defaultView||window;var o=i.getSelection&&i.getSelection();if(o&&o.rangeCount!==0){i=o.anchorNode;var n=o.anchorOffset,a=o.focusNode;o=o.focusOffset;try{i.nodeType,a.nodeType}catch{i=null;break e}var s=0,l=-1,c=-1,p=0,y=0,u=e,g=null;t:for(;;){for(var b;u!==i||n!==0&&u.nodeType!==3||(l=s+n),u!==a||o!==0&&u.nodeType!==3||(c=s+o),u.nodeType===3&&(s+=u.nodeValue.length),(b=u.firstChild)!==null;)g=u,u=b;for(;;){if(u===e)break t;if(g===i&&++p===n&&(l=s),g===a&&++y===o&&(c=s),(b=u.nextSibling)!==null)break;u=g,g=u.parentNode}u=b}i=l===-1||c===-1?null:{start:l,end:c}}else i=null}i=i||{start:0,end:0}}else i=null;for(ja={focusedElem:e,selectionRange:i},Ko=!1,C=t;C!==null;)if(t=C,e=t.child,(t.subtreeFlags&1028)!==0&&e!==null)e.return=t,C=e;else for(;C!==null;){t=C;try{var w=t.alternate;if(t.flags&1024)switch(t.tag){case 0:case 11:case 15:break;case 1:if(w!==null){var v=w.memoizedProps,x=w.memoizedState,h=t.stateNode,d=h.getSnapshotBeforeUpdate(t.elementType===t.type?v:Je(t.type,v),x);h.__reactInternalSnapshotBeforeUpdate=d}break;case 3:var m=t.stateNode.containerInfo;m.nodeType===1?m.textContent="":m.nodeType===9&&m.documentElement&&m.removeChild(m.documentElement);break;case 5:case 6:case 4:case 17:break;default:throw Error(P(163))}}catch(f){q(t,t.return,f)}if(e=t.sibling,e!==null){e.return=t.return,C=e;break}C=t.return}return w=rl,rl=!1,w}function Ri(e,t,i){var o=t.updateQueue;if(o=o!==null?o.lastEffect:null,o!==null){var n=o=o.next;do{if((n.tag&e)===e){var a=n.destroy;n.destroy=void 0,a!==void 0&&Ka(t,i,a)}n=n.next}while(n!==o)}}function An(e,t){if(t=t.updateQueue,t=t!==null?t.lastEffect:null,t!==null){var i=t=t.next;do{if((i.tag&e)===e){var o=i.create;i.destroy=o()}i=i.next}while(i!==t)}}function Qa(e){var t=e.ref;if(t!==null){var i=e.stateNode;switch(e.tag){case 5:e=i;break;default:e=i}typeof t=="function"?t(e):t.current=e}}function Td(e){var t=e.alternate;t!==null&&(e.alternate=null,Td(t)),e.child=null,e.deletions=null,e.sibling=null,e.tag===5&&(t=e.stateNode,t!==null&&(delete t[He],delete t[Yi],delete t[Da],delete t[$m],delete t[Km])),e.stateNode=null,e.return=null,e.dependencies=null,e.memoizedProps=null,e.memoizedState=null,e.pendingProps=null,e.stateNode=null,e.updateQueue=null}function Sd(e){return e.tag===5||e.tag===3||e.tag===4}function sl(e){e:for(;;){for(;e.sibling===null;){if(e.return===null||Sd(e.return))return null;e=e.return}for(e.sibling.return=e.return,e=e.sibling;e.tag!==5&&e.tag!==6&&e.tag!==18;){if(e.flags&2||e.child===null||e.tag===4)continue e;e.child.return=e,e=e.child}if(!(e.flags&2))return e.stateNode}}function Xa(e,t,i){var o=e.tag;if(o===5||o===6)e=e.stateNode,t?i.nodeType===8?i.parentNode.insertBefore(e,t):i.insertBefore(e,t):(i.nodeType===8?(t=i.parentNode,t.insertBefore(e,i)):(t=i,t.appendChild(e)),i=i._reactRootContainer,i!=null||t.onclick!==null||(t.onclick=Yo));else if(o!==4&&(e=e.child,e!==null))for(Xa(e,t,i),e=e.sibling;e!==null;)Xa(e,t,i),e=e.sibling}function Ya(e,t,i){var o=e.tag;if(o===5||o===6)e=e.stateNode,t?i.insertBefore(e,t):i.appendChild(e);else if(o!==4&&(e=e.child,e!==null))for(Ya(e,t,i),e=e.sibling;e!==null;)Ya(e,t,i),e=e.sibling}var te=null,Ne=!1;function Ye(e,t,i){for(i=i.child;i!==null;)Pd(e,t,i),i=i.sibling}function Pd(e,t,i){if(Re&&typeof Re.onCommitFiberUnmount=="function")try{Re.onCommitFiberUnmount(vn,i)}catch{}switch(i.tag){case 5:re||Qt(i,t);case 6:var o=te,n=Ne;te=null,Ye(e,t,i),te=o,Ne=n,te!==null&&(Ne?(e=te,i=i.stateNode,e.nodeType===8?e.parentNode.removeChild(i):e.removeChild(i)):te.removeChild(i.stateNode));break;case 18:te!==null&&(Ne?(e=te,i=i.stateNode,e.nodeType===8?ea(e.parentNode,i):e.nodeType===1&&ea(e,i),Vi(e)):ea(te,i.stateNode));break;case 4:o=te,n=Ne,te=i.stateNode.containerInfo,Ne=!0,Ye(e,t,i),te=o,Ne=n;break;case 0:case 11:case 14:case 15:if(!re&&(o=i.updateQueue,o!==null&&(o=o.lastEffect,o!==null))){n=o=o.next;do{var a=n,s=a.destroy;a=a.tag,s!==void 0&&(a&2||a&4)&&Ka(i,t,s),n=n.next}while(n!==o)}Ye(e,t,i);break;case 1:if(!re&&(Qt(i,t),o=i.stateNode,typeof o.componentWillUnmount=="function"))try{o.props=i.memoizedProps,o.state=i.memoizedState,o.componentWillUnmount()}catch(l){q(i,t,l)}Ye(e,t,i);break;case 21:Ye(e,t,i);break;case 22:i.mode&1?(re=(o=re)||i.memoizedState!==null,Ye(e,t,i),re=o):Ye(e,t,i);break;default:Ye(e,t,i)}}function ll(e){var t=e.updateQueue;if(t!==null){e.updateQueue=null;var i=e.stateNode;i===null&&(i=e.stateNode=new mu),t.forEach(function(o){var n=Su.bind(null,e,o);i.has(o)||(i.add(o),o.then(n,n))})}}function Ge(e,t){var i=t.deletions;if(i!==null)for(var o=0;o<i.length;o++){var n=i[o];try{var a=e,s=t,l=s;e:for(;l!==null;){switch(l.tag){case 5:te=l.stateNode,Ne=!1;break e;case 3:te=l.stateNode.containerInfo,Ne=!0;break e;case 4:te=l.stateNode.containerInfo,Ne=!0;break e}l=l.return}if(te===null)throw Error(P(160));Pd(a,s,n),te=null,Ne=!1;var c=n.alternate;c!==null&&(c.return=null),n.return=null}catch(p){q(n,t,p)}}if(t.subtreeFlags&12854)for(t=t.child;t!==null;)xd(t,e),t=t.sibling}function xd(e,t){var i=e.alternate,o=e.flags;switch(e.tag){case 0:case 11:case 14:case 15:if(Ge(t,e),De(e),o&4){try{Ri(3,e,e.return),An(3,e)}catch(v){q(e,e.return,v)}try{Ri(5,e,e.return)}catch(v){q(e,e.return,v)}}break;case 1:Ge(t,e),De(e),o&512&&i!==null&&Qt(i,i.return);break;case 5:if(Ge(t,e),De(e),o&512&&i!==null&&Qt(i,i.return),e.flags&32){var n=e.stateNode;try{Oi(n,"")}catch(v){q(e,e.return,v)}}if(o&4&&(n=e.stateNode,n!=null)){var a=e.memoizedProps,s=i!==null?i.memoizedProps:a,l=e.type,c=e.updateQueue;if(e.updateQueue=null,c!==null)try{l==="input"&&a.type==="radio"&&a.name!=null&&Zl(n,a),Pa(l,s);var p=Pa(l,a);for(s=0;s<c.length;s+=2){var y=c[s],u=c[s+1];y==="style"?Ql(n,u):y==="dangerouslySetInnerHTML"?$l(n,u):y==="children"?Oi(n,u):ur(n,y,u,p)}switch(l){case"input":va(n,a);break;case"textarea":_l(n,a);break;case"select":var g=n._wrapperState.wasMultiple;n._wrapperState.wasMultiple=!!a.multiple;var b=a.value;b!=null?Yt(n,!!a.multiple,b,!1):g!==!!a.multiple&&(a.defaultValue!=null?Yt(n,!!a.multiple,a.defaultValue,!0):Yt(n,!!a.multiple,a.multiple?[]:"",!1))}n[Yi]=a}catch(v){q(e,e.return,v)}}break;case 6:if(Ge(t,e),De(e),o&4){if(e.stateNode===null)throw Error(P(162));n=e.stateNode,a=e.memoizedProps;try{n.nodeValue=a}catch(v){q(e,e.return,v)}}break;case 3:if(Ge(t,e),De(e),o&4&&i!==null&&i.memoizedState.isDehydrated)try{Vi(t.containerInfo)}catch(v){q(e,e.return,v)}break;case 4:Ge(t,e),De(e);break;case 13:Ge(t,e),De(e),n=e.child,n.flags&8192&&(a=n.memoizedState!==null,n.stateNode.isHidden=a,!a||n.alternate!==null&&n.alternate.memoizedState!==null||(Zr=_())),o&4&&ll(e);break;case 22:if(y=i!==null&&i.memoizedState!==null,e.mode&1?(re=(p=re)||y,Ge(t,e),re=p):Ge(t,e),De(e),o&8192){if(p=e.memoizedState!==null,(e.stateNode.isHidden=p)&&!y&&e.mode&1)for(C=e,y=e.child;y!==null;){for(u=C=y;C!==null;){switch(g=C,b=g.child,g.tag){case 0:case 11:case 14:case 15:Ri(4,g,g.return);break;case 1:Qt(g,g.return);var w=g.stateNode;if(typeof w.componentWillUnmount=="function"){o=g,i=g.return;try{t=o,w.props=t.memoizedProps,w.state=t.memoizedState,w.componentWillUnmount()}catch(v){q(o,i,v)}}break;case 5:Qt(g,g.return);break;case 22:if(g.memoizedState!==null){dl(u);continue}}b!==null?(b.return=g,C=b):dl(u)}y=y.sibling}e:for(y=null,u=e;;){if(u.tag===5){if(y===null){y=u;try{n=u.stateNode,p?(a=n.style,typeof a.setProperty=="function"?a.setProperty("display","none","important"):a.display="none"):(l=u.stateNode,c=u.memoizedProps.style,s=c!=null&&c.hasOwnProperty("display")?c.display:null,l.style.display=Kl("display",s))}catch(v){q(e,e.return,v)}}}else if(u.tag===6){if(y===null)try{u.stateNode.nodeValue=p?"":u.memoizedProps}catch(v){q(e,e.return,v)}}else if((u.tag!==22&&u.tag!==23||u.memoizedState===null||u===e)&&u.child!==null){u.child.return=u,u=u.child;continue}if(u===e)break e;for(;u.sibling===null;){if(u.return===null||u.return===e)break e;y===u&&(y=null),u=u.return}y===u&&(y=null),u.sibling.return=u.return,u=u.sibling}}break;case 19:Ge(t,e),De(e),o&4&&ll(e);break;case 21:break;default:Ge(t,e),De(e)}}function De(e){var t=e.flags;if(t&2){try{e:{for(var i=e.return;i!==null;){if(Sd(i)){var o=i;break e}i=i.return}throw Error(P(160))}switch(o.tag){case 5:var n=o.stateNode;o.flags&32&&(Oi(n,""),o.flags&=-33);var a=sl(e);Ya(e,a,n);break;case 3:case 4:var s=o.stateNode.containerInfo,l=sl(e);Xa(e,l,s);break;default:throw Error(P(161))}}catch(c){q(e,e.return,c)}e.flags&=-3}t&4096&&(e.flags&=-4097)}function hu(e,t,i){C=e,kd(e)}function kd(e,t,i){for(var o=(e.mode&1)!==0;C!==null;){var n=C,a=n.child;if(n.tag===22&&o){var s=n.memoizedState!==null||Go;if(!s){var l=n.alternate,c=l!==null&&l.memoizedState!==null||re;l=Go;var p=re;if(Go=s,(re=c)&&!p)for(C=n;C!==null;)s=C,c=s.child,s.tag===22&&s.memoizedState!==null?pl(n):c!==null?(c.return=s,C=c):pl(n);for(;a!==null;)C=a,kd(a),a=a.sibling;C=n,Go=l,re=p}cl(e)}else n.subtreeFlags&8772&&a!==null?(a.return=n,C=a):cl(e)}}function cl(e){for(;C!==null;){var t=C;if(t.flags&8772){var i=t.alternate;try{if(t.flags&8772)switch(t.tag){case 0:case 11:case 15:re||An(5,t);break;case 1:var o=t.stateNode;if(t.flags&4&&!re)if(i===null)o.componentDidMount();else{var n=t.elementType===t.type?i.memoizedProps:Je(t.type,i.memoizedProps);o.componentDidUpdate(n,i.memoizedState,o.__reactInternalSnapshotBeforeUpdate)}var a=t.updateQueue;a!==null&&_s(t,a,o);break;case 3:var s=t.updateQueue;if(s!==null){if(i=null,t.child!==null)switch(t.child.tag){case 5:i=t.child.stateNode;break;case 1:i=t.child.stateNode}_s(t,s,i)}break;case 5:var l=t.stateNode;if(i===null&&t.flags&4){i=l;var c=t.memoizedProps;switch(t.type){case"button":case"input":case"select":case"textarea":c.autoFocus&&i.focus();break;case"img":c.src&&(i.src=c.src)}}break;case 6:break;case 4:break;case 12:break;case 13:if(t.memoizedState===null){var p=t.alternate;if(p!==null){var y=p.memoizedState;if(y!==null){var u=y.dehydrated;u!==null&&Vi(u)}}}break;case 19:case 17:case 21:case 22:case 23:case 25:break;default:throw Error(P(163))}re||t.flags&512&&Qa(t)}catch(g){q(t,t.return,g)}}if(t===e){C=null;break}if(i=t.sibling,i!==null){i.return=t.return,C=i;break}C=t.return}}function dl(e){for(;C!==null;){var t=C;if(t===e){C=null;break}var i=t.sibling;if(i!==null){i.return=t.return,C=i;break}C=t.return}}function pl(e){for(;C!==null;){var t=C;try{switch(t.tag){case 0:case 11:case 15:var i=t.return;try{An(4,t)}catch(c){q(t,i,c)}break;case 1:var o=t.stateNode;if(typeof o.componentDidMount=="function"){var n=t.return;try{o.componentDidMount()}catch(c){q(t,n,c)}}var a=t.return;try{Qa(t)}catch(c){q(t,a,c)}break;case 5:var s=t.return;try{Qa(t)}catch(c){q(t,s,c)}}}catch(c){q(t,t.return,c)}if(t===e){C=null;break}var l=t.sibling;if(l!==null){l.return=t.return,C=l;break}C=t.return}}var yu=Math.ceil,pn=Qe.ReactCurrentDispatcher,Or=Qe.ReactCurrentOwner,Ae=Qe.ReactCurrentBatchConfig,j=0,ee=null,$=null,ie=0,fe=0,Xt=Tt(0),Q=0,ao=null,Bt=0,Cn=0,Fr=0,Ui=null,me=null,Zr=0,mi=1/0,We=null,mn=!1,er=null,ht=null,Jo=!1,rt=null,un=0,Wi=0,tr=null,Wo=-1,qo=0;function ce(){return j&6?_():Wo!==-1?Wo:Wo=_()}function yt(e){return e.mode&1?j&2&&ie!==0?ie&-ie:Xm.transition!==null?(qo===0&&(qo=cc()),qo):(e=z,e!==0||(e=window.event,e=e===void 0?16:gc(e.type)),e):1}function ze(e,t,i,o){if(50<Wi)throw Wi=0,tr=null,Error(P(185));co(e,i,o),(!(j&2)||e!==ee)&&(e===ee&&(!(j&2)&&(Cn|=i),Q===4&&nt(e,ie)),ge(e,o),i===1&&j===0&&!(t.mode&1)&&(mi=_()+500,Pn&&St()))}function ge(e,t){var i=e.callbackNode;Xp(e,t);var o=$o(e,e===ee?ie:0);if(o===0)i!==null&&ws(i),e.callbackNode=null,e.callbackPriority=0;else if(t=o&-o,e.callbackPriority!==t){if(i!=null&&ws(i),t===1)e.tag===0?Qm(ml.bind(null,e)):zc(ml.bind(null,e)),_m(function(){!(j&6)&&St()}),i=null;else{switch(dc(o)){case 1:i=vr;break;case 4:i=sc;break;case 16:i=Vo;break;case 536870912:i=lc;break;default:i=Vo}i=Ed(i,Ad.bind(null,e))}e.callbackPriority=t,e.callbackNode=i}}function Ad(e,t){if(Wo=-1,qo=0,j&6)throw Error(P(327));var i=e.callbackNode;if(ni()&&e.callbackNode!==i)return null;var o=$o(e,e===ee?ie:0);if(o===0)return null;if(o&30||o&e.expiredLanes||t)t=hn(e,o);else{t=o;var n=j;j|=2;var a=Id();(ee!==e||ie!==t)&&(We=null,mi=_()+500,Jt(e,t));do try{vu();break}catch(l){Cd(e,l)}while(!0);Jr(),pn.current=a,j=n,$!==null?t=0:(ee=null,ie=0,t=Q)}if(t!==0){if(t===2&&(n=Ia(e),n!==0&&(o=n,t=ir(e,n))),t===1)throw i=ao,Jt(e,0),nt(e,o),ge(e,_()),i;if(t===6)nt(e,o);else{if(n=e.current.alternate,!(o&30)&&!gu(n)&&(t=hn(e,o),t===2&&(a=Ia(e),a!==0&&(o=a,t=ir(e,a))),t===1))throw i=ao,Jt(e,0),nt(e,o),ge(e,_()),i;switch(e.finishedWork=n,e.finishedLanes=o,t){case 0:case 1:throw Error(P(345));case 2:Ct(e,me,We);break;case 3:if(nt(e,o),(o&130023424)===o&&(t=Zr+500-_(),10<t)){if($o(e,0)!==0)break;if(n=e.suspendedLanes,(n&o)!==o){ce(),e.pingedLanes|=e.suspendedLanes&n;break}e.timeoutHandle=Ba(Ct.bind(null,e,me,We),t);break}Ct(e,me,We);break;case 4:if(nt(e,o),(o&4194240)===o)break;for(t=e.eventTimes,n=-1;0<o;){var s=31-je(o);a=1<<s,s=t[s],s>n&&(n=s),o&=~a}if(o=n,o=_()-o,o=(120>o?120:480>o?480:1080>o?1080:1920>o?1920:3e3>o?3e3:4320>o?4320:1960*yu(o/1960))-o,10<o){e.timeoutHandle=Ba(Ct.bind(null,e,me,We),o);break}Ct(e,me,We);break;case 5:Ct(e,me,We);break;default:throw Error(P(329))}}}return ge(e,_()),e.callbackNode===i?Ad.bind(null,e):null}function ir(e,t){var i=Ui;return e.current.memoizedState.isDehydrated&&(Jt(e,t).flags|=256),e=hn(e,t),e!==2&&(t=me,me=i,t!==null&&or(t)),e}function or(e){me===null?me=e:me.push.apply(me,e)}function gu(e){for(var t=e;;){if(t.flags&16384){var i=t.updateQueue;if(i!==null&&(i=i.stores,i!==null))for(var o=0;o<i.length;o++){var n=i[o],a=n.getSnapshot;n=n.value;try{if(!Be(a(),n))return!1}catch{return!1}}}if(i=t.child,t.subtreeFlags&16384&&i!==null)i.return=t,t=i;else{if(t===e)break;for(;t.sibling===null;){if(t.return===null||t.return===e)return!0;t=t.return}t.sibling.return=t.return,t=t.sibling}}return!0}function nt(e,t){for(t&=~Fr,t&=~Cn,e.suspendedLanes|=t,e.pingedLanes&=~t,e=e.expirationTimes;0<t;){var i=31-je(t),o=1<<i;e[i]=-1,t&=~o}}function ml(e){if(j&6)throw Error(P(327));ni();var t=$o(e,0);if(!(t&1))return ge(e,_()),null;var i=hn(e,t);if(e.tag!==0&&i===2){var o=Ia(e);o!==0&&(t=o,i=ir(e,o))}if(i===1)throw i=ao,Jt(e,0),nt(e,t),ge(e,_()),i;if(i===6)throw Error(P(345));return e.finishedWork=e.current.alternate,e.finishedLanes=t,Ct(e,me,We),ge(e,_()),null}function _r(e,t){var i=j;j|=1;try{return e(t)}finally{j=i,j===0&&(mi=_()+500,Pn&&St())}}function Dt(e){rt!==null&&rt.tag===0&&!(j&6)&&ni();var t=j;j|=1;var i=Ae.transition,o=z;try{if(Ae.transition=null,z=1,e)return e()}finally{z=o,Ae.transition=i,j=t,!(j&6)&&St()}}function Vr(){fe=Xt.current,L(Xt)}function Jt(e,t){e.finishedWork=null,e.finishedLanes=0;var i=e.timeoutHandle;if(i!==-1&&(e.timeoutHandle=-1,Zm(i)),$!==null)for(i=$.return;i!==null;){var o=i;switch(Ir(o),o.tag){case 1:o=o.type.childContextTypes,o!=null&&en();break;case 3:di(),L(he),L(se),Dr();break;case 5:Br(o);break;case 4:di();break;case 13:L(R);break;case 19:L(R);break;case 10:Nr(o.type._context);break;case 22:case 23:Vr()}i=i.return}if(ee=e,$=e=gt(e.current,null),ie=fe=t,Q=0,ao=null,Fr=Cn=Bt=0,me=Ui=null,Mt!==null){for(t=0;t<Mt.length;t++)if(i=Mt[t],o=i.interleaved,o!==null){i.interleaved=null;var n=o.next,a=i.pending;if(a!==null){var s=a.next;a.next=n,o.next=s}i.pending=o}Mt=null}return e}function Cd(e,t){do{var i=$;try{if(Jr(),Ho.current=dn,cn){for(var o=U.memoizedState;o!==null;){var n=o.queue;n!==null&&(n.pending=null),o=o.next}cn=!1}if(zt=0,Y=K=U=null,Hi=!1,io=0,Or.current=null,i===null||i.return===null){Q=1,ao=t,$=null;break}e:{var a=e,s=i.return,l=i,c=t;if(t=ie,l.flags|=32768,c!==null&&typeof c=="object"&&typeof c.then=="function"){var p=c,y=l,u=y.tag;if(!(y.mode&1)&&(u===0||u===11||u===15)){var g=y.alternate;g?(y.updateQueue=g.updateQueue,y.memoizedState=g.memoizedState,y.lanes=g.lanes):(y.updateQueue=null,y.memoizedState=null)}var b=Ys(s);if(b!==null){b.flags&=-257,el(b,s,l,a,t),b.mode&1&&Xs(a,p,t),t=b,c=p;var w=t.updateQueue;if(w===null){var v=new Set;v.add(c),t.updateQueue=v}else w.add(c);break e}else{if(!(t&1)){Xs(a,p,t),$r();break e}c=Error(P(426))}}else if(H&&l.mode&1){var x=Ys(s);if(x!==null){!(x.flags&65536)&&(x.flags|=256),el(x,s,l,a,t),Mr(pi(c,l));break e}}a=c=pi(c,l),Q!==4&&(Q=2),Ui===null?Ui=[a]:Ui.push(a),a=s;do{switch(a.tag){case 3:a.flags|=65536,t&=-t,a.lanes|=t;var h=dd(a,c,t);Zs(a,h);break e;case 1:l=c;var d=a.type,m=a.stateNode;if(!(a.flags&128)&&(typeof d.getDerivedStateFromError=="function"||m!==null&&typeof m.componentDidCatch=="function"&&(ht===null||!ht.has(m)))){a.flags|=65536,t&=-t,a.lanes|=t;var f=pd(a,l,t);Zs(a,f);break e}}a=a.return}while(a!==null)}Gd(i)}catch(S){t=S,$===i&&i!==null&&($=i=i.return);continue}break}while(!0)}function Id(){var e=pn.current;return pn.current=dn,e===null?dn:e}function $r(){(Q===0||Q===3||Q===2)&&(Q=4),ee===null||!(Bt&268435455)&&!(Cn&268435455)||nt(ee,ie)}function hn(e,t){var i=j;j|=2;var o=Id();(ee!==e||ie!==t)&&(We=null,Jt(e,t));do try{fu();break}catch(n){Cd(e,n)}while(!0);if(Jr(),j=i,pn.current=o,$!==null)throw Error(P(261));return ee=null,ie=0,Q}function fu(){for(;$!==null;)Md($)}function vu(){for(;$!==null&&!qp();)Md($)}function Md(e){var t=Nd(e.alternate,e,fe);e.memoizedProps=e.pendingProps,t===null?Gd(e):$=t,Or.current=null}function Gd(e){var t=e;do{var i=t.alternate;if(e=t.return,t.flags&32768){if(i=pu(i,t),i!==null){i.flags&=32767,$=i;return}if(e!==null)e.flags|=32768,e.subtreeFlags=0,e.deletions=null;else{Q=6,$=null;return}}else if(i=du(i,t,fe),i!==null){$=i;return}if(t=t.sibling,t!==null){$=t;return}$=t=e}while(t!==null);Q===0&&(Q=5)}function Ct(e,t,i){var o=z,n=Ae.transition;try{Ae.transition=null,z=1,bu(e,t,i,o)}finally{Ae.transition=n,z=o}return null}function bu(e,t,i,o){do ni();while(rt!==null);if(j&6)throw Error(P(327));i=e.finishedWork;var n=e.finishedLanes;if(i===null)return null;if(e.finishedWork=null,e.finishedLanes=0,i===e.current)throw Error(P(177));e.callbackNode=null,e.callbackPriority=0;var a=i.lanes|i.childLanes;if(Yp(e,a),e===ee&&($=ee=null,ie=0),!(i.subtreeFlags&2064)&&!(i.flags&2064)||Jo||(Jo=!0,Ed(Vo,function(){return ni(),null})),a=(i.flags&15990)!==0,i.subtreeFlags&15990||a){a=Ae.transition,Ae.transition=null;var s=z;z=1;var l=j;j|=4,Or.current=null,uu(e,i),xd(i,e),Hm(ja),Ko=!!Ea,ja=Ea=null,e.current=i,hu(i),Op(),j=l,z=s,Ae.transition=a}else e.current=i;if(Jo&&(Jo=!1,rt=e,un=n),a=e.pendingLanes,a===0&&(ht=null),_p(i.stateNode),ge(e,_()),t!==null)for(o=e.onRecoverableError,i=0;i<t.length;i++)n=t[i],o(n.value,{componentStack:n.stack,digest:n.digest});if(mn)throw mn=!1,e=er,er=null,e;return un&1&&e.tag!==0&&ni(),a=e.pendingLanes,a&1?e===tr?Wi++:(Wi=0,tr=e):Wi=0,St(),null}function ni(){if(rt!==null){var e=dc(un),t=Ae.transition,i=z;try{if(Ae.transition=null,z=16>e?16:e,rt===null)var o=!1;else{if(e=rt,rt=null,un=0,j&6)throw Error(P(331));var n=j;for(j|=4,C=e.current;C!==null;){var a=C,s=a.child;if(C.flags&16){var l=a.deletions;if(l!==null){for(var c=0;c<l.length;c++){var p=l[c];for(C=p;C!==null;){var y=C;switch(y.tag){case 0:case 11:case 15:Ri(8,y,a)}var u=y.child;if(u!==null)u.return=y,C=u;else for(;C!==null;){y=C;var g=y.sibling,b=y.return;if(Td(y),y===p){C=null;break}if(g!==null){g.return=b,C=g;break}C=b}}}var w=a.alternate;if(w!==null){var v=w.child;if(v!==null){w.child=null;do{var x=v.sibling;v.sibling=null,v=x}while(v!==null)}}C=a}}if(a.subtreeFlags&2064&&s!==null)s.return=a,C=s;else e:for(;C!==null;){if(a=C,a.flags&2048)switch(a.tag){case 0:case 11:case 15:Ri(9,a,a.return)}var h=a.sibling;if(h!==null){h.return=a.return,C=h;break e}C=a.return}}var d=e.current;for(C=d;C!==null;){s=C;var m=s.child;if(s.subtreeFlags&2064&&m!==null)m.return=s,C=m;else e:for(s=d;C!==null;){if(l=C,l.flags&2048)try{switch(l.tag){case 0:case 11:case 15:An(9,l)}}catch(S){q(l,l.return,S)}if(l===s){C=null;break e}var f=l.sibling;if(f!==null){f.return=l.return,C=f;break e}C=l.return}}if(j=n,St(),Re&&typeof Re.onPostCommitFiberRoot=="function")try{Re.onPostCommitFiberRoot(vn,e)}catch{}o=!0}return o}finally{z=i,Ae.transition=t}}return!1}function ul(e,t,i){t=pi(i,t),t=dd(e,t,1),e=ut(e,t,1),t=ce(),e!==null&&(co(e,1,t),ge(e,t))}function q(e,t,i){if(e.tag===3)ul(e,e,i);else for(;t!==null;){if(t.tag===3){ul(t,e,i);break}else if(t.tag===1){var o=t.stateNode;if(typeof t.type.getDerivedStateFromError=="function"||typeof o.componentDidCatch=="function"&&(ht===null||!ht.has(o))){e=pi(i,e),e=pd(t,e,1),t=ut(t,e,1),e=ce(),t!==null&&(co(t,1,e),ge(t,e));break}}t=t.return}}function wu(e,t,i){var o=e.pingCache;o!==null&&o.delete(t),t=ce(),e.pingedLanes|=e.suspendedLanes&i,ee===e&&(ie&i)===i&&(Q===4||Q===3&&(ie&130023424)===ie&&500>_()-Zr?Jt(e,0):Fr|=i),ge(e,t)}function Jd(e,t){t===0&&(e.mode&1?(t=To,To<<=1,!(To&130023424)&&(To=4194304)):t=1);var i=ce();e=$e(e,t),e!==null&&(co(e,t,i),ge(e,i))}function Tu(e){var t=e.memoizedState,i=0;t!==null&&(i=t.retryLane),Jd(e,i)}function Su(e,t){var i=0;switch(e.tag){case 13:var o=e.stateNode,n=e.memoizedState;n!==null&&(i=n.retryLane);break;case 19:o=e.stateNode;break;default:throw Error(P(314))}o!==null&&o.delete(t),Jd(e,i)}var Nd;Nd=function(e,t,i){if(e!==null)if(e.memoizedProps!==t.pendingProps||he.current)ue=!0;else{if(!(e.lanes&i)&&!(t.flags&128))return ue=!1,cu(e,t,i);ue=!!(e.flags&131072)}else ue=!1,H&&t.flags&1048576&&Bc(t,nn,t.index);switch(t.lanes=0,t.tag){case 2:var o=t.type;Uo(e,t),e=t.pendingProps;var n=si(t,se.current);oi(t,i),n=Hr(null,t,o,e,n,i);var a=Rr();return t.flags|=1,typeof n=="object"&&n!==null&&typeof n.render=="function"&&n.$$typeof===void 0?(t.tag=1,t.memoizedState=null,t.updateQueue=null,ye(o)?(a=!0,tn(t)):a=!1,t.memoizedState=n.state!==null&&n.state!==void 0?n.state:null,jr(t),n.updater=kn,t.stateNode=n,n._reactInternals=t,qa(t,o,e,i),t=Za(null,t,o,!0,a,i)):(t.tag=0,H&&a&&Cr(t),le(null,t,n,i),t=t.child),t;case 16:o=t.elementType;e:{switch(Uo(e,t),e=t.pendingProps,n=o._init,o=n(o._payload),t.type=o,n=t.tag=xu(o),e=Je(o,e),n){case 0:t=Fa(null,t,o,e,i);break e;case 1:t=ol(null,t,o,e,i);break e;case 11:t=tl(null,t,o,e,i);break e;case 14:t=il(null,t,o,Je(o.type,e),i);break e}throw Error(P(306,o,""))}return t;case 0:return o=t.type,n=t.pendingProps,n=t.elementType===o?n:Je(o,n),Fa(e,t,o,n,i);case 1:return o=t.type,n=t.pendingProps,n=t.elementType===o?n:Je(o,n),ol(e,t,o,n,i);case 3:e:{if(yd(t),e===null)throw Error(P(387));o=t.pendingProps,a=t.memoizedState,n=a.element,Wc(e,t),sn(t,o,null,i);var s=t.memoizedState;if(o=s.element,a.isDehydrated)if(a={element:o,isDehydrated:!1,cache:s.cache,pendingSuspenseBoundaries:s.pendingSuspenseBoundaries,transitions:s.transitions},t.updateQueue.baseState=a,t.memoizedState=a,t.flags&256){n=pi(Error(P(423)),t),t=nl(e,t,o,i,n);break e}else if(o!==n){n=pi(Error(P(424)),t),t=nl(e,t,o,i,n);break e}else for(ve=mt(t.stateNode.containerInfo.firstChild),be=t,H=!0,Ee=null,i=Rc(t,null,o,i),t.child=i;i;)i.flags=i.flags&-3|4096,i=i.sibling;else{if(li(),o===n){t=Ke(e,t,i);break e}le(e,t,o,i)}t=t.child}return t;case 5:return qc(t),e===null&&Ra(t),o=t.type,n=t.pendingProps,a=e!==null?e.memoizedProps:null,s=n.children,za(o,n)?s=null:a!==null&&za(o,a)&&(t.flags|=32),hd(e,t),le(e,t,s,i),t.child;case 6:return e===null&&Ra(t),null;case 13:return gd(e,t,i);case 4:return zr(t,t.stateNode.containerInfo),o=t.pendingProps,e===null?t.child=ci(t,null,o,i):le(e,t,o,i),t.child;case 11:return o=t.type,n=t.pendingProps,n=t.elementType===o?n:Je(o,n),tl(e,t,o,n,i);case 7:return le(e,t,t.pendingProps,i),t.child;case 8:return le(e,t,t.pendingProps.children,i),t.child;case 12:return le(e,t,t.pendingProps.children,i),t.child;case 10:e:{if(o=t.type._context,n=t.pendingProps,a=t.memoizedProps,s=n.value,B(an,o._currentValue),o._currentValue=s,a!==null)if(Be(a.value,s)){if(a.children===n.children&&!he.current){t=Ke(e,t,i);break e}}else for(a=t.child,a!==null&&(a.return=t);a!==null;){var l=a.dependencies;if(l!==null){s=a.child;for(var c=l.firstContext;c!==null;){if(c.context===o){if(a.tag===1){c=Ze(-1,i&-i),c.tag=2;var p=a.updateQueue;if(p!==null){p=p.shared;var y=p.pending;y===null?c.next=c:(c.next=y.next,y.next=c),p.pending=c}}a.lanes|=i,c=a.alternate,c!==null&&(c.lanes|=i),Ua(a.return,i,t),l.lanes|=i;break}c=c.next}}else if(a.tag===10)s=a.type===t.type?null:a.child;else if(a.tag===18){if(s=a.return,s===null)throw Error(P(341));s.lanes|=i,l=s.alternate,l!==null&&(l.lanes|=i),Ua(s,i,t),s=a.sibling}else s=a.child;if(s!==null)s.return=a;else for(s=a;s!==null;){if(s===t){s=null;break}if(a=s.sibling,a!==null){a.return=s.return,s=a;break}s=s.return}a=s}le(e,t,n.children,i),t=t.child}return t;case 9:return n=t.type,o=t.pendingProps.children,oi(t,i),n=Ce(n),o=o(n),t.flags|=1,le(e,t,o,i),t.child;case 14:return o=t.type,n=Je(o,t.pendingProps),n=Je(o.type,n),il(e,t,o,n,i);case 15:return md(e,t,t.type,t.pendingProps,i);case 17:return o=t.type,n=t.pendingProps,n=t.elementType===o?n:Je(o,n),Uo(e,t),t.tag=1,ye(o)?(e=!0,tn(t)):e=!1,oi(t,i),cd(t,o,n),qa(t,o,n,i),Za(null,t,o,!0,e,i);case 19:return fd(e,t,i);case 22:return ud(e,t,i)}throw Error(P(156,t.tag))};function Ed(e,t){return rc(e,t)}function Pu(e,t,i,o){this.tag=e,this.key=i,this.sibling=this.child=this.return=this.stateNode=this.type=this.elementType=null,this.index=0,this.ref=null,this.pendingProps=t,this.dependencies=this.memoizedState=this.updateQueue=this.memoizedProps=null,this.mode=o,this.subtreeFlags=this.flags=0,this.deletions=null,this.childLanes=this.lanes=0,this.alternate=null}function ke(e,t,i,o){return new Pu(e,t,i,o)}function Kr(e){return e=e.prototype,!(!e||!e.isReactComponent)}function xu(e){if(typeof e=="function")return Kr(e)?1:0;if(e!=null){if(e=e.$$typeof,e===yr)return 11;if(e===gr)return 14}return 2}function gt(e,t){var i=e.alternate;return i===null?(i=ke(e.tag,t,e.key,e.mode),i.elementType=e.elementType,i.type=e.type,i.stateNode=e.stateNode,i.alternate=e,e.alternate=i):(i.pendingProps=t,i.type=e.type,i.flags=0,i.subtreeFlags=0,i.deletions=null),i.flags=e.flags&14680064,i.childLanes=e.childLanes,i.lanes=e.lanes,i.child=e.child,i.memoizedProps=e.memoizedProps,i.memoizedState=e.memoizedState,i.updateQueue=e.updateQueue,t=e.dependencies,i.dependencies=t===null?null:{lanes:t.lanes,firstContext:t.firstContext},i.sibling=e.sibling,i.index=e.index,i.ref=e.ref,i}function Oo(e,t,i,o,n,a){var s=2;if(o=e,typeof e=="function")Kr(e)&&(s=1);else if(typeof e=="string")s=5;else e:switch(e){case Wt:return Nt(i.children,n,a,t);case hr:s=8,n|=8;break;case ua:return e=ke(12,i,t,n|2),e.elementType=ua,e.lanes=a,e;case ha:return e=ke(13,i,t,n),e.elementType=ha,e.lanes=a,e;case ya:return e=ke(19,i,t,n),e.elementType=ya,e.lanes=a,e;case ql:return In(i,n,a,t);default:if(typeof e=="object"&&e!==null)switch(e.$$typeof){case Ul:s=10;break e;case Wl:s=9;break e;case yr:s=11;break e;case gr:s=14;break e;case tt:s=16,o=null;break e}throw Error(P(130,e==null?e:typeof e,""))}return t=ke(s,i,t,n),t.elementType=e,t.type=o,t.lanes=a,t}function Nt(e,t,i,o){return e=ke(7,e,o,t),e.lanes=i,e}function In(e,t,i,o){return e=ke(22,e,o,t),e.elementType=ql,e.lanes=i,e.stateNode={isHidden:!1},e}function la(e,t,i){return e=ke(6,e,null,t),e.lanes=i,e}function ca(e,t,i){return t=ke(4,e.children!==null?e.children:[],e.key,t),t.lanes=i,t.stateNode={containerInfo:e.containerInfo,pendingChildren:null,implementation:e.implementation},t}function ku(e,t,i,o,n){this.tag=t,this.containerInfo=e,this.finishedWork=this.pingCache=this.current=this.pendingChildren=null,this.timeoutHandle=-1,this.callbackNode=this.pendingContext=this.context=null,this.callbackPriority=0,this.eventTimes=qn(0),this.expirationTimes=qn(-1),this.entangledLanes=this.finishedLanes=this.mutableReadLanes=this.expiredLanes=this.pingedLanes=this.suspendedLanes=this.pendingLanes=0,this.entanglements=qn(0),this.identifierPrefix=o,this.onRecoverableError=n,this.mutableSourceEagerHydrationData=null}function Qr(e,t,i,o,n,a,s,l,c){return e=new ku(e,t,i,l,c),t===1?(t=1,a===!0&&(t|=8)):t=0,a=ke(3,null,null,t),e.current=a,a.stateNode=e,a.memoizedState={element:o,isDehydrated:i,cache:null,transitions:null,pendingSuspenseBoundaries:null},jr(a),e}function Au(e,t,i){var o=3<arguments.length&&arguments[3]!==void 0?arguments[3]:null;return{$$typeof:Ut,key:o==null?null:""+o,children:e,containerInfo:t,implementation:i}}function jd(e){if(!e)return bt;e=e._reactInternals;e:{if(Ht(e)!==e||e.tag!==1)throw Error(P(170));var t=e;do{switch(t.tag){case 3:t=t.stateNode.context;break e;case 1:if(ye(t.type)){t=t.stateNode.__reactInternalMemoizedMergedChildContext;break e}}t=t.return}while(t!==null);throw Error(P(171))}if(e.tag===1){var i=e.type;if(ye(i))return jc(e,i,t)}return t}function zd(e,t,i,o,n,a,s,l,c){return e=Qr(i,o,!0,e,n,a,s,l,c),e.context=jd(null),i=e.current,o=ce(),n=yt(i),a=Ze(o,n),a.callback=t??null,ut(i,a,n),e.current.lanes=n,co(e,n,o),ge(e,o),e}function Mn(e,t,i,o){var n=t.current,a=ce(),s=yt(n);return i=jd(i),t.context===null?t.context=i:t.pendingContext=i,t=Ze(a,s),t.payload={element:e},o=o===void 0?null:o,o!==null&&(t.callback=o),e=ut(n,t,s),e!==null&&(ze(e,n,s,a),Lo(e,n,s)),s}function yn(e){if(e=e.current,!e.child)return null;switch(e.child.tag){case 5:return e.child.stateNode;default:return e.child.stateNode}}function hl(e,t){if(e=e.memoizedState,e!==null&&e.dehydrated!==null){var i=e.retryLane;e.retryLane=i!==0&&i<t?i:t}}function Xr(e,t){hl(e,t),(e=e.alternate)&&hl(e,t)}function Cu(){return null}var Bd=typeof reportError=="function"?reportError:function(e){console.error(e)};function Yr(e){this._internalRoot=e}Gn.prototype.render=Yr.prototype.render=function(e){var t=this._internalRoot;if(t===null)throw Error(P(409));Mn(e,t,null,null)};Gn.prototype.unmount=Yr.prototype.unmount=function(){var e=this._internalRoot;if(e!==null){this._internalRoot=null;var t=e.containerInfo;Dt(function(){Mn(null,e,null,null)}),t[Ve]=null}};function Gn(e){this._internalRoot=e}Gn.prototype.unstable_scheduleHydration=function(e){if(e){var t=uc();e={blockedOn:null,target:e,priority:t};for(var i=0;i<ot.length&&t!==0&&t<ot[i].priority;i++);ot.splice(i,0,e),i===0&&yc(e)}};function es(e){return!(!e||e.nodeType!==1&&e.nodeType!==9&&e.nodeType!==11)}function Jn(e){return!(!e||e.nodeType!==1&&e.nodeType!==9&&e.nodeType!==11&&(e.nodeType!==8||e.nodeValue!==" react-mount-point-unstable "))}function yl(){}function Iu(e,t,i,o,n){if(n){if(typeof o=="function"){var a=o;o=function(){var p=yn(s);a.call(p)}}var s=zd(t,o,e,0,null,!1,!1,"",yl);return e._reactRootContainer=s,e[Ve]=s.current,Qi(e.nodeType===8?e.parentNode:e),Dt(),s}for(;n=e.lastChild;)e.removeChild(n);if(typeof o=="function"){var l=o;o=function(){var p=yn(c);l.call(p)}}var c=Qr(e,0,!1,null,null,!1,!1,"",yl);return e._reactRootContainer=c,e[Ve]=c.current,Qi(e.nodeType===8?e.parentNode:e),Dt(function(){Mn(t,c,i,o)}),c}function Nn(e,t,i,o,n){var a=i._reactRootContainer;if(a){var s=a;if(typeof n=="function"){var l=n;n=function(){var c=yn(s);l.call(c)}}Mn(t,s,e,n)}else s=Iu(i,t,e,n,o);return yn(s)}pc=function(e){switch(e.tag){case 3:var t=e.stateNode;if(t.current.memoizedState.isDehydrated){var i=Ni(t.pendingLanes);i!==0&&(br(t,i|1),ge(t,_()),!(j&6)&&(mi=_()+500,St()))}break;case 13:Dt(function(){var o=$e(e,1);if(o!==null){var n=ce();ze(o,e,1,n)}}),Xr(e,1)}};wr=function(e){if(e.tag===13){var t=$e(e,134217728);if(t!==null){var i=ce();ze(t,e,134217728,i)}Xr(e,134217728)}};mc=function(e){if(e.tag===13){var t=yt(e),i=$e(e,t);if(i!==null){var o=ce();ze(i,e,t,o)}Xr(e,t)}};uc=function(){return z};hc=function(e,t){var i=z;try{return z=e,t()}finally{z=i}};ka=function(e,t,i){switch(t){case"input":if(va(e,i),t=i.name,i.type==="radio"&&t!=null){for(i=e;i.parentNode;)i=i.parentNode;for(i=i.querySelectorAll("input[name="+JSON.stringify(""+t)+'][type="radio"]'),t=0;t<i.length;t++){var o=i[t];if(o!==e&&o.form===e.form){var n=Sn(o);if(!n)throw Error(P(90));Fl(o),va(o,n)}}}break;case"textarea":_l(e,i);break;case"select":t=i.value,t!=null&&Yt(e,!!i.multiple,t,!1)}};ec=_r;tc=Dt;var Mu={usingClientEntryPoint:!1,Events:[mo,Zt,Sn,Xl,Yl,_r]},Mi={findFiberByHostInstance:It,bundleType:0,version:"18.3.1",rendererPackageName:"react-dom"},Gu={bundleType:Mi.bundleType,version:Mi.version,rendererPackageName:Mi.rendererPackageName,rendererConfig:Mi.rendererConfig,overrideHookState:null,overrideHookStateDeletePath:null,overrideHookStateRenamePath:null,overrideProps:null,overridePropsDeletePath:null,overridePropsRenamePath:null,setErrorHandler:null,setSuspenseHandler:null,scheduleUpdate:null,currentDispatcherRef:Qe.ReactCurrentDispatcher,findHostInstanceByFiber:function(e){return e=nc(e),e===null?null:e.stateNode},findFiberByHostInstance:Mi.findFiberByHostInstance||Cu,findHostInstancesForRefresh:null,scheduleRefresh:null,scheduleRoot:null,setRefreshHandler:null,getCurrentFiber:null,reconcilerVersion:"18.3.1-next-f1338f8080-20240426"};if(typeof __REACT_DEVTOOLS_GLOBAL_HOOK__<"u"){var No=__REACT_DEVTOOLS_GLOBAL_HOOK__;if(!No.isDisabled&&No.supportsFiber)try{vn=No.inject(Gu),Re=No}catch{}}Te.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED=Mu;Te.createPortal=function(e,t){var i=2<arguments.length&&arguments[2]!==void 0?arguments[2]:null;if(!es(t))throw Error(P(200));return Au(e,t,null,i)};Te.createRoot=function(e,t){if(!es(e))throw Error(P(299));var i=!1,o="",n=Bd;return t!=null&&(t.unstable_strictMode===!0&&(i=!0),t.identifierPrefix!==void 0&&(o=t.identifierPrefix),t.onRecoverableError!==void 0&&(n=t.onRecoverableError)),t=Qr(e,1,!1,null,null,i,!1,o,n),e[Ve]=t.current,Qi(e.nodeType===8?e.parentNode:e),new Yr(t)};Te.findDOMNode=function(e){if(e==null)return null;if(e.nodeType===1)return e;var t=e._reactInternals;if(t===void 0)throw typeof e.render=="function"?Error(P(188)):(e=Object.keys(e).join(","),Error(P(268,e)));return e=nc(t),e=e===null?null:e.stateNode,e};Te.flushSync=function(e){return Dt(e)};Te.hydrate=function(e,t,i){if(!Jn(t))throw Error(P(200));return Nn(null,e,t,!0,i)};Te.hydrateRoot=function(e,t,i){if(!es(e))throw Error(P(405));var o=i!=null&&i.hydratedSources||null,n=!1,a="",s=Bd;if(i!=null&&(i.unstable_strictMode===!0&&(n=!0),i.identifierPrefix!==void 0&&(a=i.identifierPrefix),i.onRecoverableError!==void 0&&(s=i.onRecoverableError)),t=zd(t,null,e,1,i??null,n,!1,a,s),e[Ve]=t.current,Qi(e),o)for(e=0;e<o.length;e++)i=o[e],n=i._getVersion,n=n(i._source),t.mutableSourceEagerHydrationData==null?t.mutableSourceEagerHydrationData=[i,n]:t.mutableSourceEagerHydrationData.push(i,n);return new Gn(t)};Te.render=function(e,t,i){if(!Jn(t))throw Error(P(200));return Nn(null,e,t,!1,i)};Te.unmountComponentAtNode=function(e){if(!Jn(e))throw Error(P(40));return e._reactRootContainer?(Dt(function(){Nn(null,null,e,!1,function(){e._reactRootContainer=null,e[Ve]=null})}),!0):!1};Te.unstable_batchedUpdates=_r;Te.unstable_renderSubtreeIntoContainer=function(e,t,i,o){if(!Jn(i))throw Error(P(200));if(e==null||e._reactInternals===void 0)throw Error(P(38));return Nn(e,t,i,!1,o)};Te.version="18.3.1-next-f1338f8080-20240426";function Dd(){if(!(typeof __REACT_DEVTOOLS_GLOBAL_HOOK__>"u"||typeof __REACT_DEVTOOLS_GLOBAL_HOOK__.checkDCE!="function"))try{__REACT_DEVTOOLS_GLOBAL_HOOK__.checkDCE(Dd)}catch(e){console.error(e)}}Dd(),Dl.exports=Te;var Ju=Dl.exports,Ld,gl=Ju;Ld=gl.createRoot,gl.hydrateRoot;/**
 * @remix-run/router v1.23.0
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */function ro(){return ro=Object.assign?Object.assign.bind():function(e){for(var t=1;t<arguments.length;t++){var i=arguments[t];for(var o in i)Object.prototype.hasOwnProperty.call(i,o)&&(e[o]=i[o])}return e},ro.apply(this,arguments)}var st;(function(e){e.Pop="POP",e.Push="PUSH",e.Replace="REPLACE"})(st||(st={}));const fl="popstate";function Nu(e){e===void 0&&(e={});function t(o,n){let{pathname:a,search:s,hash:l}=o.location;return nr("",{pathname:a,search:s,hash:l},n.state&&n.state.usr||null,n.state&&n.state.key||"default")}function i(o,n){return typeof n=="string"?n:gn(n)}return ju(t,i,null,e)}function V(e,t){if(e===!1||e===null||typeof e>"u")throw new Error(t)}function Hd(e,t){if(!e){typeof console<"u"&&console.warn(t);try{throw new Error(t)}catch{}}}function Eu(){return Math.random().toString(36).substr(2,8)}function vl(e,t){return{usr:e.state,key:e.key,idx:t}}function nr(e,t,i,o){return i===void 0&&(i=null),ro({pathname:typeof e=="string"?e:e.pathname,search:"",hash:""},typeof t=="string"?gi(t):t,{state:i,key:t&&t.key||o||Eu()})}function gn(e){let{pathname:t="/",search:i="",hash:o=""}=e;return i&&i!=="?"&&(t+=i.charAt(0)==="?"?i:"?"+i),o&&o!=="#"&&(t+=o.charAt(0)==="#"?o:"#"+o),t}function gi(e){let t={};if(e){let i=e.indexOf("#");i>=0&&(t.hash=e.substr(i),e=e.substr(0,i));let o=e.indexOf("?");o>=0&&(t.search=e.substr(o),e=e.substr(0,o)),e&&(t.pathname=e)}return t}function ju(e,t,i,o){o===void 0&&(o={});let{window:n=document.defaultView,v5Compat:a=!1}=o,s=n.history,l=st.Pop,c=null,p=y();p==null&&(p=0,s.replaceState(ro({},s.state,{idx:p}),""));function y(){return(s.state||{idx:null}).idx}function u(){l=st.Pop;let x=y(),h=x==null?null:x-p;p=x,c&&c({action:l,location:v.location,delta:h})}function g(x,h){l=st.Push;let d=nr(v.location,x,h);p=y()+1;let m=vl(d,p),f=v.createHref(d);try{s.pushState(m,"",f)}catch(S){if(S instanceof DOMException&&S.name==="DataCloneError")throw S;n.location.assign(f)}a&&c&&c({action:l,location:v.location,delta:1})}function b(x,h){l=st.Replace;let d=nr(v.location,x,h);p=y();let m=vl(d,p),f=v.createHref(d);s.replaceState(m,"",f),a&&c&&c({action:l,location:v.location,delta:0})}function w(x){let h=n.location.origin!=="null"?n.location.origin:n.location.href,d=typeof x=="string"?x:gn(x);return d=d.replace(/ $/,"%20"),V(h,"No window.location.(origin|href) available to create URL for href: "+d),new URL(d,h)}let v={get action(){return l},get location(){return e(n,s)},listen(x){if(c)throw new Error("A history only accepts one active listener");return n.addEventListener(fl,u),c=x,()=>{n.removeEventListener(fl,u),c=null}},createHref(x){return t(n,x)},createURL:w,encodeLocation(x){let h=w(x);return{pathname:h.pathname,search:h.search,hash:h.hash}},push:g,replace:b,go(x){return s.go(x)}};return v}var bl;(function(e){e.data="data",e.deferred="deferred",e.redirect="redirect",e.error="error"})(bl||(bl={}));function zu(e,t,i){return i===void 0&&(i="/"),Bu(e,t,i)}function Bu(e,t,i,o){let n=typeof t=="string"?gi(t):t,a=ts(n.pathname||"/",i);if(a==null)return null;let s=Rd(e);Du(s);let l=null;for(let c=0;l==null&&c<s.length;++c){let p=$u(a);l=Zu(s[c],p)}return l}function Rd(e,t,i,o){t===void 0&&(t=[]),i===void 0&&(i=[]),o===void 0&&(o="");let n=(a,s,l)=>{let c={relativePath:l===void 0?a.path||"":l,caseSensitive:a.caseSensitive===!0,childrenIndex:s,route:a};c.relativePath.startsWith("/")&&(V(c.relativePath.startsWith(o),'Absolute route path "'+c.relativePath+'" nested under path '+('"'+o+'" is not valid. An absolute child route path ')+"must start with the combined path of all its parent routes."),c.relativePath=c.relativePath.slice(o.length));let p=ft([o,c.relativePath]),y=i.concat(c);a.children&&a.children.length>0&&(V(a.index!==!0,"Index routes must not have child routes. Please remove "+('all child routes from route path "'+p+'".')),Rd(a.children,t,y,p)),!(a.path==null&&!a.index)&&t.push({path:p,score:Ou(p,a.index),routesMeta:y})};return e.forEach((a,s)=>{var l;if(a.path===""||!((l=a.path)!=null&&l.includes("?")))n(a,s);else for(let c of Ud(a.path))n(a,s,c)}),t}function Ud(e){let t=e.split("/");if(t.length===0)return[];let[i,...o]=t,n=i.endsWith("?"),a=i.replace(/\?$/,"");if(o.length===0)return n?[a,""]:[a];let s=Ud(o.join("/")),l=[];return l.push(...s.map(c=>c===""?a:[a,c].join("/"))),n&&l.push(...s),l.map(c=>e.startsWith("/")&&c===""?"/":c)}function Du(e){e.sort((t,i)=>t.score!==i.score?i.score-t.score:Fu(t.routesMeta.map(o=>o.childrenIndex),i.routesMeta.map(o=>o.childrenIndex)))}const Lu=/^:[\w-]+$/,Hu=3,Ru=2,Uu=1,Wu=10,qu=-2,wl=e=>e==="*";function Ou(e,t){let i=e.split("/"),o=i.length;return i.some(wl)&&(o+=qu),t&&(o+=Ru),i.filter(n=>!wl(n)).reduce((n,a)=>n+(Lu.test(a)?Hu:a===""?Uu:Wu),o)}function Fu(e,t){return e.length===t.length&&e.slice(0,-1).every((o,n)=>o===t[n])?e[e.length-1]-t[t.length-1]:0}function Zu(e,t,i){let{routesMeta:o}=e,n={},a="/",s=[];for(let l=0;l<o.length;++l){let c=o[l],p=l===o.length-1,y=a==="/"?t:t.slice(a.length)||"/",u=_u({path:c.relativePath,caseSensitive:c.caseSensitive,end:p},y),g=c.route;if(!u)return null;Object.assign(n,u.params),s.push({params:n,pathname:ft([a,u.pathname]),pathnameBase:Yu(ft([a,u.pathnameBase])),route:g}),u.pathnameBase!=="/"&&(a=ft([a,u.pathnameBase]))}return s}function _u(e,t){typeof e=="string"&&(e={path:e,caseSensitive:!1,end:!0});let[i,o]=Vu(e.path,e.caseSensitive,e.end),n=t.match(i);if(!n)return null;let a=n[0],s=a.replace(/(.)\/+$/,"$1"),l=n.slice(1);return{params:o.reduce((p,y,u)=>{let{paramName:g,isOptional:b}=y;if(g==="*"){let v=l[u]||"";s=a.slice(0,a.length-v.length).replace(/(.)\/+$/,"$1")}const w=l[u];return b&&!w?p[g]=void 0:p[g]=(w||"").replace(/%2F/g,"/"),p},{}),pathname:a,pathnameBase:s,pattern:e}}function Vu(e,t,i){t===void 0&&(t=!1),i===void 0&&(i=!0),Hd(e==="*"||!e.endsWith("*")||e.endsWith("/*"),'Route path "'+e+'" will be treated as if it were '+('"'+e.replace(/\*$/,"/*")+'" because the `*` character must ')+"always follow a `/` in the pattern. To get rid of this warning, "+('please change the route path to "'+e.replace(/\*$/,"/*")+'".'));let o=[],n="^"+e.replace(/\/*\*?$/,"").replace(/^\/*/,"/").replace(/[\\.*+^${}|()[\]]/g,"\\$&").replace(/\/:([\w-]+)(\?)?/g,(s,l,c)=>(o.push({paramName:l,isOptional:c!=null}),c?"/?([^\\/]+)?":"/([^\\/]+)"));return e.endsWith("*")?(o.push({paramName:"*"}),n+=e==="*"||e==="/*"?"(.*)$":"(?:\\/(.+)|\\/*)$"):i?n+="\\/*$":e!==""&&e!=="/"&&(n+="(?:(?=\\/|$))"),[new RegExp(n,t?void 0:"i"),o]}function $u(e){try{return e.split("/").map(t=>decodeURIComponent(t).replace(/\//g,"%2F")).join("/")}catch(t){return Hd(!1,'The URL path "'+e+'" could not be decoded because it is is a malformed URL segment. This is probably due to a bad percent '+("encoding ("+t+").")),e}}function ts(e,t){if(t==="/")return e;if(!e.toLowerCase().startsWith(t.toLowerCase()))return null;let i=t.endsWith("/")?t.length-1:t.length,o=e.charAt(i);return o&&o!=="/"?null:e.slice(i)||"/"}function Ku(e,t){t===void 0&&(t="/");let{pathname:i,search:o="",hash:n=""}=typeof e=="string"?gi(e):e;return{pathname:i?i.startsWith("/")?i:Qu(i,t):t,search:eh(o),hash:th(n)}}function Qu(e,t){let i=t.replace(/\/+$/,"").split("/");return e.split("/").forEach(n=>{n===".."?i.length>1&&i.pop():n!=="."&&i.push(n)}),i.length>1?i.join("/"):"/"}function da(e,t,i,o){return"Cannot include a '"+e+"' character in a manually specified "+("`to."+t+"` field ["+JSON.stringify(o)+"].  Please separate it out to the ")+("`to."+i+"` field. Alternatively you may provide the full path as ")+'a string in <Link to="..."> and the router will parse it for you.'}function Xu(e){return e.filter((t,i)=>i===0||t.route.path&&t.route.path.length>0)}function is(e,t){let i=Xu(e);return t?i.map((o,n)=>n===i.length-1?o.pathname:o.pathnameBase):i.map(o=>o.pathnameBase)}function os(e,t,i,o){o===void 0&&(o=!1);let n;typeof e=="string"?n=gi(e):(n=ro({},e),V(!n.pathname||!n.pathname.includes("?"),da("?","pathname","search",n)),V(!n.pathname||!n.pathname.includes("#"),da("#","pathname","hash",n)),V(!n.search||!n.search.includes("#"),da("#","search","hash",n)));let a=e===""||n.pathname==="",s=a?"/":n.pathname,l;if(s==null)l=i;else{let u=t.length-1;if(!o&&s.startsWith("..")){let g=s.split("/");for(;g[0]==="..";)g.shift(),u-=1;n.pathname=g.join("/")}l=u>=0?t[u]:"/"}let c=Ku(n,l),p=s&&s!=="/"&&s.endsWith("/"),y=(a||s===".")&&i.endsWith("/");return!c.pathname.endsWith("/")&&(p||y)&&(c.pathname+="/"),c}const ft=e=>e.join("/").replace(/\/\/+/g,"/"),Yu=e=>e.replace(/\/+$/,"").replace(/^\/*/,"/"),eh=e=>!e||e==="?"?"":e.startsWith("?")?e:"?"+e,th=e=>!e||e==="#"?"":e.startsWith("#")?e:"#"+e;function ih(e){return e!=null&&typeof e.status=="number"&&typeof e.statusText=="string"&&typeof e.internal=="boolean"&&"data"in e}const Wd=["post","put","patch","delete"];new Set(Wd);const oh=["get",...Wd];new Set(oh);/**
 * React Router v6.30.1
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */function so(){return so=Object.assign?Object.assign.bind():function(e){for(var t=1;t<arguments.length;t++){var i=arguments[t];for(var o in i)Object.prototype.hasOwnProperty.call(i,o)&&(e[o]=i[o])}return e},so.apply(this,arguments)}const ns=T.createContext(null),nh=T.createContext(null),Pt=T.createContext(null),En=T.createContext(null),Xe=T.createContext({outlet:null,matches:[],isDataRoute:!1}),qd=T.createContext(null);function ah(e,t){let{relative:i}=t===void 0?{}:t;fi()||V(!1);let{basename:o,navigator:n}=T.useContext(Pt),{hash:a,pathname:s,search:l}=Zd(e,{relative:i}),c=s;return o!=="/"&&(c=s==="/"?o:ft([o,s])),n.createHref({pathname:c,search:l,hash:a})}function fi(){return T.useContext(En)!=null}function vi(){return fi()||V(!1),T.useContext(En).location}function Od(e){T.useContext(Pt).static||T.useLayoutEffect(e)}function Fd(){let{isDataRoute:e}=T.useContext(Xe);return e?bh():rh()}function rh(){fi()||V(!1);let e=T.useContext(ns),{basename:t,future:i,navigator:o}=T.useContext(Pt),{matches:n}=T.useContext(Xe),{pathname:a}=vi(),s=JSON.stringify(is(n,i.v7_relativeSplatPath)),l=T.useRef(!1);return Od(()=>{l.current=!0}),T.useCallback(function(p,y){if(y===void 0&&(y={}),!l.current)return;if(typeof p=="number"){o.go(p);return}let u=os(p,JSON.parse(s),a,y.relative==="path");e==null&&t!=="/"&&(u.pathname=u.pathname==="/"?t:ft([t,u.pathname])),(y.replace?o.replace:o.push)(u,y.state,y)},[t,o,s,a,e])}function sh(){let{matches:e}=T.useContext(Xe),t=e[e.length-1];return t?t.params:{}}function Zd(e,t){let{relative:i}=t===void 0?{}:t,{future:o}=T.useContext(Pt),{matches:n}=T.useContext(Xe),{pathname:a}=vi(),s=JSON.stringify(is(n,o.v7_relativeSplatPath));return T.useMemo(()=>os(e,JSON.parse(s),a,i==="path"),[e,s,a,i])}function lh(e,t){return ch(e,t)}function ch(e,t,i,o){fi()||V(!1);let{navigator:n}=T.useContext(Pt),{matches:a}=T.useContext(Xe),s=a[a.length-1],l=s?s.params:{};s&&s.pathname;let c=s?s.pathnameBase:"/";s&&s.route;let p=vi(),y;if(t){var u;let x=typeof t=="string"?gi(t):t;c==="/"||(u=x.pathname)!=null&&u.startsWith(c)||V(!1),y=x}else y=p;let g=y.pathname||"/",b=g;if(c!=="/"){let x=c.replace(/^\//,"").split("/");b="/"+g.replace(/^\//,"").split("/").slice(x.length).join("/")}let w=zu(e,{pathname:b}),v=hh(w&&w.map(x=>Object.assign({},x,{params:Object.assign({},l,x.params),pathname:ft([c,n.encodeLocation?n.encodeLocation(x.pathname).pathname:x.pathname]),pathnameBase:x.pathnameBase==="/"?c:ft([c,n.encodeLocation?n.encodeLocation(x.pathnameBase).pathname:x.pathnameBase])})),a,i,o);return t&&v?T.createElement(En.Provider,{value:{location:so({pathname:"/",search:"",hash:"",state:null,key:"default"},y),navigationType:st.Pop}},v):v}function dh(){let e=vh(),t=ih(e)?e.status+" "+e.statusText:e instanceof Error?e.message:JSON.stringify(e),i=e instanceof Error?e.stack:null,n={padding:"0.5rem",backgroundColor:"rgba(200,200,200, 0.5)"};return T.createElement(T.Fragment,null,T.createElement("h2",null,"Unexpected Application Error!"),T.createElement("h3",{style:{fontStyle:"italic"}},t),i?T.createElement("pre",{style:n},i):null,null)}const ph=T.createElement(dh,null);class mh extends T.Component{constructor(t){super(t),this.state={location:t.location,revalidation:t.revalidation,error:t.error}}static getDerivedStateFromError(t){return{error:t}}static getDerivedStateFromProps(t,i){return i.location!==t.location||i.revalidation!=="idle"&&t.revalidation==="idle"?{error:t.error,location:t.location,revalidation:t.revalidation}:{error:t.error!==void 0?t.error:i.error,location:i.location,revalidation:t.revalidation||i.revalidation}}componentDidCatch(t,i){console.error("React Router caught the following error during render",t,i)}render(){return this.state.error!==void 0?T.createElement(Xe.Provider,{value:this.props.routeContext},T.createElement(qd.Provider,{value:this.state.error,children:this.props.component})):this.props.children}}function uh(e){let{routeContext:t,match:i,children:o}=e,n=T.useContext(ns);return n&&n.static&&n.staticContext&&(i.route.errorElement||i.route.ErrorBoundary)&&(n.staticContext._deepestRenderedBoundaryId=i.route.id),T.createElement(Xe.Provider,{value:t},o)}function hh(e,t,i,o){var n;if(t===void 0&&(t=[]),i===void 0&&(i=null),o===void 0&&(o=null),e==null){var a;if(!i)return null;if(i.errors)e=i.matches;else if((a=o)!=null&&a.v7_partialHydration&&t.length===0&&!i.initialized&&i.matches.length>0)e=i.matches;else return null}let s=e,l=(n=i)==null?void 0:n.errors;if(l!=null){let y=s.findIndex(u=>u.route.id&&(l==null?void 0:l[u.route.id])!==void 0);y>=0||V(!1),s=s.slice(0,Math.min(s.length,y+1))}let c=!1,p=-1;if(i&&o&&o.v7_partialHydration)for(let y=0;y<s.length;y++){let u=s[y];if((u.route.HydrateFallback||u.route.hydrateFallbackElement)&&(p=y),u.route.id){let{loaderData:g,errors:b}=i,w=u.route.loader&&g[u.route.id]===void 0&&(!b||b[u.route.id]===void 0);if(u.route.lazy||w){c=!0,p>=0?s=s.slice(0,p+1):s=[s[0]];break}}}return s.reduceRight((y,u,g)=>{let b,w=!1,v=null,x=null;i&&(b=l&&u.route.id?l[u.route.id]:void 0,v=u.route.errorElement||ph,c&&(p<0&&g===0?(wh("route-fallback"),w=!0,x=null):p===g&&(w=!0,x=u.route.hydrateFallbackElement||null)));let h=t.concat(s.slice(0,g+1)),d=()=>{let m;return b?m=v:w?m=x:u.route.Component?m=T.createElement(u.route.Component,null):u.route.element?m=u.route.element:m=y,T.createElement(uh,{match:u,routeContext:{outlet:y,matches:h,isDataRoute:i!=null},children:m})};return i&&(u.route.ErrorBoundary||u.route.errorElement||g===0)?T.createElement(mh,{location:i.location,revalidation:i.revalidation,component:v,error:b,children:d(),routeContext:{outlet:null,matches:h,isDataRoute:!0}}):d()},null)}var _d=function(e){return e.UseBlocker="useBlocker",e.UseRevalidator="useRevalidator",e.UseNavigateStable="useNavigate",e}(_d||{}),Vd=function(e){return e.UseBlocker="useBlocker",e.UseLoaderData="useLoaderData",e.UseActionData="useActionData",e.UseRouteError="useRouteError",e.UseNavigation="useNavigation",e.UseRouteLoaderData="useRouteLoaderData",e.UseMatches="useMatches",e.UseRevalidator="useRevalidator",e.UseNavigateStable="useNavigate",e.UseRouteId="useRouteId",e}(Vd||{});function yh(e){let t=T.useContext(ns);return t||V(!1),t}function gh(e){let t=T.useContext(nh);return t||V(!1),t}function fh(e){let t=T.useContext(Xe);return t||V(!1),t}function $d(e){let t=fh(),i=t.matches[t.matches.length-1];return i.route.id||V(!1),i.route.id}function vh(){var e;let t=T.useContext(qd),i=gh(),o=$d();return t!==void 0?t:(e=i.errors)==null?void 0:e[o]}function bh(){let{router:e}=yh(_d.UseNavigateStable),t=$d(Vd.UseNavigateStable),i=T.useRef(!1);return Od(()=>{i.current=!0}),T.useCallback(function(n,a){a===void 0&&(a={}),i.current&&(typeof n=="number"?e.navigate(n):e.navigate(n,so({fromRouteId:t},a)))},[e,t])}const Tl={};function wh(e,t,i){Tl[e]||(Tl[e]=!0)}function Th(e,t){e==null||e.v7_startTransition,e==null||e.v7_relativeSplatPath}function Sh(e){let{to:t,replace:i,state:o,relative:n}=e;fi()||V(!1);let{future:a,static:s}=T.useContext(Pt),{matches:l}=T.useContext(Xe),{pathname:c}=vi(),p=Fd(),y=os(t,is(l,a.v7_relativeSplatPath),c,n==="path"),u=JSON.stringify(y);return T.useEffect(()=>p(JSON.parse(u),{replace:i,state:o,relative:n}),[p,u,n,i,o]),null}function et(e){V(!1)}function Ph(e){let{basename:t="/",children:i=null,location:o,navigationType:n=st.Pop,navigator:a,static:s=!1,future:l}=e;fi()&&V(!1);let c=t.replace(/^\/*/,"/"),p=T.useMemo(()=>({basename:c,navigator:a,static:s,future:so({v7_relativeSplatPath:!1},l)}),[c,l,a,s]);typeof o=="string"&&(o=gi(o));let{pathname:y="/",search:u="",hash:g="",state:b=null,key:w="default"}=o,v=T.useMemo(()=>{let x=ts(y,c);return x==null?null:{location:{pathname:x,search:u,hash:g,state:b,key:w},navigationType:n}},[c,y,u,g,b,w,n]);return v==null?null:T.createElement(Pt.Provider,{value:p},T.createElement(En.Provider,{children:i,value:v}))}function xh(e){let{children:t,location:i}=e;return lh(ar(t),i)}new Promise(()=>{});function ar(e,t){t===void 0&&(t=[]);let i=[];return T.Children.forEach(e,(o,n)=>{if(!T.isValidElement(o))return;let a=[...t,n];if(o.type===T.Fragment){i.push.apply(i,ar(o.props.children,a));return}o.type!==et&&V(!1),!o.props.index||!o.props.children||V(!1);let s={id:o.props.id||a.join("-"),caseSensitive:o.props.caseSensitive,element:o.props.element,Component:o.props.Component,index:o.props.index,path:o.props.path,loader:o.props.loader,action:o.props.action,errorElement:o.props.errorElement,ErrorBoundary:o.props.ErrorBoundary,hasErrorBoundary:o.props.ErrorBoundary!=null||o.props.errorElement!=null,shouldRevalidate:o.props.shouldRevalidate,handle:o.props.handle,lazy:o.props.lazy};o.props.children&&(s.children=ar(o.props.children,a)),i.push(s)}),i}/**
 * React Router DOM v6.30.1
 *
 * Copyright (c) Remix Software Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.md file in the root directory of this source tree.
 *
 * @license MIT
 */function rr(){return rr=Object.assign?Object.assign.bind():function(e){for(var t=1;t<arguments.length;t++){var i=arguments[t];for(var o in i)Object.prototype.hasOwnProperty.call(i,o)&&(e[o]=i[o])}return e},rr.apply(this,arguments)}function kh(e,t){if(e==null)return{};var i={},o=Object.keys(e),n,a;for(a=0;a<o.length;a++)n=o[a],!(t.indexOf(n)>=0)&&(i[n]=e[n]);return i}function Ah(e){return!!(e.metaKey||e.altKey||e.ctrlKey||e.shiftKey)}function Ch(e,t){return e.button===0&&(!t||t==="_self")&&!Ah(e)}const Ih=["onClick","relative","reloadDocument","replace","state","target","to","preventScrollReset","viewTransition"],Mh="6";try{window.__reactRouterVersion=Mh}catch{}const Gh="startTransition",Sl=wp[Gh];function Jh(e){let{basename:t,children:i,future:o,window:n}=e,a=T.useRef();a.current==null&&(a.current=Nu({window:n,v5Compat:!0}));let s=a.current,[l,c]=T.useState({action:s.action,location:s.location}),{v7_startTransition:p}=o||{},y=T.useCallback(u=>{p&&Sl?Sl(()=>c(u)):c(u)},[c,p]);return T.useLayoutEffect(()=>s.listen(y),[s,y]),T.useEffect(()=>Th(o),[o]),T.createElement(Ph,{basename:t,children:i,location:l.location,navigationType:l.action,navigator:s,future:o})}const Nh=typeof window<"u"&&typeof window.document<"u"&&typeof window.document.createElement<"u",Eh=/^(?:[a-z][a-z0-9+.-]*:|\/\/)/i,Z=T.forwardRef(function(t,i){let{onClick:o,relative:n,reloadDocument:a,replace:s,state:l,target:c,to:p,preventScrollReset:y,viewTransition:u}=t,g=kh(t,Ih),{basename:b}=T.useContext(Pt),w,v=!1;if(typeof p=="string"&&Eh.test(p)&&(w=p,Nh))try{let m=new URL(window.location.href),f=p.startsWith("//")?new URL(m.protocol+p):new URL(p),S=ts(f.pathname,b);f.origin===m.origin&&S!=null?p=S+f.search+f.hash:v=!0}catch{}let x=ah(p,{relative:n}),h=jh(p,{replace:s,state:l,target:c,preventScrollReset:y,relative:n,viewTransition:u});function d(m){o&&o(m),m.defaultPrevented||h(m)}return T.createElement("a",rr({},g,{href:w||x,onClick:v||a?o:d,ref:i,target:c}))});var Pl;(function(e){e.UseScrollRestoration="useScrollRestoration",e.UseSubmit="useSubmit",e.UseSubmitFetcher="useSubmitFetcher",e.UseFetcher="useFetcher",e.useViewTransitionState="useViewTransitionState"})(Pl||(Pl={}));var xl;(function(e){e.UseFetcher="useFetcher",e.UseFetchers="useFetchers",e.UseScrollRestoration="useScrollRestoration"})(xl||(xl={}));function jh(e,t){let{target:i,replace:o,state:n,preventScrollReset:a,relative:s,viewTransition:l}=t===void 0?{}:t,c=Fd(),p=vi(),y=Zd(e,{relative:s});return T.useCallback(u=>{if(Ch(u,i)){u.preventDefault();let g=o!==void 0?o:gn(p)===gn(y);c(e,{replace:g,state:n,preventScrollReset:a,relative:s,viewTransition:l})}},[p,c,y,o,n,i,e,a,s,l])}const zh=({children:e})=>{const t=vi(),i=[{name:"Dashboard",href:"/dashboard",icon:""},{name:"Timeline",href:"/timeline",icon:""},{name:"Sessions",href:"/sessions",icon:""},{name:"Presenters",href:"/presenters",icon:""},{name:"About",href:"/about",icon:""}],o=n=>t.pathname===n;return r.jsxs("div",{className:"min-h-screen bg-gradient-to-br from-primary-50 via-white to-secondary-50",children:[r.jsxs("header",{className:"bg-gradient-to-r from-primary-600 to-primary-700 shadow-lg",children:[r.jsx("div",{className:"max-w-7xl mx-auto px-4 sm:px-6 lg:px-8",children:r.jsxs("div",{className:"flex justify-between items-center h-16",children:[r.jsx("div",{className:"flex items-center",children:r.jsx(Z,{to:"/dashboard",className:"flex items-center",children:r.jsx("h1",{className:"text-2xl font-bold text-white hover:text-primary-100 transition-colors",children:" Evomics Workshop Archive"})})}),r.jsx("nav",{className:"hidden md:flex space-x-8",children:i.map(n=>r.jsxs(Z,{to:n.href,className:`flex items-center space-x-1 px-3 py-2 rounded-md text-sm font-medium transition-colors ${o(n.href)?"bg-white/20 text-white":"text-primary-100 hover:text-white hover:bg-white/10"}`,children:[r.jsx("span",{children:n.icon}),r.jsx("span",{children:n.name})]},n.name))}),r.jsxs("div",{className:"flex space-x-4",children:[r.jsx("a",{href:"https://shandley.github.io/evomics-faculty/",target:"_blank",rel:"noopener noreferrer",className:"text-primary-100 hover:text-white transition-colors text-sm",children:" Faculty"}),r.jsx("a",{href:"https://shandley.github.io/evomics-students/",target:"_blank",rel:"noopener noreferrer",className:"text-primary-100 hover:text-white transition-colors text-sm",children:" Students"})]})]})}),r.jsx("div",{className:"md:hidden border-t border-primary-500 bg-primary-700",children:r.jsx("nav",{className:"px-4 py-2 space-x-4 overflow-x-auto",children:i.map(n=>r.jsxs(Z,{to:n.href,className:`inline-flex items-center space-x-1 px-3 py-1 rounded-md text-sm font-medium transition-colors whitespace-nowrap ${o(n.href)?"bg-white/20 text-white":"text-primary-100 hover:text-white hover:bg-white/10"}`,children:[r.jsx("span",{children:n.icon}),r.jsx("span",{children:n.name})]},n.name))})})]}),r.jsx("main",{className:"max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-8",children:e}),r.jsx("footer",{className:"bg-white border-t border-gray-200 mt-16",children:r.jsx("div",{className:"max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-8",children:r.jsxs("div",{className:"flex flex-col md:flex-row justify-between items-center",children:[r.jsx("div",{className:"text-gray-500 text-sm mb-4 md:mb-0",children:" 2025 Evomics Workshop Archive. Part of the Evomics.org ecosystem."}),r.jsxs("div",{className:"flex space-x-6 text-sm",children:[r.jsx("a",{href:"https://evomics.org",target:"_blank",rel:"noopener noreferrer",className:"text-gray-500 hover:text-gray-900 transition-colors",children:"evomics.org"}),r.jsx("a",{href:"https://github.com/shandley/evomics-faculty",target:"_blank",rel:"noopener noreferrer",className:"text-gray-500 hover:text-gray-900 transition-colors",children:"GitHub"})]})]})})})]})},Bh={WoG:"wog",WPSG:"wpsg",WPhylo:"wphylo",WMolEvo:"wmolevo",HTranscript:"htranscriptomics",HMicrobial:"hmicrobial"};function Dh(e){return Bh[e]||e.toLowerCase()}const Lh={id:"wog",name:"Workshop on Genomics",shortName:"WoG",description:"Annual workshop focusing on genomic data analysis techniques including genome assembly, annotation, alignment, and variant calling.",active:!0,startYear:2011,location:"esk Krumlov, Czech Republic"},Hh={id:"wpsg",name:"Workshop on Population and Speciation Genomics",shortName:"WPSG",description:"Biennial workshop analyzing genomic data at population and species levels.",active:!0,startYear:2016,location:"esk Krumlov, Czech Republic"},Rh={id:"wphylo",name:"Workshop on Phylogenomics",shortName:"WPhylo",description:"Workshop exploring large-scale phylogenetics and phylogenomics.",active:!0,startYear:2017,location:"esk Krumlov, Czech Republic"},Uh={id:"wmolevo",name:"Workshop on Molecular Evolution",shortName:"WMolEvo",description:"Historical workshop focusing on molecular phylogenetics and population genetics.",active:!1,startYear:2011,endYear:2015,location:"esk Krumlov, Czech Republic & Fort Collins, USA"},Wh={id:"htranscriptomics",name:"Harvard Workshop on Transcriptomics",shortName:"HTranscript",description:"Harvard University workshop on computational transcriptomics and RNA-seq analysis.",active:!1,startYear:2017,endYear:2017,location:"Ragon Institute, Boston, USA"},qh={id:"hmicrobial",name:"Harvard Workshop on Microbial Genomics",shortName:"HMicrobial",description:"Harvard University workshop on bacterial and viral genomics.",active:!1,startYear:2016,endYear:2016,location:"Harvard University, Boston, USA"},Oh={wog:Lh,wpsg:Hh,wphylo:Rh,wmolevo:Uh,htranscriptomics:Wh,hmicrobial:qh},Fh={"paris-josephine":{teaching:{totalSessions:9,workshopsHistory:{WoG:{2019:[{date:"9th Jan",time:"19-22",topic:"Genomics Lab",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Population Genomics",type:"practical",location:"House of Prelate",coPresenters:["Julian Catchen"]},{date:"16th Jan",time:"19-22",topic:"Population Genomics continued",type:"practical",location:"House of Prelate",coPresenters:["Julian Catchen"]}],2022:[{date:"23rd May",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:[]},{date:"27th May",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",coPresenters:["Emiliano Trucchi"]}],2023:[{date:"15th May",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:["Daniel Kintzl"]},{date:"19th May",time:"19-22",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",coPresenters:["Guy Leonard"]}],2024:[{date:"8th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:["Daniel Kintzl"]}],2025:[{date:"6th Jan",time:"09-12",topic:"Introduction and Orientation Town Services",type:"orientation",location:"Town Theatre",coPresenters:["Daniel Kintzl"]}]}},specializations:["General Genomics","Population Genomics"],lastTaught:2025,firstTaught:2019,yearsActive:[2019,2022,2023,2024,2025],yearRange:"2019-2025"}},"zody-mike":{teaching:{totalSessions:20,workshopsHistory:{WoG:{2011:[{date:"12th Jul",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"3rd Oct",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]}],2012:[{date:"9th Jan",time:"14-17",topic:"Genomics Study Design",type:"lecture",location:"Theater",coPresenters:[]}],2013:[{date:"9th Jan",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"15th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theater",coPresenters:[]}],2015:[{date:"14th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",coPresenters:[]}],2016:[{date:"13th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",coPresenters:[]},{date:"14th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theatre",coPresenters:[]}],2017:[{date:"11th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",coPresenters:[]},{date:"12th Jan",time:"09-12",topic:"Alignment Methods Lab",type:"practical",location:"Prelate",coPresenters:[]}],2018:[{date:"10th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Town Theatre",coPresenters:[]}],2019:[{date:"8th Jan",time:"09-12",topic:"Sequencing Technology & Study Design",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"9th Jan",time:"14-17",topic:"Read Alignment",type:"practical",location:"House of Prelate",coPresenters:[]}],2020:[{date:"7th Jan",time:"09-12",topic:"Sequencing Technology & Study Design",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"8th Jan",time:"09-12",topic:"Sequence Alignment",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"8th Jan",time:"14-17",topic:"Variant Calling",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"15th May",time:"14-17",topic:"Sequencing Technology and Study Design",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"17th May",time:"09-12",topic:"Alignment",type:"lecture",location:"House of Prelate",coPresenters:[]}],2025:[{date:"6th Jan",time:"14-17",topic:"Sequencing Technologies",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"7th Jan",time:"14-17",topic:"Essential File Types in Bioinformatics",type:"lecture",location:"House of Prelate",coPresenters:[]}]}},specializations:["Sequencing Technologies","General Genomics","Sequence Alignment","Variant Analysis"],lastTaught:2025,firstTaught:2011,yearsActive:[2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2023,2025],yearRange:"2011-2025"}},"leonard-guy":{teaching:{totalSessions:6,workshopsHistory:{WoG:{2018:[{date:"9th Jan",time:"14-17",topic:"Computer Lab Introduction",type:"practical",location:"House of Prelate",coPresenters:["Julian Catchen"]}],2023:[{date:"19th May",time:"19-22",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",coPresenters:["Josephine Paris"]}],2024:[{date:"12th Jan",time:"19-22",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",coPresenters:[]}],2025:[{date:"7th Jan",time:"09-12",topic:"AMIs, the Cloud, and Computing Introduction to UNIX",type:"lecture",location:"House of Prelate",coPresenters:["Merc Montoliu Nern"]},{date:"11th Jan",time:"14-17",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",coPresenters:[]}]},WPhylo:{2017:[{date:"22nd Jan",time:"14-17",topic:"UNIX Introduction",type:"practical",location:"Municipal Theater",coPresenters:["Jordi Paps"]}]}},specializations:["Computational Methods","General Genomics","Phylogenomics"],lastTaught:2025,firstTaught:2017,yearsActive:[2017,2018,2023,2024,2025],yearRange:"2017-2025"}},"chikhi-rayan":{teaching:{totalSessions:21,workshopsHistory:{WoG:{2013:[{date:"10th Jan",time:"09-12",topic:"Assembly Lecture",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"17th Jan",time:"09-12",topic:"Assembly",type:"lecture",location:"Theater",coPresenters:[]},{date:"17th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"16th Jan",time:"09-12",topic:"Assembly",type:"lecture",location:"Theatre",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"Prelate",coPresenters:[]}],2016:[{date:"15th Jan",time:"14-17",topic:"Genome Assembly",type:"lecture",location:"Theatre",coPresenters:[]},{date:"15th Jan",time:"19-22",topic:"Genome Assembly Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Genome Assembly Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"13th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Theatre",coPresenters:[]},{date:"13th Jan",time:"14-17",topic:"Genome Assembly Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"13th Jan",time:"19-22",topic:"Genome Assembly Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2019:[{date:"10th Jan",time:"09-12",topic:"Sequence Data Assembly",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"10th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"House of Prelate",coPresenters:[]}],2020:[{date:"11th Jan",time:"14-17",topic:"Metagenomics Assembly",type:"practical",location:"House of Prelate",coPresenters:["Sergey Nurk"]}],2022:[{date:"24th May",time:"09-12",topic:"Alignment",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"27th May",time:"09-12",topic:"BIG Data",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"20th May",time:"14-15",topic:"BIG Data",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"10th Jan",time:"09-12",topic:"Alignment",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"19th Jan",time:"09-12",topic:"BIG DATA",type:"lecture",location:"House of Prelate",coPresenters:[]}],2025:[{date:"8th Jan",time:"09-12",topic:"Alignment",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"17th Jan",time:"09-12",topic:"BIG DATA",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Sequence Alignment","Computational Methods","Genome Assembly"],lastTaught:2025,firstTaught:2013,yearsActive:[2013,2014,2015,2016,2017,2019,2020,2022,2023,2024,2025],yearRange:"2013-2025"}},"hoff-katharina":{teaching:{totalSessions:8,workshopsHistory:{WoG:{2022:[{date:"26th May",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"House of Prelate",coPresenters:[]},{date:"26th May",time:"14-17",topic:"Genome Annotation",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"19th May",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"House of Prelate",coPresenters:["Natalia Nenasheva"]},{date:"19th May",time:"14-17",topic:"Genome Annotation",type:"practical",location:"House of Prelate",coPresenters:["Natalia Nenasheva"]}],2024:[{date:"10th Jan",time:"14-17",topic:"Genome Annotation",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"10th Jan",time:"19-22",topic:"Genome Annotation practical",type:"practical",location:"House of Prelate",coPresenters:[]}],2025:[{date:"8th Jan",time:"14-17",topic:"Genome Annotation: Theory and Principles",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"8th Jan",time:"19-22",topic:"Genome Annotation practical",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Genome Annotation"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2023,2024,2025],yearRange:"2022-2025"}},"marchet-camille":{teaching:{totalSessions:8,workshopsHistory:{WoG:{2020:[{date:"9th Jan",time:"09-12",topic:"Sequence Assembly",type:"lecture",location:"Town Theatre",coPresenters:["Antoine Limasset"]},{date:"9th Jan",time:"14-17",topic:"Genome Assembly",type:"practical",location:"House of Prelate",coPresenters:["Antoine Limasset"]}],2022:[{date:"25th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",coPresenters:["Antoine Limasset"]}],2023:[{date:"18th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",coPresenters:["Antoine Limasset"]}],2024:[{date:"11th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Town Theatre",coPresenters:["Antoine Limasset"]},{date:"11th Jan",time:"14-17",topic:"Genome Assembly practical",type:"practical",location:"House of Prelate",coPresenters:["Antoine Limasset"]}],2025:[{date:"9th Jan",time:"09-12",topic:"Genome Assembly: Theory and Principles",type:"lecture",location:"Town Theatre",coPresenters:["Antoine Limasset"]},{date:"9th Jan",time:"14-17",topic:"Genome Assembly practical",type:"practical",location:"House of Prelate",coPresenters:["Antoine Limasset"]}]}},specializations:["Genome Assembly"],lastTaught:2025,firstTaught:2020,yearsActive:[2020,2022,2023,2024,2025],yearRange:"2020-2025"}},"limasset-antoine":{teaching:{totalSessions:8,workshopsHistory:{WoG:{2020:[{date:"9th Jan",time:"09-12",topic:"Sequence Assembly",type:"lecture",location:"Town Theatre",coPresenters:["Camille Marchet"]},{date:"9th Jan",time:"14-17",topic:"Genome Assembly",type:"practical",location:"House of Prelate",coPresenters:["Camille Marchet"]}],2022:[{date:"25th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",coPresenters:["Camille Marchet"]}],2023:[{date:"18th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",coPresenters:["Camille Marchet"]}],2024:[{date:"11th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Town Theatre",coPresenters:["Camille Marchet"]},{date:"11th Jan",time:"14-17",topic:"Genome Assembly practical",type:"practical",location:"House of Prelate",coPresenters:["Camille Marchet"]}],2025:[{date:"9th Jan",time:"09-12",topic:"Genome Assembly: Theory and Principles",type:"lecture",location:"Town Theatre",coPresenters:["Camille Marchet"]},{date:"9th Jan",time:"14-17",topic:"Genome Assembly practical",type:"practical",location:"House of Prelate",coPresenters:["Camille Marchet"]}]}},specializations:["Genome Assembly"],lastTaught:2025,firstTaught:2020,yearsActive:[2020,2022,2023,2024,2025],yearRange:"2020-2025"}},"sedlazeck-fritz":{teaching:{totalSessions:4,workshopsHistory:{WoG:{2022:[{date:"24th May",time:"19-22",topic:"Structural Variant Calling",type:"lecture",location:"House of Prelate",coPresenters:[]}],2023:[{date:"17th May",time:"19-22",topic:"Structural Variants",type:"lecture",location:"House of Prelate",coPresenters:[]}],2024:[{date:"12th Jan",time:"14-17",topic:"Structural Variant Calling",type:"lecture",location:"House of Prelate",coPresenters:[]}],2025:[{date:"9th Jan",time:"19-22",topic:"Structural Variant Calling",type:"lab",location:"House of Prelate",coPresenters:[]}]}},specializations:["Variant Analysis"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2023,2024,2025],yearRange:"2022-2025"}},"pettersson-olga-vinnere":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2025:[{date:"10th Jan",time:"09-12",topic:"Experimental Design in Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Experimental Design"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"garrison-erik":{teaching:{totalSessions:7,workshopsHistory:{WoG:{2022:[{date:"24th May",time:"14-17",topic:"Variant Calling",type:"lecture",location:"House of Prelate",coPresenters:[]},{date:"26th May",time:"19-22",topic:"Pangenomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"17th May",time:"14-17",topic:"Variant Calling",type:"lecture",location:"House of Prelate",coPresenters:[]},{date:"20th May",time:"09-12",topic:"Pangenomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"12th Jan",time:"09-12",topic:"Variant Calling",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"13th Jan",time:"14-17",topic:"Pangenomics",type:"lecture",location:"House of Prelate",coPresenters:[]}],2025:[{date:"10th Jan",time:"14-17",topic:"Variant Calling",type:"lab",location:"House of Prelate",coPresenters:[]}]}},specializations:["Variant Analysis","General Genomics"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2023,2024,2025],yearRange:"2022-2025"}},"wheat-chris":{teaching:{totalSessions:12,workshopsHistory:{WoG:{2013:[{date:"18th Jan",time:"14-17",topic:"Ecological Genomics",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"24th Jan",time:"09-12",topic:"Ecological Genomics (Part I and II)",type:"lecture",location:"Theater",coPresenters:[]}],2015:[{date:"23rd Jan",time:"09-12",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",coPresenters:[]}],2016:[{date:"21st Jan",time:"09-12",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",coPresenters:[]}],2017:[{date:"19th Jan",time:"14-17",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",coPresenters:[]}],2018:[{date:"19th Jan",time:"14-17",topic:"Genomics Methodology",type:"lecture",location:"Town Theatre",coPresenters:[]}],2019:[{date:"18th Jan",time:"09-12",topic:"Genomics Research Perspectives",type:"lecture",location:"Town Theatre",coPresenters:[]}],2020:[{date:"17th Jan",time:"09-12",topic:"Genomics Perspectives",type:"lecture",location:"Town Theatre",coPresenters:[]}],2022:[{date:"3rd Jun",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"26th May",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"13th Jan",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2025:[{date:"11th Jan",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["General Genomics","Ecological Genomics"],lastTaught:2025,firstTaught:2013,yearsActive:[2013,2014,2015,2016,2017,2018,2019,2020,2022,2023,2024,2025],yearRange:"2013-2025"}},"eichler-evan":{teaching:{totalSessions:12,workshopsHistory:{WoG:{2011:[{date:"11th Jul",time:"14-17",topic:"Identification and Detection of Structural Variation",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]}],2014:[{date:"18th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theater",coPresenters:[]}],2015:[{date:"17th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",coPresenters:[]}],2016:[{date:"11th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",coPresenters:[]}],2017:[{date:"9th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",coPresenters:[]}],2018:[{date:"13th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"13th Jan",time:"14-17",topic:"Structural Variation Lab",type:"practical",location:"House of Prelate",coPresenters:[]}],2020:[{date:"11th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]}],2022:[{date:"2nd Jun",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"25th May",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"18th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]}],2025:[{date:"13th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Structural Variation","General Genomics"],lastTaught:2025,firstTaught:2011,yearsActive:[2011,2014,2015,2016,2017,2018,2020,2022,2023,2024,2025],yearRange:"2011-2025"}},"haas-brian":{teaching:{totalSessions:19,workshopsHistory:{WoG:{2016:[{date:"20th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"20th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"20th Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"18th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"18th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"18th Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2018:[{date:"17th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"17th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"House of Prelate",coPresenters:[]}],2019:[{date:"14th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"14th Jan",time:"19-22",topic:"Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]}],2020:[{date:"13th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"13th Jan",time:"14-17",topic:"Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"13th Jan",time:"19-22",topic:"Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]}],2022:[{date:"31st May",time:"14-17",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"23rd May",time:"14-17",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"15th Jan",time:"09-12",topic:"Transcriptomics and Single-cell Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2025:[{date:"14th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"lab",location:"House of Prelate",coPresenters:[]}]}},specializations:["Transcriptomics"],lastTaught:2025,firstTaught:2016,yearsActive:[2016,2017,2018,2019,2020,2022,2023,2024,2025],yearRange:"2016-2025"}},"dyhrman-sonya":{teaching:{totalSessions:9,workshopsHistory:{WoG:{2016:[{date:"16th Jan",time:"09-12",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Theatre",coPresenters:[]}],2017:[{date:"14th Jan",time:"09-12",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Theatre",coPresenters:[]}],2018:[{date:"8th Jan",time:"14-17",topic:"Marine Ecosystem Phytoplankton",type:"lecture",location:"Town Theatre",coPresenters:[]}],2019:[{date:"7th Jan",time:"14-17",topic:"Marine Ecosystems",type:"lecture",location:"Town Theatre",coPresenters:[]}],2020:[{date:"6th Jan",time:"14-17",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Town Theatre",coPresenters:[]}],2022:[{date:"1st Jun",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"24th May",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"16th Jan",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",coPresenters:[]}],2025:[{date:"15th Jan",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["General Genomics","Marine Genomics"],lastTaught:2025,firstTaught:2016,yearsActive:[2016,2017,2018,2019,2020,2022,2023,2024,2025],yearRange:"2016-2025"}},"uliano-silva-marcela":{teaching:{totalSessions:6,workshopsHistory:{WoG:{2022:[{date:"25th May",time:"14-17",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"25th May",time:"19-22",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"18th May",time:"14-17",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"18th May",time:"19-22",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",coPresenters:[]}],2024:[{date:"17th Jan",time:"09-12",topic:"The wonders of genome assembly!",type:"lecture",location:"Town Theatre",coPresenters:[]}],2025:[{date:"16th Jan",time:"09-12",topic:"Wonders of Genome Assembly",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Genome Assembly"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2023,2024,2025],yearRange:"2022-2025"}},"handley-scott":{teaching:{totalSessions:19,workshopsHistory:{WoG:{2011:[{date:"11th Jul",time:"09-12",topic:"Workshop Introduction",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"19th Jul",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"19th Jul",time:"14-17",topic:"Metagenomics Laboratory",type:"practical",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"22nd Jul",time:"14-17",topic:"Workshop Wrap-up and Discussion",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"6th Oct",time:"14-18",topic:"Metagenomics Laboratory",type:"practical",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]}],2012:[{date:"9th Jan",time:"09-12",topic:"Introduction to Genomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"20th Jan",time:"14-17",topic:"Workshop Wrap-up and Discussion",type:"lecture",location:"Theater",coPresenters:[]}],2013:[{date:"7th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theater",coPresenters:[]},{date:"12th Jan",time:"09-12",topic:"Introduction to R",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"14th Jan",time:"14-17",topic:"Introduction to R",type:"practical",location:"Prelate",coPresenters:["Elin Videvall"]}],2018:[{date:"8th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:[]},{date:"18th Jan",time:"09-12",topic:"Introduction to R",type:"practical",location:"House of Prelate",coPresenters:[]}],2019:[{date:"7th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:[]}],2020:[{date:"6th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"14th Jan",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]}],2024:[{date:"8th Jan",time:"14-17",topic:"Sequencing Technology",type:"lecture",location:"Town Theatre",coPresenters:[]}]},WPhylo:{2017:[{date:"22nd Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theater",coPresenters:["Toni Gabaldn"]}]},WPSG:{2016:[{date:"24th Jan",time:"09-12",topic:"Workshop introduction",type:"orientation",location:"Town Theater",coPresenters:["Walter Salzburger"]}]}},specializations:["Sequencing Technologies","Phylogenomics","General Genomics","Microbiome Analysis","Population Genomics"],lastTaught:2024,firstTaught:2011,yearsActive:[2011,2012,2013,2016,2017,2018,2019,2020,2024],yearRange:"2011-2024"}},"steward-rachel":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2022:[{date:"31st May",time:"19-22",topic:"Differential Gene Expression",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"23rd May",time:"19-22",topic:"Differential Gene Expression",type:"practical",location:"House of Prelate",coPresenters:[]}],2024:[{date:"15th Jan",time:"19-22",topic:"Analysis of RNASeq Data",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Computational Methods","General Genomics","Transcriptomics"],lastTaught:2024,firstTaught:2022,yearsActive:[2022,2023,2024],yearRange:"2022-2024"}},"simakov-oleg":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2024:[{date:"16th Jan",time:"14-17",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"barnett-david":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2022:[{date:"1st Jun",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"24th May",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]}],2024:[{date:"16th Jan",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2024,firstTaught:2022,yearsActive:[2022,2023,2024],yearRange:"2022-2024"}},"colonna-vincenza":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2024:[{date:"17th Jan",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"ahren-dag":{teaching:{totalSessions:7,workshopsHistory:{WoG:{2015:[{date:"21st Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:["Alper Kucukural","Manuel Garber"]},{date:"21st Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Alper Kucukural","Manuel Garber"]}],2016:[{date:"11th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",coPresenters:[]}],2019:[{date:"17th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"17th Jan",time:"14-17",topic:"Evolutionary Genomics",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"17th Jan",time:"19-22",topic:"Evolutionary Genomics continued",type:"practical",location:"House of Prelate",coPresenters:[]}],2024:[{date:"18th Jan",time:"14-17",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",coPresenters:[]}]}},specializations:["Computational Methods","Evolutionary Genomics","General Genomics","Transcriptomics"],lastTaught:2024,firstTaught:2015,yearsActive:[2015,2016,2019,2024],yearRange:"2015-2024"}},"fernandez-rosa":{teaching:{totalSessions:4,workshopsHistory:{WoG:{2022:[{date:"31st May",time:"09-12",topic:"Phylogenomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"23rd May",time:"09-12",topic:"Phylogenomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2024:[{date:"18th Jan",time:"19-22",topic:"Phylogenomics",type:"lecture",location:"House of Prelate",coPresenters:[]}]},WPhylo:{2024:[{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2022,yearsActive:[2022,2023,2024],yearRange:"2022-2024"}},"nenasheva-natalia":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2023:[{date:"19th May",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"House of Prelate",coPresenters:["Katharina Hoff"]},{date:"19th May",time:"14-17",topic:"Genome Annotation",type:"practical",location:"House of Prelate",coPresenters:["Katharina Hoff"]}]}},specializations:["Genome Annotation"],lastTaught:2023,firstTaught:2023,yearsActive:[2023],yearRange:"2023-2023"}},"waterhouse-robert":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2022:[{date:"30th May",time:"09-12",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}],2023:[{date:"22nd May",time:"09-12",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2023,firstTaught:2022,yearsActive:[2022,2023],yearRange:"2022-2023"}},"cicconardi-francesco":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2022:[{date:"30th May",time:"14-17",topic:"Comparative Genomics",type:"practical",location:"House of Prelate",coPresenters:[]}],2023:[{date:"22nd May",time:"14-17",topic:"Comparative Genomics",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2023,firstTaught:2022,yearsActive:[2022,2023],yearRange:"2022-2023"}},"manghi-paolo":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2022:[{date:"1st Jun",time:"14-17",topic:"Metagenomics & Metatranscriptomics",type:"lecture",location:"House of Prelate",coPresenters:[]}],2023:[{date:"24th May",time:"14-17",topic:"Metagenomics & Metatranscriptomics",type:"lecture",location:"House of Prelate",coPresenters:["Altor Blanco Miguez"]}]}},specializations:["Transcriptomics"],lastTaught:2023,firstTaught:2022,yearsActive:[2022,2023],yearRange:"2022-2023"}},"whiting-james":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2022:[{date:"2nd Jun",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",coPresenters:[]}],2023:[{date:"25th May",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2023,firstTaught:2022,yearsActive:[2022,2023],yearRange:"2022-2023"}},"scofield-doug":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2022:[{date:"2nd Jun",time:"19-22",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",coPresenters:[]}],2023:[{date:"25th May",time:"19-22",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",coPresenters:[]}]}},specializations:["Computational Methods"],lastTaught:2023,firstTaught:2022,yearsActive:[2022,2023],yearRange:"2022-2023"}},"kintzl-daniel":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2023:[{date:"15th May",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:["Josephine Paris"]}],2024:[{date:"8th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",coPresenters:["Josephine Paris"]}]}},specializations:["General Genomics"],lastTaught:2024,firstTaught:2023,yearsActive:[2023,2024],yearRange:"2023-2024"}},"blanco-miguez-altor":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2023:[{date:"24th May",time:"14-17",topic:"Metagenomics & Metatranscriptomics",type:"lecture",location:"House of Prelate",coPresenters:["Paolo Manghi"]}]}},specializations:["Transcriptomics"],lastTaught:2023,firstTaught:2023,yearsActive:[2023],yearRange:"2023-2023"}},"matschiner-michael":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2022:[{date:"6th Jun",time:"14-17",topic:"Likelihood and probability",type:"lecture",location:"Town Theater",coPresenters:[]}],2025:[{date:"20th Jan",time:"14-17",topic:"Likelihood and Probability",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Computational Methods","Population Genomics","Statistical Methods"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2025],yearRange:"2022-2025"}},"hahn-matthew":{teaching:{totalSessions:7,workshopsHistory:{WPSG:{2022:[{date:"8th Jun",time:"09-12",topic:"Coalescent theory and introgression",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"8th Jun",time:"14-17",topic:"Multiple testing",type:"lecture",location:"Town Theater",coPresenters:[]}],2025:[{date:"21st Jan",time:"09-12",topic:"Molecular Population Genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"22nd Jan",time:"09-12",topic:"Coalescent Theory",type:"lecture",location:"Town Theater",coPresenters:[]}]},WPhylo:{2017:[{date:"1st Feb",time:"09-12",topic:"Analysis of Gene Gain and Loss",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"1st Feb",time:"14-17",topic:"CAFE Lab",type:"practical",location:"Municipal Theater",coPresenters:[]},{date:"1st Feb",time:"19-22",topic:"CAFE Lab cont. / Open Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Population Genomics","Coalescent Theory","Gene Family Evolution","Phylogenomics"],lastTaught:2025,firstTaught:2017,yearsActive:[2017,2022,2025],yearRange:"2017-2025"}},"sousa-vitor":{teaching:{totalSessions:8,workshopsHistory:{WPSG:{2018:[{date:"26th Jan",time:"09-12",topic:"Demographic inference based on Site frequency spectrum",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"26th Jan",time:"14-17",topic:"fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2020:[{date:"22nd Jan",time:"14-17",topic:"Demographic analysis with fastsimcoal2",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"22nd Jan",time:"19-22",topic:"Fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2022:[{date:"9th Jun",time:"09-12",topic:"Model-based inference of population history",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"9th Jun",time:"14-17",topic:"fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2025:[{date:"22nd Jan",time:"14-17",topic:"Model-based Inference of Population History",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"22nd Jan",time:"19-22",topic:"fastsimcoal2 Activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Population Genomics","Coalescent Theory","Demographic Inference"],lastTaught:2025,firstTaught:2018,yearsActive:[2018,2020,2022,2025],yearRange:"2018-2025"}},"suh-alexander":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2022:[{date:"13th Jun",time:"14-17",topic:"Structural variation",type:"lecture",location:"Town Theater",coPresenters:[]}],2025:[{date:"23rd Jan",time:"09-12",topic:"Structural Variation",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Structural Variation","Population Genomics"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2025],yearRange:"2022-2025"}},"peona-valentina":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2022:[{date:"13th Jun",time:"19-22",topic:"Structural variation activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2025:[{date:"23rd Jan",time:"14-17",topic:"Structural Variation Activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Structural Variation","Population Genomics"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2025],yearRange:"2022-2025"}},"leonard-alexander":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"23rd Jan",time:"19-22",topic:"Pangenomics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Structural Variation","Population Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"kern-andrew":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2025:[{date:"24th Jan",time:"09-12",topic:"Machine Learning",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"24th Jan",time:"14-17",topic:"Machine Learning Activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Machine Learning","Population Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"meier-joana":{teaching:{totalSessions:3,workshopsHistory:{WPSG:{2025:[{date:"24th Jan",time:"19-22",topic:"Experimental Design",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"27th Jan",time:"09-12",topic:"Hybridisation",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"28th Jan",time:"19-22",topic:"Selective Sweep Identification",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Experimental Design","Hybridization & Introgression","Population Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"salzburger-walter":{teaching:{totalSessions:6,workshopsHistory:{WPSG:{2016:[{date:"24th Jan",time:"09-12",topic:"Workshop introduction",type:"orientation",location:"Town Theater",coPresenters:["Scott Handley"]},{date:"30th Jan",time:"19-22",topic:"Genomics of organismal diversification",type:"lecture",location:"Town Theater",coPresenters:[]}],2018:[{date:"21st Jan",time:"09-12",topic:"Introduction and orientation",type:"orientation",location:"Town Theater",coPresenters:[]},{date:"27th Jan",time:"09-12",topic:"The genomics of organismal diversification",type:"lecture",location:"Town Theater",coPresenters:[]}],2020:[{date:"22nd Jan",time:"09-12",topic:"The genomics of organismal diversification",type:"lecture",location:"Town Theater",coPresenters:[]}],2025:[{date:"25th Jan",time:"09-12",topic:"The Genomics of Organismal Diversification",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Population Genomics","Organismal Diversification"],lastTaught:2025,firstTaught:2016,yearsActive:[2016,2018,2020,2025],yearRange:"2016-2025"}},"peichel-katie":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"28th Jan",time:"09-12",topic:"Combining Genomics with Experimental Studies of Adaptation",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"bohutinska-magdalena":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"28th Jan",time:"14-17",topic:"Detecting Positive Selection",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"tsambos-georgia":{teaching:{totalSessions:5,workshopsHistory:{WPSG:{2022:[{date:"16th Jun",time:"09-12",topic:"Simulation",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"16th Jun",time:"14-17",topic:"Simulation with msprime activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"16th Jun",time:"19-22",topic:"Inference with tsinfer activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2025:[{date:"29th Jan",time:"09-12",topic:"Coalescent Simulation and Inference Activity 1/2",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"29th Jan",time:"14-17",topic:"Coalescent Simulation and Inference Activity 2/2",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Coalescent Theory","Simulation Methods","Population Genomics"],lastTaught:2025,firstTaught:2022,yearsActive:[2022,2025],yearRange:"2022-2025"}},"petr-martin":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"29th Jan",time:"19-22",topic:"Building Intuition into Population Genetics Fundamentals and Inference",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"trucchi-emiliano":{teaching:{totalSessions:7,workshopsHistory:{WPSG:{2025:[{date:"30th Jan",time:"09-12",topic:"Conservation Genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"14-17",topic:"Genetic Load Activities",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]},WoG:{2020:[{date:"15th Jan",time:"09-12",topic:"Population Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"15th Jan",time:"14-17",topic:"Population Genomics",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"15th Jan",time:"19-22",topic:"Population Genomics continued",type:"practical",location:"House of Prelate",coPresenters:[]}],2022:[{date:"23rd May",time:"14-17",topic:"Sequencing Technology and Study Design",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"27th May",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",coPresenters:["Josephine Paris"]}]}},specializations:["Conservation Genomics","Population Genomics","Sequencing Technologies"],lastTaught:2025,firstTaught:2020,yearsActive:[2020,2022,2025],yearRange:"2020-2025"}},"hofreiter-michael":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"31st Jan",time:"09-12",topic:"Ancient Genomics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Ancient Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"westbury-michael":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2025:[{date:"31st Jan",time:"14-17",topic:"Ancient Genomics Activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Ancient Genomics"],lastTaught:2025,firstTaught:2025,yearsActive:[2025],yearRange:"2025-2025"}},"karnkowska-anna":{teaching:{totalSessions:3,workshopsHistory:{WPhylo:{2019:[{date:"31st Jan",time:"14-17",topic:"Phylogenomics presentations",type:"presentation",location:"Town Theater",coPresenters:["Martin Kolisko"]}],2024:[{date:"22nd Jan",time:"09-12",topic:"Introduction & Orientation",type:"orientation",location:"Town Theater",coPresenters:[]},{date:"2nd Feb",time:"09-12",topic:"Unravelling complexity: Exploring the evolution of microbial eukaryotes",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2019,yearsActive:[2019,2024],yearRange:"2019-2024"}},"martinez-redondo-gemma":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"22nd Jan",time:"14-17",topic:"Lab introduction & Unix",type:"practical",location:"Municipal Theater",coPresenters:["Karin Steffen"]}]}},specializations:["Computational Methods","Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"steffen-karin":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2024:[{date:"22nd Jan",time:"14-17",topic:"Lab introduction & Unix",type:"practical",location:"Municipal Theater",coPresenters:["Gemma Isabel Martnez-Redondo"]},{date:"24th Jan",time:"19-22",topic:"Partitioning and Concatenation Laboratory",type:"practical",location:"Municipal Theater",coPresenters:["Jacob Steenwyk"]}]}},specializations:["Computational Methods","Phylogenomics","Concatenation Methods"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"marcet-houben-marina":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2019:[{date:"23rd Jan",time:"14-17",topic:"Orthology and Paralogy Prediction Lab",type:"practical",location:"Municipal Theater",coPresenters:["Toni Gabaldn"]}],2024:[{date:"23rd Jan",time:"14-17",topic:"Alignment and Multiple Sequence Alignment Trimming",type:"practical",location:"Municipal Theater",coPresenters:["Jacob Steenwyk"]},{date:"24th Jan",time:"09-12",topic:"Introduction to Phylogenetics, Orthology and Paralogy",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"24th Jan",time:"14-17",topic:"Orthology and Paralogy Prediction lab",type:"practical",location:"Municipal Theater",coPresenters:["Jacob Steenwyk"]}]}},specializations:["Sequence Alignment","Phylogenomics","Orthology & Paralogy","Phylogenetic Analysis"],lastTaught:2024,firstTaught:2019,yearsActive:[2019,2024],yearRange:"2019-2024"}},"steenwyk-jacob":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2024:[{date:"23rd Jan",time:"14-17",topic:"Alignment and Multiple Sequence Alignment Trimming",type:"practical",location:"Municipal Theater",coPresenters:["Marina Marcet-Houben"]},{date:"24th Jan",time:"14-17",topic:"Orthology and Paralogy Prediction lab",type:"practical",location:"Municipal Theater",coPresenters:["Marina Marcet-Houben"]},{date:"24th Jan",time:"19-22",topic:"Partitioning and Concatenation Laboratory",type:"practical",location:"Municipal Theater",coPresenters:["Karin Steffen"]},{date:"29th Jan",time:"14-17",topic:"Incongruence detection lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Sequence Alignment","Orthology & Paralogy","Computational Methods","Phylogenetic Incongruence","Phylogenomics","Concatenation Methods"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"karlicki-michal":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"23rd Jan",time:"19-22",topic:"Tree Visualization & Tree Challenge",type:"practical",location:"Municipal Theater",coPresenters:["Eduard Ocaa-Pallars"]}]}},specializations:["Data Visualization","Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"ocana-pallares-eduard":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2024:[{date:"23rd Jan",time:"19-22",topic:"Tree Visualization & Tree Challenge",type:"practical",location:"Municipal Theater",coPresenters:["Michal Karlicki"]},{date:"30th Jan",time:"19-22",topic:"Open Lab: Phylobayes and BEAST",type:"practical",location:"Municipal Theater",coPresenters:["Gytis Dudas"]}]}},specializations:["Data Visualization","Computational Methods","Phylogenomics","Bayesian Phylogenetics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"gascuel-olivier":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"25th Jan",time:"09-12",topic:"State-of-the-art methods and software in phylogenomic inference",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"molloy-erin":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2024:[{date:"25th Jan",time:"14-17",topic:"Coalescent methods in phylogenomic inference lab",type:"practical",location:"Municipal Theater",coPresenters:[]},{date:"26th Jan",time:"14-17",topic:"Coalescent methods lab day 2",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Coalescent Methods","Multispecies Coalescent"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"kozlov-oleksiy":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2024:[{date:"25th Jan",time:"19-22",topic:"RAxML software and phylogenomic analyses",type:"practical",location:"Municipal Theater",coPresenters:[]},{date:"27th Jan",time:"14-17",topic:"Green computing",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"morlon-helene":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"26th Jan",time:"09-12",topic:"Trait evolution on trees, phenotypic and species diversification",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Trait Evolution","Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"christmas-matthew":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"26th Jan",time:"19-22",topic:"Evolution and genomic basis of shared and specialized traits in mammals",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"mclysaght-aoife":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"27th Jan",time:"09-12",topic:"Phylogenetic inference in Whole Genome Duplication context",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics","Phylogenetic Analysis"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"rokas-antonis":{teaching:{totalSessions:13,workshopsHistory:{WPhylo:{2017:[{date:"24th Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",coPresenters:[]}],2019:[{date:"21st Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theater",coPresenters:[]},{date:"22nd Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",coPresenters:[]}],2024:[{date:"29th Jan",time:"09-12",topic:"Incongruence",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"Joint lecture",type:"lecture",location:"Town Theater",coPresenters:["Toni Gabaldon"]}]},WoG:{2012:[{date:"17th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"17th Jan",time:"14-17",topic:"Evolutionary Genomics Laboratory",type:"practical",location:"Prelate",coPresenters:[]}],2013:[{date:"18th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"23rd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",coPresenters:[]}],2015:[{date:"22nd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theatre",coPresenters:[]}],2016:[{date:"22nd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theatre",coPresenters:[]}],2018:[{date:"12th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"12th Jan",time:"14-17",topic:"Evolutionary Genomics Lab",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Phylogenetic Incongruence","Phylogenomics","Evolutionary Genomics"],lastTaught:2024,firstTaught:2012,yearsActive:[2012,2013,2014,2015,2016,2017,2018,2019,2024],yearRange:"2012-2024"}},"montserrat-jordi-paps":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2017:[{date:"3rd Feb",time:"09-12",topic:"LBA & Animal Nightmares / Open Lab",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"3rd Feb",time:"14-17",topic:"Downloading data / Final Open Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2019:[{date:"31st Jan",time:"09-12",topic:"Reconstructing Ancestral Genomes to Understand Major Transitions",type:"lecture",location:"Town Theater",coPresenters:[]}],2024:[{date:"29th Jan",time:"19-22",topic:"Using genome-level processes to solve tricky nodes in phylogenetics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics","Phylogenetic Analysis"],lastTaught:2024,firstTaught:2017,yearsActive:[2017,2019,2024],yearRange:"2017-2024"}},"gabaldon-toni":{teaching:{totalSessions:10,workshopsHistory:{WPhylo:{2017:[{date:"22nd Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theater",coPresenters:["Scott Handley"]},{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenetic Inference",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"23rd Jan",time:"14-17",topic:"Orthology and Paralogy",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"23rd Jan",time:"19-22",topic:"Orthology Detection Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2019:[{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenetics, Orthology and Paralogy",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"23rd Jan",time:"14-17",topic:"Orthology and Paralogy Prediction Lab",type:"practical",location:"Municipal Theater",coPresenters:["Marina Marcet-Houben"]}],2024:[{date:"30th Jan",time:"09-12",topic:"Gene family evolution",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"Joint lecture",type:"lecture",location:"Town Theater",coPresenters:["Antonis Rokas"]}]},WoG:{2016:[{date:"21st Jan",time:"14-17",topic:"Ortholog Detection",type:"lecture",location:"Theatre",coPresenters:[]},{date:"21st Jan",time:"19-22",topic:"Ortholog Detection Lab",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Gene Family Evolution","Phylogenomics","Orthology & Paralogy","General Genomics","Phylogenetic Analysis"],lastTaught:2024,firstTaught:2016,yearsActive:[2016,2017,2019,2024],yearRange:"2016-2024"}},"dudas-gytis":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2024:[{date:"30th Jan",time:"14-17",topic:"Reticulate evolution, temporal resolution and custom data visualisations",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"19-22",topic:"Open Lab: Phylobayes and BEAST",type:"practical",location:"Municipal Theater",coPresenters:["Eduard Ocaa-Pallars"]}]}},specializations:["Data Visualization","Computational Methods","Phylogenomics","Bayesian Phylogenetics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"pokorny-lisa":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2024:[{date:"31st Jan",time:"09-12",topic:"Target Capture Sequencing Approaches in Phylogenomics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2024,firstTaught:2024,yearsActive:[2024],yearRange:"2024-2024"}},"szollosi-gergely":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2019:[{date:"1st Feb",time:"09-12",topic:"Horizontal Gene Transfer",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"1st Feb",time:"14-17",topic:"Horizontal Gene Transfer Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2024:[{date:"1st Feb",time:"09-12",topic:"Horizontal Gene Transfer",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"1st Feb",time:"14-17",topic:"Horizontal Gene Transfer Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Horizontal Gene Transfer","Phylogenomics"],lastTaught:2024,firstTaught:2019,yearsActive:[2019,2024],yearRange:"2019-2024"}},"neilsen-rasmus":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2022:[{date:"7th Jun",time:"09-12",topic:"Population genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"7th Jun",time:"14-17",topic:"ANGSD activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Thorfinn Korneliussen"]}]}},specializations:["Population Genomics","Low Coverage Analysis"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"korneliussen-thorfinn":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2022:[{date:"7th Jun",time:"14-17",topic:"ANGSD activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Rasmus Nielsen"]}]}},specializations:["Population Genomics","Low Coverage Analysis"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"moyle-leonie":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2022:[{date:"10th Jun",time:"09-12",topic:"Selection and adaptation",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"jiggins-chris":{teaching:{totalSessions:4,workshopsHistory:{WPSG:{2016:[{date:"5th Feb",time:"09-12",topic:"Divergence and gene flow between species",type:"lecture",location:"Town Theater",coPresenters:["Simon Martin"]},{date:"5th Feb",time:"14-17",topic:"Divergence and gene flow between species activity",type:"practical",location:"Computer Lab",coPresenters:["Simon Martin"]}],2018:[{date:"1st Feb",time:"09-12",topic:"Speciation with gene flow",type:"lecture",location:"Town Theater",coPresenters:[]}],2022:[{date:"13th Jun",time:"09-12",topic:"Speciation with gene flow",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Speciation Genomics"],lastTaught:2022,firstTaught:2016,yearsActive:[2016,2018,2022],yearRange:"2016-2022"}},"fumagalli-matteo":{teaching:{totalSessions:4,workshopsHistory:{WPSG:{2016:[{date:"27th Jan",time:"09-12",topic:"Genotype and SNP calling",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"27th Jan",time:"14-17",topic:"Genotype and SNP calling activity",type:"practical",location:"Computer Lab",coPresenters:[]}],2022:[{date:"14th Jun",time:"09-12",topic:"Machine Learning",type:"lecture",location:"Town Theater",coPresenters:["Flora Jay"]},{date:"14th Jun",time:"14-17",topic:"Selection inference ML activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Machine Learning","Selection & Adaptation","Population Genomics"],lastTaught:2022,firstTaught:2016,yearsActive:[2016,2022],yearRange:"2016-2022"}},"jay-flora":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2022:[{date:"14th Jun",time:"09-12",topic:"Machine Learning",type:"lecture",location:"Town Theater",coPresenters:["Matteo Fumagalli"]},{date:"14th Jun",time:"19-22",topic:"Demography inference ML activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Machine Learning","Population Genomics","Demographic Inference"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"myers-simon":{teaching:{totalSessions:4,workshopsHistory:{WPSG:{2020:[{date:"24th Jan",time:"09-12",topic:"The coalescent",type:"lecture",location:"Town Theater",coPresenters:["Leo Speidel"]},{date:"24th Jan",time:"14-17",topic:"Genome-wide genealogy inference activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Leo Speidel"]}],2022:[{date:"15th Jun",time:"09-12",topic:"Tree sequences",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"15th Jun",time:"14-17",topic:"Relate activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Jasmine Rees","Leo Speidel"]}]}},specializations:["Simulation Methods","Population Genomics","Coalescent Theory"],lastTaught:2022,firstTaught:2020,yearsActive:[2020,2022],yearRange:"2020-2022"}},"rees-jasmine":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2022:[{date:"15th Jun",time:"14-17",topic:"Relate activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Simon Myers","Leo Speidel"]}]}},specializations:["Population Genomics"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"speidel-leo":{teaching:{totalSessions:3,workshopsHistory:{WPSG:{2020:[{date:"24th Jan",time:"09-12",topic:"The coalescent",type:"lecture",location:"Town Theater",coPresenters:["Simon Myers"]},{date:"24th Jan",time:"14-17",topic:"Genome-wide genealogy inference activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Simon Myers"]}],2022:[{date:"15th Jun",time:"14-17",topic:"Relate activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:["Simon Myers","Jasmine Rees"]}]}},specializations:["Population Genomics","Coalescent Theory"],lastTaught:2022,firstTaught:2020,yearsActive:[2020,2022],yearRange:"2020-2022"}},"mafessoni-fabrizio":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2022:[{date:"17th Jun",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Ancient Genomics"],lastTaught:2022,firstTaught:2022,yearsActive:[2022],yearRange:"2022-2022"}},"zhou-xiaofen":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2017:[{date:"25th Jan",time:"09-12",topic:"Fast Phylogenetic Approaches",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"25th Jan",time:"14-17",topic:"Maximum Likelihood Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2019:[{date:"24th Jan",time:"09-12",topic:"Introduction to Methods and Software for Phylogenomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"25th Jan",time:"19-22",topic:"Phylobayes Intro",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Phylogenomics","Phylogenetic Analysis","Maximum Likelihood"],lastTaught:2019,firstTaught:2017,yearsActive:[2017,2019],yearRange:"2017-2019"}},"crotty-stephen":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2019:[{date:"24th Jan",time:"19-22",topic:"IQ-tree Introduction and Tutorial",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Phylogenomics"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"scornavacca-celine":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2017:[{date:"2nd Feb",time:"09-12",topic:"Non-Vertical Evolution and Phylogenetic Networks",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"2nd Feb",time:"14-17",topic:"Non-Vertical Evolution Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2019:[{date:"25th Jan",time:"09-12",topic:"Non-Vertical Evolution & Phylogenetic Networks",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"25th Jan",time:"14-17",topic:"Networks Lab Tutorial",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Phylogenomics","Phylogenetic Networks","Phylogenetic Analysis"],lastTaught:2019,firstTaught:2017,yearsActive:[2017,2019],yearRange:"2017-2019"}},"kubatko-laura":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2017:[{date:"31st Jan",time:"09-12",topic:"Species Tree Estimation",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"SVDquartets Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}],2019:[{date:"26th Jan",time:"09-12",topic:"Coalescent Methods",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"26th Jan",time:"14-17",topic:"Coalescent Methods Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Coalescent Methods","Phylogenomics","Species Tree Methods","Multispecies Coalescent"],lastTaught:2019,firstTaught:2017,yearsActive:[2017,2019],yearRange:"2017-2019"}},"dos-reis-mario":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2019:[{date:"28th Jan",time:"09-12",topic:"Bayesian Phylogenetic Analysis",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"28th Jan",time:"19-22",topic:"Bayesian MCMC Tutorial",type:"practical",location:"Municipal Theater",coPresenters:["Tracy Heath"]}]}},specializations:["Phylogenomics","Bayesian Phylogenetics","Phylogenetic Analysis"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"heath-tracy":{teaching:{totalSessions:6,workshopsHistory:{WPhylo:{2017:[{date:"26th Jan",time:"09-12",topic:"Bayesian Phylogenetic Inference",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"26th Jan",time:"14-17",topic:"Bayesian Phylogenetic Inference Lab",type:"practical",location:"Municipal Theater",coPresenters:[]},{date:"26th Jan",time:"19-22",topic:"Divergence Time Estimation",type:"lecture",location:"Town Theater",coPresenters:[]}],2019:[{date:"28th Jan",time:"14-17",topic:"Molecular Clocks and Dating Analysis",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"28th Jan",time:"19-22",topic:"Bayesian MCMC Tutorial",type:"practical",location:"Municipal Theater",coPresenters:["Mario dos Reis"]},{date:"29th Jan",time:"19-22",topic:"Internode certainty and concordance factors tutorial",type:"practical",location:"Municipal Theater",coPresenters:["Jacob","Marina"]}]}},specializations:["Molecular Dating","Bayesian Phylogenetics","Phylogenomics","Phylogenetic Analysis","Divergence Time Estimation"],lastTaught:2019,firstTaught:2017,yearsActive:[2017,2019],yearRange:"2017-2019"}},"garamszegi-laszlo":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2019:[{date:"29th Jan",time:"09-12",topic:"Trait evolution and ancestral state reconstruction",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"29th Jan",time:"14-17",topic:"Trait evolution and ancestral state reconstruction lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Trait Evolution","Phylogenomics"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"stajich-jason":{teaching:{totalSessions:4,workshopsHistory:{WPhylo:{2019:[{date:"30th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"14-17",topic:"Evolutionary Genomics Practical",type:"practical",location:"Municipal Theater",coPresenters:[]}]},WoG:{2011:[{date:"13th Jul",time:"14-17",topic:"Perl Programming",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"13th Jul",time:"19-22",topic:"BioPerl Laboratory",type:"practical",location:"Fort Collins (Fort Collins)",coPresenters:[]}]}},specializations:["Phylogenomics","Programming & Scripting"],lastTaught:2019,firstTaught:2011,yearsActive:[2011,2019],yearRange:"2011-2019"}},"kolisko-martin":{teaching:{totalSessions:1,workshopsHistory:{WPhylo:{2019:[{date:"31st Jan",time:"14-17",topic:"Phylogenomics presentations",type:"presentation",location:"Town Theater",coPresenters:["Anna Karnkowska"]}]}},specializations:["Phylogenomics"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"swofford-david":{teaching:{totalSessions:5,workshopsHistory:{WPhylo:{2017:[{date:"27th Jan",time:"09-12",topic:"Hypothesis Testing in Phylogenetics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"27th Jan",time:"14-17",topic:"Model Selection Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]},WPSG:{2018:[{date:"22nd Jan",time:"09-12",topic:"Likelihood and Bayesian inference",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"22nd Jan",time:"14-17",topic:"Likelihood and Bayesian inference activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"29th Jan",time:"14-17",topic:"Species tree estimation using SVDQuartets",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Phylogenomics","Selection Analysis","Statistical Methods","Population Genomics","Phylogenetic Analysis"],lastTaught:2018,firstTaught:2017,yearsActive:[2017,2018],yearRange:"2017-2018"}},"bielawski-joseph":{teaching:{totalSessions:2,workshopsHistory:{WPhylo:{2017:[{date:"30th Jan",time:"09-12",topic:"Detecting Selection",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"14-17",topic:"Detecting Selection Lab",type:"practical",location:"Municipal Theater",coPresenters:[]}]}},specializations:["Selection Analysis","Phylogenomics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"shaw-sophie":{teaching:{totalSessions:5,workshopsHistory:{WoG:{2017:[{date:"11th Jan",time:"19-22",topic:"Sequence Data Quality Control",type:"practical",location:"Prelate",coPresenters:[]}],2019:[{date:"8th Jan",time:"14-17",topic:"Unix and Software Installation",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"9th Jan",time:"09-12",topic:"Sequence Data Quality Control",type:"practical",location:"House of Prelate",coPresenters:[]}],2020:[{date:"7th Jan",time:"14-17",topic:"Unix and Sequence Data",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"7th Jan",time:"19-22",topic:"Unix/Sequence Data continued",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Computational Methods"],lastTaught:2020,firstTaught:2017,yearsActive:[2017,2019,2020],yearRange:"2017-2020"}},"griffith-malachi":{teaching:{totalSessions:5,workshopsHistory:{WoG:{2019:[{date:"11th Jan",time:"09-12",topic:"Genomic Data Visualization",type:"lecture",location:"Town Theatre",coPresenters:["Zachary Skidmore"]},{date:"11th Jan",time:"14-17",topic:"Visualization Lab Part I",type:"practical",location:"House of Prelate",coPresenters:["Zachary Skidmore"]},{date:"11th Jan",time:"19-22",topic:"Visualization Lab Part II",type:"practical",location:"House of Prelate",coPresenters:["Zachary Skidmore"]}],2020:[{date:"10th Jan",time:"14-17",topic:"Genomic Data Visualization",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"10th Jan",time:"19-22",topic:"Data Visualization continued",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Data Visualization"],lastTaught:2020,firstTaught:2019,yearsActive:[2019,2020],yearRange:"2019-2020"}},"nurk-sergey":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2020:[{date:"11th Jan",time:"14-17",topic:"Metagenomics Assembly",type:"practical",location:"House of Prelate",coPresenters:["Rayan Chikhi"]}]}},specializations:["Genome Assembly"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"wrighton-kelly":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2020:[{date:"14th Jan",time:"09-12",topic:"Microbial Ecology",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Microbial Ecology"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"schleper-christa":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2020:[{date:"16th Jan",time:"09-12",topic:"Thaumarchaeota",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"gosik-kirk":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2020:[{date:"16th Jan",time:"14-17",topic:"Single Cell Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"16th Jan",time:"19-22",topic:"Single Cell Transcriptomics",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Transcriptomics"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"tavalire-hannah":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2019:[{date:"8th Jan",time:"19-22",topic:"Introduction to R",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"10th Jan",time:"19-22",topic:"Research Documentation",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"skidmore-zachary":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2019:[{date:"11th Jan",time:"09-12",topic:"Genomic Data Visualization",type:"lecture",location:"Town Theatre",coPresenters:["Malachi Griffith"]},{date:"11th Jan",time:"14-17",topic:"Visualization Lab Part I",type:"practical",location:"House of Prelate",coPresenters:["Malachi Griffith"]},{date:"11th Jan",time:"19-22",topic:"Visualization Lab Part II",type:"practical",location:"House of Prelate",coPresenters:["Malachi Griffith"]}]}},specializations:["Data Visualization"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"cresko-bill":{teaching:{totalSessions:12,workshopsHistory:{WoG:{2013:[{date:"16th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"20th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theater",coPresenters:[]}],2015:[{date:"20th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theatre",coPresenters:[]},{date:"20th Jan",time:"14-17",topic:"STACKS Laboratory",type:"practical",location:"Prelate",coPresenters:["Julian Catchen"]},{date:"20th Jan",time:"19-22",topic:"STACKS Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Julian Catchen"]}],2016:[{date:"19th Jan",time:"09-12",topic:"RAD-Sequencing",type:"lecture",location:"Theatre",coPresenters:[]},{date:"19th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"Prelate",coPresenters:["Julian Catchen"]},{date:"19th Jan",time:"19-22",topic:"Stacks Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Julian Catchen"]}],2017:[{date:"16th Jan",time:"09-12",topic:"RAD-seq",type:"lecture",location:"Theatre",coPresenters:[]}],2018:[{date:"15th Jan",time:"09-12",topic:"RAD-seq",type:"lecture",location:"Town Theatre",coPresenters:[]}],2019:[{date:"12th Jan",time:"09-12",topic:"Statistics for Genomics Lab 1",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"12th Jan",time:"14-17",topic:"Statistics for Genomics Lab 2",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Computational Methods","General Genomics","Sequencing Technologies"],lastTaught:2019,firstTaught:2013,yearsActive:[2013,2014,2015,2016,2017,2018,2019],yearRange:"2013-2019"}},"willis-amy":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2019:[{date:"15th Jan",time:"09-12",topic:"Microbiome Analysis",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"15th Jan",time:"14-17",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"15th Jan",time:"19-22",topic:"Microbiome Analysis continued",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2019,firstTaught:2019,yearsActive:[2019],yearRange:"2019-2019"}},"catchen-julian":{teaching:{totalSessions:26,workshopsHistory:{WoG:{2012:[{date:"9th Jan",time:"19-22",topic:"Command Line Analysis Introduction",type:"practical",location:"Prelate",coPresenters:[]}],2013:[{date:"8th Jan",time:"14-17",topic:"Introduction to UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"9th Jan",time:"14-17",topic:"Advanced Unix",type:"practical",location:"Prelate",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"STACKS Lab",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"13th Jan",time:"19-22",topic:"UNIX Introduction",type:"practical",location:"Prelate",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Advanced Unix Commands",type:"practical",location:"Prelate",coPresenters:[]},{date:"20th Jan",time:"14-17",topic:"STACKS Lab",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"13th Jan",time:"14-17",topic:"Introduction to UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Advanced UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"20th Jan",time:"14-17",topic:"STACKS Laboratory",type:"practical",location:"Prelate",coPresenters:["Bill Cresko"]},{date:"20th Jan",time:"19-22",topic:"STACKS Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Bill Cresko"]}],2016:[{date:"12th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"13th Jan",time:"14-17",topic:"Advanced UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"19th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"Prelate",coPresenters:["Bill Cresko"]},{date:"19th Jan",time:"19-22",topic:"Stacks Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Bill Cresko"]}],2017:[{date:"10th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"Prelate",coPresenters:[]},{date:"11th Jan",time:"14-17",topic:"Advanced Unix",type:"practical",location:"Prelate",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"16th Jan",time:"19-22",topic:"Stacks Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2018:[{date:"9th Jan",time:"09-12",topic:"Sequencing Technology History",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"9th Jan",time:"14-17",topic:"Computer Lab Introduction",type:"practical",location:"House of Prelate",coPresenters:["Guy Leonard"]},{date:"9th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"House of Prelate",coPresenters:[]},{date:"15th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"House of Prelate",coPresenters:[]}],2019:[{date:"16th Jan",time:"09-12",topic:"Population Genomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Population Genomics",type:"practical",location:"House of Prelate",coPresenters:["Josephine Paris"]},{date:"16th Jan",time:"19-22",topic:"Population Genomics continued",type:"practical",location:"House of Prelate",coPresenters:["Josephine Paris"]}]}},specializations:["Population Genomics","Sequencing Technologies","General Genomics","Computational Methods"],lastTaught:2019,firstTaught:2012,yearsActive:[2012,2013,2014,2015,2016,2017,2018,2019],yearRange:"2012-2019"}},"buerkle-alex":{teaching:{totalSessions:5,workshopsHistory:{WPSG:{2016:[{date:"26th Jan",time:"09-12",topic:"Bayesian and likelihood inference",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"26th Jan",time:"14-17",topic:"Bayesian and likelihood inference activity",type:"practical",location:"Computer Lab",coPresenters:[]},{date:"28th Jan",time:"09-12",topic:"Introgression and admixture modeling",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"28th Jan",time:"14-17",topic:"Introgression and admixture modeling activity",type:"practical",location:"Computer Lab",coPresenters:[]}],2020:[{date:"21st Jan",time:"09-12",topic:"Likelihood, probability, and Bayesian inference",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Statistical Methods","Population Genomics"],lastTaught:2020,firstTaught:2016,yearsActive:[2016,2020],yearRange:"2016-2020"}},"dudaniec-rachael":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2020:[{date:"23rd Jan",time:"09-12",topic:"Adaptation and selection",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"nosil-patrik":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2020:[{date:"23rd Jan",time:"19-22",topic:"Predicting evolution",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"durbin-richard":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2018:[{date:"24th Jan",time:"09-12",topic:"Experimental design / Demography inference",type:"lecture",location:"Town Theater",coPresenters:[]}],2020:[{date:"24th Jan",time:"19-22",topic:"Experimental design / Demography",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Experimental Design","Demographic Inference"],lastTaught:2020,firstTaught:2018,yearsActive:[2018,2020],yearRange:"2018-2020"}},"harris-kelley":{teaching:{totalSessions:4,workshopsHistory:{WPSG:{2018:[{date:"31st Jan",time:"09-12",topic:"Evolution of the mutation spectrum",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"Mutation spectrum analysis activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2020:[{date:"25th Jan",time:"09-12",topic:"Evolution of the mutation spectrum",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"25th Jan",time:"14-17",topic:"Mutation spectrum inference activity",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Mutation Analysis"],lastTaught:2020,firstTaught:2018,yearsActive:[2018,2020],yearRange:"2018-2020"}},"falush-daniel":{teaching:{totalSessions:8,workshopsHistory:{WPSG:{2016:[{date:"25th Jan",time:"14-17",topic:"R and adegenet, STRUCTURE, smartpca",type:"practical",location:"Computer Lab",coPresenters:[]},{date:"4th Feb",time:"09-12",topic:"Introgression",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"4th Feb",time:"14-17",topic:"Introgression activity",type:"practical",location:"Computer Lab",coPresenters:[]}],2018:[{date:"23rd Jan",time:"09-12",topic:"Population structure",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"23rd Jan",time:"14-17",topic:"STRUCTURE and fineStructure workshops",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2020:[{date:"27th Jan",time:"09-12",topic:"Population structure",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"27th Jan",time:"14-17",topic:"Population structure analysis",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"27th Jan",time:"19-22",topic:"Population structure continued",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Population Genomics","Population Structure"],lastTaught:2020,firstTaught:2016,yearsActive:[2016,2018,2020],yearRange:"2016-2020"}},"knowles-lacey":{teaching:{totalSessions:3,workshopsHistory:{WPSG:{2020:[{date:"28th Jan",time:"09-12",topic:"Multi-species coalescent",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"28th Jan",time:"14-17",topic:"Multi-species coalescent analysis",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"28th Jan",time:"19-22",topic:"Multi-species coalescent continued",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Coalescent Theory"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"martin-simon":{teaching:{totalSessions:6,workshopsHistory:{WPSG:{2016:[{date:"5th Feb",time:"09-12",topic:"Divergence and gene flow between species",type:"lecture",location:"Town Theater",coPresenters:["Chris Jiggins"]},{date:"5th Feb",time:"14-17",topic:"Divergence and gene flow between species activity",type:"practical",location:"Computer Lab",coPresenters:["Chris Jiggins"]}],2018:[{date:"1st Feb",time:"14-17",topic:"ABBA BABA statistics and Topology weighting",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}],2020:[{date:"29th Jan",time:"09-12",topic:"Speciation and hybridization",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"29th Jan",time:"14-17",topic:"Hybridization analysis",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"29th Jan",time:"19-22",topic:"Hybridization continued",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Speciation Genomics","Introgression Analysis"],lastTaught:2020,firstTaught:2016,yearsActive:[2016,2018,2020],yearRange:"2016-2020"}},"nordborg-magnus":{teaching:{totalSessions:3,workshopsHistory:{WPSG:{2020:[{date:"30th Jan",time:"09-12",topic:"Genome-Wide Association Studies",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"14-17",topic:"GWAS analysis",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"30th Jan",time:"19-22",topic:"GWAS continued",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["GWAS Analysis"],lastTaught:2020,firstTaught:2020,yearsActive:[2020],yearRange:"2020-2020"}},"orlando-ludovic":{teaching:{totalSessions:5,workshopsHistory:{WPSG:{2016:[{date:"31st Jan",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"Ancient genomics activity",type:"practical",location:"Computer Lab",coPresenters:[]}],2018:[{date:"2nd Feb",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",coPresenters:[]}],2020:[{date:"31st Jan",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"31st Jan",time:"14-17",topic:"Ancient genomics analysis",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Ancient Genomics"],lastTaught:2020,firstTaught:2016,yearsActive:[2016,2018,2020],yearRange:"2016-2020"}},"hird-sarah":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2018:[{date:"16th Jan",time:"09-12",topic:"Microbiome for Evolutionary Biology",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"mcdonald-daniel":{teaching:{totalSessions:9,workshopsHistory:{WoG:{2013:[{date:"12th Jan",time:"14-17",topic:"Introduction to Python",type:"practical",location:"Prelate",coPresenters:[]},{date:"17th Jan",time:"14-17",topic:"Metagenomics Lab II: QIIME",type:"practical",location:"Prelate",coPresenters:[]},{date:"17th Jan",time:"19-22",topic:"Continued Metagenomics Lab II and Open Lab",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"23rd Jan",time:"14-17",topic:"Metagenomics Lab II: QIIME",type:"practical",location:"Prelate",coPresenters:[]},{date:"23rd Jan",time:"19-22",topic:"Continued Metagenomics Lab II and Open Lab",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"19th Jan",time:"19-22",topic:"QIIME Lab",type:"practical",location:"Prelate",coPresenters:[]}],2016:[{date:"18th Jan",time:"14-17",topic:"QIIME Lab",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"17th Jan",time:"14-17",topic:"QIIME Lab",type:"practical",location:"Prelate",coPresenters:["Yoshiki Vazquez-Baeza"]}],2018:[{date:"16th Jan",time:"14-17",topic:"QIIME 2 Lab",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["General Genomics","Microbiome Analysis","Programming & Scripting"],lastTaught:2018,firstTaught:2013,yearsActive:[2013,2014,2015,2016,2017,2018],yearRange:"2013-2018"}},"shekhar-karthik":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2018:[{date:"18th Jan",time:"14-17",topic:"Single-cell Transcriptomics",type:"lecture",location:"Town Theatre",coPresenters:[]},{date:"18th Jan",time:"19-22",topic:"Single-cell Transcriptomics Lab",type:"practical",location:"House of Prelate",coPresenters:[]}]}},specializations:["Transcriptomics"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"lorenzen-eline":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2018:[{date:"19th Jan",time:"09-12",topic:"Arctic Marine Mammals",type:"lecture",location:"Town Theatre",coPresenters:[]}]}},specializations:["Marine Genomics"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"rengefors-karin":{teaching:{totalSessions:5,workshopsHistory:{WoG:{2013:[{date:"9th Jan",time:"19-22",topic:"Participant Discussions",type:"presentation",location:"Prelate",coPresenters:[]},{date:"11th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",coPresenters:[]}],2014:[{date:"13th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theater",coPresenters:[]}],2015:[{date:"12th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",coPresenters:[]}],2017:[{date:"9th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2017,firstTaught:2013,yearsActive:[2013,2014,2015,2017],yearRange:"2013-2017"}},"paszkiewicz-konrad":{teaching:{totalSessions:16,workshopsHistory:{WoG:{2013:[{date:"8th Jan",time:"09-12",topic:"Sequencing Technology",type:"lecture",location:"Theater",coPresenters:[]},{date:"11th Jan",time:"09-12",topic:"Variant Calling",type:"lecture",location:"Theater",coPresenters:[]},{date:"11th Jan",time:"14-17",topic:"Short Read Genomics: Mapping and Variant Calling Lab",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"14th Jan",time:"09-12",topic:"Sequencing Technology",type:"lecture",location:"Theater",coPresenters:[]},{date:"16th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theater",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"12th Jan",time:"14-17",topic:"Introduction to Laboratory and AWS",type:"lecture",location:"Prelate",coPresenters:[]},{date:"13th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",coPresenters:[]},{date:"15th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theatre",coPresenters:[]},{date:"15th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",coPresenters:[]}],2016:[{date:"12th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"14th Jan",time:"19-22",topic:"Genomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"10th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",coPresenters:[]},{date:"12th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",coPresenters:[]},{date:"12th Jan",time:"19-22",topic:"Genomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Sequencing Technologies","General Genomics","Sequence Alignment","Variant Analysis"],lastTaught:2017,firstTaught:2013,yearsActive:[2013,2014,2015,2016,2017],yearRange:"2013-2017"}},"videvall-elin":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2017:[{date:"14th Jan",time:"14-17",topic:"Introduction to R",type:"practical",location:"Prelate",coPresenters:["Scott Handley"]}]}},specializations:["General Genomics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"mcmurdie-paul":{teaching:{totalSessions:4,workshopsHistory:{WoG:{2016:[{date:"18th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"18th Jan",time:"19-22",topic:"PhyloSeq Lab",type:"practical",location:"Prelate",coPresenters:[]}],2017:[{date:"17th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"17th Jan",time:"19-22",topic:"PhyloSeq Lab",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Microbiome Analysis","General Genomics"],lastTaught:2017,firstTaught:2016,yearsActive:[2016,2017],yearRange:"2016-2017"}},"vazquez-baeza-yoshiki":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2017:[{date:"17th Jan",time:"14-17",topic:"QIIME Lab",type:"practical",location:"Prelate",coPresenters:["Daniel McDonald"]}]}},specializations:["General Genomics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"stepanauskas-ramunas":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2017:[{date:"19th Jan",time:"09-12",topic:"Single Cell Genomics",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["Single Cell Genomics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"gardy-jennifer":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2017:[{date:"20th Jan",time:"09-12",topic:"Genomics in Public Health",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"uller-tobias":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2017:[{date:"20th Jan",time:"14-17",topic:"Epigenetics in Ecology and Evolution",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["Epigenetics"],lastTaught:2017,firstTaught:2017,yearsActive:[2017],yearRange:"2017-2017"}},"trynka-gosia":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2016:[{date:"15th Jan",time:"09-12",topic:"Using Epigenetic Marks for Biological Interpretation",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"rodriguez-ezpeleta-naiara":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2013:[{date:"8th Jan",time:"19-22",topic:"Quality Assessment and Quality Control Laboratory",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"15th Jan",time:"14-17",topic:"Sequence Data Quality Control",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"13th Jan",time:"19-22",topic:"Sequence Data Quality Control Lab",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Computational Methods","Quality Control"],lastTaught:2015,firstTaught:2013,yearsActive:[2013,2014,2015],yearRange:"2013-2015"}},"huttenhower-curtis":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2015:[{date:"19th Jan",time:"09-12",topic:"Metagenomics and Meta'omics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"19th Jan",time:"14-17",topic:"Metagenomics Lab",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2015,firstTaught:2015,yearsActive:[2015],yearRange:"2015-2015"}},"garber-manuel":{teaching:{totalSessions:13,workshopsHistory:{WoG:{2011:[{date:"16th Jul",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"16th Jul",time:"14-17",topic:"Transcriptomics Lab: Scripture",type:"practical",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"7th Oct",time:"09-12",topic:"Transcriptomics Lecture",type:"lecture",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]},{date:"7th Oct",time:"14-18",topic:"Transcriptomics Lab: Scripture",type:"practical",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]}],2012:[{date:"13th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"13th Jan",time:"19-22",topic:"Transcriptomics Laboratory",type:"practical",location:"Prelate",coPresenters:[]}],2013:[{date:"14th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"22nd Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"22nd Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"21st Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",coPresenters:[]},{date:"21st Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:["Dag Ahrn","Alper Kucukural"]},{date:"21st Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Dag Ahrn","Alper Kucukural"]}]}},specializations:["Transcriptomics"],lastTaught:2015,firstTaught:2011,yearsActive:[2011,2012,2013,2014,2015],yearRange:"2011-2015"}},"kucukural-alper":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2015:[{date:"21st Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",coPresenters:["Dag Ahrn","Manuel Garber"]},{date:"21st Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",coPresenters:["Dag Ahrn","Manuel Garber"]}]}},specializations:["Transcriptomics"],lastTaught:2015,firstTaught:2015,yearsActive:[2015],yearRange:"2015-2015"}},"woyke-tanja":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2015:[{date:"22nd Jan",time:"14-17",topic:"Single-cell Genomics",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2015,firstTaught:2015,yearsActive:[2015],yearRange:"2015-2015"}},"kamoun-sophien":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2012:[{date:"18th Jan",time:"09-12",topic:"Pathogenomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"18th Jan",time:"14-17",topic:"Pathogenomics Laboratory",type:"practical",location:"Prelate",coPresenters:[]}],2015:[{date:"23rd Jan",time:"14-17",topic:"Pathogenomics",type:"lecture",location:"Theatre",coPresenters:[]}]}},specializations:["Pathogenomics"],lastTaught:2015,firstTaught:2012,yearsActive:[2012,2015],yearRange:"2012-2015"}},"knight-robert":{teaching:{totalSessions:4,workshopsHistory:{WoG:{2013:[{date:"15th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"15th Jan",time:"14-17",topic:"Metagenomics Lab I",type:"practical",location:"Prelate",coPresenters:[]},{date:"15th Jan",time:"19-22",topic:"Continued Metagenomics Lab",type:"practical",location:"Prelate",coPresenters:[]}],2014:[{date:"18th Jan",time:"09-12",topic:"Metagenomics/Microbial Ecology",type:"lecture",location:"Theater",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2014,firstTaught:2013,yearsActive:[2013,2014],yearRange:"2013-2014"}},"loman-nick":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2013:[{date:"17th Jan",time:"09-12",topic:"Clinical Metagenomics",type:"lecture",location:"Theater",coPresenters:[]}],2014:[{date:"21st Jan",time:"09-12",topic:"Clinical Metagenomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"21st Jan",time:"14-17",topic:"Metagenomics Lab I",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Microbiome Analysis"],lastTaught:2014,firstTaught:2013,yearsActive:[2013,2014],yearRange:"2013-2014"}},"hall-neil":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2014:[{date:"24th Jan",time:"14-17",topic:"Genomics and Pathogens",type:"lecture",location:"Theater",coPresenters:[]}]}},specializations:["Pathogenomics"],lastTaught:2014,firstTaught:2014,yearsActive:[2014],yearRange:"2014-2014"}},"ponting-chris":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2013:[{date:"7th Jan",time:"14-17",topic:"Comparison of Mammalian Genomes",type:"lecture",location:"Theater",coPresenters:[]}]}},specializations:["Evolutionary Genomics"],lastTaught:2013,firstTaught:2013,yearsActive:[2013],yearRange:"2013-2013"}},"grabherr-manfred":{teaching:{totalSessions:4,workshopsHistory:{WoG:{2012:[{date:"10th Jan",time:"09-12",topic:"Quality Assessment and Control",type:"lecture",location:"Theater",coPresenters:[]},{date:"10th Jan",time:"14-17",topic:"Quality Assessment Laboratory",type:"practical",location:"Prelate",coPresenters:[]},{date:"11th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Theater",coPresenters:[]},{date:"11th Jan",time:"14-17",topic:"Assembly Laboratory",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Quality Control","Genome Assembly"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"kaessman-henrik":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2012:[{date:"12th Jan",time:"09-12",topic:"Next-gen Sequencing",type:"lecture",location:"Theater",coPresenters:[]}]}},specializations:["Sequencing Technologies"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"lunter-gerton":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2012:[{date:"12th Jan",time:"14-17",topic:"Read Mapping and Variant Calling",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Variant Analysis"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"taylor-james":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2012:[{date:"14th Jan",time:"09-12",topic:"Galaxy Platform",type:"lecture",location:"Theater",coPresenters:[]},{date:"14th Jan",time:"14-17",topic:"Galaxy Platform Laboratory",type:"practical",location:"Prelate",coPresenters:[]}]}},specializations:["Computational Platforms"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"mchardy-alice":{teaching:{totalSessions:3,workshopsHistory:{WoG:{2012:[{date:"16th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theater",coPresenters:[]},{date:"16th Jan",time:"14-17",topic:"Metagenomics Laboratory",type:"practical",location:"Prelate",coPresenters:["Johannes Droge"]},{date:"16th Jan",time:"19-22",topic:"Continued Metagenomics Lab",type:"practical",location:"Prelate",coPresenters:["Johannes Droge"]}]}},specializations:["Microbiome Analysis"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"droge-johannes":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2012:[{date:"16th Jan",time:"14-17",topic:"Metagenomics Laboratory",type:"practical",location:"Prelate",coPresenters:["Alice McHardy"]},{date:"16th Jan",time:"19-22",topic:"Continued Metagenomics Lab",type:"practical",location:"Prelate",coPresenters:["Alice McHardy"]}]}},specializations:["Microbiome Analysis"],lastTaught:2012,firstTaught:2012,yearsActive:[2012],yearRange:"2012-2012"}},"mckay-sheldon":{teaching:{totalSessions:2,workshopsHistory:{WoG:{2011:[{date:"18th Jul",time:"09-12",topic:"GMOD/GBrowse",type:"lecture",location:"Fort Collins (Fort Collins)",coPresenters:[]},{date:"18th Jul",time:"14-17",topic:"GMOD/GBrowse Laboratory",type:"practical",location:"Fort Collins (Fort Collins)",coPresenters:[]}]}},specializations:["Genome Annotation"],lastTaught:2011,firstTaught:2011,yearsActive:[2011],yearRange:"2011-2011"}},"cummings-michael":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2011:[{date:"3rd Oct",time:"09-12",topic:"Workshop Introduction",type:"lecture",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]}]}},specializations:["General Genomics"],lastTaught:2011,firstTaught:2011,yearsActive:[2011],yearRange:"2011-2011"}},"nekrutenko-anton":{teaching:{totalSessions:1,workshopsHistory:{WoG:{2011:[{date:"4th Oct",time:"14-18",topic:"Galaxy: Introduction and Introductory Laboratory",type:"practical",location:"Smithsonian Institution (Smithsonian)",coPresenters:[]}]}},specializations:["Computational Platforms"],lastTaught:2011,firstTaught:2011,yearsActive:[2011],yearRange:"2011-2011"}},"gaggiotti-oscar":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2016:[{date:"29th Jan",time:"09-12",topic:"Natural selection and adaptation",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"29th Jan",time:"14-17",topic:"Natural selection and adaptation activity",type:"practical",location:"Computer Lab",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"gutenkurst-ryan":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2016:[{date:"30th Jan",time:"09-12",topic:"Population demography",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"30th Jan",time:"14-17",topic:"Population demography activity",type:"practical",location:"Computer Lab",coPresenters:[]}]}},specializations:["Demographic Inference"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"drummond-alexei":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2016:[{date:"1st Feb",time:"09-12",topic:"Demographic analysis with BEAST2",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"1st Feb",time:"14-17",topic:"Demographic analysis with BEAST2 activity",type:"practical",location:"Computer Lab",coPresenters:[]}]}},specializations:["Demographic Inference"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"berner-daniel":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2016:[{date:"2nd Feb",time:"09-12",topic:"Evolutionary population genomics",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"2nd Feb",time:"14-17",topic:"Evolutionary population genomics activity",type:"practical",location:"Computer Lab",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"jones-felicity":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2016:[{date:"3rd Feb",time:"09-12",topic:"Detecting selective sweeps",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"3rd Feb",time:"14-17",topic:"Detecting selective sweeps activity",type:"practical",location:"Computer Lab",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2016,firstTaught:2016,yearsActive:[2016],yearRange:"2016-2016"}},"vaughan-tim":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2018:[{date:"24th Jan",time:"14-17",topic:"Demography inference with BEAST2",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]},{date:"26th Jan",time:"19-22",topic:"Structured coalescent analyses with BEAST2",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Demographic Inference","Population Structure"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"albrechtsen-anders":{teaching:{totalSessions:2,workshopsHistory:{WPSG:{2018:[{date:"25th Jan",time:"09-12",topic:"Inferring population history with low-coverage data",type:"lecture",location:"Town Theater",coPresenters:[]},{date:"25th Jan",time:"14-17",topic:"ANGSD Lab",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Population Genomics","Low Coverage Analysis"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"bank-claudia":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2018:[{date:"29th Jan",time:"09-12",topic:"Positive and negative selection",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Selection & Adaptation"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"hancock-angela":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2018:[{date:"30th Jan",time:"09-12",topic:"Genomics basis of adaptive evolution",type:"lecture",location:"Town Theater",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}},"ogilvie-huw":{teaching:{totalSessions:1,workshopsHistory:{WPSG:{2018:[{date:"30th Jan",time:"14-17",topic:"Species delimitation with SNAPP",type:"practical",location:"Egon Schiele Art Center",coPresenters:[]}]}},specializations:["Population Genomics"],lastTaught:2018,firstTaught:2018,yearsActive:[2018],yearRange:"2018-2018"}}},Zh="WoG",_h=2011,Vh="Fort Collins, Colorado, USA",$h="July 10-23, 2011",Kh=[{week:1,sessions:[{date:"10th Jul",time:"19-22",topic:"Workshop Reception",type:"social",location:"Fort Collins",presenters:["Workshop Team"]},{date:"11th Jul",time:"09-12",topic:"Workshop Introduction",type:"lecture",location:"Fort Collins",presenters:["Scott Handley"]},{date:"11th Jul",time:"14-17",topic:"Identification and Detection of Structural Variation",type:"lecture",location:"Fort Collins",presenters:["Evan Eichler"]},{date:"11th Jul",time:"19-22",topic:"Laboratory Introduction",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"12th Jul",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Fort Collins",presenters:["Mike Zody"]},{date:"12th Jul",time:"14-17",topic:"Life Technologies-Ion Torrent and Roche 454 Lectures",type:"lecture",location:"Fort Collins",presenters:["Craig Cummings","Sepideh Rogers"]},{date:"12th Jul",time:"19-22",topic:"Unix Tutorial",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"13th Jul",time:"09-12",topic:"Data Quality Assessment and Quality Control",type:"lecture",location:"Fort Collins",presenters:["Workshop Team"]},{date:"13th Jul",time:"14-17",topic:"Perl Programming",type:"lecture",location:"Fort Collins",presenters:["Jason Stajich"]},{date:"13th Jul",time:"19-22",topic:"BioPerl Laboratory",type:"practical",location:"Fort Collins",presenters:["Jason Stajich"]},{date:"14th Jul",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Fort Collins",presenters:["Shaun Jackman"]},{date:"14th Jul",time:"14-17",topic:"ABySS Laboratory I",type:"practical",location:"Fort Collins",presenters:["Shaun Jackman"]},{date:"14th Jul",time:"19-22",topic:"ABySS Laboratory II",type:"practical",location:"Fort Collins",presenters:["Shaun Jackman"]},{date:"15th Jul",time:"09-12",topic:"Genome Assembly Lab: Velvet and Bowtie I",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"15th Jul",time:"14-17",topic:"Genome Assembly Lab: Velvet and Bowtie II",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"15th Jul",time:"19-22",topic:"Open Lab",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"16th Jul",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Fort Collins",presenters:["Manuel Garber"]},{date:"16th Jul",time:"14-17",topic:"Transcriptomics Lab: Scripture",type:"practical",location:"Fort Collins",presenters:["Manuel Garber"]},{date:"16th Jul",time:"19-22",topic:"Workshop Cookout",type:"social",location:"Fort Collins",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"17th Jul",time:"all day",topic:"Free Day",type:"social",location:"Fort Collins",presenters:["Workshop Team"]},{date:"18th Jul",time:"09-12",topic:"GMOD/GBrowse",type:"lecture",location:"Fort Collins",presenters:["Sheldon McKay"]},{date:"18th Jul",time:"14-17",topic:"GMOD/GBrowse Laboratory",type:"practical",location:"Fort Collins",presenters:["Sheldon McKay"]},{date:"18th Jul",time:"19-22",topic:"Open Lab",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"19th Jul",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Fort Collins",presenters:["Scott Handley"]},{date:"19th Jul",time:"14-17",topic:"Metagenomics Laboratory",type:"practical",location:"Fort Collins",presenters:["Scott Handley"]},{date:"19th Jul",time:"19-22",topic:"Individual Projects",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"20th Jul",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"Fort Collins",presenters:["Kendra Kerr Nightingale"]},{date:"20th Jul",time:"14-17",topic:"Genome Annotation Laboratory",type:"practical",location:"Fort Collins",presenters:["Kendra Kerr Nightingale"]},{date:"20th Jul",time:"19-22",topic:"Individual Projects",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"21st Jul",time:"09-12",topic:"Participant Presentations",type:"presentation",location:"Fort Collins",presenters:["Workshop Team"]},{date:"21st Jul",time:"14-17",topic:"Individual Projects",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"21st Jul",time:"19-22",topic:"Open Lab",type:"practical",location:"Fort Collins",presenters:["Workshop Team"]},{date:"22nd Jul",time:"09-12",topic:"Participant Presentations",type:"presentation",location:"Fort Collins",presenters:["Workshop Team"]},{date:"22nd Jul",time:"14-17",topic:"Workshop Wrap-up and Discussion",type:"lecture",location:"Fort Collins",presenters:["Scott Handley"]},{date:"22nd Jul",time:"19-22",topic:"End of Workshop Dinner",type:"social",location:"Fort Collins",presenters:["Workshop Team"]},{date:"23rd Jul",time:"09-12",topic:"Departure Day",type:"social",location:"Fort Collins",presenters:["Workshop Team"]}]}],Qh={workshop:Zh,year:_h,location:Vh,dates:$h,weeks:Kh},Xh="WoG",Yh=2011,ey="Smithsonian Institution National Museum of Natural History, Washington, DC, USA",ty="October 3-7, 2011",iy=[{week:1,sessions:[{date:"3rd Oct",time:"09-12",topic:"Workshop Introduction",type:"lecture",location:"Smithsonian Institution",presenters:["Michael Cummings"]},{date:"3rd Oct",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Smithsonian Institution",presenters:["Mike Zody"]},{date:"3rd Oct",time:"14-16",topic:"Laboratory Introduction",type:"practical",location:"Smithsonian Institution",presenters:["Workshop Team"]},{date:"3rd Oct",time:"16-18",topic:"Unix Tutorial",type:"practical",location:"Smithsonian Institution",presenters:["Workshop Team"]},{date:"4th Oct",time:"09-12",topic:"Data Quality Assessment and Control",type:"lecture",location:"Smithsonian Institution",presenters:["Workshop Team"]},{date:"4th Oct",time:"09-12",topic:"File Formats",type:"lecture",location:"Smithsonian Institution",presenters:["Workshop Team"]},{date:"4th Oct",time:"14-18",topic:"Galaxy: Introduction and Introductory Laboratory",type:"practical",location:"Smithsonian Institution",presenters:["Anton Nekrutenko"]},{date:"5th Oct",time:"09-12",topic:"Assembly Lecture",type:"lecture",location:"Smithsonian Institution",presenters:["Inanc Birol","Shaun Jackman"]},{date:"5th Oct",time:"14-18",topic:"ABySS Laboratory",type:"practical",location:"Smithsonian Institution",presenters:["Shaun Jackman","Inanc Birol"]},{date:"6th Oct",time:"09-12",topic:"Metagenomics Lecture",type:"lecture",location:"Smithsonian Institution",presenters:["Jay Evans","Scott Cornman"]},{date:"6th Oct",time:"14-18",topic:"Metagenomics Laboratory",type:"practical",location:"Smithsonian Institution",presenters:["Scott Handley"]},{date:"7th Oct",time:"09-12",topic:"Transcriptomics Lecture",type:"lecture",location:"Smithsonian Institution",presenters:["Manuel Garber"]},{date:"7th Oct",time:"14-18",topic:"Transcriptomics Lab: Scripture",type:"practical",location:"Smithsonian Institution",presenters:["Manuel Garber"]}]}],oy={workshop:Xh,year:Yh,location:ey,dates:ty,weeks:iy},ny="WoG",ay=2012,ry="esk Krumlov, Czech Republic",sy="January 8-21, 2012",ly=[{week:1,sessions:[{date:"8th Jan",time:"reception",topic:"Workshop Reception",type:"social",location:"Hotel",presenters:["Workshop Team"]},{date:"9th Jan",time:"09-12",topic:"Introduction to Genomics",type:"lecture",location:"Theater",presenters:["Scott Handley"]},{date:"9th Jan",time:"14-17",topic:"Genomics Study Design",type:"lecture",location:"Theater",presenters:["Mike Zody"]},{date:"9th Jan",time:"19-22",topic:"Command Line Analysis Introduction",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"10th Jan",time:"09-12",topic:"Quality Assessment and Control",type:"lecture",location:"Theater",presenters:["Manfred Grabherr"]},{date:"10th Jan",time:"14-17",topic:"Quality Assessment Laboratory",type:"practical",location:"Prelate",presenters:["Manfred Grabherr"]},{date:"10th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"11th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Theater",presenters:["Manfred Grabherr"]},{date:"11th Jan",time:"14-17",topic:"Assembly Laboratory",type:"practical",location:"Prelate",presenters:["Manfred Grabherr"]},{date:"11th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",presenters:["Workshop Team"]},{date:"12th Jan",time:"09-12",topic:"Next-gen Sequencing",type:"lecture",location:"Theater",presenters:["Henrik Kaessmann"]},{date:"12th Jan",time:"14-17",topic:"Read Mapping and Variant Calling",type:"practical",location:"Prelate",presenters:["Gerton Lunter"]},{date:"12th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"13th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",presenters:["Manuel Garber"]},{date:"13th Jan",time:"14-17",topic:"Group Photo and Activity",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"13th Jan",time:"19-22",topic:"Transcriptomics Laboratory",type:"practical",location:"Prelate",presenters:["Manuel Garber"]},{date:"14th Jan",time:"09-12",topic:"Galaxy Platform",type:"lecture",location:"Theater",presenters:["James Taylor"]},{date:"14th Jan",time:"14-17",topic:"Galaxy Platform Laboratory",type:"practical",location:"Prelate",presenters:["James Taylor"]},{date:"14th Jan",time:"19-22",topic:"Workshop Party",type:"social",location:"Zlaty Andel",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"15th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"16th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theater",presenters:["Alice McHardy"]},{date:"16th Jan",time:"14-17",topic:"Metagenomics Laboratory",type:"practical",location:"Prelate",presenters:["Alice McHardy","Johannes Droge"]},{date:"16th Jan",time:"19-22",topic:"Continued Metagenomics Lab",type:"practical",location:"Prelate",presenters:["Alice McHardy","Johannes Droge"]},{date:"17th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",presenters:["Antonis Rokas"]},{date:"17th Jan",time:"14-17",topic:"Evolutionary Genomics Laboratory",type:"practical",location:"Prelate",presenters:["Antonis Rokas"]},{date:"17th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"18th Jan",time:"09-12",topic:"Pathogenomics",type:"lecture",location:"Theater",presenters:["Sophien Kamoun"]},{date:"18th Jan",time:"14-17",topic:"Pathogenomics Laboratory",type:"practical",location:"Prelate",presenters:["Sophien Kamoun"]},{date:"18th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",presenters:["Workshop Team"]},{date:"19th Jan",time:"09-12",topic:"Advanced Topic Sessions Planning",type:"lecture",location:"Theater",presenters:["Workshop Team"]},{date:"19th Jan",time:"14-17",topic:"Open Lab and Clinics",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"19th Jan",time:"19-22",topic:"Open Lab and Clinics",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"20th Jan",time:"09-12",topic:"Participant Presentations",type:"presentation",location:"Theater",presenters:["Workshop Team"]},{date:"20th Jan",time:"14-17",topic:"Workshop Wrap-up and Discussion",type:"lecture",location:"Theater",presenters:["Scott Handley"]},{date:"20th Jan",time:"19-22",topic:"End of Workshop Dinner",type:"social",location:"Theater",presenters:["Workshop Team"]},{date:"21st Jan",time:"09-12",topic:"Departure Day",type:"social",location:"Hotel",presenters:["Workshop Team"]}]}],cy={workshop:ny,year:ay,location:ry,dates:sy,weeks:ly},dy="WoG",py=2013,my="esk Krumlov, Czech Republic",uy="January 7-18, 2013",hy=[{week:1,sessions:[{date:"7th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theater",presenters:["Scott Handley"]},{date:"7th Jan",time:"14-17",topic:"Comparison of Mammalian Genomes",type:"lecture",location:"Theater",presenters:["Chris Ponting"]},{date:"7th Jan",time:"19-22",topic:"Laboratory Introduction",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"8th Jan",time:"09-12",topic:"Sequencing Technology",type:"lecture",location:"Theater",presenters:["Konrad Paszkiewicz"]},{date:"8th Jan",time:"14-17",topic:"Introduction to UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"8th Jan",time:"19-22",topic:"Quality Assessment and Quality Control Laboratory",type:"practical",location:"Prelate",presenters:["Naiara Rodrguez-Ezpeleta"]},{date:"9th Jan",time:"09-12",topic:"Genomic Study Design",type:"lecture",location:"Theater",presenters:["Mike Zody"]},{date:"9th Jan",time:"14-17",topic:"Advanced Unix",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"9th Jan",time:"19-22",topic:"Participant Discussions",type:"presentation",location:"Prelate",presenters:["Karin Rengefors"]},{date:"10th Jan",time:"09-12",topic:"Assembly Lecture",type:"lecture",location:"Theater",presenters:["Rayan Chikhi"]},{date:"10th Jan",time:"14-17",topic:"Assembly Lab I",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"10th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"11th Jan",time:"09-12",topic:"Variant Calling",type:"lecture",location:"Theater",presenters:["Konrad Paszkiewicz"]},{date:"11th Jan",time:"14-17",topic:"Short Read Genomics: Mapping and Variant Calling Lab",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"11th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",presenters:["Karin Rengefors"]},{date:"12th Jan",time:"09-12",topic:"Introduction to R",type:"practical",location:"Prelate",presenters:["Scott Handley"]},{date:"12th Jan",time:"14-17",topic:"Introduction to Python",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"12th Jan",time:"18-22",topic:"Brewery Tour & Workshop Party",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"13th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"14th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",presenters:["Manuel Garber"]},{date:"14th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",presenters:["Manuel Garber"]},{date:"14th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"15th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theater",presenters:["Rob Knight"]},{date:"15th Jan",time:"14-17",topic:"Metagenomics Lab I",type:"practical",location:"Prelate",presenters:["Rob Knight"]},{date:"15th Jan",time:"19-22",topic:"Continued Metagenomics Lab",type:"practical",location:"Prelate",presenters:["Rob Knight"]},{date:"16th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theater",presenters:["Bill Cresko"]},{date:"16th Jan",time:"14-17",topic:"STACKS Lab",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"16th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"17th Jan",time:"09-12",topic:"Clinical Metagenomics",type:"lecture",location:"Theater",presenters:["Nick Loman"]},{date:"17th Jan",time:"14-17",topic:"Metagenomics Lab II: QIIME",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"17th Jan",time:"19-22",topic:"Continued Metagenomics Lab II and Open Lab",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"18th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",presenters:["Antonis Rokas"]},{date:"18th Jan",time:"14-17",topic:"Ecological Genomics",type:"lecture",location:"Theater",presenters:["Chris Wheat"]},{date:"18th Jan",time:"19-22",topic:"End of Workshop Dinner and Party",type:"social",location:"Theater",presenters:["Workshop Team"]}]}],yy={workshop:dy,year:py,location:my,dates:uy,weeks:hy},gy="WoG",fy=2014,vy="esk Krumlov, Czech Republic",by="January 12-24, 2014",wy=[{week:1,sessions:[{date:"12th Jan",time:"18-22",topic:"Reception",type:"social",location:"Zlaty Andel",presenters:["Workshop Team"]},{date:"13th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theater",presenters:["Karin Rengefors"]},{date:"13th Jan",time:"14-17",topic:"Laboratory Setup",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"13th Jan",time:"19-22",topic:"UNIX Introduction",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"14th Jan",time:"09-12",topic:"Sequencing Technology",type:"lecture",location:"Theater",presenters:["Konrad Paszkiewicz"]},{date:"14th Jan",time:"14-17",topic:"Advanced Unix Commands",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"14th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",presenters:["Workshop Participants"]},{date:"15th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theater",presenters:["Mike Zody"]},{date:"15th Jan",time:"14-17",topic:"Sequence Data Quality Control",type:"practical",location:"Prelate",presenters:["Naiara Rodrguez-Ezpeleta"]},{date:"15th Jan",time:"19-22",topic:"Participant Presentations",type:"presentation",location:"Prelate",presenters:["Workshop Participants"]},{date:"16th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theater",presenters:["Konrad Paszkiewicz"]},{date:"16th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"16th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"17th Jan",time:"09-12",topic:"Assembly",type:"lecture",location:"Theater",presenters:["Rayan Chikhi"]},{date:"17th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"17th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"18th Jan",time:"09-12",topic:"Metagenomics/Microbial Ecology",type:"lecture",location:"Theater",presenters:["Rob Knight"]},{date:"18th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theater",presenters:["Evan Eichler"]},{date:"18th Jan",time:"19-22",topic:"Social Activity",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"19th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"20th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theater",presenters:["Bill Cresko"]},{date:"20th Jan",time:"14-17",topic:"STACKS Lab",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"20th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"21st Jan",time:"09-12",topic:"Clinical Metagenomics",type:"lecture",location:"Theater",presenters:["Nick Loman"]},{date:"21st Jan",time:"14-17",topic:"Metagenomics Lab I",type:"practical",location:"Prelate",presenters:["Nick Loman"]},{date:"21st Jan",time:"19-22",topic:"Python Programming Introduction",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"22nd Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theater",presenters:["Manuel Garber"]},{date:"22nd Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",presenters:["Manuel Garber"]},{date:"22nd Jan",time:"19-22",topic:"CREST Tutorial",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"23rd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theater",presenters:["Antonis Rokas"]},{date:"23rd Jan",time:"14-17",topic:"Metagenomics Lab II: QIIME",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"23rd Jan",time:"19-22",topic:"Continued Metagenomics Lab II and Open Lab",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"24th Jan",time:"09-12",topic:"Ecological Genomics (Part I and II)",type:"lecture",location:"Theater",presenters:["Chris Wheat"]},{date:"24th Jan",time:"14-17",topic:"Genomics and Pathogens",type:"lecture",location:"Theater",presenters:["Neil Hall"]},{date:"24th Jan",time:"19-22",topic:"End of Workshop Dinner and Party",type:"social",location:"Theater",presenters:["Workshop Team"]}]}],Ty={workshop:gy,year:fy,location:vy,dates:by,weeks:wy},Sy="WoG",Py=2015,xy="esk Krumlov, Czech Republic",ky="January 11-24, 2015",Ay=[{week:1,sessions:[{date:"11th Jan",time:"18-22",topic:"Reception",type:"orientation",location:"Zlaty Andel",presenters:["Workshop Team"]},{date:"12th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",presenters:["Karin Rengefors"]},{date:"12th Jan",time:"14-17",topic:"Introduction to Laboratory and AWS",type:"lecture",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"12th Jan",time:"19-22",topic:"Scientific Speed Networking",type:"orientation",location:"Prelate",presenters:["Workshop Team"]},{date:"13th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",presenters:["Konrad Paszkiewicz"]},{date:"13th Jan",time:"14-17",topic:"Introduction to UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"13th Jan",time:"19-22",topic:"Sequence Data Quality Control Lab",type:"practical",location:"Prelate",presenters:["Naiara Rodriquez-Ezpeleta"]},{date:"14th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",presenters:["Mike Zody"]},{date:"14th Jan",time:"14-17",topic:"Advanced UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"14th Jan",time:"19-22",topic:"Advanced UNIX Continuation",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"15th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theatre",presenters:["Konrad Paszkiewicz"]},{date:"15th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"15th Jan",time:"19-22",topic:"Genomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"16th Jan",time:"09-12",topic:"Assembly",type:"lecture",location:"Theatre",presenters:["Rayan Chikhi"]},{date:"16th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"16th Jan",time:"19-22",topic:"Assembly Lab Continuation",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"17th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",presenters:["Evan Eichler"]},{date:"17th Jan",time:"14-17",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"17th Jan",time:"17:30-22",topic:"Social Activity in Town/Brewery",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"18th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"19th Jan",time:"09-12",topic:"Metagenomics and Meta'omics",type:"lecture",location:"Theatre",presenters:["Curtis Huttenhower"]},{date:"19th Jan",time:"14-17",topic:"Metagenomics Lab",type:"practical",location:"Prelate",presenters:["Curtis Huttenhower"]},{date:"19th Jan",time:"19-22",topic:"QIIME Lab",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"20th Jan",time:"09-12",topic:"RAD-sequencing",type:"lecture",location:"Theatre",presenters:["Bill Cresko"]},{date:"20th Jan",time:"14-17",topic:"STACKS Laboratory",type:"practical",location:"Prelate",presenters:["Bill Cresko","Julian Catchen"]},{date:"20th Jan",time:"19-22",topic:"STACKS Lab Continuation",type:"practical",location:"Prelate",presenters:["Workshop Team","Bill Cresko","Julian Catchen"]},{date:"21st Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",presenters:["Manuel Garber"]},{date:"21st Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",presenters:["Dag Ahren","Alper Kucukural","Manuel Garber"]},{date:"21st Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Dag Ahren","Alper Kucukural","Manuel Garber"]},{date:"22nd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theatre",presenters:["Antonis Rokas"]},{date:"22nd Jan",time:"14-17",topic:"Single-cell Genomics",type:"lecture",location:"Theatre",presenters:["Tanja Woyke"]},{date:"22nd Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"23rd Jan",time:"09-12",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",presenters:["Chris Wheat"]},{date:"23rd Jan",time:"14-17",topic:"Pathogenomics",type:"lecture",location:"Theatre",presenters:["Sophien Kamoun"]},{date:"23rd Jan",time:"19-22",topic:"End of Workshop Dinner and Party",type:"social",location:"Theatre",presenters:["Workshop Team"]},{date:"24th Jan",time:"all day",topic:"Departure",type:"travel",location:"esk Krumlov",presenters:["Workshop Team"]}]}],Cy={workshop:Sy,year:Py,location:xy,dates:ky,weeks:Ay},Iy="WoG",My=2016,Gy="esk Krumlov, Czech Republic",Jy="January 10-23, 2016",Ny=[{week:1,sessions:[{date:"10th Jan",time:"18-22",topic:"Reception",type:"orientation",location:"Hotel Zlaty Andel",presenters:["Workshop Team"]},{date:"11th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",presenters:["Dag Ahrn"]},{date:"11th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",presenters:["Evan Eichler"]},{date:"11th Jan",time:"19:30-22",topic:"Scientific Speed Networking",type:"orientation",location:"Theatre",presenters:["Workshop Team"]},{date:"12th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",presenters:["Konrad Paszkiewicz"]},{date:"12th Jan",time:"14-17",topic:"Cloud Computing and AWS Introduction",type:"lecture",location:"Theatre",presenters:["Workshop Team"]},{date:"12th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"13th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",presenters:["Mike Zody"]},{date:"13th Jan",time:"14-17",topic:"Advanced UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"13th Jan",time:"19-22",topic:"Sequence Data QC and R Introduction",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"14th Jan",time:"09-12",topic:"Alignment Methods",type:"lecture",location:"Theatre",presenters:["Mike Zody"]},{date:"14th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"14th Jan",time:"19-22",topic:"Genomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"15th Jan",time:"09-12",topic:"Using Epigenetic Marks for Biological Interpretation",type:"lecture",location:"Theatre",presenters:["Gosia Trynka"]},{date:"15th Jan",time:"14-17",topic:"Genome Assembly",type:"lecture",location:"Theatre",presenters:["Rayan Chikhi"]},{date:"15th Jan",time:"19-22",topic:"Genome Assembly Lab",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"16th Jan",time:"09-12",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Theatre",presenters:["Sonya Dyhrman"]},{date:"16th Jan",time:"14-17",topic:"Genome Assembly Lab Continuation",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"16th Jan",time:"18-22",topic:"Beer Tasting and Dinner",type:"social",location:"Eggenberg Brewery",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"17th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"18th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theatre",presenters:["Paul McMurdie"]},{date:"18th Jan",time:"14-17",topic:"QIIME Lab",type:"practical",location:"Prelate",presenters:["Daniel McDonald"]},{date:"18th Jan",time:"19-22",topic:"PhyloSeq Lab",type:"practical",location:"Prelate",presenters:["Paul McMurdie"]},{date:"19th Jan",time:"09-12",topic:"RAD-Sequencing",type:"lecture",location:"Theatre",presenters:["Bill Cresko"]},{date:"19th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"Prelate",presenters:["Bill Cresko","Julian Catchen"]},{date:"19th Jan",time:"19-22",topic:"Stacks Lab Continuation",type:"practical",location:"Prelate",presenters:["Bill Cresko","Julian Catchen"]},{date:"20th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",presenters:["Brian Haas"]},{date:"20th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",presenters:["Brian Haas"]},{date:"20th Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Brian Haas"]},{date:"21st Jan",time:"09-12",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",presenters:["Chris Wheat"]},{date:"21st Jan",time:"14-17",topic:"Ortholog Detection",type:"lecture",location:"Theatre",presenters:["Toni Gabaldn"]},{date:"21st Jan",time:"19-22",topic:"Ortholog Detection Lab",type:"practical",location:"Prelate",presenters:["Toni Gabaldn"]},{date:"22nd Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Theatre",presenters:["Antonis Rokas"]},{date:"22nd Jan",time:"14-17",topic:"Final Open Lab and Data Downloading",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"22nd Jan",time:"19-22",topic:"End of Workshop Dinner and Party",type:"social",location:"Theatre",presenters:["Workshop Team"]},{date:"23rd Jan",time:"all day",topic:"Departure",type:"travel",location:"esk Krumlov",presenters:["Workshop Team"]}]}],Ey={workshop:Iy,year:My,location:Gy,dates:Jy,weeks:Ny},jy="WoG",zy=2017,By="esk Krumlov, Czech Republic",Dy="January 8-21, 2017",Ly=[{week:1,sessions:[{date:"8th Jan",time:"18-22",topic:"Reception",type:"orientation",location:"Hotel Zlaty Andel",presenters:["Workshop Team"]},{date:"9th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Theatre",presenters:["Karin Rengefors"]},{date:"9th Jan",time:"14-17",topic:"Genome Structural Variation",type:"lecture",location:"Theatre",presenters:["Evan Eichler"]},{date:"9th Jan",time:"19:30-22",topic:"Scientific Speed Networking",type:"orientation",location:"Theatre",presenters:["Workshop Team"]},{date:"10th Jan",time:"09-12",topic:"History and New Developments in Sequencing Technology",type:"lecture",location:"Theatre",presenters:["Konrad Paszkiewicz"]},{date:"10th Jan",time:"14-17",topic:"Cloud Computing and Computing Hardware",type:"lecture",location:"Theatre",presenters:["Workshop Team"]},{date:"10th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"11th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Theatre",presenters:["Mike Zody"]},{date:"11th Jan",time:"14-17",topic:"Advanced Unix",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"11th Jan",time:"19-22",topic:"Sequence Data Quality Control",type:"practical",location:"Prelate",presenters:["Sophie Shaw"]},{date:"12th Jan",time:"09-12",topic:"Alignment Methods Lab",type:"practical",location:"Prelate",presenters:["Mike Zody"]},{date:"12th Jan",time:"14-17",topic:"Genomics Lab",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"12th Jan",time:"19-22",topic:"Genomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Konrad Paszkiewicz"]},{date:"13th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Theatre",presenters:["Rayan Chikhi"]},{date:"13th Jan",time:"14-17",topic:"Genome Assembly Lab",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"13th Jan",time:"19-22",topic:"Genome Assembly Lab Continuation",type:"practical",location:"Prelate",presenters:["Rayan Chikhi"]},{date:"14th Jan",time:"09-12",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Theatre",presenters:["Sonya Dyhrman"]},{date:"14th Jan",time:"14-17",topic:"Introduction to R",type:"practical",location:"Prelate",presenters:["Elin Videvall","Scott Handley"]},{date:"14th Jan",time:"18-22",topic:"Town Tour and Dinner",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"15th Jan",time:"all day",topic:"Free Day",type:"social",location:"esk Krumlov",presenters:["Workshop Team"]},{date:"16th Jan",time:"09-12",topic:"RAD-seq",type:"lecture",location:"Theatre",presenters:["Bill Cresko"]},{date:"16th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"16th Jan",time:"19-22",topic:"Stacks Lab Continuation",type:"practical",location:"Prelate",presenters:["Julian Catchen"]},{date:"17th Jan",time:"09-12",topic:"Metagenomics",type:"lecture",location:"Theatre",presenters:["Paul McMurdie"]},{date:"17th Jan",time:"14-17",topic:"QIIME Lab",type:"practical",location:"Prelate",presenters:["Daniel McDonald","Yoshiki Vazquez-Baeza"]},{date:"17th Jan",time:"19-22",topic:"PhyloSeq Lab",type:"practical",location:"Prelate",presenters:["Paul McMurdie"]},{date:"18th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Theatre",presenters:["Brian Haas"]},{date:"18th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"Prelate",presenters:["Brian Haas"]},{date:"18th Jan",time:"19-22",topic:"Transcriptomics Lab Continuation",type:"practical",location:"Prelate",presenters:["Brian Haas"]},{date:"19th Jan",time:"09-12",topic:"Single Cell Genomics",type:"lecture",location:"Theatre",presenters:["Ramunas Stepanauskas"]},{date:"19th Jan",time:"14-17",topic:"Lies, Damn Lies and Genomics",type:"lecture",location:"Theatre",presenters:["Chris Wheat"]},{date:"19th Jan",time:"19-22",topic:"Final Open Lab",type:"practical",location:"Prelate",presenters:["Workshop Team"]},{date:"20th Jan",time:"09-12",topic:"Genomics in Public Health",type:"lecture",location:"Theatre",presenters:["Jennifer Gardy"]},{date:"20th Jan",time:"14-17",topic:"Epigenetics in Ecology and Evolution",type:"lecture",location:"Theatre",presenters:["Tobias Uller"]},{date:"20th Jan",time:"19-22",topic:"End of Workshop Dinner and Party",type:"social",location:"Theatre",presenters:["Workshop Team"]},{date:"21st Jan",time:"all day",topic:"Departure",type:"travel",location:"esk Krumlov",presenters:["Workshop Team"]}]}],Hy={workshop:jy,year:zy,location:By,dates:Dy,weeks:Ly},Ry="WoG",Uy=2018,Wy="esk Krumlov, Czech Republic",qy="January 7-20, 2018",Oy=[{week:1,sessions:[{date:"8th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Scott Handley"]},{date:"8th Jan",time:"14-17",topic:"Marine Ecosystem Phytoplankton",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"8th Jan",time:"19:30-22",topic:"Scientific Speed Networking",type:"orientation",location:"Town Theatre",presenters:["Workshop Team"]},{date:"9th Jan",time:"09-12",topic:"Sequencing Technology History",type:"lecture",location:"Town Theatre",presenters:["Julian Catchen"]},{date:"9th Jan",time:"14-17",topic:"Computer Lab Introduction",type:"practical",location:"House of Prelate",presenters:["Guy Leonard","Julian Catchen"]},{date:"9th Jan",time:"19-22",topic:"Introduction to UNIX",type:"practical",location:"House of Prelate",presenters:["Julian Catchen"]},{date:"10th Jan",time:"09-12",topic:"Genomics Study Design",type:"lecture",location:"Town Theatre",presenters:["Mike Zody"]},{date:"10th Jan",time:"14-17",topic:"Sequence Data Quality Control",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]},{date:"11th Jan",time:"09-12",topic:"Read Alignment",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]},{date:"11th Jan",time:"14-17",topic:"Genome Assembly",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]},{date:"12th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theatre",presenters:["Antonis Rokas"]},{date:"12th Jan",time:"14-17",topic:"Evolutionary Genomics Lab",type:"practical",location:"House of Prelate",presenters:["Antonis Rokas"]},{date:"13th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",presenters:["Evan Eichler"]},{date:"13th Jan",time:"14-17",topic:"Structural Variation Lab",type:"practical",location:"House of Prelate",presenters:["Evan Eichler"]}]},{week:2,sessions:[{date:"15th Jan",time:"09-12",topic:"RAD-seq",type:"lecture",location:"Town Theatre",presenters:["Bill Cresko"]},{date:"15th Jan",time:"14-17",topic:"Stacks Lab",type:"practical",location:"House of Prelate",presenters:["Julian Catchen"]},{date:"16th Jan",time:"09-12",topic:"Microbiome for Evolutionary Biology",type:"lecture",location:"Town Theatre",presenters:["Sarah Hird"]},{date:"16th Jan",time:"14-17",topic:"QIIME 2 Lab",type:"practical",location:"House of Prelate",presenters:["Daniel McDonald"]},{date:"17th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"17th Jan",time:"14-17",topic:"Transcriptomics Lab",type:"practical",location:"House of Prelate",presenters:["Brian Haas"]},{date:"18th Jan",time:"09-12",topic:"Introduction to R",type:"practical",location:"House of Prelate",presenters:["Scott Handley"]},{date:"18th Jan",time:"14-17",topic:"Single-cell Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Karthik Shekhar"]},{date:"18th Jan",time:"19-22",topic:"Single-cell Transcriptomics Lab",type:"practical",location:"House of Prelate",presenters:["Karthik Shekhar"]},{date:"19th Jan",time:"09-12",topic:"Arctic Marine Mammals",type:"lecture",location:"Town Theatre",presenters:["Eline Lorenzen"]},{date:"19th Jan",time:"14-17",topic:"Genomics Methodology",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]},{date:"20th Jan",time:"09-12",topic:"Final Open Lab",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]}]}],Fy={workshop:Ry,year:Uy,location:Wy,dates:qy,weeks:Oy},Zy="WoG",_y=2019,Vy="esk Krumlov, Czech Republic",$y="January 6-19, 2019",Ky=[{week:1,sessions:[{date:"7th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Scott Handley"]},{date:"7th Jan",time:"14-17",topic:"Marine Ecosystems",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"8th Jan",time:"09-12",topic:"Sequencing Technology & Study Design",type:"lecture",location:"Town Theatre",presenters:["Mike Zody"]},{date:"8th Jan",time:"14-17",topic:"Unix and Software Installation",type:"practical",location:"House of Prelate",presenters:["Sophie Shaw"]},{date:"8th Jan",time:"19-22",topic:"Introduction to R",type:"practical",location:"House of Prelate",presenters:["Hannah Tavalire"]},{date:"9th Jan",time:"09-12",topic:"Sequence Data Quality Control",type:"practical",location:"House of Prelate",presenters:["Sophie Shaw"]},{date:"9th Jan",time:"14-17",topic:"Read Alignment",type:"practical",location:"House of Prelate",presenters:["Mike Zody"]},{date:"9th Jan",time:"19-22",topic:"Genomics Lab",type:"practical",location:"House of Prelate",presenters:["Josephine Paris"]},{date:"10th Jan",time:"09-12",topic:"Sequence Data Assembly",type:"lecture",location:"Town Theatre",presenters:["Rayan Chikhi"]},{date:"10th Jan",time:"14-17",topic:"Assembly Lab",type:"practical",location:"House of Prelate",presenters:["Rayan Chikhi"]},{date:"10th Jan",time:"19-22",topic:"Research Documentation",type:"practical",location:"House of Prelate",presenters:["Hannah Tavalire"]},{date:"11th Jan",time:"09-12",topic:"Genomic Data Visualization",type:"lecture",location:"Town Theatre",presenters:["Malachi Griffith","Zachary Skidmore"]},{date:"11th Jan",time:"14-17",topic:"Visualization Lab Part I",type:"practical",location:"House of Prelate",presenters:["Malachi Griffith","Zachary Skidmore"]},{date:"11th Jan",time:"19-22",topic:"Visualization Lab Part II",type:"practical",location:"House of Prelate",presenters:["Malachi Griffith","Zachary Skidmore"]},{date:"12th Jan",time:"09-12",topic:"Statistics for Genomics Lab 1",type:"practical",location:"House of Prelate",presenters:["Bill Cresko"]},{date:"12th Jan",time:"14-17",topic:"Statistics for Genomics Lab 2",type:"practical",location:"House of Prelate",presenters:["Bill Cresko"]}]},{week:2,sessions:[{date:"14th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"14th Jan",time:"14-17",topic:"Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Brian Haas"]},{date:"14th Jan",time:"19-22",topic:"Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Brian Haas"]},{date:"15th Jan",time:"09-12",topic:"Microbiome Analysis",type:"lecture",location:"Town Theatre",presenters:["Amy Willis"]},{date:"15th Jan",time:"14-17",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["Amy Willis"]},{date:"15th Jan",time:"19-22",topic:"Microbiome Analysis continued",type:"practical",location:"House of Prelate",presenters:["Amy Willis"]},{date:"16th Jan",time:"09-12",topic:"Population Genomics",type:"lecture",location:"Town Theatre",presenters:["Julian Catchen"]},{date:"16th Jan",time:"14-17",topic:"Population Genomics",type:"practical",location:"House of Prelate",presenters:["Julian Catchen","Josephine Paris"]},{date:"16th Jan",time:"19-22",topic:"Population Genomics continued",type:"practical",location:"House of Prelate",presenters:["Julian Catchen","Josephine Paris"]},{date:"17th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theatre",presenters:["Dag Ahren"]},{date:"17th Jan",time:"14-17",topic:"Evolutionary Genomics",type:"practical",location:"House of Prelate",presenters:["Dag Ahren"]},{date:"17th Jan",time:"19-22",topic:"Evolutionary Genomics continued",type:"practical",location:"House of Prelate",presenters:["Dag Ahren"]},{date:"18th Jan",time:"09-12",topic:"Genomics Research Perspectives",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]},{date:"18th Jan",time:"14-17",topic:"Final Open Lab",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]}]}],Qy={workshop:Zy,year:_y,location:Vy,dates:$y,weeks:Ky},Xy="WoG",Yy=2020,eg="esk Krumlov, Czech Republic",tg="January 5-18, 2020",ig=[{week:1,sessions:[{date:"6th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Scott Handley"]},{date:"6th Jan",time:"14-17",topic:"Phytoplankton Physiological Ecology",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"7th Jan",time:"09-12",topic:"Sequencing Technology & Study Design",type:"lecture",location:"Town Theatre",presenters:["Mike Zody"]},{date:"7th Jan",time:"14-17",topic:"Unix and Sequence Data",type:"practical",location:"House of Prelate",presenters:["Sophie Shaw"]},{date:"7th Jan",time:"19-22",topic:"Unix/Sequence Data continued",type:"practical",location:"House of Prelate",presenters:["Sophie Shaw"]},{date:"8th Jan",time:"09-12",topic:"Sequence Alignment",type:"practical",location:"House of Prelate",presenters:["Mike Zody"]},{date:"8th Jan",time:"14-17",topic:"Variant Calling",type:"practical",location:"House of Prelate",presenters:["Mike Zody"]},{date:"9th Jan",time:"09-12",topic:"Sequence Assembly",type:"lecture",location:"Town Theatre",presenters:["Antoine Limasset","Camille Marchet"]},{date:"9th Jan",time:"14-17",topic:"Genome Assembly",type:"practical",location:"House of Prelate",presenters:["Antoine Limasset","Camille Marchet"]},{date:"9th Jan",time:"19-22",topic:"Genomics Lab",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]},{date:"10th Jan",time:"09-12",topic:"Introduction to R",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]},{date:"10th Jan",time:"14-17",topic:"Genomic Data Visualization",type:"practical",location:"House of Prelate",presenters:["Malachi Griffith"]},{date:"10th Jan",time:"19-22",topic:"Data Visualization continued",type:"practical",location:"House of Prelate",presenters:["Malachi Griffith"]},{date:"11th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",presenters:["Evan Eichler"]},{date:"11th Jan",time:"14-17",topic:"Metagenomics Assembly",type:"practical",location:"House of Prelate",presenters:["Rayan Chikhi","Sergey Nurk"]}]},{week:2,sessions:[{date:"13th Jan",time:"09-12",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"13th Jan",time:"14-17",topic:"Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Brian Haas"]},{date:"13th Jan",time:"19-22",topic:"Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Brian Haas"]},{date:"14th Jan",time:"09-12",topic:"Microbial Ecology",type:"lecture",location:"Town Theatre",presenters:["Kelly Wrighton"]},{date:"14th Jan",time:"14-17",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["Scott Handley"]},{date:"14th Jan",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["Scott Handley"]},{date:"15th Jan",time:"09-12",topic:"Population Genomics",type:"lecture",location:"Town Theatre",presenters:["Emiliano Trucchi"]},{date:"15th Jan",time:"14-17",topic:"Population Genomics",type:"practical",location:"House of Prelate",presenters:["Emiliano Trucchi"]},{date:"15th Jan",time:"19-22",topic:"Population Genomics continued",type:"practical",location:"House of Prelate",presenters:["Emiliano Trucchi"]},{date:"16th Jan",time:"09-12",topic:"Thaumarchaeota",type:"lecture",location:"Town Theatre",presenters:["Christa Schleper"]},{date:"16th Jan",time:"14-17",topic:"Single Cell Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Kirk Gosik"]},{date:"16th Jan",time:"19-22",topic:"Single Cell Transcriptomics",type:"practical",location:"House of Prelate",presenters:["Kirk Gosik"]},{date:"17th Jan",time:"09-12",topic:"Genomics Perspectives",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]},{date:"17th Jan",time:"14-17",topic:"Closing Open Lab",type:"practical",location:"House of Prelate",presenters:["Workshop Team"]}]}],og={workshop:Xy,year:Yy,location:eg,dates:tg,weeks:ig},ng="WoG",ag=2022,rg="esk Krumlov, Czech Republic",sg="May 22 - June 4, 2022",lg=[{week:1,sessions:[{date:"23rd May",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Josephine Paris"]},{date:"23rd May",time:"14-17",topic:"Sequencing Technology and Study Design",type:"lecture",location:"Town Theatre",presenters:["Emiliano Trucchi"]},{date:"24th May",time:"09-12",topic:"Alignment",type:"lecture",location:"Town Theatre",presenters:["Rayan Chikhi"]},{date:"24th May",time:"14-17",topic:"Variant Calling",type:"lecture",location:"House of Prelate",presenters:["Erik Garrison"]},{date:"24th May",time:"19-22",topic:"Structural Variant Calling",type:"lecture",location:"House of Prelate",presenters:["Fritz Sedlazeck"]},{date:"25th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",presenters:["Camille Marchet","Antoine Limasset"]},{date:"25th May",time:"14-17",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",presenters:["Marcela Uliano-Silva"]},{date:"25th May",time:"19-22",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",presenters:["Marcela Uliano-Silva"]},{date:"26th May",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"House of Prelate",presenters:["Katharina Hoff"]},{date:"26th May",time:"14-17",topic:"Genome Annotation",type:"practical",location:"House of Prelate",presenters:["Katharina Hoff"]},{date:"26th May",time:"19-22",topic:"Pangenomics",type:"lecture",location:"Town Theatre",presenters:["Erik Garrison"]},{date:"27th May",time:"09-12",topic:"BIG Data",type:"lecture",location:"Town Theatre",presenters:["Rayan Chikhi"]},{date:"27th May",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",presenters:["Josephine Paris","Emiliano Trucchi"]}]},{week:2,sessions:[{date:"30th May",time:"09-12",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",presenters:["Robert Waterhouse"]},{date:"30th May",time:"14-17",topic:"Comparative Genomics",type:"practical",location:"House of Prelate",presenters:["Francesco Cicconardi"]},{date:"31st May",time:"09-12",topic:"Phylogenomics",type:"lecture",location:"Town Theatre",presenters:["Rosa Fernndez"]},{date:"31st May",time:"14-17",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"31st May",time:"19-22",topic:"Differential Gene Expression",type:"practical",location:"House of Prelate",presenters:["Rachel Steward"]},{date:"1st Jun",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"1st Jun",time:"14-17",topic:"Metagenomics & Metatranscriptomics",type:"lecture",location:"House of Prelate",presenters:["Paolo Manghi"]},{date:"1st Jun",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["David Barnett"]},{date:"2nd Jun",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",presenters:["Evan Eichler"]},{date:"2nd Jun",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",presenters:["James Whiting"]},{date:"2nd Jun",time:"19-22",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",presenters:["Doug Scofield"]},{date:"3rd Jun",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]}]}],cg={workshop:ng,year:ag,location:rg,dates:sg,weeks:lg},dg="WoG",pg=2023,mg="esk Krumlov, Czech Republic",ug="May 14-27, 2023",hg=[{week:1,sessions:[{date:"15th May",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Josie Paris","Daniel Kintzl"]},{date:"15th May",time:"14-17",topic:"Sequencing Technology and Study Design",type:"lecture",location:"Town Theatre",presenters:["Mike Zody"]},{date:"17th May",time:"09-12",topic:"Alignment",type:"lecture",location:"House of Prelate",presenters:["Mike Zody"]},{date:"17th May",time:"14-17",topic:"Variant Calling",type:"lecture",location:"House of Prelate",presenters:["Erik Garrison"]},{date:"17th May",time:"19-22",topic:"Structural Variants",type:"lecture",location:"House of Prelate",presenters:["Fritz Sedlazeck"]},{date:"18th May",time:"09-12",topic:"Genome Assembly: theory and principles",type:"lecture",location:"Town Theatre",presenters:["Camille Marchet","Antoine Limasset"]},{date:"18th May",time:"14-17",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",presenters:["Marcela Uliano-Silva"]},{date:"18th May",time:"19-22",topic:"Genome Assembly: from reads to a genome",type:"practical",location:"House of Prelate",presenters:["Marcela Uliano-Silva"]},{date:"19th May",time:"09-12",topic:"Genome Annotation",type:"lecture",location:"House of Prelate",presenters:["Katharina Hoff","Natalia Nenasheva"]},{date:"19th May",time:"14-17",topic:"Genome Annotation",type:"practical",location:"House of Prelate",presenters:["Katharina Hoff","Natalia Nenasheva"]},{date:"19th May",time:"19-22",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",presenters:["Josie Paris","Guy Leonard"]},{date:"20th May",time:"09-12",topic:"Pangenomics",type:"lecture",location:"Town Theatre",presenters:["Erik Garrison"]},{date:"20th May",time:"14-15",topic:"BIG Data",type:"lecture",location:"Town Theatre",presenters:["Rayan Chikhi"]}]},{week:2,sessions:[{date:"22nd May",time:"09-12",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",presenters:["Robert Waterhouse"]},{date:"22nd May",time:"14-17",topic:"Comparative Genomics",type:"practical",location:"House of Prelate",presenters:["Francesco Cicconardi"]},{date:"23rd May",time:"09-12",topic:"Phylogenomics",type:"lecture",location:"Town Theatre",presenters:["Rosa Fernndez"]},{date:"23rd May",time:"14-17",topic:"Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"23rd May",time:"19-22",topic:"Differential Gene Expression",type:"practical",location:"House of Prelate",presenters:["Rachel Steward"]},{date:"24th May",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"24th May",time:"14-17",topic:"Metagenomics & Metatranscriptomics",type:"lecture",location:"House of Prelate",presenters:["Paolo Manghi","Aitor Blanco Miguez"]},{date:"24th May",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["David Barnett"]},{date:"25th May",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",presenters:["Evan Eichler"]},{date:"25th May",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",presenters:["James Whiting"]},{date:"25th May",time:"19-22",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",presenters:["Doug Scofield"]},{date:"26th May",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]}]}],yg={workshop:dg,year:pg,location:mg,dates:ug,weeks:hg},gg="WoG",fg=2024,vg="esk Krumlov, Czech Republic",bg="January 7-20, 2024",wg=[{week:1,sessions:[{date:"8th Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theatre",presenters:["Josie Paris","Daniel Kintzl"]},{date:"8th Jan",time:"14-17",topic:"Sequencing Technology",type:"lecture",location:"Town Theatre",presenters:["Scott Handley"]},{date:"10th Jan",time:"09-12",topic:"Alignment",type:"lecture",location:"Town Theatre",presenters:["Rayan Chikhi"]},{date:"10th Jan",time:"14-17",topic:"Genome Annotation",type:"lecture",location:"Town Theatre",presenters:["Katharina Hoff"]},{date:"10th Jan",time:"19-22",topic:"Genome Annotation practical",type:"practical",location:"House of Prelate",presenters:["Katharina Hoff"]},{date:"11th Jan",time:"09-12",topic:"Genome Assembly",type:"lecture",location:"Town Theatre",presenters:["Camille Marchet","Antoine Limasset"]},{date:"11th Jan",time:"14-17",topic:"Genome Assembly practical",type:"practical",location:"House of Prelate",presenters:["Camille Marchet","Antoine Limasset"]},{date:"12th Jan",time:"09-12",topic:"Variant Calling",type:"lecture",location:"Town Theatre",presenters:["Erik Garrison"]},{date:"12th Jan",time:"14-17",topic:"Structural Variant Calling",type:"lecture",location:"House of Prelate",presenters:["Fritz Sedlazeck"]},{date:"12th Jan",time:"19-22",topic:"Genomics Adventure",type:"practical",location:"House of Prelate",presenters:["Guy Leonard"]},{date:"13th Jan",time:"09-12",topic:"Lies, Damn Lies, and Genomics",type:"lecture",location:"Town Theatre",presenters:["Chris Wheat"]},{date:"13th Jan",time:"14-17",topic:"Pangenomics",type:"lecture",location:"House of Prelate",presenters:["Erik Garrison"]}]},{week:2,sessions:[{date:"15th Jan",time:"09-12",topic:"Transcriptomics and Single-cell Transcriptomics",type:"lecture",location:"Town Theatre",presenters:["Brian Haas"]},{date:"15th Jan",time:"19-22",topic:"Analysis of RNASeq Data",type:"practical",location:"House of Prelate",presenters:["Rachel Steward"]},{date:"16th Jan",time:"09-12",topic:"Genomics in the Ocean",type:"lecture",location:"Town Theatre",presenters:["Sonya Dyhrman"]},{date:"16th Jan",time:"14-17",topic:"Comparative Genomics",type:"lecture",location:"Town Theatre",presenters:["Oleg Simakov"]},{date:"16th Jan",time:"19-22",topic:"Microbiome Analysis",type:"practical",location:"House of Prelate",presenters:["David Barnett"]},{date:"17th Jan",time:"09-12",topic:"The wonders of genome assembly!",type:"lecture",location:"Town Theatre",presenters:["Marcela Uliano-Silva"]},{date:"17th Jan",time:"14-17",topic:"Population Genomics",type:"lecture",location:"House of Prelate",presenters:["Vincenza Colonna"]},{date:"18th Jan",time:"09-12",topic:"Genome Structural Variation",type:"lecture",location:"Town Theatre",presenters:["Evan Eichler"]},{date:"18th Jan",time:"14-17",topic:"Best Practices in Handling Genomic Data",type:"lecture",location:"House of Prelate",presenters:["Dag Ahrn"]},{date:"18th Jan",time:"19-22",topic:"Phylogenomics",type:"lecture",location:"House of Prelate",presenters:["Rosa Fernndez"]},{date:"19th Jan",time:"09-12",topic:"BIG DATA",type:"lecture",location:"House of Prelate",presenters:["Rayan Chikhi"]}]}],Tg={workshop:gg,year:fg,location:vg,dates:bg,weeks:wg},Sg="WPSG",Pg=2016,xg="esk Krumlov, Czech Republic",kg="January 24-February 6, 2016",Ag=[{week:1,sessions:[{date:"24th Jan",time:"09-12",topic:"Workshop introduction",type:"orientation",location:"Town Theater",presenters:["Scott Handley","Walter Salzburger"]},{date:"24th Jan",time:"14-17",topic:"UNIX computer lab",type:"practical",location:"Computer Lab",presenters:["Workshop Team"]},{date:"25th Jan",time:"09-12",topic:"File formats, VCFtools, Plink",type:"lecture",location:"Town Theater",presenters:["Workshop Team"]},{date:"25th Jan",time:"14-17",topic:"R and adegenet, STRUCTURE, smartpca",type:"practical",location:"Computer Lab",presenters:["Daniel Falush"]},{date:"26th Jan",time:"09-12",topic:"Bayesian and likelihood inference",type:"lecture",location:"Town Theater",presenters:["Alex Buerkle"]},{date:"26th Jan",time:"14-17",topic:"Bayesian and likelihood inference activity",type:"practical",location:"Computer Lab",presenters:["Alex Buerkle"]},{date:"27th Jan",time:"09-12",topic:"Genotype and SNP calling",type:"lecture",location:"Town Theater",presenters:["Matteo Fumagalli"]},{date:"27th Jan",time:"14-17",topic:"Genotype and SNP calling activity",type:"practical",location:"Computer Lab",presenters:["Matteo Fumagalli"]},{date:"28th Jan",time:"09-12",topic:"Introgression and admixture modeling",type:"lecture",location:"Town Theater",presenters:["Alex Buerkle"]},{date:"28th Jan",time:"14-17",topic:"Introgression and admixture modeling activity",type:"practical",location:"Computer Lab",presenters:["Alex Buerkle"]},{date:"29th Jan",time:"09-12",topic:"Natural selection and adaptation",type:"lecture",location:"Town Theater",presenters:["Oscar Gaggiotti"]},{date:"29th Jan",time:"14-17",topic:"Natural selection and adaptation activity",type:"practical",location:"Computer Lab",presenters:["Oscar Gaggiotti"]},{date:"30th Jan",time:"09-12",topic:"Population demography",type:"lecture",location:"Town Theater",presenters:["Ryan Gutenkunst"]},{date:"30th Jan",time:"14-17",topic:"Population demography activity",type:"practical",location:"Computer Lab",presenters:["Ryan Gutenkunst"]},{date:"30th Jan",time:"19-22",topic:"Genomics of organismal diversification",type:"lecture",location:"Town Theater",presenters:["Walter Salzburger"]}]},{week:2,sessions:[{date:"31st Jan",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",presenters:["Ludovic Orlando"]},{date:"31st Jan",time:"14-17",topic:"Ancient genomics activity",type:"practical",location:"Computer Lab",presenters:["Ludovic Orlando"]},{date:"1st Feb",time:"09-12",topic:"Demographic analysis with BEAST2",type:"lecture",location:"Town Theater",presenters:["Alexei Drummond"]},{date:"1st Feb",time:"14-17",topic:"Demographic analysis with BEAST2 activity",type:"practical",location:"Computer Lab",presenters:["Alexei Drummond"]},{date:"2nd Feb",time:"09-12",topic:"Evolutionary population genomics",type:"lecture",location:"Town Theater",presenters:["Daniel Berner"]},{date:"2nd Feb",time:"14-17",topic:"Evolutionary population genomics activity",type:"practical",location:"Computer Lab",presenters:["Daniel Berner"]},{date:"3rd Feb",time:"09-12",topic:"Detecting selective sweeps",type:"lecture",location:"Town Theater",presenters:["Felicity Jones"]},{date:"3rd Feb",time:"14-17",topic:"Detecting selective sweeps activity",type:"practical",location:"Computer Lab",presenters:["Felicity Jones"]},{date:"4th Feb",time:"09-12",topic:"Introgression",type:"lecture",location:"Town Theater",presenters:["Daniel Falush"]},{date:"4th Feb",time:"14-17",topic:"Introgression activity",type:"practical",location:"Computer Lab",presenters:["Daniel Falush"]},{date:"5th Feb",time:"09-12",topic:"Divergence and gene flow between species",type:"lecture",location:"Town Theater",presenters:["Chris Jiggins","Simon Martin"]},{date:"5th Feb",time:"14-17",topic:"Divergence and gene flow between species activity",type:"practical",location:"Computer Lab",presenters:["Chris Jiggins","Simon Martin"]},{date:"6th Feb",time:"09-12",topic:"Wrap-up session",type:"discussion",location:"Town Theater",presenters:["Workshop Team"]}]}],Cg={workshop:Sg,year:Pg,location:xg,dates:kg,weeks:Ag},Ig="WPSG",Mg=2018,Gg="esk Krumlov, Czech Republic",Jg="January 21-February 3, 2018",Ng=[{week:1,sessions:[{date:"21st Jan",time:"09-12",topic:"Introduction and orientation",type:"orientation",location:"Town Theater",presenters:["Walter Salzburger","Workshop Team"]},{date:"21st Jan",time:"14-17",topic:"UNIX introduction",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"22nd Jan",time:"09-12",topic:"Likelihood and Bayesian inference",type:"lecture",location:"Town Theater",presenters:["David Swofford"]},{date:"22nd Jan",time:"14-17",topic:"Likelihood and Bayesian inference activity",type:"practical",location:"Egon Schiele Art Center",presenters:["David Swofford"]},{date:"23rd Jan",time:"09-12",topic:"Population structure",type:"lecture",location:"Town Theater",presenters:["Daniel Falush"]},{date:"23rd Jan",time:"14-17",topic:"STRUCTURE and fineStructure workshops",type:"practical",location:"Egon Schiele Art Center",presenters:["Daniel Falush"]},{date:"24th Jan",time:"09-12",topic:"Experimental design / Demography inference",type:"lecture",location:"Town Theater",presenters:["Richard Durbin"]},{date:"24th Jan",time:"14-17",topic:"Demography inference with BEAST2",type:"practical",location:"Egon Schiele Art Center",presenters:["Tim Vaughan"]},{date:"25th Jan",time:"09-12",topic:"Inferring population history with low-coverage data",type:"lecture",location:"Town Theater",presenters:["Anders Albrechtsen"]},{date:"25th Jan",time:"14-17",topic:"ANGSD Lab",type:"practical",location:"Egon Schiele Art Center",presenters:["Anders Albrechtsen"]},{date:"26th Jan",time:"09-12",topic:"Demographic inference based on Site frequency spectrum",type:"lecture",location:"Town Theater",presenters:["Vitor Sousa"]},{date:"26th Jan",time:"14-17",topic:"fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Vitor Sousa"]},{date:"26th Jan",time:"19-22",topic:"Structured coalescent analyses with BEAST2",type:"practical",location:"Egon Schiele Art Center",presenters:["Tim Vaughan"]},{date:"27th Jan",time:"09-12",topic:"The genomics of organismal diversification",type:"lecture",location:"Town Theater",presenters:["Walter Salzburger"]}]},{week:2,sessions:[{date:"29th Jan",time:"09-12",topic:"Positive and negative selection",type:"lecture",location:"Town Theater",presenters:["Claudia Bank"]},{date:"29th Jan",time:"14-17",topic:"Species tree estimation using SVDQuartets",type:"practical",location:"Egon Schiele Art Center",presenters:["David Swofford"]},{date:"30th Jan",time:"09-12",topic:"Genomics basis of adaptive evolution",type:"lecture",location:"Town Theater",presenters:["Angela Hancock"]},{date:"30th Jan",time:"14-17",topic:"Species delimitation with SNAPP",type:"practical",location:"Egon Schiele Art Center",presenters:["Huw Ogilvie"]},{date:"31st Jan",time:"09-12",topic:"Evolution of the mutation spectrum",type:"lecture",location:"Town Theater",presenters:["Kelley Harris"]},{date:"31st Jan",time:"14-17",topic:"Mutation spectrum analysis activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Kelley Harris"]},{date:"1st Feb",time:"09-12",topic:"Speciation with gene flow",type:"lecture",location:"Town Theater",presenters:["Chris Jiggins"]},{date:"1st Feb",time:"14-17",topic:"ABBA BABA statistics and Topology weighting",type:"practical",location:"Egon Schiele Art Center",presenters:["Simon Martin"]},{date:"2nd Feb",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",presenters:["Ludovic Orlando"]},{date:"2nd Feb",time:"14-17",topic:"Wrap-up session",type:"discussion",location:"Town Theater",presenters:["Workshop Team"]}]}],Eg={workshop:Ig,year:Mg,location:Gg,dates:Jg,weeks:Ng},jg="WPSG",zg=2020,Bg="esk Krumlov, Czech Republic",Dg="January 19 - February 1, 2020",Lg=[{week:1,sessions:[{date:"20th Jan",time:"09-12",topic:"Introduction and orientation",type:"orientation",location:"Town Theater",presenters:["Workshop Team"]},{date:"20th Jan",time:"14-17",topic:"Computer lab / UNIX introduction",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"21st Jan",time:"09-12",topic:"Likelihood, probability, and Bayesian inference",type:"lecture",location:"Town Theater",presenters:["Alex Buerkle"]},{date:"21st Jan",time:"14-17",topic:"Probability in population genomics activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"21st Jan",time:"19-22",topic:"First steps in genomic data analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"22nd Jan",time:"09-12",topic:"The genomics of organismal diversification",type:"lecture",location:"Town Theater",presenters:["Walter Salzburger"]},{date:"22nd Jan",time:"14-17",topic:"Demographic analysis with fastsimcoal2",type:"practical",location:"Egon Schiele Art Center",presenters:["Vtor Sousa"]},{date:"22nd Jan",time:"19-22",topic:"Fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Vtor Sousa"]},{date:"23rd Jan",time:"09-12",topic:"Adaptation and selection",type:"lecture",location:"Town Theater",presenters:["Rachael Dudaniec"]},{date:"23rd Jan",time:"14-17",topic:"Environmental selection detection activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"23rd Jan",time:"19-22",topic:"Predicting evolution",type:"lecture",location:"Town Theater",presenters:["Patrik Nosil"]},{date:"24th Jan",time:"09-12",topic:"The coalescent",type:"lecture",location:"Town Theater",presenters:["Leo Speidel","Simon Myers"]},{date:"24th Jan",time:"14-17",topic:"Genome-wide genealogy inference activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Leo Speidel","Simon Myers"]},{date:"24th Jan",time:"19-22",topic:"Experimental design / Demography",type:"lecture",location:"Town Theater",presenters:["Richard Durbin"]},{date:"25th Jan",time:"09-12",topic:"Evolution of the mutation spectrum",type:"lecture",location:"Town Theater",presenters:["Kelley Harris"]},{date:"25th Jan",time:"14-17",topic:"Mutation spectrum inference activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Kelley Harris"]}]},{week:2,sessions:[{date:"27th Jan",time:"09-12",topic:"Population structure",type:"lecture",location:"Town Theater",presenters:["Daniel Falush"]},{date:"27th Jan",time:"14-17",topic:"Population structure analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Daniel Falush"]},{date:"27th Jan",time:"19-22",topic:"Population structure continued",type:"practical",location:"Egon Schiele Art Center",presenters:["Daniel Falush"]},{date:"28th Jan",time:"09-12",topic:"Multi-species coalescent",type:"lecture",location:"Town Theater",presenters:["Lacey Knowles"]},{date:"28th Jan",time:"14-17",topic:"Multi-species coalescent analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Lacey Knowles"]},{date:"28th Jan",time:"19-22",topic:"Multi-species coalescent continued",type:"practical",location:"Egon Schiele Art Center",presenters:["Lacey Knowles"]},{date:"29th Jan",time:"09-12",topic:"Speciation and hybridization",type:"lecture",location:"Town Theater",presenters:["Simon Martin"]},{date:"29th Jan",time:"14-17",topic:"Hybridization analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Simon Martin"]},{date:"29th Jan",time:"19-22",topic:"Hybridization continued",type:"practical",location:"Egon Schiele Art Center",presenters:["Simon Martin"]},{date:"30th Jan",time:"09-12",topic:"Genome-Wide Association Studies",type:"lecture",location:"Town Theater",presenters:["Magnus Nordborg"]},{date:"30th Jan",time:"14-17",topic:"GWAS analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Magnus Nordborg"]},{date:"30th Jan",time:"19-22",topic:"GWAS continued",type:"practical",location:"Egon Schiele Art Center",presenters:["Magnus Nordborg"]},{date:"31st Jan",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",presenters:["Ludovic Orlando"]},{date:"31st Jan",time:"14-17",topic:"Ancient genomics analysis",type:"practical",location:"Egon Schiele Art Center",presenters:["Ludovic Orlando"]}]}],Hg={workshop:jg,year:zg,location:Bg,dates:Dg,weeks:Lg},Rg="WPSG",Ug=2022,Wg="esk Krumlov, Czech Republic",qg="June 5-18, 2022",Og=[{week:1,sessions:[{date:"5th Jun",time:"14-17",topic:"UNIX primer",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"6th Jun",time:"09-12",topic:"Workshop introduction",type:"orientation",location:"Town Theater",presenters:["Workshop Team"]},{date:"6th Jun",time:"14-17",topic:"Likelihood and probability",type:"lecture",location:"Town Theater",presenters:["Michael Matschiner"]},{date:"6th Jun",time:"19-22",topic:"AMI/Jupyter tutorial & Probability activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"7th Jun",time:"09-12",topic:"Population genomics",type:"lecture",location:"Town Theater",presenters:["Rasmus Nielsen"]},{date:"7th Jun",time:"14-17",topic:"ANGSD activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Rasmus Nielsen","Thorfinn Korneliussen"]},{date:"8th Jun",time:"09-12",topic:"Coalescent theory and introgression",type:"lecture",location:"Town Theater",presenters:["Matthew Hahn"]},{date:"8th Jun",time:"14-17",topic:"Multiple testing",type:"lecture",location:"Town Theater",presenters:["Matthew Hahn"]},{date:"8th Jun",time:"19-22",topic:"D-statistics activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"9th Jun",time:"09-12",topic:"Model-based inference of population history",type:"lecture",location:"Town Theater",presenters:["Vitor Sousa"]},{date:"9th Jun",time:"14-17",topic:"fastsimcoal2 activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Vitor Sousa"]},{date:"9th Jun",time:"19-22",topic:"Species tree inference activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"10th Jun",time:"09-12",topic:"Selection and adaptation",type:"lecture",location:"Town Theater",presenters:["Leonie Moyle"]}]},{week:2,sessions:[{date:"13th Jun",time:"09-12",topic:"Speciation with gene flow",type:"lecture",location:"Town Theater",presenters:["Chris Jiggins"]},{date:"13th Jun",time:"14-17",topic:"Structural variation",type:"lecture",location:"Town Theater",presenters:["Alexander Suh"]},{date:"13th Jun",time:"19-22",topic:"Structural variation activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Valentina Peona"]},{date:"14th Jun",time:"09-12",topic:"Machine Learning",type:"lecture",location:"Town Theater",presenters:["Matteo Fumagalli","Flora Jay"]},{date:"14th Jun",time:"14-17",topic:"Selection inference ML activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Matteo Fumagalli"]},{date:"14th Jun",time:"19-22",topic:"Demography inference ML activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Flora Jay"]},{date:"15th Jun",time:"09-12",topic:"Tree sequences",type:"lecture",location:"Town Theater",presenters:["Simon Myers"]},{date:"15th Jun",time:"14-17",topic:"Relate activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Simon Myers","Jasmin Rees","Leo Speidel"]},{date:"15th Jun",time:"19-22",topic:"Open lab",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"16th Jun",time:"09-12",topic:"Simulation",type:"lecture",location:"Town Theater",presenters:["Georgia Tsambos"]},{date:"16th Jun",time:"14-17",topic:"Simulation with msprime activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Georgia Tsambos"]},{date:"16th Jun",time:"19-22",topic:"Inference with tsinfer activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Georgia Tsambos"]},{date:"17th Jun",time:"09-12",topic:"Ancient genomics",type:"lecture",location:"Town Theater",presenters:["Fabrizio Mafessoni"]},{date:"17th Jun",time:"14-17",topic:"Wrap-up",type:"discussion",location:"Town Theater",presenters:["Workshop Team"]}]}],Fg={workshop:Rg,year:Ug,location:Wg,dates:qg,weeks:Og},Zg="WPSG",_g=2025,Vg="esk Krumlov, Czech Republic",$g="January 19-February 1, 2025",Kg=[{week:1,sessions:[{date:"19th Jan",time:"14-17",topic:"UNIX Primer",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"20th Jan",time:"09-12",topic:"Workshop Introduction",type:"orientation",location:"Town Theater",presenters:["Workshop Team"]},{date:"20th Jan",time:"14-17",topic:"Likelihood and Probability",type:"lecture",location:"Town Theater",presenters:["Michael Matschiner"]},{date:"20th Jan",time:"19-22",topic:"AMI Introduction & Probability Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"21st Jan",time:"09-12",topic:"Molecular Population Genomics",type:"lecture",location:"Town Theater",presenters:["Matthew Hahn"]},{date:"21st Jan",time:"14-17",topic:"Population Genomics Primer I",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"21st Jan",time:"19-22",topic:"Population Genomics Primer II",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"22nd Jan",time:"09-12",topic:"Coalescent Theory",type:"lecture",location:"Town Theater",presenters:["Matthew Hahn"]},{date:"22nd Jan",time:"14-17",topic:"Model-based Inference of Population History",type:"lecture",location:"Town Theater",presenters:["Vitor Sousa"]},{date:"22nd Jan",time:"19-22",topic:"fastsimcoal2 Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Vitor Sousa"]},{date:"23rd Jan",time:"09-12",topic:"Structural Variation",type:"lecture",location:"Town Theater",presenters:["Alex Suh"]},{date:"23rd Jan",time:"14-17",topic:"Structural Variation Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Valentina Peona"]},{date:"23rd Jan",time:"19-22",topic:"Pangenomics",type:"lecture",location:"Town Theater",presenters:["Alexander Leonard"]},{date:"24th Jan",time:"09-12",topic:"Machine Learning",type:"lecture",location:"Town Theater",presenters:["Andrew Kern"]},{date:"24th Jan",time:"14-17",topic:"Machine Learning Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Andrew Kern"]},{date:"24th Jan",time:"19-22",topic:"Experimental Design",type:"lecture",location:"Town Theater",presenters:["Joana Meier","Workshop Team"]},{date:"25th Jan",time:"09-12",topic:"The Genomics of Organismal Diversification",type:"lecture",location:"Town Theater",presenters:["Walter Salzburger"]},{date:"25th Jan",time:"14-17",topic:"Open Lab",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]}]},{week:2,sessions:[{date:"27th Jan",time:"09-12",topic:"Hybridisation",type:"lecture",location:"Town Theater",presenters:["Joana Meier"]},{date:"27th Jan",time:"14-17",topic:"D-statistics Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"27th Jan",time:"19-22",topic:"Tree-based Introgression Detection Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"28th Jan",time:"09-12",topic:"Combining Genomics with Experimental Studies of Adaptation",type:"lecture",location:"Town Theater",presenters:["Katie Peichel"]},{date:"28th Jan",time:"14-17",topic:"Detecting Positive Selection",type:"practical",location:"Egon Schiele Art Center",presenters:["Magdalena Bohutnsk"]},{date:"28th Jan",time:"19-22",topic:"Selective Sweep Identification",type:"practical",location:"Egon Schiele Art Center",presenters:["Joana Meier"]},{date:"29th Jan",time:"09-12",topic:"Coalescent Simulation and Inference Activity 1/2",type:"practical",location:"Egon Schiele Art Center",presenters:["Georgia Tsambos"]},{date:"29th Jan",time:"14-17",topic:"Coalescent Simulation and Inference Activity 2/2",type:"practical",location:"Egon Schiele Art Center",presenters:["Georgia Tsambos"]},{date:"29th Jan",time:"19-22",topic:"Building Intuition into Population Genetics Fundamentals and Inference",type:"lecture",location:"Town Theater",presenters:["Martin Petr"]},{date:"30th Jan",time:"09-12",topic:"Conservation Genomics",type:"lecture",location:"Town Theater",presenters:["Emiliano Trucchi"]},{date:"30th Jan",time:"14-17",topic:"Genetic Load Activities",type:"practical",location:"Egon Schiele Art Center",presenters:["Emiliano Trucchi"]},{date:"30th Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Egon Schiele Art Center",presenters:["Workshop Team"]},{date:"31st Jan",time:"09-12",topic:"Ancient Genomics",type:"lecture",location:"Town Theater",presenters:["Michael Hofreiter"]},{date:"31st Jan",time:"14-17",topic:"Ancient Genomics Activity",type:"practical",location:"Egon Schiele Art Center",presenters:["Michael Westbury"]}]}],Qg={workshop:Zg,year:_g,location:Vg,dates:$g,weeks:Kg},Xg="WPhylo",Yg=2017,ef="esk Krumlov, Czech Republic",tf="January 22-February 4, 2017",of=[{week:1,sessions:[{date:"22nd Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theater",presenters:["Scott Handley","Toni Gabaldn"]},{date:"22nd Jan",time:"14-17",topic:"UNIX Introduction",type:"practical",location:"Municipal Theater",presenters:["Guy Leonard","Jordi Paps"]},{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenetic Inference",type:"lecture",location:"Town Theater",presenters:["Toni Gabaldn"]},{date:"23rd Jan",time:"14-17",topic:"Orthology and Paralogy",type:"lecture",location:"Town Theater",presenters:["Toni Gabaldn"]},{date:"23rd Jan",time:"19-22",topic:"Orthology Detection Lab",type:"practical",location:"Municipal Theater",presenters:["Toni Gabaldn"]},{date:"24th Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",presenters:["Antonis Rokas"]},{date:"24th Jan",time:"14-17",topic:"Alignment and Trimming Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"24th Jan",time:"19-22",topic:"Alignment Concatenation Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"25th Jan",time:"09-12",topic:"Fast Phylogenetic Approaches",type:"lecture",location:"Town Theater",presenters:["Xiaofan Zhou"]},{date:"25th Jan",time:"14-17",topic:"Maximum Likelihood Lab",type:"practical",location:"Municipal Theater",presenters:["Xiaofan Zhou"]},{date:"25th Jan",time:"19-22",topic:"Maximum Likelihood Tools Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"26th Jan",time:"09-12",topic:"Bayesian Phylogenetic Inference",type:"lecture",location:"Town Theater",presenters:["Tracy Heath"]},{date:"26th Jan",time:"14-17",topic:"Bayesian Phylogenetic Inference Lab",type:"practical",location:"Municipal Theater",presenters:["Tracy Heath"]},{date:"26th Jan",time:"19-22",topic:"Divergence Time Estimation",type:"lecture",location:"Town Theater",presenters:["Tracy Heath"]},{date:"27th Jan",time:"09-12",topic:"Hypothesis Testing in Phylogenetics",type:"lecture",location:"Town Theater",presenters:["David Swofford"]},{date:"27th Jan",time:"14-17",topic:"Model Selection Lab",type:"practical",location:"Municipal Theater",presenters:["David Swofford"]}]},{week:2,sessions:[{date:"30th Jan",time:"09-12",topic:"Detecting Selection",type:"lecture",location:"Town Theater",presenters:["Joe Bielawski"]},{date:"30th Jan",time:"14-17",topic:"Detecting Selection Lab",type:"practical",location:"Municipal Theater",presenters:["Joe Bielawski"]},{date:"30th Jan",time:"19-22",topic:"Techniques for Generating Phylogenomic Data Matrices",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"31st Jan",time:"09-12",topic:"Species Tree Estimation",type:"lecture",location:"Town Theater",presenters:["Laura Kubatko"]},{date:"31st Jan",time:"14-17",topic:"SVDquartets Lab",type:"practical",location:"Municipal Theater",presenters:["Laura Kubatko"]},{date:"31st Jan",time:"19-22",topic:"Tree Editing and Visualization / Open Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"1st Feb",time:"09-12",topic:"Analysis of Gene Gain and Loss",type:"lecture",location:"Town Theater",presenters:["Matthew Hahn"]},{date:"1st Feb",time:"14-17",topic:"CAFE Lab",type:"practical",location:"Municipal Theater",presenters:["Matthew Hahn"]},{date:"1st Feb",time:"19-22",topic:"CAFE Lab cont. / Open Lab",type:"practical",location:"Municipal Theater",presenters:["Matthew Hahn"]},{date:"2nd Feb",time:"09-12",topic:"Non-Vertical Evolution and Phylogenetic Networks",type:"lecture",location:"Town Theater",presenters:["Cline Scornavacca"]},{date:"2nd Feb",time:"14-17",topic:"Non-Vertical Evolution Lab",type:"practical",location:"Municipal Theater",presenters:["Cline Scornavacca"]},{date:"2nd Feb",time:"19-22",topic:"Open Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Instructors"]},{date:"3rd Feb",time:"09-12",topic:"LBA & Animal Nightmares / Open Lab",type:"lecture",location:"Town Theater",presenters:["Jordi Paps Montserrat"]},{date:"3rd Feb",time:"14-17",topic:"Downloading data / Final Open Lab",type:"practical",location:"Municipal Theater",presenters:["Jordi Paps Montserrat"]}]}],nf={workshop:Xg,year:Yg,location:ef,dates:tf,weeks:of},af="WPhylo",rf=2019,sf="esk Krumlov, Czech Republic",lf="January 20-February 1, 2019",cf=[{week:1,sessions:[{date:"21st Jan",time:"09-12",topic:"Introduction and Orientation",type:"orientation",location:"Town Theater",presenters:["Antonis Rokas"]},{date:"21st Jan",time:"14-17",topic:"Lab Introduction",type:"practical",location:"Municipal Theater",presenters:["Eric","Anna"]},{date:"22nd Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",presenters:["Antonis Rokas"]},{date:"22nd Jan",time:"14-17",topic:"Alignment and Alignment Trimming",type:"practical",location:"Municipal Theater",presenters:["Lisa","Marina"]},{date:"22nd Jan",time:"19-22",topic:"Tree Visualization & Tree Challenge",type:"practical",location:"Municipal Theater",presenters:["Lisa","Marina"]},{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenetics, Orthology and Paralogy",type:"lecture",location:"Town Theater",presenters:["Toni Gabaldn"]},{date:"23rd Jan",time:"14-17",topic:"Orthology and Paralogy Prediction Lab",type:"practical",location:"Municipal Theater",presenters:["Toni Gabaldn","Marina Marcet-Houben"]},{date:"23rd Jan",time:"19-22",topic:"Partitioning and Concatenation Laboratory",type:"practical",location:"Municipal Theater",presenters:["Jacob"]},{date:"24th Jan",time:"09-12",topic:"Introduction to Methods and Software for Phylogenomics",type:"lecture",location:"Town Theater",presenters:["Xiaofan Zhou"]},{date:"24th Jan",time:"14-17",topic:"RAxML-NG Introduction and Laboratory",type:"practical",location:"Municipal Theater",presenters:["Alexey Kozlov"]},{date:"24th Jan",time:"19-22",topic:"IQ-tree Introduction and Tutorial",type:"practical",location:"Municipal Theater",presenters:["Stephen Crotty"]},{date:"25th Jan",time:"09-12",topic:"Non-Vertical Evolution & Phylogenetic Networks",type:"lecture",location:"Town Theater",presenters:["Celine Scornavacca"]},{date:"25th Jan",time:"14-17",topic:"Networks Lab Tutorial",type:"practical",location:"Municipal Theater",presenters:["Celine Scornavacca"]},{date:"25th Jan",time:"19-22",topic:"Phylobayes Intro",type:"practical",location:"Municipal Theater",presenters:["Xiaofan Zhou"]},{date:"26th Jan",time:"09-12",topic:"Coalescent Methods",type:"lecture",location:"Town Theater",presenters:["Laura Kubatko"]},{date:"26th Jan",time:"14-17",topic:"Coalescent Methods Lab",type:"practical",location:"Municipal Theater",presenters:["Laura Kubatko"]}]},{week:2,sessions:[{date:"28th Jan",time:"09-12",topic:"Bayesian Phylogenetic Analysis",type:"lecture",location:"Town Theater",presenters:["Mario dos Reis"]},{date:"28th Jan",time:"14-17",topic:"Molecular Clocks and Dating Analysis",type:"lecture",location:"Town Theater",presenters:["Tracy Heath"]},{date:"28th Jan",time:"19-22",topic:"Bayesian MCMC Tutorial",type:"practical",location:"Municipal Theater",presenters:["Tracy Heath","Mario dos Reis"]},{date:"29th Jan",time:"09-12",topic:"Trait evolution and ancestral state reconstruction",type:"lecture",location:"Town Theater",presenters:["Laszlo Garamszegi"]},{date:"29th Jan",time:"14-17",topic:"Trait evolution and ancestral state reconstruction lab",type:"practical",location:"Municipal Theater",presenters:["Laszlo Garamszegi"]},{date:"29th Jan",time:"19-22",topic:"Internode certainty and concordance factors tutorial",type:"practical",location:"Municipal Theater",presenters:["Jacob","Tracy Heath","Marina"]},{date:"30th Jan",time:"09-12",topic:"Evolutionary Genomics",type:"lecture",location:"Town Theater",presenters:["Jason Stajich"]},{date:"30th Jan",time:"14-17",topic:"Evolutionary Genomics Practical",type:"practical",location:"Municipal Theater",presenters:["Jason Stajich"]},{date:"30th Jan",time:"19-22",topic:"Discussion on phylogenomic pipelines",type:"discussion",location:"Municipal Theater",presenters:["Marina","Ania"]},{date:"31st Jan",time:"09-12",topic:"Reconstructing Ancestral Genomes to Understand Major Transitions",type:"lecture",location:"Town Theater",presenters:["Jordi Paps Montserrat"]},{date:"31st Jan",time:"14-17",topic:"Phylogenomics presentations",type:"presentation",location:"Town Theater",presenters:["Martin Kolisko","Anna Karnkowska"]},{date:"31st Jan",time:"19-22",topic:"Open Lab",type:"practical",location:"Municipal Theater",presenters:["Workshop Team"]},{date:"1st Feb",time:"09-12",topic:"Horizontal Gene Transfer",type:"lecture",location:"Town Theater",presenters:["Gergely Szllsi"]},{date:"1st Feb",time:"14-17",topic:"Horizontal Gene Transfer Lab",type:"practical",location:"Municipal Theater",presenters:["Gergely Szllsi"]}]}],df={workshop:af,year:rf,location:sf,dates:lf,weeks:cf},pf="WPhylo",mf=2024,uf="esk Krumlov, Czech Republic",hf="January 21-February 2, 2024",yf=[{week:1,sessions:[{date:"22nd Jan",time:"09-12",topic:"Introduction & Orientation",type:"orientation",location:"Town Theater",presenters:["Anna Karnkowska"]},{date:"22nd Jan",time:"14-17",topic:"Lab introduction & Unix",type:"practical",location:"Municipal Theater",presenters:["Gemma Martnez-Redondo","Karin Steffen"]},{date:"23rd Jan",time:"09-12",topic:"Introduction to Phylogenomics",type:"lecture",location:"Town Theater",presenters:["Rosa Fernndez"]},{date:"23rd Jan",time:"14-17",topic:"Alignment and Multiple Sequence Alignment Trimming",type:"practical",location:"Municipal Theater",presenters:["Marina Marcet-Houben","Jacob L Steenwyk"]},{date:"23rd Jan",time:"19-22",topic:"Tree Visualization & Tree Challenge",type:"practical",location:"Municipal Theater",presenters:["Micha Karlicki","Eduard Ocaa-Pallars"]},{date:"24th Jan",time:"09-12",topic:"Introduction to Phylogenetics, Orthology and Paralogy",type:"lecture",location:"Town Theater",presenters:["Marina Marcet-Houben"]},{date:"24th Jan",time:"14-17",topic:"Orthology and Paralogy Prediction lab",type:"practical",location:"Municipal Theater",presenters:["Marina Marcet-Houben","Jacob L Steenwyk"]},{date:"24th Jan",time:"19-22",topic:"Partitioning and Concatenation Laboratory",type:"practical",location:"Municipal Theater",presenters:["Jacob L Steenwyk","Karin Steffen"]},{date:"25th Jan",time:"09-12",topic:"State-of-the-art methods and software in phylogenomic inference",type:"lecture",location:"Town Theater",presenters:["Olivier Gascuel"]},{date:"25th Jan",time:"14-17",topic:"Coalescent methods in phylogenomic inference lab",type:"practical",location:"Municipal Theater",presenters:["Erin Molloy"]},{date:"25th Jan",time:"19-22",topic:"RAxML software and phylogenomic analyses",type:"practical",location:"Municipal Theater",presenters:["Oleksiy Kozlov"]},{date:"26th Jan",time:"09-12",topic:"Trait evolution on trees, phenotypic and species diversification",type:"lecture",location:"Town Theater",presenters:["Hlne Morlon"]},{date:"26th Jan",time:"14-17",topic:"Coalescent methods lab day 2",type:"practical",location:"Municipal Theater",presenters:["Erin Molloy"]},{date:"26th Jan",time:"19-22",topic:"Evolution and genomic basis of shared and specialized traits in mammals",type:"lecture",location:"Town Theater",presenters:["Matthew Christmas"]},{date:"27th Jan",time:"09-12",topic:"Phylogenetic inference in Whole Genome Duplication context",type:"lecture",location:"Town Theater",presenters:["Aoife McLysaght"]},{date:"27th Jan",time:"14-17",topic:"Green computing",type:"practical",location:"Municipal Theater",presenters:["Oleksyi Kozlov"]}]},{week:2,sessions:[{date:"29th Jan",time:"09-12",topic:"Incongruence",type:"lecture",location:"Town Theater",presenters:["Antonis Rokas"]},{date:"29th Jan",time:"14-17",topic:"Incongruence detection lab",type:"practical",location:"Municipal Theater",presenters:["Jacob Steenwyk"]},{date:"29th Jan",time:"19-22",topic:"Using genome-level processes to solve tricky nodes in phylogenetics",type:"lecture",location:"Town Theater",presenters:["Jordi Paps Montserrat"]},{date:"30th Jan",time:"09-12",topic:"Gene family evolution",type:"lecture",location:"Town Theater",presenters:["Toni Gabaldon"]},{date:"30th Jan",time:"14-17",topic:"Reticulate evolution, temporal resolution and custom data visualisations",type:"lecture",location:"Town Theater",presenters:["Gytis Dudas"]},{date:"30th Jan",time:"19-22",topic:"Open Lab: Phylobayes and BEAST",type:"practical",location:"Municipal Theater",presenters:["Eduard Ocaa-Pallars","Gytis Dudas"]},{date:"31st Jan",time:"09-12",topic:"Target Capture Sequencing Approaches in Phylogenomics",type:"lecture",location:"Town Theater",presenters:["Lisa Pokorny"]},{date:"31st Jan",time:"14-17",topic:"Joint lecture",type:"lecture",location:"Town Theater",presenters:["Antonis Rokas","Toni Gabaldon"]},{date:"31st Jan",time:"19-22",topic:"Discussion on phylogenomic pipelines & tree thinking",type:"discussion",location:"Municipal Theater",presenters:["Workshop Team"]},{date:"1st Feb",time:"09-12",topic:"Horizontal Gene Transfer",type:"lecture",location:"Town Theater",presenters:["Gergely Szllsi"]},{date:"1st Feb",time:"14-17",topic:"Horizontal Gene Transfer Lab",type:"practical",location:"Municipal Theater",presenters:["Gergely Szllsi"]},{date:"1st Feb",time:"19-22",topic:"Open Lab: Workshop summary & software installation",type:"practical",location:"Municipal Theater",presenters:["Workshop Team"]},{date:"2nd Feb",time:"09-12",topic:"Unravelling complexity: Exploring the evolution of microbial eukaryotes",type:"lecture",location:"Town Theater",presenters:["Anna Karnkowska"]}]}],gf={workshop:pf,year:mf,location:uf,dates:hf,weeks:yf};function bi(){return Oh}function ff(){return Fh}function as(){return{wog:[Qh,oy,cy,yy,Ty,Cy,Ey,Hy,Fy,Qy,og,cg,yg,Tg],wpsg:[Cg,Eg,Hg,Fg,Qg],wphylo:[nf,df,gf]}}function jn(){const e=ff(),t=bi(),i=[];return Object.entries(e).forEach(([o,n])=>{var a;(a=n.teaching)!=null&&a.workshopsHistory&&Object.entries(n.teaching.workshopsHistory).forEach(([s,l])=>{const c=Dh(s);Object.entries(l).forEach(([p,y])=>{y.forEach((u,g)=>{const b=t[c];b&&i.push({id:`${o}-${c}-${p}-${g}`,workshopId:c,workshopName:b.name,year:parseInt(p),date:u.date,time:u.time,topic:u.topic,type:u.type,location:u.location,presenters:[o],coPresenters:u.coPresenters})})})})}),i}function vf(){const e=bi(),t=jn(),i=new Set,o=new Set,n={};return Object.keys(e).forEach(s=>{n[s]={sessions:0,years:new Set,presenters:new Set,topics:new Set}}),t.forEach(s=>{i.add(s.year),o.add(s.presenters[0]);const l=n[s.workshopId];l&&(l.sessions++,l.years.add(s.year),l.presenters.add(s.presenters[0]),l.topics.add(s.topic))}),Object.keys(n).forEach(s=>{const l=n[s];l.years=Array.from(l.years).sort((c,p)=>c-p),l.presenters=l.presenters.size,l.topics=Array.from(l.topics)}),{totalWorkshops:Object.keys(n).filter(s=>n[s].sessions>0).length,totalSessions:t.length,totalPresenters:o.size,totalYears:i.size,yearRange:{start:Math.min(...i),end:Math.max(...i)},workshopStats:n}}function bf(){const e=jn(),t=new Map;return e.forEach(i=>{t.has(i.year)||t.set(i.year,{year:i.year,workshops:{}});const o=t.get(i.year);o.workshops[i.workshopId]||(o.workshops[i.workshopId]={sessions:0,topics:[],presenters:[]});const n=o.workshops[i.workshopId];n.sessions++,n.topics.includes(i.topic)||n.topics.push(i.topic),n.presenters.includes(i.presenters[0])||n.presenters.push(i.presenters[0])}),Array.from(t.values()).sort((i,o)=>i.year-o.year)}function wf(e,t,i={}){return e.filter(o=>{if(t){const n=t.toLowerCase();if(!(o.topic.toLowerCase().includes(n)||o.presenters.some(s=>s.toLowerCase().includes(n))||o.workshopName.toLowerCase().includes(n)))return!1}return!(i.workshop&&i.workshop.length>0&&!i.workshop.includes(o.workshopId)||i.year&&o.year!==i.year||i.sessionType&&i.sessionType.length>0&&!i.sessionType.includes(o.type)||i.presenter&&i.presenter.length>0&&!i.presenter.some(a=>o.presenters.includes(a)))})}function Tf(e){const t=new Set,i=new Set,o=new Set,n=new Set,a=new Set;return e.forEach(s=>{t.add(s.workshopId),i.add(s.year),o.add(s.type),s.presenters.forEach(l=>n.add(l)),a.add(s.topic)}),{workshops:Array.from(t).sort(),years:Array.from(i).sort((s,l)=>l-s),sessionTypes:Array.from(o).sort(),presenters:Array.from(n).sort(),topics:Array.from(a).sort()}}const Sf=()=>{const e=T.useMemo(()=>vf(),[]),t=T.useMemo(()=>bi(),[]);return r.jsxs("div",{className:"space-y-8",children:[r.jsxs("div",{className:"text-center",children:[r.jsx("h1",{className:"text-4xl font-bold text-gray-900 mb-4",children:"Workshop Curriculum Archive"}),r.jsxs("p",{className:"text-xl text-gray-600 max-w-3xl mx-auto",children:["Explore ",e.totalYears," years of genomics education excellence. Browse ",e.totalSessions," teaching sessions from ",e.totalPresenters," faculty across ",e.totalWorkshops," workshop series."]})]}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-6",children:[r.jsxs("div",{className:"bg-gradient-to-br from-primary-500 to-primary-600 rounded-xl shadow-lg p-6 text-center text-white transform hover:scale-105 transition-transform",children:[r.jsx("div",{className:"text-3xl font-bold mb-2",children:e.totalSessions}),r.jsx("div",{className:"font-medium",children:"Teaching Sessions"}),r.jsx("div",{className:"text-primary-100 text-sm mt-1",children:"Archived & Searchable"})]}),r.jsxs("div",{className:"bg-gradient-to-br from-green-500 to-green-600 rounded-xl shadow-lg p-6 text-center text-white transform hover:scale-105 transition-transform",children:[r.jsx("div",{className:"text-3xl font-bold mb-2",children:e.totalPresenters}),r.jsx("div",{className:"font-medium",children:"Faculty Presenters"}),r.jsx("div",{className:"text-green-100 text-sm mt-1",children:"Contributing Experts"})]}),r.jsxs("div",{className:"bg-gradient-to-br from-secondary-500 to-secondary-600 rounded-xl shadow-lg p-6 text-center text-white transform hover:scale-105 transition-transform",children:[r.jsx("div",{className:"text-3xl font-bold mb-2",children:e.totalWorkshops}),r.jsx("div",{className:"font-medium",children:"Workshop Series"}),r.jsx("div",{className:"text-secondary-100 text-sm mt-1",children:"Active & Historical"})]}),r.jsxs("div",{className:"bg-gradient-to-br from-orange-500 to-orange-600 rounded-xl shadow-lg p-6 text-center text-white transform hover:scale-105 transition-transform",children:[r.jsx("div",{className:"text-3xl font-bold mb-2",children:e.yearRange.end-e.yearRange.start+1}),r.jsx("div",{className:"font-medium",children:"Years Covered"}),r.jsxs("div",{className:"text-orange-100 text-sm mt-1",children:[e.yearRange.start,"",e.yearRange.end]})]})]}),r.jsxs("div",{className:"bg-white/80 backdrop-blur rounded-xl shadow-lg p-6 border border-white/20",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Workshop Series"}),r.jsx("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6",children:Object.entries(t).map(([i,o])=>{const n=e.workshopStats[i];return!n||n.sessions===0?null:r.jsxs("div",{className:"border border-gray-200 rounded-lg p-4 hover:shadow-md transition-shadow",children:[r.jsxs("div",{className:"flex items-center justify-between mb-3",children:[r.jsx("h3",{className:"font-bold text-gray-900",children:o.shortName}),r.jsx("span",{className:`px-2 py-1 rounded-full text-xs font-medium ${o.active?"bg-green-100 text-green-800":"bg-gray-100 text-gray-600"}`,children:o.active?"Active":"Historical"})]}),r.jsx("p",{className:"text-gray-600 text-sm mb-4 line-clamp-2",children:o.description}),n&&r.jsxs("div",{className:"space-y-2 text-sm",children:[r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{className:"text-gray-500",children:"Sessions:"}),r.jsx("span",{className:"font-medium",children:n.sessions})]}),r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{className:"text-gray-500",children:"Presenters:"}),r.jsx("span",{className:"font-medium",children:n.presenters})]}),r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{className:"text-gray-500",children:"Years:"}),r.jsx("span",{className:"font-medium",children:n.years.length>0?`${Math.min(...n.years)}${Math.max(...n.years)}`:"No data"})]})]})]},i)})})]}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6",children:[r.jsxs(Z,{to:"/sessions",className:"bg-gradient-to-br from-primary-50 to-primary-100 border border-primary-200 rounded-xl p-6 hover:from-primary-100 hover:to-primary-200 transition-all transform hover:scale-105 shadow-lg group",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-primary-500 rounded-lg flex items-center justify-center mr-4 shadow-md",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-xl font-bold text-primary-900",children:"Browse Sessions"})]}),r.jsxs("p",{className:"text-primary-700",children:["Search and filter through all ",e.totalSessions," teaching sessions with advanced filters."]})]}),r.jsxs(Z,{to:"/timeline",className:"bg-gradient-to-br from-green-50 to-green-100 border border-green-200 rounded-xl p-6 hover:from-green-100 hover:to-green-200 transition-all transform hover:scale-105 shadow-lg group",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-green-500 rounded-lg flex items-center justify-center mr-4 shadow-md",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-xl font-bold text-green-900",children:"Timeline View"})]}),r.jsxs("p",{className:"text-green-700",children:["Explore curriculum evolution and teaching contributions over ",e.totalYears," years."]})]}),r.jsxs("a",{href:"https://shandley.github.io/evomics-faculty/",target:"_blank",rel:"noopener noreferrer",className:"bg-gradient-to-br from-secondary-50 to-secondary-100 border border-secondary-200 rounded-xl p-6 hover:from-secondary-100 hover:to-secondary-200 transition-all transform hover:scale-105 shadow-lg group",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-secondary-500 rounded-lg flex items-center justify-center mr-4 shadow-md",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-xl font-bold text-secondary-900",children:"Faculty Profiles"})]}),r.jsx("p",{className:"text-secondary-700",children:"View detailed faculty profiles with enriched information and teaching histories."})]})]}),r.jsxs("div",{className:"bg-white/80 backdrop-blur rounded-xl shadow-lg p-6 border border-white/20",children:[r.jsxs("div",{className:"flex items-center justify-between mb-6",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900",children:"Archive Highlights"}),r.jsx(Z,{to:"/sessions",className:"text-blue-600 hover:text-blue-800 font-medium text-sm",children:"View All Sessions "})]}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 gap-6",children:[r.jsxs("div",{className:"border-l-4 border-blue-500 bg-blue-50 p-4 rounded-r-lg",children:[r.jsx("h4",{className:"font-semibold text-blue-900 mb-2",children:"Most Recent Workshop"}),r.jsx("p",{className:"text-blue-800",children:"Workshop on Population and Speciation Genomics 2025"}),r.jsx("p",{className:"text-blue-700 text-sm mt-1",children:"Latest curriculum content available"})]}),r.jsxs("div",{className:"border-l-4 border-green-500 bg-green-50 p-4 rounded-r-lg",children:[r.jsx("h4",{className:"font-semibold text-green-900 mb-2",children:"Archive Scope"}),r.jsxs("p",{className:"text-green-800",children:[e.yearRange.start,"",e.yearRange.end," Coverage"]}),r.jsx("p",{className:"text-green-700 text-sm mt-1",children:"Complete historical documentation"})]})]})]})]})},Pf=()=>{const e=T.useMemo(()=>bf(),[]),t=T.useMemo(()=>bi(),[]),i=n=>({wog:"bg-blue-500",wpsg:"bg-green-500",wphylo:"bg-purple-500",wmolevo:"bg-orange-500",htranscriptomics:"bg-pink-500",hmicrobial:"bg-indigo-500"})[n]||"bg-gray-500",o=n=>({wog:"bg-blue-100 border-blue-300",wpsg:"bg-green-100 border-green-300",wphylo:"bg-purple-100 border-purple-300",wmolevo:"bg-orange-100 border-orange-300",htranscriptomics:"bg-pink-100 border-pink-300",hmicrobial:"bg-indigo-100 border-indigo-300"})[n]||"bg-gray-100 border-gray-300";return r.jsxs("div",{className:"space-y-8",children:[r.jsxs("div",{className:"text-center",children:[r.jsx("h1",{className:"text-3xl font-bold text-gray-900 mb-4",children:"Workshop Timeline"}),r.jsxs("p",{className:"text-xl text-gray-600 max-w-3xl mx-auto",children:["Explore the evolution of workshop curricula across ",e.length," years of genomics education."]})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow p-6",children:[r.jsx("h2",{className:"text-lg font-semibold text-gray-900 mb-4",children:"Workshop Series"}),r.jsx("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4",children:Object.entries(t).map(([n,a])=>r.jsxs("div",{className:"flex items-center space-x-3",children:[r.jsx("div",{className:`w-4 h-4 rounded-full ${i(n)}`}),r.jsxs("div",{children:[r.jsx("span",{className:"font-medium text-gray-900",children:a.shortName}),r.jsx("span",{className:`ml-2 px-2 py-1 rounded-full text-xs ${a.active?"bg-green-100 text-green-800":"bg-gray-100 text-gray-600"}`,children:a.active?"Active":"Historical"})]})]},n))})]}),r.jsxs("div",{className:"relative",children:[r.jsx("div",{className:"absolute left-8 top-0 bottom-0 w-0.5 bg-gray-300"}),r.jsx("div",{className:"space-y-8",children:e.map((n,a)=>r.jsxs("div",{className:"relative flex items-start",children:[r.jsx("div",{className:"flex-shrink-0 w-16 h-16 bg-white border-4 border-gray-300 rounded-full flex items-center justify-center shadow-lg z-10",children:r.jsx("span",{className:"text-sm font-bold text-gray-700",children:n.year})}),r.jsx("div",{className:"ml-6 flex-1",children:r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-6",children:[r.jsxs("h3",{className:"text-xl font-bold text-gray-900 mb-4",children:[n.year," Workshop Activities"]}),r.jsx("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4",children:Object.entries(n.workshops).map(([s,l])=>{var c;return r.jsxs("div",{className:`border-2 rounded-lg p-4 ${o(s)}`,children:[r.jsxs("div",{className:"flex items-center mb-3",children:[r.jsx("div",{className:`w-3 h-3 rounded-full ${i(s)} mr-2`}),r.jsx("h4",{className:"font-semibold text-gray-900",children:((c=t[s])==null?void 0:c.shortName)||s})]}),r.jsxs("div",{className:"space-y-2 text-sm text-gray-700",children:[r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{children:"Sessions:"}),r.jsx("span",{className:"font-medium",children:l.sessions})]}),r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{children:"Presenters:"}),r.jsx("span",{className:"font-medium",children:l.presenters.length})]}),r.jsxs("div",{className:"flex justify-between",children:[r.jsx("span",{children:"Topics:"}),r.jsx("span",{className:"font-medium",children:l.topics.length})]})]}),l.topics.length>0&&r.jsxs("div",{className:"mt-3",children:[r.jsx("h5",{className:"text-xs font-medium text-gray-600 mb-1",children:"Key Topics:"}),r.jsxs("div",{className:"space-y-1",children:[l.topics.slice(0,3).map((p,y)=>r.jsxs("div",{className:"text-xs text-gray-600 truncate",children:[" ",p]},y)),l.topics.length>3&&r.jsxs("div",{className:"text-xs text-gray-500",children:["+",l.topics.length-3," more"]})]})]})]},s)})}),r.jsx("div",{className:"mt-4 pt-4 border-t border-gray-200",children:r.jsxs("div",{className:"grid grid-cols-2 md:grid-cols-4 gap-4 text-center text-sm",children:[r.jsxs("div",{children:[r.jsx("div",{className:"font-semibold text-gray-900",children:Object.values(n.workshops).reduce((s,l)=>s+l.sessions,0)}),r.jsx("div",{className:"text-gray-600",children:"Total Sessions"})]}),r.jsxs("div",{children:[r.jsx("div",{className:"font-semibold text-gray-900",children:Object.keys(n.workshops).length}),r.jsx("div",{className:"text-gray-600",children:"Active Workshops"})]}),r.jsxs("div",{children:[r.jsx("div",{className:"font-semibold text-gray-900",children:new Set(Object.values(n.workshops).flatMap(s=>s.presenters)).size}),r.jsx("div",{className:"text-gray-600",children:"Unique Presenters"})]}),r.jsxs("div",{children:[r.jsx("div",{className:"font-semibold text-gray-900",children:new Set(Object.values(n.workshops).flatMap(s=>s.topics)).size}),r.jsx("div",{className:"text-gray-600",children:"Topics Covered"})]})]})})]})})]},n.year))})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-6",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Timeline Summary"}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-6",children:[r.jsxs("div",{className:"text-center",children:[r.jsx("div",{className:"text-3xl font-bold text-blue-600 mb-2",children:e.length}),r.jsx("div",{className:"text-gray-600 font-medium",children:"Years of Data"})]}),r.jsxs("div",{className:"text-center",children:[r.jsx("div",{className:"text-3xl font-bold text-green-600 mb-2",children:e.reduce((n,a)=>n+Object.values(a.workshops).reduce((s,l)=>s+l.sessions,0),0)}),r.jsx("div",{className:"text-gray-600 font-medium",children:"Total Sessions"})]}),r.jsxs("div",{className:"text-center",children:[r.jsx("div",{className:"text-3xl font-bold text-purple-600 mb-2",children:new Set(e.flatMap(n=>Object.values(n.workshops).flatMap(a=>a.presenters))).size}),r.jsx("div",{className:"text-gray-600 font-medium",children:"Faculty Contributors"})]}),r.jsxs("div",{className:"text-center",children:[r.jsx("div",{className:"text-3xl font-bold text-orange-600 mb-2",children:Object.keys(t).length}),r.jsx("div",{className:"text-gray-600 font-medium",children:"Workshop Series"})]})]})]})]})},kl={"ahren-dag":{id:"ahren-dag",name:"Dag Ahrn",enrichment:{lastUpdated:"2025-07-01T13:00:00.000Z",confidence:"high",professional:{title:"Senior lecturer, Bioinformatician",affiliation:"Lund University",department:"Department of Biology / Molecular Biosciences"},academic:{orcid:"0000-0003-4713-0032",researchAreas:{raw:["bioinformatics","computational biology","genomics","NGS","evolution","metagenomics","fungal host interactions","nematode-trapping fungi"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]}],secondary:[{id:"fungal-evolution",label:"Fungal Evolution",level:3,parentId:"microbial-evolution",description:"Evolution of fungal species",synonyms:["mycological evolution","fungal phylogeny"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Dag Ahrn studies genetic and genomic mechanisms related to fungal host interactions using genomics and transcriptomics approaches coupled with bioinformatics analyses, particularly focused on nematode-trapping fungi.",source:"Lund University Research Portal"}}},"albrechtsen-anders":{id:"albrechtsen-anders",name:"Anders Albrechtsen",enrichment:{lastUpdated:"2025-07-01T14:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of Copenhagen",department:"Department of Biology, Computational and RNA Biology"},academic:{orcid:"0000-0001-7306-031X",researchAreas:{raw:["statistical genetics","computational methods","genomic data analysis","population genetics","association studies","population stratification","natural selection"],standardized:{primary:[{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"genomic-data-analysis",label:"Genomic Data Analysis",level:3,parentId:"data-analysis",description:"Analysis of genome-scale datasets",synonyms:["genome data processing","genomic bioinformatics"]},{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Anders Albrechtsen is an Associate Professor at the Bioinformatics Centre, University of Copenhagen. His group develops statistical and computational methods for analysis of genomic data including multi-loci association studies and detecting natural selection.",source:"University of Copenhagen Research Portal"}}},"almeida-king-jeff":{id:"almeida-king-jeff",name:"Jeff Almeida-King",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"medium",professional:{title:"Scientist",affiliation:"Genedata",department:"Bioinformatics"},academic:{researchAreas:{raw:["genomics","bioinformatics","MHC haplotypes","BioMart","Ensembl Genomes","data warehousing"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Jeff Almeida-King is a commercially-minded scientist at Genedata. He contributed to major genomics projects including the MHC Haplotype Project and BioMart/Ensembl Genomes development. His work includes gene annotation and developing data access tools for genomic databases.",source:"Genedata, EMBL-EBI"}}},"bank-claudia":{id:"bank-claudia",name:"Claudia Bank",enrichment:{lastUpdated:"2025-07-02T19:27:49.709Z",confidence:"high",professional:{title:"Group Leader",affiliation:"SIB Swiss Institute of Bioinformatics",department:"Theoretical Ecology and Evolution"},academic:{researchAreas:{raw:["theoretical ecology","evolution","population genetics","evolutionary forces","genomics","diversity"],standardized:{primary:[{id:"biodiversity",label:"Biodiversity",level:2,parentId:"ecology-environmental",description:"Study of biological diversity at all levels",synonyms:["biological diversity","species diversity"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4730-758X"},profile:{shortBio:"Claudia Bank leads the Theoretical Ecology and Evolution group at SIB. Her overarching goal is to quantify the relative roles of ecological and evolutionary forces for the buildup and maintenance of diversity.",source:"SIB website, Evolution and Genomics website"}}},"barnett-david":{id:"barnett-david",name:"David Barnett",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Research Bioinformatician",affiliation:"Maastricht University",department:"Maastricht Centre for Systems Biology (MaCSBio)"},academic:{researchAreas:{raw:["microbiome analysis","molecular epidemiology","R statistical programming","gut microbiota","child health","data visualization"],standardized:{primary:[{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[{id:"molecular-epidemiology",label:"Molecular Epidemiology",level:3,parentId:"medical-clinical",description:"Use of molecular biology in epidemiological studies",synonyms:["genetic epidemiology","genomic epidemiology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-1961-7206"},profile:{shortBio:"David Barnett is a Research Bioinformatician at Maastricht University working on infant/child gut microbiome projects. He developed the R package microViz for microbiome data analysis and visualization. He teaches microbiome analysis methods at genomics workshops.",source:"Maastricht University, Workshop on Genomics"}}},"barth-julia":{id:"barth-julia",name:"Julia M.I. Barth",enrichment:{lastUpdated:"2025-07-01T14:00:00.000Z",confidence:"high",professional:{title:"Guest Researcher",affiliation:"University of Oslo / University of Basel",department:"CEES - Centre for Ecological and Evolutionary Synthesis"},academic:{researchAreas:{raw:["population genomics","speciation genomics","evolutionary biology","marine fish genomics","Atlantic cod","conservation genetics","adaptation"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5125-819X"},profile:{shortBio:"Julia M.I. Barth studies genomic differentiation and demographic histories of Atlantic and Indo-Pacific yellowfin tuna populations. She is involved in various population genomic projects including the Aqua Genome project, studying connectivity, gene flow, and adaptation in marine fish species.",source:"University of Oslo CEES / Evolution and Genomics website"}}},"beiko-robert":{id:"beiko-robert",name:"Robert Beiko",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor and Associate Dean",affiliation:"Dalhousie University",department:"Faculty of Computer Science",labWebsite:"https://beikolab.cs.dal.ca"},academic:{researchAreas:{raw:["bioinformatics","microbial evolution","microbial ecology","horizontal gene transfer","antimicrobial resistance","metagenomics","phylogenetics","machine learning"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbial-evolution",label:"Microbial Evolution",level:2,parentId:"microbiology-microbiome",description:"Evolution of microorganisms",synonyms:["bacterial evolution","prokaryotic evolution"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"horizontal-gene-transfer",label:"Horizontal Gene Transfer",level:3,parentId:"microbial-evolution",description:"Transfer of genetic material between organisms",synonyms:["HGT","lateral gene transfer"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0949-4471"},profile:{shortBio:"Robert Beiko is a Canada Research Chair in bioinformatics at Dalhousie University. His lab researches ways to understand microbial ecosystems, with expertise in amplicon sequencing, metagenomics, phylogenetics, and machine learning approaches.",source:"Dalhousie University, Google Scholar"}}},"berner-daniel":{id:"berner-daniel",name:"Daniel Berner",enrichment:{lastUpdated:"2025-07-03T00:00:00.000Z",confidence:"high",professional:{title:"PD Dr. Lecturer",affiliation:"University of Basel",department:"Department of Environmental Sciences"},academic:{researchAreas:{raw:["population genomics","adaptive divergence","stickleback evolution","speciation","evolutionary ecology","range expansion genetics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5995-6494"},profile:{shortBio:"Daniel Berner is a population genomics researcher at University of Basel studying adaptive divergence in stickleback fish, speciation, and the genetic basis of adaptation and range expansion in butterflies.",source:"University of Basel Department Page"}}},"bielawski-joseph":{id:"bielawski-joseph",name:"Joseph Bielawski",enrichment:{lastUpdated:"2025-07-02T21:45:51.000Z",confidence:"medium",professional:{affiliation:"Dalhousie University",department:"Biology",title:"Professor"},academic:{researchAreas:{raw:["molecular evolution","phylogenetics","selection analysis","computational biology"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1621-1772"},profile:{source:"Inferred from workshop participation"}}},"blanco-miguez-altor":{id:"blanco-miguez-altor",name:"Aitor Blanco Miguez",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Coordinator Microbiome Bioinformatic Core",affiliation:"PreBiomics",department:"Microbiome Bioinformatics",previousAffiliation:"University of Trento"},academic:{researchAreas:{raw:["microbiome","metagenomics","computational biology","bioinformatics","human microbiome","food microbiomes"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-7386-5572"},profile:{shortBio:"Aitor Blanco-Mguez is Coordinator of Microbiome Bioinformatic Core at PreBiomics. Previously at University of Trento, he has contributed to MetaPhlAn 4 and studies on microbiome-health relationships. His research includes developing bioinformatics tools for microbiome analysis and database curation.",source:"PreBiomics, University of Trento"}}},"bohutinska-magdalena":{id:"bohutinska-magdalena",name:"Magdalena Bohutinska",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Charles University in Prague",department:"Department of Botany, Faculty of Science",additionalAffiliation:"University of Bern"},academic:{researchAreas:{raw:["evolutionary genomics","polyploidy","whole genome duplication","speciation","Arabidopsis","repeated adaptation"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"chromosome-evolution",label:"Chromosome Evolution",level:3,parentId:"evolutionary-genomics",description:"Evolution of chromosome structure and number",synonyms:["karyotype evolution","chromosomal evolution"]},{id:"model-organisms",label:"Model Organisms",level:3,parentId:"comparative-genomics",description:"Organisms used as models for biological research",synonyms:["model species","research organisms"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Magdalena Bohutnsk is a researcher at Charles University in Prague studying evolutionary genomics. Her work focuses on polyploidy and whole genome duplication in plant speciation, particularly using Arabidopsis species. She investigates how organisms repeatedly adapt to similar environmental challenges through genomic mechanisms.",source:"Charles University in Prague, Faculty of Science"}}},"bos-kirsten":{id:"bos-kirsten",name:"Kirsten Bos",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Group Leader, Molecular Palaeopathology",affiliation:"Max Planck Institute for Evolutionary Anthropology",department:"Department of Archaeogenetics",labWebsite:"https://kirstenbos.ca"},academic:{researchAreas:{raw:["ancient DNA","infectious diseases","paleopathology","Black Death","tuberculosis","syphilis","pathogen genomics"],standardized:{primary:[{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-2937-3006"},profile:{shortBio:"Kirsten Bos is a Canadian physical anthropologist and Group Leader of Molecular Palaeopathology at the Max Planck Institute for Evolutionary Anthropology. Her research focuses on ancient DNA retrieval to reconstruct ancient pathogen genomes, studying the changing landscape of infectious diseases over time, host-pathogen coevolution, and the biological consequences of European and New World contact.",source:"Max Planck Institute for Evolutionary Anthropology, Google Scholar"}}},"buerkle-alex":{id:"buerkle-alex",name:"Alex Buerkle",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Staff Scientist",affiliation:"Illumina",department:"Bioinformatics for Clinical Genomics"},academic:{researchAreas:{raw:["population genomics","computational biology","evolutionary genetics","statistical genomics","low coverage sequencing","Bayesian methods"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-4222-8858"},profile:{shortBio:"C. Alex Buerkle is a Staff Scientist at Illumina working on bioinformatics for clinical genomics. Formerly Professor at the University of Wyoming (2004-2022), he is a geneticist, data scientist, and evolutionary biologist specializing in evolutionary and ecological genomics.",source:"Illumina, University of Wyoming"}}},"catchen-julian":{id:"catchen-julian",name:"Julian Catchen",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"high",professional:{affiliation:"University of Illinois",labWebsite:"https://catchenlab.life.illinois.edu",title:"Associate Professor",department:"Department of Evolution, Ecology, and Behavior"},academic:{researchAreas:{raw:["population genomics","RAD-seq","genetic mapping","restriction enzyme-based data","stacks software","evolutionary genomics"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]},{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-4798-660X"},profile:{shortBio:"Julian Catchen developed Stacks, an analysis tool set for population genomics. Stacks was developed to work with restriction enzyme-based data such as RAD-seq, for building genetic maps and conducting population genomics analyses.",source:"Catchen Lab website, PubMed"}}},"chevreux-bastien":{id:"chevreux-bastien",name:"Bastien Chevreux",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Data Scientist and Bioinformatics Expert",affiliation:"Independent Consultant",department:"Data - Insights - Products - Strategy",labWebsite:"https://www.chevreux.org"},academic:{researchAreas:{raw:["genome assembly","bioinformatics","sequence assembly","MIRA assembler","computational biology","EST assembly","SNP detection"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"variant-calling",label:"Variant Calling",level:3,parentId:"computational-methods",description:"Identification of genetic variants from sequencing data",synonyms:["SNP calling","variant detection"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Bastien Chevreux is the developer of the MIRA genome assembler, a versatile tool for sequence assembly that has been continuously maintained since 1997. His work includes over 20 biotechnology patents and numerous high-impact publications. MIRA supports multiple sequencing technologies including Sanger, 454, Illumina, IonTorrent, and PacBio, making it a Swiss army knife of sequence assembly.",source:"chevreux.org, MIRA documentation"}}},"chikhi-rayan":{id:"chikhi-rayan",name:"Rayan Chikhi",enrichment:{lastUpdated:"2025-07-02T20:47:51.165369Z",confidence:"high",professional:{title:"G5 Group Leader",affiliation:"Institut Pasteur",department:"Sequence Bioinformatics"},academic:{researchAreas:{raw:["algorithms","big biological data","next-generation sequencing","genome assembly","k-mers","metagenomics","pan-genomics","transcriptomics"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"pangenomics",label:"Pangenomics",level:2,parentId:"genomics-omics",description:"Study of all genes within a species or clade",synonyms:["pan-genome","core genome"],children:[]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-1099-8735"},profile:{shortBio:"Rayan Chikhi leads a computational team researching algorithms for big biological data at Institut Pasteur. The group creates and implements algorithms and data structures into software tools for genomics and metagenomics, including methods for improving bacterial genome assemblies with long reads.",source:"Institut Pasteur Research Portal"}}},"christmas-matthew":{id:"christmas-matthew",name:"Matthew Christmas",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Uppsala University",department:"Department of Medical Biochemistry and Microbiology"},academic:{researchAreas:{raw:["evolutionary genomics","population genomics","bioinformatics","comparative genomics","alpine bumblebees","common quails","placental mammals"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Matthew Christmas is a post-doctoral researcher at Uppsala University working with Kerstin Lindblad-Toh. His research focuses on evolutionary genomics, contributing to major studies on genome evolution in common quails, evolutionary constraint across placental mammals, and population genomics of alpine bumblebees.",source:"Uppsala University, Google Scholar"}}},"cicconardi-francesco":{id:"cicconardi-francesco",name:"Francesco Cicconardi",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Senior Research Associate",affiliation:"University of Bristol",department:"School of Biological Sciences"},academic:{researchAreas:{raw:["evolutionary biology","comparative genomics","bioinformatics","molecular evolution","entomology","alpine adaptation"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]}],secondary:[{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-8395-8057"},profile:{shortBio:"Francesco Cicconardi is a Senior Research Associate at University of Bristol. His research focuses on evolutionary adaptation to extreme environments, particularly studying how ants adapted to alpine habitats through comparative genomics. Previously at University of Cambridge and University of Innsbruck.",source:"University of Bristol, School of Biological Sciences"}}},"clark-andrew":{id:"clark-andrew",name:"Andrew Clark",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Jacob Gould Schurman Professor",affiliation:"Cornell University",department:"Department of Molecular Biology and Genetics",labWebsite:"https://blogs.cornell.edu/andyclarklab"},academic:{researchAreas:{raw:["population genetics","evolutionary genetics","Drosophila genetics","human genetics","cardiovascular disease","innate immunity","genomic imprinting"],standardized:{primary:[{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"model-organisms",label:"Model Organisms",level:3,parentId:"comparative-genomics",description:"Organisms used as models for biological research",synonyms:["model species","research organisms"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-0622-6611"},profile:{shortBio:"Andrew G. Clark is the Jacob Gould Schurman Professor at Cornell University and a member of the National Academy of Sciences. His research focuses on empirical and analytical problems in population genetics, with particular emphasis on Drosophila and human systems. He has published over 360 papers and co-authored the textbook Principles of Population Genetics with Dan Hartl.",source:"Cornell University, National Academy of Sciences"}}},"clauw-pieter":{id:"clauw-pieter",name:"Pieter Clauw",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Gregor Mendel Institute of Molecular Plant Biology",department:"Austrian Academy of Sciences"},academic:{researchAreas:{raw:["plant genomics","Arabidopsis thaliana","cold adaptation","metabolomics","drought stress","natural variation","GWAS"],standardized:{primary:[{id:"metabolomics",label:"Metabolomics",level:2,parentId:"genomics-omics",description:"Large-scale study of metabolites",synonyms:["metabolome analysis","metabolic profiling"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"genetic-variation",label:"Genetic Variation",level:3,parentId:"population-genetics",description:"Differences in DNA sequences among individuals",synonyms:["genetic diversity","allelic variation"]},{id:"model-organisms",label:"Model Organisms",level:3,parentId:"comparative-genomics",description:"Organisms used as models for biological research",synonyms:["model species","research organisms"]},{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Pieter Clauw researches natural variation in plant responses to environmental stresses at the Gregor Mendel Institute. His work focuses on cold temperature adaptation, metabolome plasticity, and drought stress responses in Arabidopsis thaliana, using genomic and metabolomic approaches to understand how plants adapt to environmental challenges.",source:"Gregor Mendel Institute, Google Scholar"}}},"colonna-vincenza":{id:"colonna-vincenza",name:"Vincenza Colonna",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Group Leader",affiliation:"Institute of Genetics and Biophysics, CNR",department:"Italian National Research Council",additionalAffiliation:"University of Tennessee Health Science Center"},academic:{researchAreas:{raw:["human population genomics","genetic variation","consanguineous populations","isolated populations","Southern Italy genetics","natural selection"],standardized:{primary:[{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]}],secondary:[{id:"genetic-variation",label:"Genetic Variation",level:3,parentId:"population-genetics",description:"Differences in DNA sequences among individuals",synonyms:["genetic diversity","allelic variation"]},{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-3966-0474"},profile:{shortBio:"Vincenza Colonna is Group Leader at the Institute of Genetics and Biophysics of the Italian National Research Council in Naples. Her research focuses on human genetic variation, particularly in isolated populations of Southern Italy. She graduated from University of Naples Federico II and did postdoctoral work at Wellcome Trust Sanger Institute.",source:"Institute of Genetics and Biophysics, CNR, Naples"}}},"cresko-bill":{id:"cresko-bill",name:"Bill Cresko",enrichment:{lastUpdated:"2025-07-02T20:47:51.165383Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Oregon",department:"Institute of Ecology and Evolution"},academic:{researchAreas:{raw:["evolutionary genomics","stickleback fish","population genomics","adaptation","speciation"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"model-organisms",label:"Model Organisms",level:3,parentId:"comparative-genomics",description:"Organisms used as models for biological research",synonyms:["model species","research organisms"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-3496-8074"},profile:{shortBio:"Bill Cresko studies evolutionary genomics using threespine stickleback fish as a model system. Co-organizer of the Workshop on Population and Speciation Genomics.",source:"Workshop information"}}},"crotty-stephen":{id:"crotty-stephen",name:"Stephen Crotty",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Professor and Chief Scientific Officer",affiliation:"La Jolla Institute for Immunology",department:"Center for Infectious Disease and Vaccine Research",labWebsite:"https://www.lji.org/labs/crotty-lab"},academic:{researchAreas:{raw:["immunology","virology","T follicular helper cells","vaccine immunology","COVID-19","HIV","germinal centers"],standardized:{primary:[{id:"immunology",label:"Immunology",level:2,parentId:"medical-clinical",description:"Study of immune systems and immune responses",synonyms:["immunogenetics","immune system biology"],children:[]},{id:"viral-genomics",label:"Viral Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of viruses and viral evolution",synonyms:["virus genomics","viral genome analysis"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-3484-4068"},profile:{shortBio:"Shane Crotty is a Professor and Chief Scientific Officer at La Jolla Institute for Immunology. His lab is best known for understanding T follicular helper cells and their role in antibody responses. He made crucial contributions to COVID-19 research, publishing the first detailed analysis of immune responses to SARS-CoV-2. He has been a Highly Cited Researcher for 8 consecutive years.",source:"La Jolla Institute for Immunology, Wikipedia"}}},"cummings-michael":{id:"cummings-michael",name:"Michael Cummings",enrichment:{lastUpdated:"2025-07-01T14:15:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Maryland",department:"Department of Biology",additionalRoles:["Director, Center for Bioinformatics and Computational Biology, UMIACS","Director, Master of Science Program in Bioinformatics and Computational Biology"]},academic:{researchAreas:{raw:["bioinformatics","computational biology","data analysis","molecular evolution","biomedical data science","machine learning"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]}],secondary:[{id:"data-analysis",label:"Data Analysis",level:3,parentId:"data-science",description:"General biological data analysis methods",synonyms:["biological data analysis","bioinformatics analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Michael Cummings is a Professor of Biology at the University of Maryland. His research involves bioinformatics and computational biology with a particular focus on data analysis that spans a broad range of biomedical problems using machine learning, statistical and other methods.",source:"University of Maryland Department of Biology"}}},"danacek-petr":{id:"danacek-petr",name:"Petr Danacek",enrichment:{lastUpdated:"2025-07-02T23:47:36.078Z",confidence:"high",professional:{title:"Senior Bioinformatician",affiliation:"Wellcome Sanger Institute",department:"Genomics",orcid:"0000-0002-0667-3788"},academic:{researchAreas:{raw:["methods development","data analysis","genomics","BCFtools","SAMtools","HTSlib","large-scale sequencing projects","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"data-analysis",label:"Data Analysis",level:3,parentId:"data-science",description:"General biological data analysis methods",synonyms:["biological data analysis","bioinformatics analysis"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-4159-1666"},profile:{shortBio:"Petr Danacek is a Senior Bioinformatician at the Wellcome Sanger Institute with a background in computer science and PhD in biophysics. He is one of the lead developers of BCFtools/SAMtools/HTSlib and was involved in large-scale sequencing projects such as 1000 Genomes Project, UK10k, or Deciphering Developmental Disorders.",source:"Wellcome Sanger Institute, ResearchGate"}}},"dos-reis-mario":{id:"dos-reis-mario",name:"Mario dos Reis",enrichment:{lastUpdated:"2025-07-03T00:00:00.000Z",confidence:"high",professional:{title:"Dr",affiliation:"Queen Mary University of London",department:"School of Biological and Behavioural Sciences",labWebsite:"https://www.qmul.ac.uk/sbcs/staff/mariodosreisbarros.html"},academic:{researchAreas:{raw:["molecular evolution","phylogenetics","human genetics","genomics","molecular clock","divergence time estimation"],standardized:{primary:[{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-9514-3976"},profile:{shortBio:"Dr Mario Dos Reis Barros teaches Human Genetics and Genomics at the Joint Programme with Nanchang University. His research focuses on molecular evolution, phylogenetics, and using genomic data to understand evolutionary processes and timing.",source:"Queen Mary University of London"}}},"droge-johannes":{id:"droge-johannes",name:"Johannes Droge",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"high",professional:{affiliation:"Chalmers University of Technology",department:"Life Sciences",orcid:"0000-0002-6752-2204",title:"Bioinformatics Specialist"},academic:{researchAreas:{raw:["algorithmic bioinformatics","environmental research","metagenomics","next-generation sequencing","computational biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Johannes Drge specializes in algorithmic bioinformatics with a focus on environmental research and metagenomics using next-generation sequencing data.",source:"Chalmers Research, ORCID, Google Scholar"}}},"drummond-alexei":{id:"drummond-alexei",name:"Alexei Drummond",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor of Computational Biology",affiliation:"University of Auckland",department:"School of Biological Sciences & School of Computer Science",labWebsite:"https://www.cs.auckland.ac.nz/~alexei"},academic:{researchAreas:{raw:["computational biology","statistical phylogenetics","Bayesian inference","phylodynamics","molecular evolution","population genetics","coalescent theory","infectious disease epidemiology"],standardized:{primary:[{id:"bayesian-methods",label:"Bayesian Methods",level:2,parentId:"mathematical-statistical",description:"Bayesian statistical approaches in biology",synonyms:["bayesian statistics","bayesian inference"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"coalescent-theory",label:"Coalescent Theory",level:3,parentId:"population-genetics",description:"Mathematical model of gene genealogies",synonyms:["coalescence","gene genealogy"]},{id:"phylodynamics",label:"Phylodynamics",level:3,parentId:"phylogenetics",description:"Study of interaction between epidemiology and pathogen evolution",synonyms:["viral phylodynamics","epidemiological phylogenetics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4454-2576"},profile:{shortBio:"Alexei Drummond is Director of the Centre for Computational Evolution at University of Auckland. He co-invented BEAST and BEAST 2 software packages and co-founded Geneious. He is a Clarivate Highly Cited Researcher and James Cook Research Fellow.",source:"University of Auckland, Google Scholar"}}},"duchoslav-miroslav":{id:"duchoslav-miroslav",name:"Milos Duchoslav",enrichment:{lastUpdated:"2025-07-01T14:30:00.000Z",confidence:"high",professional:{title:"Postdoc",affiliation:"Charles University",department:"Department of Botany"},academic:{orcid:"0000-0003-0715-3029",researchAreas:{raw:["plant molecular biology","bioinformatics","proteomics","photosystem II","plant ecological genomics","evolution","molecular biology of plants"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Milos Duchoslav is a Postdoc in the Plant ecological genomics group at Charles University in Prague. His specialties include plant molecular biology, bioinformatics, proteomics, and photosystem II research.",source:"Charles University Faculty Page"}}},"dudaniec-rachael":{id:"dudaniec-rachael",name:"Rachael Dudaniec",enrichment:{lastUpdated:"2025-07-02T23:47:36.077Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"Macquarie University",department:"Biological Sciences (Conservation Biology)",labWebsite:"https://dudanieclab.weebly.com"},academic:{researchAreas:{raw:["landscape genetics","evolutionary genetics","conservation genetics","host-parasite ecology","environmental genomics","genetic connectivity"],standardized:{primary:[{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1854-6808"},profile:{shortBio:"A/Prof Rachael Dudaniec examines how environmental conditions affect genetic connectivity and evolutionary processes within species. She links this information with data on pathogen impacts and diversity to understand how species persist and evolve across diverse landscapes.",source:"Macquarie University, Google Scholar"}}},"dudas-gytis":{id:"dudas-gytis",name:"Gytis Dudas",enrichment:{lastUpdated:"2025-07-02T23:47:36.078Z",confidence:"high",professional:{title:"Scientist",affiliation:"Vilnius University Life Sciences Center",department:"Institute of Biotechnology",labWebsite:"https://evogytis.github.io"},academic:{researchAreas:{raw:["viral evolution","RNA virus ecology","phylogenetics","outbreak analysis","molecular epidemiology","genomics"],standardized:{primary:[{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"viral-genomics",label:"Viral Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of viruses and viral evolution",synonyms:["virus genomics","viral genome analysis"],children:[]}],secondary:[{id:"molecular-epidemiology",label:"Molecular Epidemiology",level:3,parentId:"medical-clinical",description:"Use of molecular biology in epidemiological studies",synonyms:["genetic epidemiology","genomic epidemiology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0227-4158"},profile:{shortBio:"Gytis Dudas is a scientist at Vilnius University's Life Sciences Center in Lithuania who works on viruses. His interests broadly cover the evolution and ecology of RNA viruses from the most closely related lineages in an outbreak setting to radical changes viruses undergo over evolutionary time.",source:"Personal website, PubMed"}}},"durbin-richard":{id:"durbin-richard",name:"Richard Durbin",enrichment:{lastUpdated:"2025-01-30T12:00:00Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Cambridge",department:"Department of Genetics / Wellcome Sanger Institute"},academic:{researchAreas:{raw:["computational genomics","genome assembly","population genetics","sequence analysis","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-9130-1006"},profile:{shortBio:"Richard Durbin is a computational biologist and one of the founders of the field of computational genomics. Key contributor to the Human Genome Project.",source:"Academic profile"}}},"dyhrman-sonya":{id:"dyhrman-sonya",name:"Sonya Dyhrman",enrichment:{lastUpdated:"2025-07-02T22:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"Columbia University",department:"Department of Earth and Environmental Sciences",labWebsite:"https://dyhrman.ldeo.columbia.edu"},academic:{researchAreas:{raw:["microbial oceanography","marine microbiology","phytoplankton ecology","biogeochemistry","climate change","ocean acidification"],standardized:{primary:[{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"environmental-microbiology",label:"Environmental Microbiology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in environmental contexts",synonyms:["microbial ecology","environmental microbiome"],children:[]}],secondary:[{id:"marine-ecology",label:"Marine Ecology",level:3,parentId:"ecology",description:"Ecology of marine organisms and ecosystems",synonyms:["ocean ecology","marine biology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-1323-3307"},profile:{shortBio:"Sonya Dyhrman is a microbial oceanographer at Lamont-Doherty Earth Observatory and Associate Professor at Columbia University. She studies how marine microbes interact with their environment using molecular tools to examine the intersection of microbial physiology and biogeochemistry. She is a Fellow of the American Academy of Microbiology.",source:"Columbia University, Lamont-Doherty Earth Observatory"}}},"eichler-evan":{id:"eichler-evan",name:"Evan Eichler",enrichment:{lastUpdated:"2025-07-02T20:47:51.165378Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Washington",department:"Department of Genome Sciences"},academic:{researchAreas:{raw:["genome evolution","structural variation","segmental duplications","primate genomics","human genetics"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]}],secondary:[{id:"mammalian-genomics",label:"Mammalian Genomics",level:3,parentId:"vertebrate-genomics",description:"Genomics of mammalian species",synonyms:["mammal genomics"]},{id:"structural-variation",label:"Structural Variation",level:3,parentId:"population-genomics",description:"Large-scale genomic rearrangements and variations",synonyms:["SV analysis","genomic structural variants"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-8246-4014"},profile:{shortBio:"Evan Eichler is a leader in primate genome evolution and structural variation. Howard Hughes Medical Institute Investigator.",source:"Academic profile"}}},"falush-daniel":{id:"falush-daniel",name:"Daniel Falush",enrichment:{lastUpdated:"2025-07-02T22:00:00.000Z",confidence:"high",professional:{title:"Principal Investigator",affiliation:"Institut Pasteur of Shanghai",department:"Centre for Microbes, Development and Health"},academic:{researchAreas:{raw:["population genetics","bacterial evolution","population structure","computational methods","Helicobacter pylori","microbial genomics"],standardized:{primary:[{id:"microbial-evolution",label:"Microbial Evolution",level:2,parentId:"microbiology-microbiome",description:"Evolution of microorganisms",synonyms:["bacterial evolution","prokaryotic evolution"],children:[]},{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-0747-4187"},profile:{shortBio:"Daniel Falush is a leading population geneticist at Institut Pasteur of Shanghai. He co-developed influential methods for inferring population structure from genetic data. His research focuses on bacterial population genetics, particularly Helicobacter pylori evolution and antibiotic resistance.",source:"Institut Pasteur of Shanghai, Chinese Academy of Sciences"}}},"fernandez-rosa":{id:"fernandez-rosa",name:"Rosa Fernndez",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Group Leader and Senior Research Scientist",affiliation:"Institute of Evolutionary Biology (IBE, CSIC-UPF)",department:"Metazoa Phylogenomics Lab",labWebsite:"https://www.metazomics.com"},academic:{researchAreas:{raw:["phylogenomics","comparative genomics","terrestrialization","invertebrate evolution","adaptation","functional annotation"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-3724-6096"},profile:{shortBio:"Rosa Fernndez is a Group Leader at the Institute of Evolutionary Biology (CSIC-UPF) in Barcelona. She was awarded the 2024 National Research Award for Young Scientists and an ERC Starting Grant. Her research focuses on understanding how animals adapted to terrestrial environments through genomic approaches.",source:"Institute of Evolutionary Biology (IBE), CSIC-UPF"}}},"ferrer-obiol-joan":{id:"ferrer-obiol-joan",name:"Joan Ferrer Obiol",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"Colorado State University",department:"The Bird Genoscape Project, Department of Biology"},academic:{researchAreas:{raw:["population genomics","conservation genomics","phylogenomics","ornithology","migratory behaviour","seabird evolution"],standardized:{primary:[{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"bird-genomics",label:"Bird Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of avian species",synonyms:["avian genomics","ornithological genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-1184-5434"},profile:{shortBio:"Joan Ferrer Obiol is a postdoctoral researcher at Colorado State University's Bird Genoscape Project. He completed his PhD on shearwater phylogeny using UCEs and ddRADseq data. His research uses genomic approaches to understand bird evolution, particularly seabirds, and the genetic basis of migratory behavior.",source:"Colorado State University, The Bird Genoscape Project"}}},"fillault-daniel":{id:"fillault-daniel",name:"Daniel Fillault",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"low",professional:{title:"Researcher",affiliation:"Unknown"},academic:{researchAreas:{raw:["genomics","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Limited information available about Daniel Fillault's current research activities and affiliations in genomics and bioinformatics.",source:"Web search - no specific results found"}}},"fumagalli-matteo":{id:"fumagalli-matteo",name:"Matteo Fumagalli",enrichment:{lastUpdated:"2025-07-01T14:15:00.000Z",confidence:"high",professional:{title:"Senior Lecturer",affiliation:"Queen Mary University of London",department:"School of Biological and Behavioural Sciences",previousAffiliation:"Imperial College London"},academic:{researchAreas:{raw:["population genetics","bioinformatics","evolutionary genomics","quantitative evolution","human genetics","statistical genetics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-4084-2953"},profile:{shortBio:"Matteo Fumagalli moved to Imperial College London in 2016 and then became Lecturer in Quantitative Evolution at Silwood Park in 2017. He joined Queen Mary University of London as Senior Lecturer in Genetics in 2021. His research focuses on population genetics and bioinformatics.",source:"Queen Mary University of London, Google Scholar"}}},"gabaldon-toni":{id:"gabaldon-toni",name:"Toni Gabaldn",enrichment:{lastUpdated:"2025-07-01T13:00:00.000Z",confidence:"high",professional:{title:"Senior Group Leader (Bases of disease)",affiliation:"Institut de Recerca Biomdica",department:"Barcelona Supercomputing Center"},academic:{orcid:"0000-0003-0019-1735",researchAreas:{raw:["comparative genomics","phylogenomics","fungal genomics","evolutionary biology","systems biology","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]},{id:"systems-biology",label:"Systems Biology",level:2,parentId:"molecular-cellular",description:"Holistic study of biological systems and their interactions",synonyms:["biological systems analysis","integrative biology"],children:[]}],secondary:[{id:"fungal-genomics",label:"Fungal Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of fungi",synonyms:["mycological genomics","fungal genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Toni Gabaldn is a Senior Group Leader at the Institut de Recerca Biomdica in Barcelona, specializing in comparative genomics, phylogenomics, and fungal evolution.",source:"ORCID profile"}}},"gaggiotti-oscar":{id:"gaggiotti-oscar",name:"Oscar Gaggiotti",enrichment:{lastUpdated:"2025-07-03T03:06:03.049Z",confidence:"high",professional:{title:"MASTS Professor",affiliation:"University of St Andrews",department:"Scottish Oceans Institute and School of Biology",labWebsite:"https://www.st-andrews.ac.uk/biology/people/oeg"},academic:{researchAreas:{raw:["population genetics","metapopulation dynamics","Bayesian statistics","marine biology","local adaptation","spatial genetics","conservation genetics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"bayesian-methods",label:"Bayesian Methods",level:2,parentId:"mathematical-statistical",description:"Bayesian statistical approaches in biology",synonyms:["bayesian statistics","bayesian inference"],children:[]},{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"landscape-genetics",label:"Landscape Genetics",level:3,parentId:"population-genetics",description:"Spatial patterns of genetic variation",synonyms:["spatial genetics","geographic genetics"]},{id:"marine-ecology",label:"Marine Ecology",level:3,parentId:"ecology",description:"Ecology of marine organisms and ecosystems",synonyms:["ocean ecology","marine biology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-1827-1493"},profile:{shortBio:"Oscar Gaggiotti is a MASTS professor at the University of St Andrews studying spatial patterns of genetic diversity. He develops ecologically realistic population genetics theory and methods using metapopulation paradigms and Bayesian statistics, with applications mainly to marine species including fish, marine mammals, and macroalgae.",source:"University of St Andrews, Google Scholar"}}},"garamszegi-laszlo":{id:"garamszegi-laszlo",name:"Laszlo Garamszegi",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Director",affiliation:"Centre for Ecological Research, Budapest",department:"Institute of Ecology and Botany",previousAffiliation:"Estacin Biolgica de Doana - CSIC, Spain"},academic:{researchAreas:{raw:["behavioral ecology","evolutionary biology","bird song evolution","host-parasite interactions","animal personality","phylogenetic comparative methods","statistical ecology"],standardized:{primary:[{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-8920-2183"},profile:{shortBio:"Lszl Zsolt Garamszegi is Director at the Centre for Ecological Research in Budapest. His research focuses on the evolutionary ecology of birds, including the evolution of bird song, behavioral adaptation, and host-parasite interactions. He develops statistical approaches for behavioral ecology and has been cited over 12,000 times.",source:"Centre for Ecological Research, CSIC"}}},"garber-manuel":{id:"garber-manuel",name:"Manuel Garber",enrichment:{lastUpdated:"2025-07-02T22:00:00.000Z",confidence:"high",professional:{title:"Professor, Bioinformatics Core Director",affiliation:"UMass Chan Medical School",department:"Department of Genomics and Computational Biology",labWebsite:"https://www.umassmed.edu/garberlab"},academic:{researchAreas:{raw:["evolutionary non-coding genomics","bioinformatics","RNA-seq","epigenomics","gene regulation","single cell genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"functional-genomics",label:"Functional Genomics",level:2,parentId:"genomics-omics",description:"Study of gene function and interactions at genome scale",synonyms:["genome function","functional genome analysis"],children:[]},{id:"single-cell-genomics",label:"Single-Cell Genomics",level:2,parentId:"genomics-omics",description:"Analysis of genomes, transcriptomes, and epigenomes at single-cell resolution",synonyms:["single cell genomics","single-cell sequencing"],children:[]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-8732-1293"},profile:{shortBio:"Manuel Garber is Professor and Bioinformatics Core Director at UMass Chan Medical School. His lab focuses on evolutionary non-coding genomics, developing computational techniques to integrate comparative and functional genomics data. He developed Siphy and Scripture algorithms and continues work on single cell RNA-seq and epigenomics.",source:"UMass Chan Medical School"}}},"gardy-jennifer":{id:"gardy-jennifer",name:"Jennifer Gardy",enrichment:{lastUpdated:"2025-01-02T20:00:00.000Z",confidence:"high",professional:{title:"Deputy Director for Surveillance, Data, and Epidemiology",affiliation:"Bill & Melinda Gates Foundation",department:"Malaria Team",previousAffiliation:"University of British Columbia and BC Centre for Disease Control"},academic:{researchAreas:{raw:["genomic epidemiology","infectious disease surveillance","pathogen transmission","tuberculosis genomics","public health genomics","bioinformatics","malaria"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]}],secondary:[{id:"molecular-epidemiology",label:"Molecular Epidemiology",level:3,parentId:"medical-clinical",description:"Use of molecular biology in epidemiological studies",synonyms:["genetic epidemiology","genomic epidemiology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-6061-8740"},profile:{shortBio:"Jennifer Gardy is Deputy Director at the Gates Foundation, previously Canada Research Chair in Public Health Genomics at UBC. She pioneered genomic epidemiology, using genome sequencing to track disease outbreaks. Her team was first to show how genomics could reconstruct person-to-person TB transmission. She's also a science communicator and occasional CBC host.",source:"Gates Foundation, University of British Columbia"}}},"garrison-erik":{id:"garrison-erik",name:"Erik Garrison",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"University of Tennessee Health Science Center",department:"Department of Genetics, Genomics and Informatics"},academic:{researchAreas:{raw:["pangenomics","variant calling","genome graphs","sequence analysis","bioinformatics tools"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"pangenomics",label:"Pangenomics",level:2,parentId:"genomics-omics",description:"Study of all genes within a species or clade",synonyms:["pan-genome","core genome"],children:[]}],secondary:[{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]},{id:"variant-calling",label:"Variant Calling",level:3,parentId:"computational-methods",description:"Identification of genetic variants from sequencing data",synonyms:["SNP calling","variant detection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-3821-631X"},profile:{shortBio:"Erik Garrison develops computational methods for pangenomics and variation graphs. Creator of FreeBayes and other genomics tools.",source:"Academic profile"}}},"gascuel-olivier":{id:"gascuel-olivier",name:"Olivier Gascuel",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"high",professional:{title:"Research Director",affiliation:"CNRS and Musum National d'Histoire Naturelle",department:"ISYEB (Institut de Systmatique, volution, Biodiversit)",labWebsite:"https://www.lirmm.fr/~gascuel"},academic:{researchAreas:{raw:["phylogenetics","bioinformatics","evolutionary biology","mathematical biology","sequence analysis","phylogenetic tree building","computational biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"mathematical-biology",label:"Mathematical Biology",level:2,parentId:"mathematical-statistical",description:"Mathematical modeling of biological systems",synonyms:["biomathematics","theoretical biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Olivier Gascuel is a Research Director at CNRS focusing on phylogeny and evolutionary biology. He co-developed PhyML, a widely-used phylogenetic tree building software with over 20,000 citations. He received the 2017 Grand Prix Inria  Acadmie des Sciences and has made significant contributions to mathematical and computational approaches in evolution.",source:"CNRS, LIRMM, Wikipedia"}}},"gosik-kirk":{id:"gosik-kirk",name:"Kirk Gosik",enrichment:{lastUpdated:"2025-07-01T14:30:00.000Z",confidence:"high",professional:{title:"Ph.D.",affiliation:"Constantiam Biosciences",previousAffiliation:"Penn State College of Medicine"},academic:{researchAreas:{raw:["statistical genetics","bioinformatics","computational biology","genotype-phenotype maps","complex traits","biostatistics","systems biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"biostatistics",label:"Biostatistics",level:2,parentId:"mathematical-statistical",description:"Statistical methods in biological research",synonyms:["biological statistics","statistical biology"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"quantitative-genetics",label:"Quantitative Genetics",level:2,parentId:"population-quantitative",description:"Study of complex traits influenced by multiple genes",synonyms:["polygenic traits","complex trait genetics"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]},{id:"systems-biology",label:"Systems Biology",level:2,parentId:"molecular-cellular",description:"Holistic study of biological systems and their interactions",synonyms:["biological systems analysis","integrative biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Kirk is a statistical geneticist by training with experience in bioinformatics and computational biology working with various levels of omics data for inferring the architecture of genotype-phenotype maps for complex traits and diseases.",source:"Constantiam Biosciences, Google Scholar"}}},"grabherr-manfred":{id:"grabherr-manfred",name:"Manfred Grabherr",enrichment:{lastUpdated:"2025-07-01T14:30:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"Uppsala University",department:"Department of Medical Biochemistry and Microbiology",additionalRole:"Nathaniel Robert Street Professor, Ume Plant Science Centre",orcid:"0000-0001-8792-6508"},academic:{researchAreas:{raw:["bioinformatics","genomics","genome assembly","comparative genomics","evolutionary genomics","plant genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Manfred Grabherr is Head of NGS and bioinformatics at State Laboratory Schleswig-Holstein, Germany. Previously at Uppsala University, Sweden & IPI, Polish Academy of Sciences, Poland. Patric Jern Uppsala University. His research includes genome assembly and evolutionary genomics.",source:"Uppsala University, Google Scholar, ResearchGate"}}},"griffith-malachi":{id:"griffith-malachi",name:"Malachi Griffith",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Associate Professor of Medicine and Genetics",affiliation:"Washington University in St. Louis",department:"McDonnell Genome Institute & School of Medicine",additionalRole:"Assistant Director of the McDonnell Genome Institute",labWebsite:"https://griffithlab.org"},academic:{researchAreas:{raw:["cancer genomics","bioinformatics","personalized medicine","data mining","cancer research","precision oncology","genomic medicine"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"cancer-genomics",label:"Cancer Genomics",level:2,parentId:"medical-clinical",description:"Genomic analysis of cancer",synonyms:["oncogenomics","tumor genomics"],children:[]},{id:"clinical-genomics",label:"Clinical Genomics",level:2,parentId:"medical-clinical",description:"Application of genomics in clinical settings",synonyms:["medical genomics","clinical genetics"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"precision-medicine",label:"Precision Medicine",level:2,parentId:"medical-clinical",description:"Tailored medical treatment based on genetic information",synonyms:["personalized medicine","genomic medicine"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-6388-446X"},profile:{shortBio:"Dr. Griffith has more than 15 years of experience in the fields of genomics, bioinformatics, data mining, and cancer research. His research is focused on the development of personalized medicine strategies for cancer using genomic technologies.",source:"Washington University School of Medicine"}}},"guido-roderic":{id:"guido-roderic",name:"Roderic Guigo",enrichment:{lastUpdated:"2025-07-02T23:47:36.078Z",confidence:"high",professional:{affiliation:"Centre for Genomic Regulation",department:"Bioinformatics and Genomics Program",title:"Professor of Bioinformatics"},academic:{researchAreas:{raw:["computational biology","genomics","bioinformatics","RNA processing","genome annotation"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5738-4477"},profile:{shortBio:"Roderic Guigo is Professor of Bioinformatics at the Centre for Genomic Regulation and Universitat Pompeu Fabra. His research focuses on computational biology of RNA processing and understanding genomic sequence information.",source:"Centre for Genomic Regulation, Google Scholar"}}},"gutenkurst-ryan":{id:"gutenkurst-ryan",name:"Ryan Gutenkurst",enrichment:{lastUpdated:"2025-07-01T14:30:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of Arizona",department:"Molecular and Cellular Biology",additionalAffiliations:["BIO5 Institute","Departments of Ecology and Evolutionary Biology","Graduate Programs in Applied Biosciences"]},academic:{researchAreas:{raw:["population genomics","systems biology","evolutionary genetics","molecular mechanisms","computational biology","statistical genetics"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]},{id:"systems-biology",label:"Systems Biology",level:2,parentId:"molecular-cellular",description:"Holistic study of biological systems and their interactions",synonyms:["biological systems analysis","integrative biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Ryan Gutenkunst joined the faculty in Molecular and Cellular Biology at the University of Arizona in Fall 2010. His research program integrates population genomics and systems biology to incorporate molecular mechanisms into evolutionary genetics.",source:"University of Arizona, Zuckerman College of Public Health"}}},"haas-brian":{id:"haas-brian",name:"Brian Haas",enrichment:{lastUpdated:"2025-07-02T20:47:51.165381Z",confidence:"high",professional:{title:"Principal Computational Scientist",affiliation:"Broad Institute of MIT and Harvard",department:"Cancer Informatics",labWebsite:"https://personal.broadinstitute.org/bhaas"},academic:{researchAreas:{raw:["transcriptomics","genomics","bioinformatics tools","cancer informatics","gene discovery","genome annotation"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"cancer-genomics",label:"Cancer Genomics",level:2,parentId:"medical-clinical",description:"Genomic analysis of cancer",synonyms:["oncogenomics","tumor genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-6609-4973"},profile:{shortBio:"Brian Haas is a Principal Computational Scientist at Broad Institute of MIT and Harvard. He is a Bioinformatics software developer and data analyst with a focus on leveraging transcriptomics for studies in genomics and cancer.",source:"Broad Institute, ResearchGate"}}},"hahn-matthew":{id:"hahn-matthew",name:"Matthew Hahn",enrichment:{lastUpdated:"2025-07-02T21:22:16.667Z",confidence:"high",professional:{title:"Distinguished Professor",affiliation:"Indiana University",department:"Department of Biology and Computer Science"},academic:{researchAreas:{raw:["molecular evolution","population genetics","comparative genomics","speciation","computational biology"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5731-8808"},profile:{shortBio:"Matthew Hahn is a Distinguished Professor studying molecular evolution and population genetics. Author of 'Molecular Population Genetics' textbook.",source:"Academic profile"}}},"hall-neil":{id:"hall-neil",name:"Neil Hall",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"Director",affiliation:"The Earlham Institute",department:"Genomics",labWebsite:"https://www.earlham.ac.uk/profile/neil-hall"},academic:{researchAreas:{raw:["comparative genomics","evolutionary genomics","pathogen genomics","parasitic protists","bacterial genomics","genome sequencing"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-2808-0009"},profile:{shortBio:"Neil Hall is Director of The Earlham Institute. His research focuses on comparative and evolutionary genomics in pathogens, particularly parasitic protists. He led development of an efficient pipeline for large-scale bacterial genome sequencing with the 10KSG consortium.",source:"The Earlham Institute, Google Scholar"}}},"hancock-angela":{id:"hancock-angela",name:"Angela Hancock",enrichment:{lastUpdated:"2025-07-02T23:47:36.077Z",confidence:"high",professional:{title:"Group Leader",affiliation:"Max Planck Institute for Plant Breeding Research (moving to Purdue University in 2025)",department:"Department of Plant Developmental Biology",labWebsite:"https://www.mpipz.mpg.de/hancock"},academic:{researchAreas:{raw:["population genetics","evolutionary biology","quantitative genetics","climate adaptation","Arabidopsis genomics","GWAS","complex trait evolution"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"quantitative-genetics",label:"Quantitative Genetics",level:2,parentId:"population-quantitative",description:"Study of complex traits influenced by multiple genes",synonyms:["polygenic traits","complex trait genetics"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-4768-3377"},profile:{shortBio:"Angela Hancock leads a research group at MPI for Plant Biology studying how complex traits evolve. Her work integrates computational and experimental approaches to understand adaptation, particularly in Arabidopsis populations from Africa and Cape Verde Islands. She received her PhD from University of Chicago in Human Genetics.",source:"MPI Plant Biology, Google Scholar"}}},"handley-scott":{id:"handley-scott",name:"Scott Handley",enrichment:{lastUpdated:"2025-07-02T20:47:51.165259Z",confidence:"high",professional:{title:"Professor",affiliation:"Washington University in St. Louis",department:"Edison Family Center for Genome Sciences & Systems Biology"},academic:{researchAreas:{raw:["microbiome","virome","metagenomics","computational biology","systems biology","genomics"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]},{id:"systems-biology",label:"Systems Biology",level:2,parentId:"molecular-cellular",description:"Holistic study of biological systems and their interactions",synonyms:["biological systems analysis","integrative biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-2143-6570"},profile:{shortBio:"Scott Handley is the founder and director of the Workshop on Genomics and co-founder of workshops on Population & Speciation Genomics and Phylogenomics. His research focuses on the human virome and microbiome interactions.",source:"Workshop organizer information"}}},"harris-kelley":{id:"harris-kelley",name:"Kelley Harris",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of Washington",department:"Department of Genome Sciences",labWebsite:"https://sites.google.com/site/harriskelley/home"},academic:{researchAreas:{raw:["population genetics","mutation rate evolution","human evolution","ancient DNA","computational genomics","Neanderthal introgression"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-0302-2523"},profile:{shortBio:"Kelley Harris is Associate Professor of Genome Sciences at University of Washington. She uses population genetic theory to study evolutionary history, particularly the evolution of mutagenesis and human-Neanderthal interactions. She received a Sloan Fellowship and Pew Biomedical Scholar award for her innovative research.",source:"University of Washington, Department of Genome Sciences"}}},"heath-tracy":{id:"heath-tracy",name:"Tracy Heath",enrichment:{lastUpdated:"2025-07-01T14:45:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"Iowa State University",department:"Department of Ecology, Evolution, and Organismal Biology",labWebsite:"https://phyloworks.org"},academic:{researchAreas:{raw:["phylogenetics","statistical phylogenetics","Bayesian inference","computational methods","evolutionary biology","molecular evolution"],standardized:{primary:[{id:"bayesian-methods",label:"Bayesian Methods",level:2,parentId:"mathematical-statistical",description:"Bayesian statistical approaches in biology",synonyms:["bayesian statistics","bayesian inference"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0087-2541"},profile:{shortBio:"Tracy Heath's research program focuses on developing and applying computational methods for understanding evolutionary processes in a phylogenetic context. Her primary approach to developing phylogenetic tools is to apply new statistical models as priors using Bayesian inference methods.",source:"Iowa State University EEOB"}}},"hird-sarah":{id:"hird-sarah",name:"Sarah Hird",enrichment:{lastUpdated:"2025-07-02T23:47:36.077Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"University of Connecticut",department:"Department of Molecular and Cell Biology",additionalRole:"CLAS Molecular and Cell Biology Website"},academic:{researchAreas:{raw:["microbiome evolution","gut microbiota","bioinformatics","microbial ecology","ornithology","phylogeography"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbial-evolution",label:"Microbial Evolution",level:2,parentId:"microbiology-microbiome",description:"Evolution of microorganisms",synonyms:["bacterial evolution","prokaryotic evolution"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"bird-genomics",label:"Bird Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of avian species",synonyms:["avian genomics","ornithological genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1998-9387"},profile:{shortBio:"Sarah Hird's research focuses on host-microbiome evolution, gut microbiota, bioinformatics, microbial ecology, ornithology, and phylogeography. She is Assistant Professor at the Department of Molecular and Cell Biology at University of Connecticut.",source:"University of Connecticut MCB"}}},"hoff-katharina":{id:"hoff-katharina",name:"Katharina Hoff",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"PD Dr.",affiliation:"Universitt Greifswald",department:"Institut fr Mathematik und Informatik",additionalRole:"Postdoktorandinnen 2019 Externe Mentorin"},academic:{researchAreas:{raw:["bioinformatics","genome annotation","structural gene prediction","computational biology","genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7333-8390"},profile:{shortBio:"PD Dr. Katharina Hoff is a Privatdozentin at the Institut fr Mathematik und Informatik, Universitt Greifswald. Her research focuses on bioinformatics, particularly genome annotation and structural gene prediction.",source:"Universitt Greifswald"}}},"hofreiter-michael":{id:"hofreiter-michael",name:"Michael Hofreiter",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"Professor of Evolutionary Adaptive Genomics",affiliation:"University of Potsdam",department:"Institute of Biochemistry and Biology",labWebsite:"https://www.uni-potsdam.de/en/ibb-genomics/group/hofreiter.html"},academic:{researchAreas:{raw:["paleogenomics","ancient DNA","evolutionary biology","molecular ecology","conservation genetics","adaptive genomics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"molecular-ecology",label:"Molecular Ecology",level:2,parentId:"ecology-environmental",description:"Molecular approaches to ecological questions",synonyms:["ecological genetics","molecular ecological analysis"],children:[]},{id:"paleogenomics",label:"Paleogenomics",level:2,parentId:"genomics-omics",description:"Genomic analysis of ancient organisms",synonyms:["ancient genomics","paleogenetics"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-0441-4705"},profile:{shortBio:"Michael Hofreiter at University of Potsdam is an evolutionary biologist specializing in genomic approaches on extinct species. His research elucidates the genetic basis of animal adaptations and the effect of environmental changes on genetic diversity.",source:"University of Potsdam, Google Scholar"}}},"huson-daniel":{id:"huson-daniel",name:"Daniel Huson",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"Professor of Algorithms in Bioinformatics",affiliation:"University of Tbingen",department:"Center for Bioinformatics and Department of Computer Science",labWebsite:"https://uni-tuebingen.de/en/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics"},academic:{researchAreas:{raw:["bioinformatics algorithms","phylogenetics","metagenomics","sequence analysis","phylogenetic networks","computational biology"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"phylogenetic-networks",label:"Phylogenetic Networks",level:3,parentId:"phylogenetics",description:"Network representations of evolutionary relationships",synonyms:["reticulate evolution","phylogenetic graphs"]},{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-2961-604X"},profile:{shortBio:"Daniel Huson is Professor of Algorithms in Bioinformatics at University of Tbingen. His research focuses on algorithms for sequence analysis, methods for inferring phylogenetic networks, and tools for metagenomic analysis. Developer of widely-used bioinformatics software.",source:"University of Tbingen, Google Scholar"}}},"huttenhower-curtis":{id:"huttenhower-curtis",name:"Curtis Huttenhower",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"high",professional:{title:"Professor of Computational Biology and Bioinformatics",affiliation:"Harvard T.H. Chan School of Public Health",department:"Department of Biostatistics",labWebsite:"https://huttenhower.sph.harvard.edu"},academic:{researchAreas:{raw:["microbiome","computational biology","metagenomics","human health","machine learning","functional genomics","inflammatory bowel disease"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"functional-genomics",label:"Functional Genomics",level:2,parentId:"genomics-omics",description:"Study of gene function and interactions at genome scale",synonyms:["genome function","functional genome analysis"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"medical-clinical",label:"Medical & Clinical Sciences",level:1,description:"Application of genomics and biology to human health and disease",icon:"",children:["human-genetics","medical-genetics","clinical-genomics","precision-medicine","cancer-genomics","pharmacogenomics","genetic-counseling","rare-diseases","complex-diseases"]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-1110-0096"},profile:{shortBio:"Curtis Huttenhower is a Professor at Harvard School of Public Health and Associate Member at the Broad Institute. He co-leads the HMP2 Center studying the gut microbiome in IBD and directs the Harvard Microbiome Analysis Core. His work on computational methods for microbiome research has been cited over 162,000 times.",source:"Harvard T.H. Chan School of Public Health, Google Scholar"}}},"jay-flora":{id:"jay-flora",name:"Flora Jay",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"high",professional:{title:"CNRS Junior Researcher",affiliation:"Universit Paris-Saclay, CNRS, INRIA",department:"Laboratoire Interdisciplinaire des Sciences du Numrique (LISN)",labWebsite:"http://flora-jay.blogspot.com"},academic:{researchAreas:{raw:["population genetics","machine learning","demographic inference","deep learning","artificial genomes","ancient DNA","Bayesian models"],standardized:{primary:[{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"demographic-inference",label:"Demographic Inference",level:3,parentId:"population-genetics",description:"Inference of population demographic history",synonyms:["population history","demographic modeling"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Flora Jay is a CNRS researcher bridging computer science and genetics. She develops machine learning methods for population genetics, including pioneering work on artificial genome generation using deep learning. Her research reconstructs demographic histories and studies admixture patterns in human evolution, with over 8,700 citations.",source:"CNRS, Google Scholar, ResearchGate"}}},"jiggins-chris":{id:"jiggins-chris",name:"Chris Jiggins",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor of Evolutionary Biology",affiliation:"University of Cambridge",department:"Department of Zoology",labWebsite:"https://heliconius.zoo.cam.ac.uk"},academic:{researchAreas:{raw:["butterfly genomics","speciation","adaptive evolution","mimicry","Heliconius butterflies","evolutionary developmental biology"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"developmental-biology",label:"Developmental Biology",level:2,parentId:"molecular-cellular",description:"Study of organism development from embryo to adult",synonyms:["developmental genetics","embryology"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-7809-062X"},profile:{shortBio:"Chris Jiggins is Professor of Evolutionary Biology at Cambridge University, leading the Insect Evolution and Genomics Group. He has worked with Heliconius butterflies since 1993 and led the Heliconius genome sequencing project. His research focuses on speciation, mimicry, and the genetic basis of adaptation.",source:"University of Cambridge, Department of Zoology"}}},"jones-felicity":{id:"jones-felicity",name:"Felicity Jones",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"high",professional:{title:"Group Leader",affiliation:"University of Groningen (previously Max Planck Institute)",department:"Evolutionary Genetics",labWebsite:"https://research.rug.nl/en/persons/felicity-jones"},academic:{researchAreas:{raw:["evolutionary genomics","stickleback fish","adaptation","functional genomics","regulatory evolution","ancient DNA","speciation"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"functional-genomics",label:"Functional Genomics",level:2,parentId:"genomics-omics",description:"Study of gene function and interactions at genome scale",synonyms:["genome function","functional genome analysis"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"model-organisms",label:"Model Organisms",level:3,parentId:"comparative-genomics",description:"Organisms used as models for biological research",synonyms:["model species","research organisms"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-5027-1031"},profile:{shortBio:"Felicity Jones studies genome function in naturally evolving organisms, particularly using threespine stickleback fish. She led landmark work sequencing 21 stickleback genomes (Nature 2012) creating high-resolution adaptive loci maps. Her research combines functional genomics with evolutionary biology to understand adaptation and speciation.",source:"University of Groningen, Google Scholar"}}},"kaessman-henrik":{id:"kaessman-henrik",name:"Henrik Kaessmann",enrichment:{lastUpdated:"2025-07-02T23:36:51.3NZ",confidence:"high",professional:{title:"Professor of Evolutionary Genomics",affiliation:"Heidelberg University",department:"Center for Molecular Biology (ZMBH)",labWebsite:"https://www.zmbh.uni-heidelberg.de/Kaessmann"},academic:{researchAreas:{raw:["evolutionary genomics","gene expression evolution","mammalian organ development","brain evolution","single-cell genomics","new gene evolution","comparative genomics"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"single-cell-genomics",label:"Single-Cell Genomics",level:2,parentId:"genomics-omics",description:"Analysis of genomes, transcriptomes, and epigenomes at single-cell resolution",synonyms:["single cell genomics","single-cell sequencing"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-7563-839X"},profile:{shortBio:"Henrik Kaessmann is Professor at Heidelberg University studying mammalian organ evolution through gene expression. His lab examines transcriptome evolution across development in multiple species, pioneering single-cell approaches. He has received three ERC Grants, EMBO membership, and the Clotta Prize for his contributions.",source:"Heidelberg University, NOMIS Foundation"}}},"kamoun-sophien":{id:"kamoun-sophien",name:"Sophien Kamoun",enrichment:{lastUpdated:"2025-07-01T15:00:00.000Z",confidence:"high",professional:{title:"Senior Scientist",affiliation:"The Sainsbury Laboratory",additionalRole:"Professor of Biology at the University of East Anglia",labWebsite:"https://kamounlab.net"},academic:{researchAreas:{raw:["plant pathology","plant-pathogen interactions","Phytophthora infestans","potato famine","rice blast fungus","Magnaporthe oryzae"],standardized:{primary:[{id:"pathogen-genomics",label:"Pathogen Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of pathogenic organisms",synonyms:["pathogen genetics","infectious disease genomics"],children:[]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0290-0315"},profile:{shortBio:"Sophien Kamoun is a Tunisian biologist and senior scientist at The Sainsbury Laboratory. The Kamoun lab studies the interactions between plants and filamentous pathogens, notably the potato famine organism Phytophthora infestans and the rice and wheat blast fungus Magnaporthe oryzae.",source:"The Sainsbury Laboratory, Wikipedia"}}},"karlicki-michal":{id:"karlicki-michal",name:"Michal Karlicki",enrichment:{lastUpdated:"2025-07-01T15:00:00.000Z",confidence:"high",professional:{title:"PhD Student (former)",affiliation:"University of Warsaw",department:"Institute of Evolutionary Biology, Field of Bioinformatics",period:"October 2015 - September 2017"},academic:{researchAreas:{raw:["bioinformatics","protistology","metagenomics","machine learning","chloroplast genomics","plastid genomes"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Michal Karlicki worked on MetaPlastHunter: An efficient pipeline for classifying chloroplast reads from metagenomic data. His research focused on searching for Plastid Genomes in the Metagenomic Data.",source:"University of Warsaw IBE, ResearchGate, GitHub"}}},"karnkowska-anna":{id:"karnkowska-anna",name:"Anna Karnkowska",enrichment:{lastUpdated:"2025-07-01T15:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of Warsaw",department:"Institute of Evolutionary Biology, Faculty of Biology",labWebsite:"https://karnkowskalab.com"},academic:{researchAreas:{raw:["protists","eukaryotic microorganisms","evolution","diversity","ecology","mitochondria","lignocellulose digestion"],standardized:{primary:[{id:"biodiversity",label:"Biodiversity",level:2,parentId:"ecology-environmental",description:"Study of biological diversity at all levels",synonyms:["biological diversity","species diversity"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"organellar-genomics",label:"Organellar Genomics",level:2,parentId:"genomics-omics",description:"Genomics of cellular organelles (mitochondria, chloroplasts)",synonyms:["organelle genomics","mitochondrial genomics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-3709-7873"},profile:{shortBio:"Dr hab. Anna Karnkowska's research focuses on the diversity, evolution and ecology of eukaryotic microorganisms (aka protists). The lab is part of the Institute of Evolutionary Biology at Faculty of Biology at University of Warsaw and the Diversity of Eukaryotic Microorganisms Consortium (DEMON).",source:"University of Warsaw IBE, Personal Lab Website"}}},"kern-andrew":{id:"kern-andrew",name:"Andrew Kern",enrichment:{lastUpdated:"2025-07-02T23:47:36.077Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Oregon",department:"Institute of Ecology and Evolution",additionalRole:"Evergreen Professor"},academic:{researchAreas:{raw:["population genetics","evolutionary biology","machine learning","computational biology","genomics","probabilistic approaches","simulation"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4381-4680"},profile:{shortBio:"Andrew Kern is an academic expert in population genetics, evolutionary biology, machine learning, computational biology, and genomics. His research combines modern machine learning methods with classical probabilistic approaches and large-scale simulation to gain insight into evolutionary processes.",source:"University of Oregon Institute of Ecology and Evolution"}}},"knight-robert":{id:"knight-robert",name:"Robert Knight",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor and Founding Director",affiliation:"UC San Diego",department:"Pediatrics, Bioengineering, Computer Science & Engineering",labWebsite:"https://knightlab.ucsd.edu"},academic:{researchAreas:{raw:["microbiome","metagenomics","computational biology","microbial ecology","human health","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"medical-clinical",label:"Medical & Clinical Sciences",level:1,description:"Application of genomics and biology to human health and disease",icon:"",children:["human-genetics","medical-genetics","clinical-genomics","precision-medicine","cancer-genomics","pharmacogenomics","genetic-counseling","rare-diseases","complex-diseases"]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0975-9019"},profile:{shortBio:"Rob Knight is Founding Director of the Center for Microbiome Innovation at UC San Diego. He developed QIIME and UniFrac, leads the American Gut Project, and was elected to the National Academy of Engineering in 2024.",source:"UC San Diego, Google Scholar"}}},"knowles-lacey":{id:"knowles-lacey",name:"Lacey Knowles",enrichment:{lastUpdated:"2025-07-03T23:53:50.016Z",confidence:"low",professional:{affiliation:"Unknown"}}},"kolisko-martin":{id:"kolisko-martin",name:"Martin Kolisko",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"korneliussen-thorfinn":{id:"korneliussen-thorfinn",name:"Thorfinn Korneliussen",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"kosunsky-ilya":{id:"kosunsky-ilya",name:"Ilya Kosunsky",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"kozlov-oleksiy":{id:"kozlov-oleksiy",name:"Oleksiy Kozlov",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"kubatko-laura":{id:"kubatko-laura",name:"Laura Kubatko",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"medium",professional:{affiliation:"Ohio State University",department:"Statistics and Evolution, Ecology and Organismal Biology",title:"Professor"},academic:{researchAreas:{raw:["phylogenetics","statistical phylogenetics","coalescent theory","species tree inference"],standardized:{primary:[{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"coalescent-theory",label:"Coalescent Theory",level:3,parentId:"population-genetics",description:"Mathematical model of gene genealogies",synonyms:["coalescence","gene genealogy"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5215-7144"},profile:{source:"Inferred from workshop participation"}}},"kucukural-alper":{id:"kucukural-alper",name:"Alper Kucukural",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"lanzen-anders":{id:"lanzen-anders",name:"Anders Lanzen",enrichment:{lastUpdated:"2025-07-03T23:53:50.017Z",confidence:"low",professional:{affiliation:"Unknown"}}},"leonard-alexander":{id:"leonard-alexander",name:"Alexander Leonard",enrichment:{lastUpdated:"2025-07-02T23:47:36.078Z",confidence:"medium",professional:{affiliation:"University of Exeter",title:"Researcher"},academic:{researchAreas:{raw:["bioinformatics","genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-8425-5630"},profile:{shortBio:"Alexander Leonard is a researcher at the University of Exeter with expertise in bioinformatics and genomics.",source:"ORCID"}}},"leonard-guy":{id:"leonard-guy",name:"Guy Leonard",enrichment:{lastUpdated:"2025-07-02T19:27:49.708Z",confidence:"high",professional:{title:"Research Fellow",affiliation:"University of Oxford",department:"Department of Biology",labWebsite:"https://protists.co.uk"},academic:{researchAreas:{raw:["comparative genomics","phylogenomics","genome assembly","evolutionary relationships","eukaryotes","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-4607-2064"},profile:{shortBio:"Guy Leonard works with comparative genomics, phylogenomics and genome assembly with an interest in the evolutionary relationships of eukaryotes. Lead Developer and Co-Director of the Workshop on Genomics.",source:"protists.co.uk, Google Scholar"}}},"limasset-antoine":{id:"limasset-antoine",name:"Antoine Limasset",enrichment:{lastUpdated:"2025-07-02T22:00:00.000Z",confidence:"high",professional:{title:"CNRS Researcher",affiliation:"University of Lille",department:"CRIStAL - Centre de Recherche en Informatique, Signal et Automatique de Lille",labWebsite:"https://malfoy.github.io"},academic:{researchAreas:{raw:["bioinformatics","algorithms","metagenomics","k-mer analysis","de Bruijn graphs","sequence analysis"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]}],secondary:[{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-7848-8701"},profile:{shortBio:"Antoine Limasset is a CNRS Researcher at University of Lille working on methods and data structures in sequence bioinformatics. His research focuses on efficient algorithms for k-mer analysis, de Bruijn graphs, and scalable sequence analysis tools. He has developed key contributions to sequence assembly and correction methods.",source:"CRIStAL, University of Lille"}}},"loman-nick":{id:"loman-nick",name:"Nick Loman",enrichment:{lastUpdated:"2025-07-02T23:47:36.078Z",confidence:"high",professional:{affiliation:"University of Birmingham",department:"Institute for Microbiology and Infection",title:"Professor of Microbial Genomics and Bioinformatics"},academic:{researchAreas:{raw:["microbial genomics","bioinformatics","infectious disease","metagenomics","nanopore sequencing","pathogen surveillance"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]},{id:"pathogen-genomics",label:"Pathogen Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of pathogenic organisms",synonyms:["pathogen genetics","infectious disease genomics"],children:[]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-9843-8988"},profile:{shortBio:"Nick Loman is Professor of Microbial Genomics and Bioinformatics at the University of Birmingham. His research explores the use of cutting-edge genomics and metagenomics approaches to infectious disease diagnosis, treatment and surveillance.",source:"University of Birmingham, COVID-19 Genomics UK Consortium"}}},"lorenzen-eline":{id:"lorenzen-eline",name:"Eline Lorenzen",enrichment:{lastUpdated:"2025-07-02T23:12:10.649Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Copenhagen",department:"GLOBE Institute, Section for GeoGenetics"},academic:{researchAreas:{raw:["conservation genomics","ancient DNA","population genomics","biodiversity","environmental genomics","wildlife genetics"],standardized:{primary:[{id:"biodiversity",label:"Biodiversity",level:2,parentId:"ecology-environmental",description:"Study of biological diversity at all levels",synonyms:["biological diversity","species diversity"],children:[]},{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-6353-2819"},profile:{shortBio:"Eline Lorenzen is a Professor at the GLOBE Institute, University of Copenhagen, specializing in conservation genomics and ancient DNA research to understand biodiversity patterns and inform conservation efforts.",source:"University of Copenhagen"}}},"lunter-gerton":{id:"lunter-gerton",name:"Gerton Lunter",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor of Medical Statistics",affiliation:"University Medical Center Groningen / University of Oxford",department:"Epidemiology / Weatherall Institute of Molecular Medicine",labWebsite:"https://www.chg.ox.ac.uk/~gerton"},academic:{researchAreas:{raw:["bioinformatics","population genetics","statistical genetics","Bayesian statistics","machine learning","computational biology","next-generation sequencing","non-coding DNA evolution"],standardized:{primary:[{id:"bayesian-methods",label:"Bayesian Methods",level:2,parentId:"mathematical-statistical",description:"Bayesian statistical approaches in biology",synonyms:["bayesian statistics","bayesian inference"],children:[]},{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-3798-2058"},profile:{shortBio:"Gerton Lunter is Professor of Medical Statistics at UMCG and Visiting Professor at Oxford. He contributed to the 1000 Genomes Project, co-founded Genomics PLC, and develops methods to study evolution of non-coding DNA.",source:"University of Groningen, Oxford University"}}},"mafessoni-fabrizio":{id:"mafessoni-fabrizio",name:"Fabrizio Mafessoni",enrichment:{lastUpdated:"2025-07-02T23:47:36.077Z",confidence:"high",professional:{title:"Research Group Leader",affiliation:"Max Planck Institute for Evolutionary Anthropology",department:"Department of Evolutionary Genetics"},academic:{researchAreas:{raw:["evolutionary anthropology","ancient DNA","population genetics","computational biology","human evolution","archaic humans"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4319-2076"},profile:{shortBio:"Fabrizio Mafessoni leads a research group at the Max Planck Institute for Evolutionary Anthropology, focusing on computational approaches to understand human evolution through ancient DNA and population genetics.",source:"Max Planck Institute"}}},"malinsky-milan":{id:"malinsky-milan",name:"Milan Malinsky",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"medium",professional:{affiliation:"University of Basel",title:"Group Leader",department:"Evolutionary Genomics"},academic:{researchAreas:{raw:["population genomics","speciation","cichlid evolution","genomic differentiation"],standardized:{primary:[{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1462-6317"},profile:{source:"Inferred from WPSG participation"}}},"manghi-paolo":{id:"manghi-paolo",name:"Paolo Manghi",enrichment:{lastUpdated:"2025-07-01T15:30:00.000Z",confidence:"high",professional:{title:"Guest",affiliation:"University of Trento",department:"Department of Cellular, Computational and Integrative Biology - CIBIO"},academic:{researchAreas:{raw:["microbiome","computational biology","metagenomics","bioinformatics","microbial ecology","gut microbiota"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0846-6529"},profile:{shortBio:"Paolo Manghi is a computational biologist at University of Trento's CIBIO department, specializing in microbiome research and metagenomic analysis of microbial communities.",source:"University of Trento"}}},"marcet-houben-marina":{id:"marcet-houben-marina",name:"Marina Marcet-Houben",enrichment:{lastUpdated:"2025-07-02T20:38:09.907Z",confidence:"high",professional:{affiliation:"Barcelona Supercomputing Center",department:"BSC",title:"Bioinformatician"},academic:{researchAreas:{raw:["phylogenomics","fungal evolution","next generation sequencing","genomics","bioinformatics","microsporidians"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"microbial-evolution",label:"Microbial Evolution",level:2,parentId:"microbiology-microbiome",description:"Evolution of microorganisms",synonyms:["bacterial evolution","prokaryotic evolution"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"fungal-evolution",label:"Fungal Evolution",level:3,parentId:"microbial-evolution",description:"Evolution of fungal species",synonyms:["mycological evolution","fungal phylogeny"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4838-187X"},profile:{shortBio:"Marina Marcet-Houben at Barcelona Supercomputing Center. Phylogenomics supports microsporidia as the earliest diverging clade of sequenced fungi. Skills in Next Generation Sequencing, Genomics, and Bioinformatics.",source:"Google Scholar, ResearchGate"}}},"marchet-camille":{id:"marchet-camille",name:"Camille Marchet",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"medium",professional:{affiliation:"CNRS / University of Lille",title:"CNRS Researcher",department:"CRIStAL"},academic:{researchAreas:{raw:["bioinformatics","k-mer analysis","genome assembly","transcriptomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7235-7346"},profile:{source:"Inferred from recent WoG participation"}}},"martin-simon":{id:"martin-simon",name:"Simon Martin",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Researcher",affiliation:"University of Cambridge",department:"Department of Zoology"},academic:{researchAreas:{raw:["evolutionary genomics","speciation","butterfly evolution","population genomics","Heliconius butterflies","natural selection"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]},{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0747-7456"},profile:{shortBio:"Simon Martin studies evolutionary genomics and speciation at the University of Cambridge, with a focus on understanding how natural selection shapes genetic diversity in butterfly populations.",source:"University of Cambridge"}}},"martinez-redondo-gemma":{id:"martinez-redondo-gemma",name:"Gemma Isabel Martnez-Redondo",enrichment:{lastUpdated:"2025-07-02T20:15:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Institute of Evolutionary Biology (IBE, CSIC-UPF)",department:"Evolutionary Biology"},academic:{researchAreas:{raw:["terrestrialization","phylogenomics","comparative genomics","bioinformatics","macroevolution","dark proteome"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-5633-8690"},profile:{shortBio:"Gemma Isabel Martnez-Redondo is a researcher at the Institute of Evolutionary Biology in Barcelona. Her work focuses on terrestrialization processes, phylogenomics, and comparative genomics. She has contributed to major projects including the European Reference Genome Atlas and studies on the dark proteome across the Animal Tree of Life.",source:"Google Scholar, ORCID, IBE"}}},"matschiner-michael":{id:"matschiner-michael",name:"Michael Matschiner",enrichment:{lastUpdated:"2025-07-01T15:30:00.000Z",confidence:"high",professional:{title:"Adjunct Professor",affiliation:"Natural History Museum, University of Oslo",department:"Sex and Evolution Research Group"},academic:{researchAreas:{raw:["phylogenomics","molecular dating","fish evolution","speciation genomics","evolutionary biology","comparative genomics"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4741-3884"},profile:{shortBio:"Michael Matschiner is an Adjunct Professor at the Natural History Museum, University of Oslo, specializing in phylogenomics and molecular dating to understand the evolutionary history of fishes and other organisms.",source:"Natural History Museum Oslo"}}},"mcdonald-daniel":{id:"mcdonald-daniel",name:"Daniel McDonald",enrichment:{lastUpdated:"2025-07-02T19:27:49.709Z",confidence:"high",professional:{title:"Ph.D. in Computer Science",affiliation:"University of Colorado Boulder",department:"Computer Science"},academic:{researchAreas:{raw:["microbiome data science","microbiome analysis","genomics","QIIME 2","complex microbial communities","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0876-9060"},profile:{shortBio:"Daniel McDonald holds a Ph.D. in Computer Science from UC Boulder. He specializes as a developer using popular microbiome software and works on reproducible, interactive, scalable and extensible microbiome data science using QIIME 2.",source:"The Microsetta Initiative, Google Scholar"}}},"mchardy-alice":{id:"mchardy-alice",name:"Alice McHardy",enrichment:{lastUpdated:"2025-07-03T18:15:00.000Z",confidence:"high",professional:{title:"Professor / Head of Research Group",affiliation:"Helmholtz Centre for Infection Research",department:"Department of Computational Biology for Infection Research",orcid:"0000-0003-2370-3430"},academic:{researchAreas:{raw:["computational biology","metagenomics","infection research","bioinformatics","microbial genomics","algorithmic bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Alice McHardy is a Professor and Head of the Department of Computational Biology for Infection Research at the Helmholtz Centre. She develops computational methods for analyzing microbial communities and understanding infection processes.",source:"Helmholtz Centre for Infection Research"}}},"mckay-sheldon":{id:"mckay-sheldon",name:"Sheldon McKay",enrichment:{lastUpdated:"2025-07-01T16:00:00.000Z",confidence:"high",professional:{title:"Scientist - Bioinformatics",affiliation:"Cold Spring Harbor Laboratory",previousAffiliation:"University of Arizona"},academic:{researchAreas:{raw:["bioinformatics","genomics","genome sequencing","genome annotation","functional genomics","genome browsers"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"functional-genomics",label:"Functional Genomics",level:2,parentId:"genomics-omics",description:"Study of gene function and interactions at genome scale",synonyms:["genome function","functional genome analysis"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-4011-3160"},profile:{shortBio:"Sheldon McKay is a bioinformatics scientist specializing in genome sequencing and annotation. He develops functional genomics tools to apply high-throughput techniques for understanding gene function and interactions.",source:"LinkedIn, ResearchGate"}}},"mclysaght-aoife":{id:"mclysaght-aoife",name:"Aoife McLysaght",enrichment:{lastUpdated:"2025-07-02T23:25:26.304Z",confidence:"high",professional:{title:"Professor of Genetics",affiliation:"Trinity College Dublin",department:"School of Genetics and Microbiology"},academic:{researchAreas:{raw:["molecular evolution","comparative genomics","gene duplication","genome evolution","human genetics","evolutionary genetics"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-2552-6220"},profile:{shortBio:"Aoife McLysaght is a Professor of Genetics at Trinity College Dublin. Her research focuses on molecular evolution, comparative genomics, and understanding how genomes evolve through gene duplication and other mechanisms.",source:"Trinity College Dublin"}}},"mcmurdie-paul":{id:"mcmurdie-paul",name:"Paul McMurdie",enrichment:{lastUpdated:"2025-07-01T16:00:00.000Z",confidence:"high",professional:{title:"Bioinformatics Scientist",affiliation:"Independent/Industry",previousAffiliation:"Stanford University"},academic:{researchAreas:{raw:["microbiome analysis","bioinformatics","statistical methods","R programming","phyloseq package","microbial ecology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"biostatistics",label:"Biostatistics",level:2,parentId:"mathematical-statistical",description:"Statistical methods in biological research",synonyms:["biological statistics","statistical biology"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-8879-3954"},profile:{shortBio:"Paul McMurdie is the developer of phyloseq, an R package for reproducible interactive analysis and graphics of microbiome census data. He specializes in statistical methods for microbiome analysis.",source:"phyloseq documentation, Bioconductor"}}},"meier-joana":{id:"meier-joana",name:"Joana Meier",enrichment:{lastUpdated:"2025-07-03T00:00:00.000Z",confidence:"high",professional:{title:"Group Leader",affiliation:"Wellcome Sanger Institute",department:"Tree of Life Programme"},academic:{researchAreas:{raw:["speciation genomics","evolutionary genomics","population genetics","cichlid evolution","hybridization","adaptive radiation"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"adaptive-radiation",label:"Adaptive Radiation",level:3,parentId:"speciation",description:"Rapid evolution of diverse species from common ancestor",synonyms:["evolutionary radiation","species radiation"]},{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-7726-2875"},profile:{shortBio:"Joana Meier leads a research group at the Wellcome Sanger Institute studying speciation genomics. She teaches at workshops including the Workshop on Population and Speciation Genomics, focusing on understanding genomic mechanisms of speciation.",source:"Wellcome Sanger Institute"}}},"meyer-britta":{id:"meyer-britta",name:"Britta Meyer",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Computational Biologist",affiliation:"University of Hamburg",department:"Faculty of Mathematics, Informatics and Natural Sciences"},academic:{researchAreas:{raw:["evolutionary immunogenomics","computational biology","population genomics","speciation genomics","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-2549-1825"},profile:{shortBio:"Britta Meyer is a computational biologist at University of Hamburg specializing in evolutionary immunogenomics. She is an instructor for the Workshop on Population and Speciation Genomics.",source:"University of Hamburg, evomics.org"}}},"molloy-erin":{id:"molloy-erin",name:"Erin Molloy",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"University of Maryland",department:"Department of Computer Science"},academic:{researchAreas:{raw:["computational phylogenetics","algorithms","phylogenetic trees","species tree estimation","computational biology","bioinformatics"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5553-3312"},profile:{shortBio:"Erin Molloy is an Assistant Professor in Computer Science at University of Maryland. Her research focuses on developing algorithms for phylogenetic tree estimation and computational methods for evolutionary biology.",source:"University of Maryland CS"}}},"montoliu-nern-merc":{id:"montoliu-nern-merc",name:"Merc Montoliu-Nerin",enrichment:{lastUpdated:"2025-07-02T21:22:16.670Z",confidence:"high",professional:{title:"Researcher",affiliation:"Uppsala University",department:"Department of Ecology and Genetics"},academic:{researchAreas:{raw:["arbuscular mycorrhizal fungi","genomics","evolutionary biology","symbiosis","phylogenomics","fungal biology"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"fungal-genomics",label:"Fungal Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of fungi",synonyms:["mycological genomics","fungal genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-5200-0411"},profile:{shortBio:"Merc Montoliu-Nerin specializes in arbuscular mycorrhizal fungi genomics at Uppsala University. She led groundbreaking phylogenomic analysis of AM fungi using comprehensive de novo genome assemblies, advancing our understanding of these crucial plant symbionts.",source:"Google Scholar, Uppsala University"}}},"montserrat-jordi-paps":{id:"montserrat-jordi-paps",name:"Jordi Paps Montserrat",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Dr",affiliation:"University of Bristol",department:"School of Biological Sciences"},academic:{researchAreas:{raw:["evolution","genomics","phylogenetics","animal kingdom evolution","land plants evolution","molecular evolution"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-2636-6636"},profile:{shortBio:"Dr Jordi Paps Montserrat studies major evolutionary transitions using molecular and genomic approaches. His lab aims to unravel the molecular and genomic changes underlying evolutionary shifts like the origins of animals or plants.",source:"University of Bristol"}}},"morlon-helene":{id:"morlon-helene",name:"Hlne Morlon",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"CNRS Research Director",affiliation:"cole Normale Suprieure",department:"Institute of Biology"},academic:{researchAreas:{raw:["macroevolution","macroecology","phylogenetics","biodiversity modeling","species diversification","computational ecology"],standardized:{primary:[{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"macroevolution",label:"Macroevolution",level:2,parentId:"evolutionary-biology",description:"Large-scale evolutionary patterns above species level",synonyms:["macroevolutionary patterns"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-3195-7521"},profile:{shortBio:"Hlne Morlon is CNRS Research Director at cole Normale Suprieure. A mathematician and ecologist specializing in biodiversity computational modeling, she studies ecological and evolutionary mechanisms shaping biological diversity on geological time scales. Recipient of CNRS Bronze (2015) and Silver (2021) medals.",source:"cole Normale Suprieure, Google Scholar"}}},"moyle-leonie":{id:"moyle-leonie",name:"Leonie Moyle",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Professor",affiliation:"Indiana University",department:"Department of Biology"},academic:{researchAreas:{raw:["evolutionary genetics","speciation","reproductive isolation","plant evolution","population genetics","tomato genetics"],standardized:{primary:[{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4960-8001"},profile:{shortBio:"Leonie Moyle is a Professor of Biology at Indiana University studying the genetics of speciation and reproductive isolation, particularly in wild tomato species. Her research focuses on understanding how new species evolve.",source:"Indiana University Biology"}}},"myers-simon":{id:"myers-simon",name:"Simon Myers",enrichment:{lastUpdated:"2025-07-01T16:00:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Oxford",department:"Department of Statistics and Wellcome Centre for Human Genetics"},academic:{researchAreas:{raw:["statistical genetics","population genetics","recombination","human evolution","computational methods","genetic variation"],standardized:{primary:[{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"genetic-variation",label:"Genetic Variation",level:3,parentId:"population-genetics",description:"Differences in DNA sequences among individuals",synonyms:["genetic diversity","allelic variation"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-2585-9626"},profile:{shortBio:"Simon Myers is a Professor at Oxford University developing statistical and computational methods to analyze genetic variation. His research focuses on understanding recombination, mutation, and selection in shaping genetic diversity.",source:"University of Oxford Statistics"}}},"neilsen-rasmus":{id:"neilsen-rasmus",name:"Rasmus Nielsen",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor of Computational Biology",affiliation:"UC Berkeley / University of Copenhagen",department:"Integrative Biology and Statistics",labWebsite:"https://nielsen-lab.github.io"},academic:{researchAreas:{raw:["population genetics","evolutionary genetics","statistical genomics","molecular evolution","human evolution","natural selection","adaptation","medical genetics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"clinical-genomics",label:"Clinical Genomics",level:2,parentId:"medical-clinical",description:"Application of genomics in clinical settings",synonyms:["medical genomics","clinical genetics"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]},{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0513-6591"},profile:{shortBio:"Rasmus Nielsen is a professor at UC Berkeley and University of Copenhagen. His research includes discovering genetic adaptations in Tibetans and Inuit populations. He develops statistical methods for genomic analyses including IM, ANGSD, and PAML.",source:"UC Berkeley, Google Scholar"}}},"nekrutenko-anton":{id:"nekrutenko-anton",name:"Anton Nekrutenko",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor and Huck Chair in Genomics",affiliation:"Pennsylvania State University",department:"Biochemistry and Molecular Biology",labWebsite:"https://nekrut.github.io/lab_site"},academic:{researchAreas:{raw:["bioinformatics","genomics","computational biology","reproducible research","data science","evolutionary genetics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5987-8032"},profile:{shortBio:"Anton Nekrutenko is the founding member of Galaxy Project, an open web-based platform for accessible, reproducible computational research. He holds the Dorothy Foher Huck and J. Lloyd Huck Chair in Genomics at Penn State.",source:"Penn State University, Google Scholar"}}},"nenasheva-natalia":{id:"nenasheva-natalia",name:"Natalia Nenasheva",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"medium",professional:{title:"PhD Student",affiliation:"University of Greifswald",department:"Institute of Mathematics and Computer Science"},academic:{researchAreas:{raw:["metagenomics","genome annotation","bioinformatics","gene prediction","computational biology","applied mathematics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"mathematical-biology",label:"Mathematical Biology",level:2,parentId:"mathematical-statistical",description:"Mathematical modeling of biological systems",synonyms:["biomathematics","theoretical biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]}],secondary:[{id:"gene-prediction",label:"Gene Prediction",level:3,parentId:"genome-annotation",description:"Computational identification of genes in genomes",synonyms:["gene finding","gene identification"]},{id:"genome-annotation",label:"Genome Annotation",level:3,parentId:"computational-methods",description:"Identification and labeling of genomic features",synonyms:["gene annotation","functional annotation"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Natalia Nenasheva is a PhD student at University of Greifswald working on gene prediction in eukaryotic metagenomes. She co-developed GALBA, a genome annotation pipeline using miniprot and AUGUSTUS. Her background includes Applied Mathematics and Physics from Moscow Institute of Physics and Technology.",source:"University of Greifswald, BMC Bioinformatics"}}},"nordborg-magnus":{id:"nordborg-magnus",name:"Magnus Nordborg",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"Scientific Director",affiliation:"Gregor Mendel Institute",department:"Austrian Academy of Sciences"},academic:{researchAreas:{raw:["population genetics","evolutionary biology","statistical genetics","Arabidopsis genomics","natural variation","quantitative genetics"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"quantitative-genetics",label:"Quantitative Genetics",level:2,parentId:"population-quantitative",description:"Study of complex traits influenced by multiple genes",synonyms:["polygenic traits","complex trait genetics"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"genetic-variation",label:"Genetic Variation",level:3,parentId:"population-genetics",description:"Differences in DNA sequences among individuals",synonyms:["genetic diversity","allelic variation"]},{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-7178-9748"},profile:{shortBio:'Magnus Nordborg is Scientific Director of the Gregor Mendel Institute. A population geneticist specializing in quantitative and computational analysis of genomic data, he was elected fellow of AAAS (2010) and member of EMBO (2015). He describes GMI as "a refuge for people with ideas".',source:"Gregor Mendel Institute, Google Scholar"}}},"nosil-patrik":{id:"nosil-patrik",name:"Patrik Nosil",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Director of Research",affiliation:"CEFE, CNRS",department:"Centre d'Ecologie Fonctionnelle et Evolutive",labWebsite:"https://www.cefe.cnrs.fr/fr/recherche/ee/gee/800-c/3615-patrik-nosil"},academic:{researchAreas:{raw:["speciation","evolution","adaptation","ecological genomics","population genetics","eco-evolutionary dynamics","stick insects"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"ecological-genomics",label:"Ecological Genomics",level:2,parentId:"ecology-environmental",description:"Integration of genomics and ecology",synonyms:["genomic ecology","eco-evolutionary genomics"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-8271-9005"},profile:{shortBio:"Patrik Nosil is an ERC Principal Investigator studying the evolutionary processes driving speciation, ecological effects of rapid evolution, and genetics of adaptation using Timema stick insects as a model system.",source:"CEFE CNRS, Google Scholar"}}},"nurk-sergey":{id:"nurk-sergey",name:"Sergey Nurk",enrichment:{lastUpdated:"2025-07-03T00:00:00.000Z",confidence:"high",professional:{title:"Staff Scientist",affiliation:"National Institutes of Health (NIH)",department:"Genome Informatics Section"},academic:{researchAreas:{raw:["genome assembly","telomere-to-telomere sequencing","bioinformatics","algorithms","T2T consortium","complete genome assembly"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-1301-5749"},profile:{shortBio:"Sergey Nurk is a Staff Scientist at NIH working on genome assembly algorithms. He is a key member of the Telomere-to-Telomere (T2T) Consortium that produced the first complete human genome assembly.",source:"NIH Genome Informatics"}}},"ocana-pallares-eduard":{id:"ocana-pallares-eduard",name:"Eduard Ocaa-Pallars",enrichment:{lastUpdated:"2025-07-03T03:06:03.050Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"Barcelona Supercomputing Center",department:"Life Sciences"},academic:{researchAreas:{raw:["evolutionary genomics","protist evolution","horizontal gene transfer","opisthokonta","comparative genomics","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]}],secondary:[{id:"horizontal-gene-transfer",label:"Horizontal Gene Transfer",level:3,parentId:"microbial-evolution",description:"Transfer of genetic material between organisms",synonyms:["HGT","lateral gene transfer"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-6040-4216"},profile:{shortBio:"Eduard Ocaa-Pallars studies the evolutionary trajectories of major eukaryotic lineages. His work on divergent genomic trajectories predating the origin of animals and fungi was published in Nature, revealing how Metazoa and Fungi evolved distinct genomic features.",source:"Google Scholar, Nature publications"}}},"ogilvie-huw":{id:"ogilvie-huw",name:"Huw Ogilvie",enrichment:{lastUpdated:"2025-07-03T03:06:03.050Z",confidence:"high",professional:{title:"Assistant Research Professor",affiliation:"Rice University",department:"Department of Computer Science"},academic:{researchAreas:{raw:["computational biology","phylogenetics","species tree inference","BEAST software","StarBEAST2","evolutionary algorithms"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-1589-6885"},profile:{shortBio:"Huw Ogilvie is the lead author of StarBEAST2, a widely used method for joint inference of species and gene trees. He develops computational methods for phylogenetic inference and helps coordinate development of PhyloNet software.",source:"Rice University CS"}}},"orlando-ludovic":{id:"orlando-ludovic",name:"Ludovic Orlando",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Director of Research",affiliation:"CNRS / University of Toulouse",department:"Centre for Anthropobiology and Genomics of Toulouse"},academic:{researchAreas:{raw:["paleogenomics","ancient DNA","evolutionary genomics","archaeogenetics","horse domestication","molecular archaeology"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"paleogenomics",label:"Paleogenomics",level:2,parentId:"genomics-omics",description:"Genomic analysis of ancient organisms",synonyms:["ancient genomics","paleogenetics"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-3936-1850"},profile:{shortBio:"Ludovic Orlando is a leading paleogeneticist and Director of Research at CNRS. He leads the Centre for Anthropobiology and Genomics of Toulouse, specializing in ancient DNA and the genomic history of domestication.",source:"CNRS Toulouse"}}},"paris-josephine":{id:"paris-josephine",name:"Josephine Paris",enrichment:{lastUpdated:"2025-07-02T20:47:51.165380Z",confidence:"high",professional:{title:"Workshop Director",affiliation:"University of Oxford",additionalRole:"Marie Skodowska-Curie Fellow"},academic:{researchAreas:{raw:["genomics","evolution","workshop organization","bioinformatics training","scientific education"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6868-3416"},profile:{shortBio:"Josephine Paris is the Workshop Director for the Workshop on Genomics. She is a Marie Skodowska-Curie Fellow at University of Oxford organizing intensive training opportunities in genomics.",source:"Evolution and Genomics workshops"}}},"paszkiewicz-konrad":{id:"paszkiewicz-konrad",name:"Konrad Paszkiewicz",enrichment:{lastUpdated:"2025-07-01T16:30:00.000Z",confidence:"high",professional:{title:"Head of Sequencing Service",affiliation:"University of Exeter",department:"Wellcome Trust Biomedical Informatics Hub"},academic:{researchAreas:{raw:["bioinformatics","sequencing technologies","genomics","data analysis","next-generation sequencing"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[{id:"data-analysis",label:"Data Analysis",level:3,parentId:"data-science",description:"General biological data analysis methods",synonyms:["biological data analysis","bioinformatics analysis"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Konrad Paszkiewicz heads the University of Exeter Sequencing Service and Wellcome Trust Biomedical Informatics Hub. He specializes in empowering biologists to interpret their genomic data.",source:"University of Exeter"}}},"pearson-william":{id:"pearson-william",name:"William Pearson",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Virginia",department:"Department of Biochemistry and Molecular Genetics"},academic:{researchAreas:{raw:["sequence analysis","FASTA algorithm","bioinformatics","protein similarity","computational biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]}],secondary:[{id:"sequence-analysis",label:"Sequence Analysis",level:3,parentId:"computational-methods",description:"Computational analysis of biological sequences",synonyms:["sequence alignment","sequence comparison"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-0727-3680"},profile:{shortBio:"William Pearson is the developer of the FASTA algorithm for sequence similarity searching, one of the foundational tools in bioinformatics. He is a Professor at University of Virginia.",source:"Academic profile"}}},"peichel-katie":{id:"peichel-katie",name:"Katie Peichel",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor and Head of Division",affiliation:"University of Bern",department:"Institute of Ecology and Evolution",labWebsite:"https://www.ee.iee.unibe.ch"},academic:{researchAreas:{raw:["evolutionary genetics","stickleback evolution","speciation","adaptation","sex chromosome evolution","behavioral evolution","genomics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"chromosome-evolution",label:"Chromosome Evolution",level:3,parentId:"evolutionary-genomics",description:"Evolution of chromosome structure and number",synonyms:["karyotype evolution","chromosomal evolution"]},{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7731-8944"},profile:{shortBio:"Catherine (Katie) Peichel pioneered the development of stickleback fish as a model system for evolutionary genetics. She is a member of the National Academy of Sciences and received an SNSF Advanced Grant.",source:"University of Bern, Google Scholar"}}},"peona-valentina":{id:"peona-valentina",name:"Valentina Peona",enrichment:{lastUpdated:"2025-07-02T19:45:00.000Z",confidence:"high",professional:{title:"Postdoc",affiliation:"Swedish Natural Museum / Swiss Vogelwarte",department:"Department of Organismal Biology; Systematic Biology"},academic:{researchAreas:{raw:["evolutionary biology","transposable elements","avian genomics","sex chromosome evolution","genome assembly","structural variation"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]}],secondary:[{id:"bird-genomics",label:"Bird Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of avian species",synonyms:["avian genomics","ornithological genomics"]},{id:"chromosome-evolution",label:"Chromosome Evolution",level:3,parentId:"evolutionary-genomics",description:"Evolution of chromosome structure and number",synonyms:["karyotype evolution","chromosomal evolution"]},{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"structural-variation",label:"Structural Variation",level:3,parentId:"population-genomics",description:"Large-scale genomic rearrangements and variations",synonyms:["SV analysis","genomic structural variants"]},{id:"transposable-elements",label:"Transposable Elements",level:3,parentId:"functional-genomics",description:"Mobile genetic elements",synonyms:["TEs","jumping genes","transposons"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5119-1837"},profile:{shortBio:"Valentina Peona is a postdoc at the Swedish Natural Museum and Swiss Vogelwarte, with visiting researcher status at Uppsala University. Her research focuses on how transposable elements and chromosome rearrangements shape genome evolution, particularly in birds. She has made significant contributions to understanding tuatara genome evolution and avian genome assembly.",source:"Google Scholar, Uppsala University"}}},"petr-martin":{id:"petr-martin",name:"Martin Petr",enrichment:{lastUpdated:"2025-07-01T16:30:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"University of Copenhagen",department:"Globe Institute"},academic:{researchAreas:{raw:["population genetics","admixtools","ancient DNA","human evolution","computational methods","R programming"],standardized:{primary:[{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Martin Petr developed the admixr R package providing an easy interface to ADMIXTOOLS for population genetics analyses. He specializes in computational methods for studying human evolution.",source:"admixr documentation"}}},"pettersson-olga-vinnere":{id:"pettersson-olga-vinnere",name:"Olga Vinnere Pettersson",enrichment:{lastUpdated:"2025-07-01T16:30:00.000Z",confidence:"high",professional:{title:"Project Manager",affiliation:"Uppsala University",department:"Department of Immunology, Genetics and Pathology",additionalRole:"Uppsala Genome Center"},academic:{researchAreas:{raw:["genomics","project management","sequencing facilities","bioinformatics infrastructure"],standardized:{primary:[{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[{id:"computational-infrastructure",label:"Computational Infrastructure",level:2,parentId:"technology-methods",description:"Infrastructure and platforms for computational biology",synonyms:["bioinformatics infrastructure","computational platforms"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5597-1870"},profile:{shortBio:"Olga Vinnere Pettersson is Project Manager at the Department of Immunology, Genetics and Pathology and involved with Uppsala Genome Center, supporting genomics research infrastructure.",source:"Uppsala University"}}},"pokorny-lisa":{id:"pokorny-lisa",name:"Lisa Pokorny",enrichment:{lastUpdated:"2025-07-01T16:30:00.000Z",confidence:"high",professional:{title:"Senior Research Leader",affiliation:"Royal Botanic Gardens, Kew",department:"Centre for Plant Biotechnology and Genomics"},academic:{researchAreas:{raw:["plant genomics","phylogenetics","cornales evolution","computational biology","botanical research"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-2478-8555"},profile:{shortBio:"Lisa Pokorny is a Senior Research Leader at Royal Botanic Gardens, Kew. Her research focuses on phylogenetic reconstruction and understanding plant evolution through genomic approaches.",source:"Royal Botanic Gardens, Kew"}}},"ponting-chris":{id:"ponting-chris",name:"Chris Ponting",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Edinburgh",department:"MRC Human Genetics Unit"},academic:{researchAreas:{raw:["computational genomics","functional genomics","human genetics","genome evolution","non-coding RNA"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"functional-genomics",label:"Functional Genomics",level:2,parentId:"genomics-omics",description:"Study of gene function and interactions at genome scale",synonyms:["genome function","functional genome analysis"],children:[]},{id:"human-genetics",label:"Human Genetics",level:2,parentId:"medical-clinical",description:"Study of human genetic variation and inheritance",synonyms:["human genetic analysis","human genomics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0202-7816"},profile:{shortBio:"Chris Ponting is a Professor at the MRC Human Genetics Unit, University of Edinburgh. His research focuses on computational approaches to understand genome function and evolution.",source:"University of Edinburgh"}}},"rees-jasmine":{id:"rees-jasmine",name:"Jasmine Rees",enrichment:{lastUpdated:"2025-01-02T21:00:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"University of Pennsylvania",department:"Tishkoff Lab"},academic:{researchAreas:{raw:["evolutionary genetics","computational biology","human adaptation","convergent evolution","population genomics","diet-related adaptation","trace metals"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genetics",label:"Evolutionary Genetics",level:2,parentId:"evolutionary-biology",description:"Genetic basis of evolutionary change",synonyms:["genetics of evolution","evolutionary genetic analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"convergent-evolution",label:"Convergent Evolution",level:3,parentId:"molecular-evolution",description:"Independent evolution of similar traits",synonyms:["evolutionary convergence","parallel evolution"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Jasmin Rees (note: spelled without 'e') is an evolutionary geneticist at University of Pennsylvania studying how environments influence genome evolution. She focuses on local adaptation in humans, particularly regarding micronutrients and diet. She completed her PhD at UCL and has taught at the Workshop on Population and Speciation Genomics.",source:"University of Pennsylvania, personal website"}}},"rengefors-karin":{id:"rengefors-karin",name:"Karin Rengefors",enrichment:{lastUpdated:"2025-07-01T16:30:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"Lund University",department:"Department of Biology"},academic:{researchAreas:{raw:["phytoplankton ecology","aquatic microbiology","algal genomics","microbial evolution","harmful algal blooms"],standardized:{primary:[{id:"environmental-microbiology",label:"Environmental Microbiology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in environmental contexts",synonyms:["microbial ecology","environmental microbiome"],children:[]},{id:"microbial-evolution",label:"Microbial Evolution",level:2,parentId:"microbiology-microbiome",description:"Evolution of microorganisms",synonyms:["bacterial evolution","prokaryotic evolution"],children:[]}],secondary:[{id:"marine-ecology",label:"Marine Ecology",level:3,parentId:"ecology",description:"Ecology of marine organisms and ecosystems",synonyms:["ocean ecology","marine biology"]},{id:"plant-genomics",label:"Plant Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of plants",synonyms:["botanical genomics","plant genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-5209-3160"},profile:{shortBio:"Karin Rengefors is a Professor at Lund University studying phytoplankton ecology and evolution, with a focus on understanding harmful algal blooms and microbial diversity in aquatic systems.",source:"Lund University Research Portal"}}},"rodriguez-ezpeleta-naiara":{id:"rodriguez-ezpeleta-naiara",name:"Naiara Rodriguez-Ezpeleta",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Head of Molecular Ecology and Biotechnology Unit",affiliation:"AZTI",department:"Marine Research Division"},academic:{researchAreas:{raw:["marine genomics","molecular ecology","biotechnology","transcriptomics","bioinformatics","fisheries genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"molecular-ecology",label:"Molecular Ecology",level:2,parentId:"ecology-environmental",description:"Molecular approaches to ecological questions",synonyms:["ecological genetics","molecular ecological analysis"],children:[]},{id:"synthetic-biology",label:"Synthetic Biology",level:2,parentId:"molecular-cellular",description:"Engineering of biological systems and organisms",synonyms:["synbio","biological engineering"],children:[]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6735-6755"},profile:{shortBio:"Naiara Rodriguez-Ezpeleta leads the Molecular Ecology and Biotechnology Unit at AZTI, integrating genomics, transcriptomics and bioinformatics approaches for marine environment research.",source:"AZTI, personal website"}}},"rokas-antonis":{id:"rokas-antonis",name:"Antonis Rokas",enrichment:{lastUpdated:"2025-07-02T20:47:51.165374Z",confidence:"high",professional:{title:"Cornelius Vanderbilt Chair, Professor",affiliation:"Vanderbilt University",department:"Department of Biological Sciences and Department of Biomedical Informatics"},academic:{researchAreas:{raw:["molecular evolution","phylogenetics","fungal genomics","computational biology","evolutionary genomics","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"fungal-genomics",label:"Fungal Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of fungi",synonyms:["mycological genomics","fungal genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7248-6551"},profile:{shortBio:"Antonis Rokas holds the Cornelius Vanderbilt Chair at Vanderbilt University. His research focuses on understanding the molecular foundations of the tree of life and the evolution of human pregnancy.",source:"Vanderbilt University"}}},"salzburger-walter":{id:"salzburger-walter",name:"Walter Salzburger",enrichment:{lastUpdated:"2025-07-01T17:00:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Basel",department:"Department of Environmental Sciences"},academic:{researchAreas:{raw:["evolutionary biology","cichlid fish","speciation","adaptive radiation","genomics","African Great Lakes"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"adaptive-radiation",label:"Adaptive Radiation",level:3,parentId:"speciation",description:"Rapid evolution of diverse species from common ancestor",synonyms:["evolutionary radiation","species radiation"]},{id:"fish-evolution",label:"Fish Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of fish species",synonyms:["ichthyological evolution","piscine evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-9988-1674"},profile:{shortBio:"Walter Salzburger studies the evolution of cichlid fishes, particularly their adaptive radiation in African Great Lakes. His research focuses on understanding the molecular basis of adaptation, innovation, and speciation.",source:"University of Basel"}}},"schleper-christa":{id:"schleper-christa",name:"Christa Schleper",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Professor, Head of Unit",affiliation:"University of Vienna",department:"Archaea Biology and Ecogenomics Unit"},academic:{researchAreas:{raw:["archaea biology","microbiology","ecogenomics","evolution","ammonia oxidation","environmental microbiology"],standardized:{primary:[{id:"ecological-genomics",label:"Ecological Genomics",level:2,parentId:"ecology-environmental",description:"Integration of genomics and ecology",synonyms:["genomic ecology","eco-evolutionary genomics"],children:[]},{id:"environmental-microbiology",label:"Environmental Microbiology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in environmental contexts",synonyms:["microbial ecology","environmental microbiome"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"microbiology-microbiome",label:"Microbiology & Microbiome Sciences",level:1,description:"Study of microorganisms and microbial communities including their diversity and interactions",icon:"",children:["microbiome-research","microbial-ecology","microbial-genomics","virology","bacteriology","mycology","parasitology","microbial-evolution","antimicrobial-resistance"]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1918-2735"},profile:{shortBio:"Christa Schleper is Head of the Archaea Biology and Ecogenomics Unit at University of Vienna. Her research focuses on archaea, particularly ammonia-oxidizing archaea and their role in global nitrogen cycling.",source:"University of Vienna"}}},"scofield-doug":{id:"scofield-doug",name:"Doug Scofield",enrichment:{lastUpdated:"2025-07-02T20:38:09.908Z",confidence:"medium",professional:{affiliation:"Uppsala University",department:"Evolutionary Biology Centre",title:"Senior Bioinformatician"},academic:{researchAreas:{raw:["bioinformatics","genomics","population genetics","evolutionary biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5235-6461"},profile:{source:"Workshop participation records"}}},"scornavacca-celine":{id:"scornavacca-celine",name:"Celine Scornavacca",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Senior Researcher",affiliation:"CNRS",department:"ISEM - Institute of Evolutionary Sciences"},academic:{researchAreas:{raw:["phylogenetic networks","computational phylogenetics","evolutionary biology","algorithms","mathematical biology"],standardized:{primary:[{id:"algorithm-development",label:"Algorithm Development",level:2,parentId:"computational-sciences",description:"Development of algorithms for biological data analysis",synonyms:["algorithmic bioinformatics","computational algorithm design"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"mathematical-biology",label:"Mathematical Biology",level:2,parentId:"mathematical-statistical",description:"Mathematical modeling of biological systems",synonyms:["biomathematics","theoretical biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[{id:"phylogenetic-networks",label:"Phylogenetic Networks",level:3,parentId:"phylogenetics",description:"Network representations of evolutionary relationships",synonyms:["reticulate evolution","phylogenetic graphs"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-3758-273X"},profile:{shortBio:"Celine Scornavacca is a Senior Researcher at CNRS working on phylogenetic networks and computational methods for understanding reticulate evolution.",source:"ISEM, CNRS"}}},"sedlazeck-fritz":{id:"sedlazeck-fritz",name:"Fritz Sedlazeck",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"Baylor College of Medicine",department:"Human Genome Sequencing Center"},academic:{researchAreas:{raw:["structural variation","genome assembly","long-read sequencing","computational genomics","bioinformatics tools","variant calling"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"structural-variation",label:"Structural Variation",level:3,parentId:"population-genomics",description:"Large-scale genomic rearrangements and variations",synonyms:["SV analysis","genomic structural variants"]},{id:"variant-calling",label:"Variant Calling",level:3,parentId:"computational-methods",description:"Identification of genetic variants from sequencing data",synonyms:["SNP calling","variant detection"]}],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6040-2691"},profile:{shortBio:"Fritz Sedlazeck develops computational methods for detecting structural variations using long-read sequencing technologies. He is a key developer of tools for analyzing complex genomic variations.",source:"Academic profile"}}},"setter-derek":{id:"setter-derek",name:"Derek Setter",enrichment:{lastUpdated:"2025-07-01T17:00:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"University of Edinburgh",department:"MRC Human Genetics Unit"},academic:{researchAreas:{raw:["population genetics","evolutionary genomics","natural selection","demography","statistical inference","genomic data analysis"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"demographic-inference",label:"Demographic Inference",level:3,parentId:"population-genetics",description:"Inference of population demographic history",synonyms:["population history","demographic modeling"]},{id:"genomic-data-analysis",label:"Genomic Data Analysis",level:3,parentId:"data-analysis",description:"Analysis of genome-scale datasets",synonyms:["genome data processing","genomic bioinformatics"]},{id:"natural-selection",label:"Natural Selection",level:3,parentId:"adaptation",description:"Process of differential survival and reproduction",synonyms:["selection pressure","adaptive selection"]},{id:"statistical-inference",label:"Statistical Inference",level:3,parentId:"biostatistics",description:"Drawing conclusions from biological data",synonyms:["inferential statistics","hypothesis testing"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7101-443X"},profile:{shortBio:"Derek Setter is a theoretical population geneticist interested in how evolutionary processes shape genetic variation. He develops inference frameworks for understanding selection and demography from genomic data.",source:"University of Edinburgh Research Explorer"}}},"shaw-sophie":{id:"shaw-sophie",name:"Sophie Shaw",enrichment:{lastUpdated:"2025-07-02T21:00:00.000Z",confidence:"high",professional:{title:"Lead Bioinformatician for Research and Development",affiliation:"All Wales Medical Genomics Service",department:"NHS Wales"},academic:{researchAreas:{raw:["bioinformatics","next generation sequencing","microbiome","metagenomics","clinical genomics","microbial ecology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"clinical-genomics",label:"Clinical Genomics",level:2,parentId:"medical-clinical",description:"Application of genomics in clinical settings",synonyms:["medical genomics","clinical genetics"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Sophie Shaw is the Lead Bioinformatician for Research and Development at the All Wales Medical Genomics Service. Her research focuses on microbiome studies, particularly in clinical contexts such as ulcerative colitis and trans-kingdom microbial communication. She specializes in next-generation sequencing and metagenomics applications.",source:"Google Scholar, NHS Wales"}}},"shekhar-karthik":{id:"shekhar-karthik",name:"Karthik Shekhar",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"John F. Heil, Jr. Professor",affiliation:"University of California, Berkeley",department:"Chemical and Biomolecular Engineering",additionalAffiliation:"Helen Wills Neuroscience Institute"},academic:{researchAreas:{raw:["single-cell genomics","neuroscience","retinal biology","computational biology","systems biology","cell type diversity"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"neurobiology",label:"Neurobiology",level:2,parentId:"molecular-cellular",description:"Study of the nervous system and neural mechanisms",synonyms:["neuroscience","neural biology"],children:[]},{id:"single-cell-genomics",label:"Single-Cell Genomics",level:2,parentId:"genomics-omics",description:"Analysis of genomes, transcriptomes, and epigenomes at single-cell resolution",synonyms:["single cell genomics","single-cell sequencing"],children:[]},{id:"systems-biology",label:"Systems Biology",level:2,parentId:"molecular-cellular",description:"Holistic study of biological systems and their interactions",synonyms:["biological systems analysis","integrative biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4349-6600"},profile:{shortBio:"Karthik Shekhar uses single-cell genomics to understand cell type diversity in the nervous system, particularly in the retina. He combines experimental and computational approaches to map cellular diversity.",source:"UC Berkeley"}}},"simakov-oleg":{id:"simakov-oleg",name:"Oleg Simakov",enrichment:{lastUpdated:"2025-07-03T03:06:03.050Z",confidence:"high",professional:{title:"Univ.-Prof. Dr.",affiliation:"University of Vienna",department:"Faculty of Life Sciences, Department of Neuroscience and Developmental Biology"},academic:{researchAreas:{raw:["evolutionary genomics","comparative genomics","cephalopod biology","animal evolution","genome evolution","developmental biology"],standardized:{primary:[{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"developmental-biology",label:"Developmental Biology",level:2,parentId:"molecular-cellular",description:"Study of organism development from embryo to adult",synonyms:["developmental genetics","embryology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-3585-4511"},profile:{shortBio:"Oleg Simakov studies genome evolution and the origins of animal complexity, with a particular focus on cephalopods. His research uses comparative genomics to understand major evolutionary transitions.",source:"University of Vienna"}}},"skidmore-zachary":{id:"skidmore-zachary",name:"Zachary Skidmore",enrichment:{lastUpdated:"2025-07-03T03:06:03.050Z",confidence:"high",professional:{title:"Staff Scientist",affiliation:"Washington University in St. Louis",department:"McDonell Genome Institute, Department of Medicine"},academic:{researchAreas:{raw:["cancer genomics","bioinformatics","precision oncology","variant analysis","clinical genomics","computational biology"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"cancer-genomics",label:"Cancer Genomics",level:2,parentId:"medical-clinical",description:"Genomic analysis of cancer",synonyms:["oncogenomics","tumor genomics"],children:[]},{id:"clinical-genomics",label:"Clinical Genomics",level:2,parentId:"medical-clinical",description:"Application of genomics in clinical settings",synonyms:["medical genomics","clinical genetics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"precision-medicine",label:"Precision Medicine",level:2,parentId:"medical-clinical",description:"Tailored medical treatment based on genetic information",synonyms:["personalized medicine","genomic medicine"],children:[]}],secondary:[{id:"variant-calling",label:"Variant Calling",level:3,parentId:"computational-methods",description:"Identification of genetic variants from sequencing data",synonyms:["SNP calling","variant detection"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5811-0200"},profile:{shortBio:"Zachary Skidmore is a staff scientist at the McDonell Genome Institute specializing in oncology bioinformatics. He develops tools and analyses for cancer genomics and precision medicine.",source:"Griffith Lab, Washington University"}}},"sousa-vitor":{id:"sousa-vitor",name:"Vitor Sousa",enrichment:{lastUpdated:"2025-07-02T21:08:43.665Z",confidence:"high",professional:{title:"Researcher",affiliation:"University of Lisbon",department:"Centre for Ecology, Evolution and Environmental Changes (cE3c)"},academic:{researchAreas:{raw:["population genetics","evolutionary biology","demography","speciation genetics","conservation genetics","statistical inference"],standardized:{primary:[{id:"conservation-genetics",label:"Conservation Genetics",level:2,parentId:"population-quantitative",description:"Application of genetics to conservation biology",synonyms:["conservation genomics","genetic conservation"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"speciation",label:"Speciation",level:2,parentId:"evolutionary-biology",description:"Study of the formation of new species",synonyms:["species formation","speciation process"],children:[]}],secondary:[{id:"demographic-inference",label:"Demographic Inference",level:3,parentId:"population-genetics",description:"Inference of population demographic history",synonyms:["population history","demographic modeling"]},{id:"statistical-inference",label:"Statistical Inference",level:3,parentId:"biostatistics",description:"Drawing conclusions from biological data",synonyms:["inferential statistics","hypothesis testing"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-3575-0875"},profile:{shortBio:"Vitor Sousa uses population genetic data and theory to reconstruct evolutionary history and understand the roles of demography and selection in structuring genetic diversity.",source:"University of Lisbon, cE3c"}}},"speidel-leo":{id:"speidel-leo",name:"Leo Speidel",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"RIKEN ECL Research Unit Leader",affiliation:"RIKEN",department:"Mathematical Genomics, iTHEMS"},academic:{researchAreas:{raw:["population genetics","statistical genetics","ancient DNA","genealogy estimation","human evolution","computational biology"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]},{id:"human-evolution",label:"Human Evolution",level:3,parentId:"evolutionary-biology",description:"Evolution of humans and their ancestors",synonyms:["hominin evolution","anthropological genetics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0002-4644-8033"},profile:{shortBio:"Leo Speidel is a RIKEN ECL Research Unit Leader in Mathematical Genomics at RIKEN iTHEMS. He develops statistical tools to reconstruct human genetic history using modern and ancient genomes, focusing on past migrations, introgression, and adaptation.",source:"University of Oxford, RIKEN, Nature Genetics"}}},"stajich-jason":{id:"stajich-jason",name:"Jason Stajich",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Professor",affiliation:"University of California, Riverside",department:"Microbiology and Plant Pathology",labWebsite:"https://lab.stajich.org"},academic:{researchAreas:{raw:["fungal evolution","fungal genomics","mycology","bioinformatics","comparative genomics","microbe-microbe interactions","extremophile fungi","molecular evolution"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]}],secondary:[{id:"fungal-evolution",label:"Fungal Evolution",level:3,parentId:"microbial-evolution",description:"Evolution of fungal species",synonyms:["mycological evolution","fungal phylogeny"]},{id:"fungal-genomics",label:"Fungal Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of fungi",synonyms:["mycological genomics","fungal genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-7591-0020"},profile:{shortBio:"Jason Stajich studies fungal evolution using comparative, computational, and experimental tools. He was elected as a 2020 AAAS fellow and leads research on zygomycete and chytrid fungi genomics at UC Riverside.",source:"UC Riverside, Google Scholar"}}},"stanke-mario":{id:"stanke-mario",name:"Mario Stanke",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Professor of Bioinformatics",affiliation:"University of Greifswald",department:"Institute of Mathematics and Computer Science"},academic:{researchAreas:{raw:["gene prediction","comparative genome analysis","machine learning","probabilistic graphical models","molecular evolution","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"machine-learning",label:"Machine Learning",level:2,parentId:"mathematical-statistical",description:"Machine learning applications in biology",synonyms:["ML in biology","artificial intelligence in biology"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]}],secondary:[{id:"gene-prediction",label:"Gene Prediction",level:3,parentId:"genome-annotation",description:"Computational identification of genes in genomes",synonyms:["gene finding","gene identification"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-8696-0384"},profile:{shortBio:"Mario Stanke is Professor of Bioinformatics at the University of Greifswald. He is the developer of AUGUSTUS, a gene prediction software, and works on genome annotation pipelines including BRAKER.",source:"Google Scholar, University of Greifswald"}}},"steenwyk-jacob":{id:"steenwyk-jacob",name:"Jacob Steenwyk",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"UC Berkeley / HHMI",department:"Evolutionary Studies"},academic:{researchAreas:{raw:["evolutionary biology","genomics","bioinformatics","phylogenomics","molecular evolution","fungal genomics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"fungal-genomics",label:"Fungal Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of fungi",synonyms:["mycological genomics","fungal genome analysis"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-8436-595X"},profile:{shortBio:"Jacob Steenwyk is Chair of Alumni Affairs, Evolutionary Studies Postdoctoral Researcher at UC Berkeley / HHMI. His research focuses on evolutionary biology, genomics, and bioinformatics, with a particular emphasis on the evolution and function of genomes.",source:"Vanderbilt University, LinkedIn"}}},"steffen-karin":{id:"steffen-karin",name:"Karin Steffen",enrichment:{lastUpdated:"2025-07-02T20:30:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"Vanderbilt University",department:"Department of Biological Sciences"},academic:{researchAreas:{raw:["genomics","metabolomics","evolution","marine biology","sponge biology","host-microbe interactions"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"metabolomics",label:"Metabolomics",level:2,parentId:"genomics-omics",description:"Large-scale study of metabolites",synonyms:["metabolome analysis","metabolic profiling"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]}],secondary:[{id:"marine-ecology",label:"Marine Ecology",level:3,parentId:"ecology",description:"Ecology of marine organisms and ecosystems",synonyms:["ocean ecology","marine biology"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0003-0499-1430"},profile:{shortBio:"Karin Steffen is a postdoctoral researcher at Vanderbilt University specializing in genomics, metabolomics, and evolution. Her research focuses on marine organisms, particularly sponges and their associated microbiomes, studying convergent morphology in Alpineae and host-defense mechanisms in marine environments.",source:"Google Scholar, ORCID, Vanderbilt University"}}},"stefka-jan":{id:"stefka-jan",name:"Jan Stefka",enrichment:{lastUpdated:"2025-07-02T18:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of South Bohemia",department:"Faculty of Science, Biology Centre CAS"},academic:{researchAreas:{raw:["population genomics","parasitic evolution","phylogenetics","evolutionary biology","fish parasites","coevolution"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-9735-8479"},profile:{shortBio:"Jan Stefka is an Associate Professor at the Faculty of Science, University of South Bohemia. His research focuses on population genomics of pathogenic fish parasites and understanding the role of population structure and selection in parasitic evolution.",source:"Evomics.org, Fulbright Scholars"}}},"stepanauskas-ramunas":{id:"stepanauskas-ramunas",name:"Ramunas Stepanauskas",enrichment:{lastUpdated:"2025-07-03T18:15:00.000Z",confidence:"high",professional:{title:"Senior Research Scientist",affiliation:"Bigelow Laboratory for Ocean Sciences",department:"Single Cell Genomics Center",labWebsite:"https://www.bigelow.org",orcid:"0000-0003-4458-3108"},academic:{researchAreas:{raw:["single cell genomics","marine microbiology","environmental genomics","microbial ecology","synthetic biology","bioremediation"],standardized:{primary:[{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"environmental-microbiology",label:"Environmental Microbiology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in environmental contexts",synonyms:["microbial ecology","environmental microbiome"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]},{id:"single-cell-genomics",label:"Single-Cell Genomics",level:2,parentId:"genomics-omics",description:"Analysis of genomes, transcriptomes, and epigenomes at single-cell resolution",synonyms:["single cell genomics","single-cell sequencing"],children:[]},{id:"synthetic-biology",label:"Synthetic Biology",level:2,parentId:"molecular-cellular",description:"Engineering of biological systems and organisms",synonyms:["synbio","biological engineering"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Ramunas Stepanauskas is Senior Research Scientist and Director of the Single Cell Genomics Center at Bigelow Laboratory for Ocean Sciences. He pioneered single cell genomics techniques to study uncultured microorganisms in marine environments.",source:"Bigelow Laboratory for Ocean Sciences"}}},"steward-rachel":{id:"steward-rachel",name:"Rachel Steward",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"Stockholm University",department:"Department of Biological Sciences"},academic:{researchAreas:{raw:["evolution","plasticity","environmental biology","entomology","behavioral ecology","insect-plant interactions"],standardized:{primary:[{id:"developmental-biology",label:"Developmental Biology",level:2,parentId:"molecular-cellular",description:"Study of organism development from embryo to adult",synonyms:["developmental genetics","embryology"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]}],secondary:[{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-8610-334X"},profile:{shortBio:"Rachel Steward studies evolution and plasticity in response to variable environments. Her PhD focused on genetic, environmental, and behavioral constraints contributing to persistent maladaptive use of an invasive host plant by a native insect.",source:"Lund University, Stockholm University"}}},"suh-alexander":{id:"suh-alexander",name:"Alexander Suh",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Uppsala University",department:"Department of Organismal Biology"},academic:{researchAreas:{raw:["evolutionary genomics","convergent evolution","phylogenomics","transposable elements","chromosome evolution","avian genomics"],standardized:{primary:[{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"phylogenomics",label:"Phylogenomics",level:2,parentId:"evolutionary-biology",description:"Phylogenetic analysis using genome-scale data",synonyms:["genomic phylogenetics","phylogenomic analysis"],children:[]}],secondary:[{id:"bird-genomics",label:"Bird Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of avian species",synonyms:["avian genomics","ornithological genomics"]},{id:"chromosome-evolution",label:"Chromosome Evolution",level:3,parentId:"evolutionary-genomics",description:"Evolution of chromosome structure and number",synonyms:["karyotype evolution","chromosomal evolution"]},{id:"convergent-evolution",label:"Convergent Evolution",level:3,parentId:"molecular-evolution",description:"Independent evolution of similar traits",synonyms:["evolutionary convergence","parallel evolution"]},{id:"transposable-elements",label:"Transposable Elements",level:3,parentId:"functional-genomics",description:"Mobile genetic elements",synonyms:["TEs","jumping genes","transposons"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-8979-9992"},profile:{shortBio:"Alexander Suh is a researcher at Uppsala University focusing on evolutionary genomics. His work provides insights into the processes underpinning convergent evolution and the contributions of ancestral, introgressed, and novel genetic variation to phenotypic evolution.",source:"ResearchGate"}}},"swofford-david":{id:"swofford-david",name:"David Swofford",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Professor Emeritus",affiliation:"Duke University",department:"Department of Biology"},academic:{researchAreas:{raw:["phylogenetics","systematics","computational biology","evolutionary biology","statistical methods","bioinformatics"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"biostatistics",label:"Biostatistics",level:2,parentId:"mathematical-statistical",description:"Statistical methods in biological research",synonyms:["biological statistics","statistical biology"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0396-1395"},profile:{shortBio:"David Swofford is the author and developer of PAUP* (Phylogenetic Analysis Using Parsimony), one of the most widely used software packages for phylogenetic analysis. He is a pioneer in computational phylogenetics and systematic biology.",source:"PAUP* website"}}},"szollosi-gergely":{id:"szollosi-gergely",name:"Gergely Szollosi",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"Etvs Lornd University",department:"Institute of Physics"},academic:{researchAreas:{raw:["evolutionary genomics","phylogenetics","gene family evolution","computational biology","reconciliation methods","molecular evolution"],standardized:{primary:[{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-8556-845X"},profile:{shortBio:"Gergely Szllsi heads the Evolutionary Genomics Research Group at Etvs University in Budapest. He develops computational methods to reconstruct gene family evolution and was a Marie Curie Fellow at the Bioinformatics, Phylogenetics and Evolutionary Genomics group.",source:"ResearchGate, LinkedIn"}}},"tavalire-hannah":{id:"tavalire-hannah",name:"Hannah Tavalire",enrichment:{lastUpdated:"2025-07-03T18:15:00.000Z",confidence:"high",professional:{title:"Postdoctoral Scholar",affiliation:"University of Oregon",department:"Prevention Science Institute",orcid:"0000-0003-2360-1119"},academic:{researchAreas:{raw:["gut microbiome","childhood development","genetics","environmental health","early growth","microbiome composition"],standardized:{primary:[{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Hannah Tavalire, PhD, is a postdoctoral scholar in the Leve research group at the Prevention Science Institute. Her research centers on characterizing the influences of genetics and rearing environment on gut microbiome composition in children and how the microbiome affects overall health.",source:"Prevention Science Institute, University of Oregon"}}},"taylor-james":{id:"taylor-james",name:"James Taylor",enrichment:{lastUpdated:"2025-07-02T18:30:00.000Z",confidence:"high",professional:{title:"Professor (1943-2020)",affiliation:"Johns Hopkins University",department:"Department of Biology"},academic:{researchAreas:{raw:["bioinformatics","computational biology","genomics","Galaxy platform","data science","open science"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"James Taylor was a pioneer in computational biology and creator of the Galaxy platform for genomic data analysis. He revolutionized bioinformatics by making computational tools accessible to researchers without programming expertise. He passed away in 2020.",source:"Johns Hopkins University, bio.jhu.edu"}}},"trucchi-emiliano":{id:"trucchi-emiliano",name:"Emiliano Trucchi",enrichment:{lastUpdated:"2025-07-02T20:47:51.165384Z",confidence:"high",professional:{title:"Researcher",affiliation:"Marche Polytechnic University",department:"Department of Life and Environmental Sciences"},academic:{researchAreas:{raw:["evolutionary genomics","population genomics","bioinformatics","non-model species","demographic inference","adaptation genomics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"demographic-inference",label:"Demographic Inference",level:3,parentId:"population-genetics",description:"Inference of population demographic history",synonyms:["population history","demographic modeling"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-1270-5273"},profile:{shortBio:"Emiliano Trucchi specializes in evolutionary dynamics in non-model species. His expertise in non-model species genomics has led to novel approaches for estimating demographic histories and identifying genome-wide methylation patterns to test models of genetic diversity decay.",source:"Google Scholar, evomics.org"}}},"trynka-gosia":{id:"trynka-gosia",name:"Gosia Trynka",enrichment:{lastUpdated:"2025-07-03T18:15:00.000Z",confidence:"high",professional:{title:"Group Leader",affiliation:"Wellcome Sanger Institute",department:"Human Genetics",orcid:"0000-0002-6955-9529"},academic:{researchAreas:{raw:["complex genetics","genomics","bioinformatics","immunogenetics","epigenetics","autoimmune diseases"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"immunology",label:"Immunology",level:2,parentId:"medical-clinical",description:"Study of immune systems and immune responses",synonyms:["immunogenetics","immune system biology"],children:[]},{id:"quantitative-genetics",label:"Quantitative Genetics",level:2,parentId:"population-quantitative",description:"Study of complex traits influenced by multiple genes",synonyms:["polygenic traits","complex trait genetics"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Gosia Trynka is a Group Leader at the Wellcome Sanger Institute. Her group aims to understand the role of genetic variants in modulating the immune system, with the ultimate goal to identify mechanisms through which genetic variants predispose to autoimmune diseases.",source:"Wellcome Sanger Institute, Cambridge Immunology Network"}}},"tsambos-georgia":{id:"tsambos-georgia",name:"Georgia Tsambos",enrichment:{lastUpdated:"2025-07-02T22:06:27.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"University of Washington",department:"Genome Sciences"},academic:{researchAreas:{raw:["computational genomics","bioinformatics","population genetics","ancestry simulation","statistical genetics","tree sequences"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]},{id:"statistical-genetics",label:"Statistical Genetics",level:2,parentId:"population-quantitative",description:"Statistical methods for analyzing genetic data",synonyms:["genetic statistics","quantitative genetics methods"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"},orcid:"0000-0001-7001-2275"},profile:{shortBio:"Georgia Tsambos is a Postdoc in Kelley Harris lab at the University of Washington. She completed her PhD in statistical genetics at the University of Melbourne, focusing on genetic signatures of admixture and local ancestry.",source:"University of Melbourne, bioRxiv"}}},"uliano-silva-marcela":{id:"uliano-silva-marcela",name:"Marcela Uliano-Silva",enrichment:{lastUpdated:"2025-07-02T21:22:16.669Z",confidence:"high",professional:{title:"Senior Bioinformatician",affiliation:"Wellcome Sanger Institute",department:"Tree of Life Programme"},academic:{researchAreas:{raw:["genome assembly","bioinformatics","comparative genomics","eukaryotic species","Darwin Tree of Life","chromosome-level genomes"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6723-4715"},profile:{shortBio:"Marcela Uliano-Silva is a TED Fellow and Senior Bioinformatician at the Wellcome Sanger Institute, specializing in genome assembly and bioinformatics. She plays a crucial role in the Darwin Tree of Life Project, focusing on creating chromosome-level genomes for various eukaryotic species.",source:"Wellcome Sanger Institute, TED Fellows"}}},"uller-tobias":{id:"uller-tobias",name:"Tobias Uller",enrichment:{lastUpdated:"2025-07-02T23:25:26.306Z",confidence:"high",professional:{title:"Professor",affiliation:"Lund University",department:"Department of Biology"},academic:{researchAreas:{raw:["evolutionary biology","developmental biology","ecology","developmental plasticity","evolutionary theory","mathematical modeling"],standardized:{primary:[{id:"developmental-biology",label:"Developmental Biology",level:2,parentId:"molecular-cellular",description:"Study of organism development from embryo to adult",synonyms:["developmental genetics","embryology"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"mathematical-biology",label:"Mathematical Biology",level:2,parentId:"mathematical-statistical",description:"Mathematical modeling of biological systems",synonyms:["biomathematics","theoretical biology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-1293-5842"},profile:{shortBio:"Tobias Uller studies how evolution works through an integrative approach spanning from molecular and developmental biology to ecology, guided by mathematical modeling and conceptual analysis. His research focuses on development, physiology and behavior in evolutionary processes.",source:"Lund University"}}},"vaughan-tim":{id:"vaughan-tim",name:"Tim Vaughan",enrichment:{lastUpdated:"2025-07-03T00:00:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Centre for Computational Evolution",department:"Computational Evolution"},academic:{researchAreas:{raw:["computational phylogenetics","BEAST 2","evolutionary biology","population genetics","phylodynamics","molecular evolution"],standardized:{primary:[{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"molecular-evolution",label:"Molecular Evolution",level:2,parentId:"evolutionary-biology",description:"Evolution at the molecular level including DNA, RNA, and protein sequences",synonyms:["molecular evolutionary biology","evolutionary molecular biology"],children:[]},{id:"phylogenetics",label:"Phylogenetics",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary relationships among organisms",synonyms:["phylogenetic analysis","phylogenetic inference"],children:[]},{id:"population-genetics",label:"Population Genetics",level:2,parentId:"population-quantitative",description:"Study of genetic variation and change within populations",synonyms:["population genetic analysis","genetic population structure"],children:[]}],secondary:[{id:"phylodynamics",label:"Phylodynamics",level:3,parentId:"phylogenetics",description:"Study of interaction between epidemiology and pathogen evolution",synonyms:["viral phylodynamics","epidemiological phylogenetics"]}],techniques:[{id:"computational-methods",label:"Computational Methods",level:2,parentId:"technology-methods",description:"Computational approaches and tools for biological analysis",synonyms:["computational tools","bioinformatics methods"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-5755-0060"},profile:{shortBio:"Tim Vaughan specializes in computational phylogenetics using BEAST 2. His work focuses on evolutionary processes that give rise to biodiversity at every level of biological organization, from species to individual organisms and genes.",source:"GitHub, Centre for Computational Evolution"}}},"vazquez-baeza-yoshiki":{id:"vazquez-baeza-yoshiki",name:"Yoshiki Vazquez-Baeza",enrichment:{lastUpdated:"2025-07-03T03:06:03.050Z",confidence:"high",professional:{title:"Bioinformatics Lead",affiliation:"BiomeSense",department:"Computational Biology and Data Engineering"},academic:{researchAreas:{raw:["microbiome","bioinformatics","data visualization","high-dimensional data","computational biology","statistical analysis"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"biostatistics",label:"Biostatistics",level:2,parentId:"mathematical-statistical",description:"Statistical methods in biological research",synonyms:["biological statistics","statistical biology"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"data-science",label:"Data Science",level:2,parentId:"computational-sciences",description:"Data science approaches in biology",synonyms:["biological data science","biomedical data science"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6014-2009"},profile:{shortBio:"Yoshiki Vazquez-Baeza leads computational biology and data engineering at BiomeSense. Previously Associate Director of Bioinformatics at UC San Diego's Center for Microbiome Innovation, he develops open source statistical analysis and visualization tools for microbiome research.",source:"LinkedIn, The Org"}}},"videvall-elin":{id:"videvall-elin",name:"Elin Videvall",enrichment:{lastUpdated:"2025-07-03T18:15:00.000Z",confidence:"high",professional:{title:"Researcher",affiliation:"Uppsala University",department:"Department of Ecology and Genetics",labWebsite:"http://www.videvall.com",orcid:"0000-0002-9998-3689"},academic:{researchAreas:{raw:["microbiome","genomics","transcriptomics","avian malaria","host-pathogen interactions","gut microbiota"],standardized:{primary:[{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]},{id:"pathogen-genomics",label:"Pathogen Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of pathogenic organisms",synonyms:["pathogen genetics","infectious disease genomics"],children:[]},{id:"transcriptomics",label:"Transcriptomics",level:2,parentId:"genomics-omics",description:"Study of RNA transcripts produced by the genome",synonyms:["RNA-seq analysis","gene expression profiling"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"}},profile:{shortBio:"Elin Videvall leads the Videvall Lab studying animals and their microbes. She uses high-throughput sequencing and bioinformatics to study the evolution of genomes, transcriptomes and microbiomes. She is Associate Editor for Molecular Ecology and Molecular Ecology Resources.",source:"Videvall Lab website, Uppsala University"}}},"waterhouse-robert":{id:"waterhouse-robert",name:"Robert Waterhouse",enrichment:{lastUpdated:"2025-07-02T23:33:56.657Z",confidence:"high",professional:{title:"Professor",affiliation:"University of Lausanne",department:"Department of Ecology and Evolution",labWebsite:"https://www.rmwaterhouse.org"},academic:{researchAreas:{raw:["environmental bioinformatics","comparative genomics","evolutionary genomics","genome assembly","arthropod genomics","biodiversity genomics"],standardized:{primary:[{id:"biodiversity",label:"Biodiversity",level:2,parentId:"ecology-environmental",description:"Study of biological diversity at all levels",synonyms:["biological diversity","species diversity"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]}],secondary:[{id:"genome-assembly",label:"Genome Assembly",level:3,parentId:"sequencing-technology",description:"Computational reconstruction of genomes from sequencing data",synonyms:["genome reconstruction","sequence assembly"]},{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-4199-9052"},profile:{shortBio:"Robert Waterhouse leads the Environmental Bioinformatics Group at SIB Swiss Institute of Bioinformatics and University of Lausanne. His research focuses on elucidating interactions between gene evolution and gene function through computational approaches, with emphasis on genomics data for planetary preservation.",source:"SIB Swiss Institute of Bioinformatics, University of Lausanne"}}},"westbury-michael":{id:"westbury-michael",name:"Michael Westbury",enrichment:{lastUpdated:"2025-07-02T23:33:56.658Z",confidence:"high",professional:{title:"Assistant Professor",affiliation:"University of Copenhagen",department:"Globe Institute, Section for Molecular Ecology and Evolution"},academic:{researchAreas:{raw:["paleogenomics","population genomics","evolutionary genomics","ancient DNA","extinct species","climate adaptation"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"paleogenomics",label:"Paleogenomics",level:2,parentId:"genomics-omics",description:"Genomic analysis of ancient organisms",synonyms:["ancient genomics","paleogenetics"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"ancient-dna",label:"Ancient DNA",level:3,parentId:"paleogenomics",description:"DNA recovered from ancient specimens",synonyms:["aDNA","paleoDNA"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0136-5889"},profile:{shortBio:"Michael Westbury at University of Copenhagen specializes in paleogenomics and evolutionary genomics. He uses DNA from ancient and historical specimens to understand population dynamics, speciation, hybridization, and adaptation in response to climate change.",source:"University of Copenhagen, Google Scholar"}}},"wheat-chris":{id:"wheat-chris",name:"Chris Wheat",enrichment:{lastUpdated:"2025-07-02T20:47:51.165376Z",confidence:"high",professional:{title:"Professor",affiliation:"Stockholm University",department:"Department of Zoology"},academic:{researchAreas:{raw:["ecological genomics","evolutionary biology","butterfly genomics","adaptation","population genomics","lepidoptera"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"ecological-genomics",label:"Ecological Genomics",level:2,parentId:"ecology-environmental",description:"Integration of genomics and ecology",synonyms:["genomic ecology","eco-evolutionary genomics"],children:[]},{id:"evolutionary-biology",label:"Evolutionary Biology",level:1,description:"Study of evolutionary processes, patterns, and history of life including phylogenetics and speciation",icon:"",children:["molecular-evolution","phylogenetics","phylogenomics","speciation","adaptation","evolutionary-genetics","macroevolution","microevolution","evolutionary-ecology"]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"insect-genomics",label:"Insect Genomics",level:3,parentId:"comparative-genomics",description:"Genomic studies of insects",synonyms:["entomological genomics","arthropod genomics"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-1863-2340"},profile:{shortBio:"Chris Wheat at Stockholm University specializes in Ecological Genomics, studying how genomic variation affects ecological and evolutionary processes in butterflies and other organisms. His work combines field studies with genomic approaches.",source:"Evolution and Genomics, Stockholm University"}}},"whiting-james":{id:"whiting-james",name:"James Whiting",enrichment:{lastUpdated:"2025-07-02T19:00:00.000Z",confidence:"high",professional:{title:"Postdoctoral Researcher",affiliation:"University of Exeter",department:"Biosciences"},academic:{researchAreas:{raw:["evolutionary genomics","convergent evolution","adaptation","population genomics","parallel evolution","guppy genomics"],standardized:{primary:[{id:"adaptation",label:"Adaptation",level:2,parentId:"evolutionary-biology",description:"Study of evolutionary adaptation to environments",synonyms:["adaptive evolution","evolutionary adaptation"],children:[]},{id:"evolutionary-genomics",label:"Evolutionary Genomics",level:2,parentId:"genomics-omics",description:"Study of genome evolution and evolutionary processes using genomic data",synonyms:["genome evolution","evolutionary genome analysis"],children:[]},{id:"population-genomics",label:"Population Genomics",level:2,parentId:"genomics-omics",description:"Genome-wide analysis of genetic variation within and between populations",synonyms:["population-level genomics","genomic population genetics"],children:[]}],secondary:[{id:"convergent-evolution",label:"Convergent Evolution",level:3,parentId:"molecular-evolution",description:"Independent evolution of similar traits",synonyms:["evolutionary convergence","parallel evolution"]}],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-8936-4991"},profile:{shortBio:"James Whiting studies evolution within natural systems and the role of the local environment in manipulating evolution at the gene level. He investigates convergent genomic changes in Trinidadian guppies that have repeatedly adapted to different predation environments.",source:"University of Exeter, ResearchGate"}}},"willis-amy":{id:"willis-amy",name:"Amy Willis",enrichment:{lastUpdated:"2025-01-02T00:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"University of Washington",department:"Biostatistics",labWebsite:"http://statisticaldiversitylab.com"},academic:{researchAreas:{raw:["biostatistics","microbiome","computational biology","statistical ecology","biodiversity","metagenomics","statistical methods"],standardized:{primary:[{id:"biodiversity",label:"Biodiversity",level:2,parentId:"ecology-environmental",description:"Study of biological diversity at all levels",synonyms:["biological diversity","species diversity"],children:[]},{id:"biostatistics",label:"Biostatistics",level:2,parentId:"mathematical-statistical",description:"Statistical methods in biological research",synonyms:["biological statistics","statistical biology"],children:[]},{id:"computational-biology",label:"Computational Biology",level:2,parentId:"computational-sciences",description:"Development and application of theoretical and computational methods in biology",synonyms:["in silico biology","theoretical biology"],children:[]},{id:"ecology",label:"Ecology",level:2,parentId:"ecology-environmental",description:"Study of organisms and their environment",synonyms:["ecological science","environmental biology"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbiome-research",label:"Microbiome Research",level:2,parentId:"microbiology-microbiome",description:"Study of microbial communities and their interactions with hosts and environments",synonyms:["microbiome analysis","microbiota research"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-2802-4317"},profile:{shortBio:"Amy Willis develops rigorous statistical methods for microbiome and biodiversity data analysis. She leads the Statistical Diversity Lab and has developed tools like corncob and betta for microbiome research.",source:"University of Washington, Google Scholar"}}},"woyke-tanja":{id:"woyke-tanja",name:"Tanja Woyke",enrichment:{lastUpdated:"2025-07-02T23:33:56.656Z",confidence:"high",professional:{title:"Deputy of User Programs",affiliation:"DOE Joint Genome Institute",department:"Microbial Program"},academic:{researchAreas:{raw:["single-cell genomics","microbial genomics","metagenomics","uncultured microorganisms","genome technology","environmental genomics"],standardized:{primary:[{id:"environmental-genomics",label:"Environmental Genomics",level:2,parentId:"ecology-environmental",description:"Genomic approaches to environmental biology",synonyms:["ecogenomics","environmental DNA"],children:[]},{id:"metagenomics",label:"Metagenomics",level:2,parentId:"genomics-omics",description:"Study of genetic material recovered directly from environmental samples",synonyms:["environmental genomics","community genomics"],children:[]},{id:"microbial-genomics",label:"Microbial Genomics",level:2,parentId:"microbiology-microbiome",description:"Genomic analysis of microorganisms",synonyms:["bacterial genomics","prokaryotic genomics"],children:[]},{id:"single-cell-genomics",label:"Single-Cell Genomics",level:2,parentId:"genomics-omics",description:"Analysis of genomes, transcriptomes, and epigenomes at single-cell resolution",synonyms:["single cell genomics","single-cell sequencing"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0002-9485-5637"},profile:{shortBio:"Tanja Woyke is Deputy of User Programs at DOE Joint Genome Institute and Head of the JGI Microbial Program. She pioneered single-cell genomics approaches to access genetic material of uncultivated taxa and received the 2020 van Niel International Prize for Studies in Bacterial Systematics.",source:"DOE Joint Genome Institute, Google Scholar"}}},"wrighton-kelly":{id:"wrighton-kelly",name:"Kelly Wrighton",enrichment:{lastUpdated:"2025-07-02T23:33:56.657Z",confidence:"high",professional:{title:"Gordon and Joan Bishop Professor",affiliation:"Colorado State University",department:"Department of Soil and Crop Sciences"},academic:{researchAreas:{raw:["microbial ecology","environmental microbiology","carbon cycling","nitrogen cycling","methane metabolism","greenhouse gas emissions"],standardized:{primary:[{id:"environmental-microbiology",label:"Environmental Microbiology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in environmental contexts",synonyms:["microbial ecology","environmental microbiome"],children:[]},{id:"microbial-ecology",label:"Microbial Ecology",level:2,parentId:"microbiology-microbiome",description:"Study of microorganisms in their natural environments",synonyms:["environmental microbiology","microbial community ecology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0003-0434-4217"},profile:{shortBio:"Kelly Wrighton is the Gordon and Joan Bishop Professor at Colorado State University. She investigates how microorganisms contribute to ecosystem processes, with particular interest in carbon and nitrogen cycling. Her work has uncovered previously unknown metabolic pathways explaining microbial influence on global climate.",source:"Colorado State University, Google Scholar"}}},"zhou-xiaofen":{id:"zhou-xiaofen",name:"Xiaofen Zhou",enrichment:{lastUpdated:"2025-07-02T19:00:00.000Z",confidence:"high",professional:{title:"Associate Professor",affiliation:"Wuhan Technology and Business University",department:"Department of Technology"},academic:{researchAreas:{raw:["cancer genomics","thyroid cancer","bioinformatics","SP1BN2","apoptosis","clinical medicine"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"cancer-genomics",label:"Cancer Genomics",level:2,parentId:"medical-clinical",description:"Genomic analysis of cancer",synonyms:["oncogenomics","tumor genomics"],children:[]},{id:"cell-biology",label:"Cell Biology",level:2,parentId:"molecular-cellular",description:"Study of cell structure, function, and behavior",synonyms:["cellular biology","cytology"],children:[]}],secondary:[],techniques:[]},lastMigrated:"2025-07-03T15:33:18.706Z"}},profile:{shortBio:"Xiaofen Zhou conducts research on cancer genomics with 155 citations and 809 reads. Her work includes studying SP1BN2's role in promoting the progression of thyroid cancer by accelerating G1/S transition and inhibiting apoptosis.",source:"ResearchGate, Loop Frontiers"}}},"zody-mike":{id:"zody-mike",name:"Mike Zody",enrichment:{lastUpdated:"2025-07-02T20:47:51.165372Z",confidence:"high",professional:{title:"Senior Computational Biologist",affiliation:"Broad Institute",department:"Genomics Platform"},academic:{researchAreas:{raw:["genomics","sequencing technology","comparative genomics","bioinformatics","genome biology","illumina sequencing"],standardized:{primary:[{id:"bioinformatics",label:"Bioinformatics",level:2,parentId:"computational-sciences",description:"Application of computational methods to analyze biological data",synonyms:["biomedical informatics","computational genomics"],children:[]},{id:"comparative-genomics",label:"Comparative Genomics",level:2,parentId:"genomics-omics",description:"Comparison of genome sequences and structures across different species",synonyms:["genome comparison","comparative genome analysis"],children:[]},{id:"genomics-omics",label:"Genomics & Omics Sciences",level:1,description:"Large-scale study of genomes and other molecular data including sequencing, assembly, and multi-omics integration",icon:"",children:["comparative-genomics","functional-genomics","population-genomics","evolutionary-genomics","structural-genomics","single-cell-genomics","metagenomics","transcriptomics","proteomics","metabolomics","epigenomics","pangenomics"]}],secondary:[],techniques:[{id:"sequencing-technology",label:"Sequencing Technology",level:2,parentId:"technology-methods",description:"Development and application of DNA/RNA sequencing technologies",synonyms:["next-generation sequencing","NGS","high-throughput sequencing"],children:[]}]},lastMigrated:"2025-07-03T15:33:18.705Z"},orcid:"0000-0001-6594-7199"},profile:{shortBio:"Mike Zody is a senior computational biologist at the Broad Institute. He teaches workshops on comparative genomics and sequencing technologies, with expertise in genome biology and sequencing study design.",source:"Evolution and Genomics workshops, Broad Institute"}}},"danecek-petr":{id:"danecek-petr",name:"Petr Danek",enrichment:{lastUpdated:"2025-07-05T19:27:59.613Z",confidence:"low",professional:{},academic:{},profile:{source:"Automated search"}}},"kintzl-daniel":{id:"kintzl-daniel",name:"Daniel Kintzl",enrichment:{lastUpdated:"2025-07-05T19:28:00.616Z",confidence:"low",professional:{},academic:{},profile:{source:"Automated search"}}},"blanco-miguez-aitor":{id:"blanco-miguez-aitor",name:"Aitor Blanco Miguez",enrichment:{lastUpdated:"2025-07-05T19:28:01.617Z",confidence:"low",professional:{},academic:{},profile:{source:"Automated search"}}},"montoliu-nerin-merce":{id:"montoliu-nerin-merce",name:"Merc Montoliu Nern",enrichment:{lastUpdated:"2025-07-05T19:28:02.617Z",confidence:"low",professional:{},academic:{},profile:{source:"Automated search"}}},"beerli-peter":{id:"beerli-peter",name:"Peter Beerli",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Florida State University",department:"Department of Scientific Computing",title:"Professor"},academic:{orcid:"0000-0002-9090-662X"},profile:{}}},"hoehna-sebastian":{id:"hoehna-sebastian",name:"Sebastian Hoehna",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Ludwig-Maximilians-Universitt Mnchen",department:"Department of Earth and Environmental Sciences",title:"Professor"},academic:{orcid:"0000-0001-5135-5212"},profile:{}}},"lewis-paul":{id:"lewis-paul",name:"Paul Lewis",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"University of Connecticut",department:"Department of Ecology and Evolutionary Biology",title:"Professor"},academic:{orcid:"0000-0001-9852-8759"},profile:{}}},"pennell-matt":{id:"pennell-matt",name:"Matt Pennell",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"University of British Columbia",department:"Department of Zoology",title:"Associate Professor"},academic:{orcid:"0000-0001-8396-8103"},profile:{}}},"stadler-tanja":{id:"stadler-tanja",name:"Tanja Stadler",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"ETH Zurich",department:"Department of Biosystems Science and Engineering",title:"Professor"},academic:{orcid:"0000-0003-1815-0706"},profile:{}}},"desjardins-christopher":{id:"desjardins-christopher",name:"Christopher Desjardins",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Broad Institute",department:"Infectious Disease and Microbiome Program",title:"Computational Biologist"},academic:{},profile:{}}},"eren-murat":{id:"eren-murat",name:"A. Murat Eren",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"University of Chicago",department:"Department of Medicine",title:"Assistant Professor"},academic:{orcid:"0000-0001-9013-4827"},profile:{}}},"grad-yonatan":{id:"grad-yonatan",name:"Yonatan Grad",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Harvard T.H. Chan School of Public Health",department:"Department of Immunology and Infectious Diseases",title:"Associate Professor"},academic:{orcid:"0000-0001-5646-1314"},profile:{}}},"kharchenko-peter":{id:"kharchenko-peter",name:"Peter Kharchenko",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Harvard Medical School",department:"Department of Biomedical Informatics",title:"Associate Professor"},academic:{orcid:"0000-0002-0930-4948"},profile:{}}},"kosakovsky-pond-sergei":{id:"kosakovsky-pond-sergei",name:"Sergei L Kosakovsky Pond",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Temple University",department:"Institute for Genomics and Evolutionary Medicine",title:"Professor"},academic:{orcid:"0000-0002-2047-8863"},profile:{}}},"reyes-alejandro":{id:"reyes-alejandro",name:"Alejandro Reyes",enrichment:{lastUpdated:"2025-07-05",confidence:"high",professional:{affiliation:"Dana-Farber Cancer Institute",department:"Department of Data Science",title:"Principal Research Scientist"},academic:{orcid:"0000-0001-8717-6612"},profile:{}}}};function ai(e){return e.toLowerCase().replace(/[^\w\s]/g,"").replace(/\s+/g," ").trim()}function Kd(e){return e.toLowerCase().replace(/[^\w\s]/g,"").replace(/\s+/g,"-")}function xf(e){const t=ai(e);for(const[o,n]of Object.entries(kl)){const a=n.name;if(ai(a)===t)return{id:o,...n}}const i=t.split(" ");if(i.length>=2){const o=i[0],n=i[i.length-1];for(const[a,s]of Object.entries(kl)){const c=ai(s.name).split(" ");if(c.length>=2){const p=c[0],y=c[c.length-1];if(o===p&&n===y)return{id:a,...s}}}}return null}function kf(e,t){const i=ai(e);let o=0;const n=[],a=[],s={},l={},c=[];for(const[g,b]of Object.entries(t)){let w=0;for(const v of b){const x=v.year;let h=0;for(const d of v.weeks)for(const m of d.sessions){const f=m.presenters.map(A=>ai(A)),S=m.coPresenters||[];[...f,...S.map(A=>ai(A))].includes(i)&&(o++,w++,h++,m.topic&&!c.includes(m.topic)&&c.push(m.topic))}h>0&&(a.push(x),l[x.toString()]=(l[x.toString()]||0)+h)}w>0&&(n.push(g),s[g]=w)}const p=[...new Set(a)].sort(),y=new Date().getFullYear(),u=p.some(g=>y-g<=3);return{totalSessions:o,workshopsParticipated:n,yearsActive:p,recentActivity:u,sessionsByWorkshop:s,sessionsByYear:l,firstTaught:p.length>0?Math.min(...p):0,lastTaught:p.length>0?Math.max(...p):0,primaryTopics:c.slice(0,10)}}function Af(e){var n,a,s,l;const t=[],i=[],o=[];if((a=(n=e==null?void 0:e.enrichment)==null?void 0:n.academic)!=null&&a.researchAreas){const c=e.enrichment.academic.researchAreas;if(c.raw&&t.push(...c.raw.slice(0,5)),(s=c.standardized)!=null&&s.primary){const y=c.standardized.primary.map(u=>u.label).slice(0,3);t.push(...y)}const p=((l=c.raw)==null?void 0:l.filter(y=>y.toLowerCase().includes("seq")||y.toLowerCase().includes("analysis")||y.toLowerCase().includes("assembly")||y.toLowerCase().includes("alignment")||y.toLowerCase().includes("tree")||y.toLowerCase().includes("blast")||y.toLowerCase().includes("pipeline")))||[];i.push(...p.slice(0,5))}return{primaryAreas:[...new Set(t)],techniques:[...new Set(i)],workshopSpecializations:o}}function rs(e){const t=new Map,i=new Set;for(const o of Object.values(e))for(const n of o)for(const a of n.weeks)for(const s of a.sessions)s.presenters.forEach(l=>i.add(l)),s.coPresenters&&s.coPresenters.forEach(l=>i.add(l));for(const o of i){const n=Kd(o),a=xf(o),s=kf(o,e),l=Af(a);l.workshopSpecializations=s.primaryTopics.slice(0,5);const c={id:n,name:o,displayName:o,enrichment:a==null?void 0:a.enrichment,teaching:s,expertise:l};t.set(n,c)}return Object.fromEntries(t)}function Cf(e,t){const i=Kd(e);return t[i]||null}function Qd(e,t){const i=e.toLowerCase();return Object.values(t).filter(o=>[...o.expertise.primaryAreas,...o.expertise.techniques,...o.expertise.workshopSpecializations].map(a=>a.toLowerCase()).some(a=>a.includes(i)))}function Xd(e,t,i=5){const o=[...e.expertise.primaryAreas,...e.expertise.techniques].map(a=>a.toLowerCase());return Object.values(t).filter(a=>a.id!==e.id).map(a=>{const s=[...a.expertise.primaryAreas,...a.expertise.techniques].map(c=>c.toLowerCase()),l=o.filter(c=>s.some(p=>p.includes(c)||c.includes(p))).length;return{presenter:a,score:l}}).filter(a=>a.score>0).sort((a,s)=>s.score-a.score).slice(0,i).map(a=>a.presenter)}function lt(e){if(!e||typeof e!="string")return e;const t=e.split("-");if(t.length>=2){const i=t[0],o=t[1],n=a=>a.charAt(0).toUpperCase()+a.slice(1).toLowerCase();return`${n(o)} ${n(i)}`}return e.charAt(0).toUpperCase()+e.slice(1).toLowerCase()}function pa(e){const t=lt(e);return`https://shandley.github.io/evomics-faculty/?search=${encodeURIComponent(t)}`}const If=()=>{const e=T.useMemo(()=>jn(),[]),t=T.useMemo(()=>bi(),[]),i=T.useMemo(()=>Tf(e),[e]),o=T.useMemo(()=>{const d=as();return rs(d)},[]),n=T.useMemo(()=>{const d=new Set;return Object.values(o).forEach(m=>{m.expertise.primaryAreas.forEach(f=>d.add(f)),m.expertise.techniques.forEach(f=>d.add(f))}),Array.from(d).sort()},[o]),[a,s]=T.useState({search:"",workshop:[],year:null,sessionType:[],presenter:[],topic:[]}),[l,c]=T.useState(""),[p,y]=T.useState("date"),[u,g]=T.useState("desc"),b=T.useMemo(()=>{let d=wf(e,a.search,{workshop:a.workshop,year:a.year,sessionType:a.sessionType,presenter:a.presenter});if(l){const m=Qd(l,o),f=new Set(m.map(S=>S.name.toLowerCase()));d=d.filter(S=>S.presenters.some(k=>f.has(k.toLowerCase()))||(S.coPresenters||[]).some(k=>f.has(k.toLowerCase())))}return d.sort((m,f)=>{var k;let S=0;switch(p){case"date":S=m.year-f.year;break;case"topic":S=m.topic.localeCompare(f.topic);break;case"presenter":S=((k=m.presenters[0])==null?void 0:k.localeCompare(f.presenters[0]||""))||0;break}return u==="asc"?S:-S}),d},[e,a,l,o,p,u]),w=(d,m)=>{s(f=>({...f,[d]:m}))},v=(d,m)=>{s(f=>{const S=f[d],k=S.includes(m)?S.filter(A=>A!==m):[...S,m];return{...f,[d]:k}})},x=()=>{s({search:"",workshop:[],year:null,sessionType:[],presenter:[],topic:[]}),c("")},h=d=>({wog:"bg-blue-100 text-blue-800",wpsg:"bg-green-100 text-green-800",wphylo:"bg-purple-100 text-purple-800",wmolevo:"bg-orange-100 text-orange-800",htranscriptomics:"bg-pink-100 text-pink-800",hmicrobial:"bg-indigo-100 text-indigo-800"})[d]||"bg-gray-100 text-gray-800";return r.jsxs("div",{className:"space-y-6",children:[r.jsxs("div",{className:"flex flex-col md:flex-row md:items-center md:justify-between",children:[r.jsxs("div",{children:[r.jsx("h1",{className:"text-3xl font-bold text-gray-900",children:"Workshop Sessions"}),r.jsxs("p",{className:"text-gray-600 mt-1",children:["Showing ",b.length," of ",e.length," sessions"]})]}),r.jsxs("div",{className:"flex items-center space-x-4 mt-4 md:mt-0",children:[r.jsxs("select",{value:p,onChange:d=>y(d.target.value),className:"border border-gray-300 rounded-md px-3 py-2 text-sm",children:[r.jsx("option",{value:"date",children:"Sort by Date"}),r.jsx("option",{value:"topic",children:"Sort by Topic"}),r.jsx("option",{value:"presenter",children:"Sort by Presenter"})]}),r.jsx("button",{onClick:()=>g(d=>d==="asc"?"desc":"asc"),className:"border border-gray-300 rounded-md px-3 py-2 text-sm hover:bg-gray-50",children:u==="asc"?"":""})]})]}),r.jsxs("div",{className:"bg-white/80 backdrop-blur rounded-xl shadow-lg p-6 border border-white/20",children:[r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4",children:[r.jsxs("div",{children:[r.jsx("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:"Search"}),r.jsx("input",{type:"text",value:a.search,onChange:d=>w("search",d.target.value),placeholder:"Search topics, presenters...",className:"w-full border border-gray-300 rounded-md px-3 py-2 text-sm focus:outline-none focus:ring-2 focus:ring-blue-500"})]}),r.jsxs("div",{children:[r.jsx("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:"Year"}),r.jsxs("select",{value:a.year||"",onChange:d=>w("year",d.target.value?parseInt(d.target.value):null),className:"w-full border border-gray-300 rounded-md px-3 py-2 text-sm focus:outline-none focus:ring-2 focus:ring-blue-500",children:[r.jsx("option",{value:"",children:"All Years"}),i.years.map(d=>r.jsx("option",{value:d,children:d},d))]})]}),r.jsxs("div",{children:[r.jsx("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:"Workshop"}),r.jsx("div",{className:"space-y-1 max-h-24 overflow-y-auto",children:i.workshops.map(d=>{var m;return r.jsxs("label",{className:"flex items-center text-sm",children:[r.jsx("input",{type:"checkbox",checked:a.workshop.includes(d),onChange:()=>v("workshop",d),className:"mr-2"}),((m=t[d])==null?void 0:m.shortName)||d]},d)})})]}),r.jsxs("div",{children:[r.jsx("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:"Session Type"}),r.jsx("div",{className:"space-y-1 max-h-24 overflow-y-auto",children:i.sessionTypes.map(d=>r.jsxs("label",{className:"flex items-center text-sm",children:[r.jsx("input",{type:"checkbox",checked:a.sessionType.includes(d),onChange:()=>v("sessionType",d),className:"mr-2"}),d]},d))})]}),r.jsxs("div",{children:[r.jsxs("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:["Presenter Expertise",r.jsx("span",{className:"ml-1 text-xs text-gray-500",children:"(filter by research area)"})]}),r.jsxs("select",{value:l,onChange:d=>c(d.target.value),className:"w-full border border-gray-300 rounded-md px-3 py-2 text-sm focus:outline-none focus:ring-2 focus:ring-blue-500",children:[r.jsx("option",{value:"",children:"All Expertise Areas"}),n.map(d=>r.jsx("option",{value:d,children:d},d))]})]}),r.jsxs("div",{children:[r.jsxs("label",{className:"block text-sm font-medium text-gray-700 mb-2",children:["Presenter Name",r.jsx(Z,{to:"/presenters",className:"ml-2 text-xs text-blue-600 hover:text-blue-800",children:"Browse all "})]}),r.jsxs("div",{className:"space-y-1 max-h-24 overflow-y-auto",children:[i.presenters.slice(0,10).map(d=>r.jsxs("label",{className:"flex items-center text-sm",children:[r.jsx("input",{type:"checkbox",checked:a.presenter.includes(d),onChange:()=>v("presenter",d),className:"mr-2"}),r.jsx("span",{className:"truncate",children:lt(d)})]},d)),i.presenters.length>10&&r.jsxs("div",{className:"text-xs text-gray-500 pt-1",children:["... and ",i.presenters.length-10," more"]})]})]})]}),r.jsx("div",{className:"mt-4 flex justify-end",children:r.jsx("button",{onClick:x,className:"text-sm text-blue-600 hover:text-blue-800",children:"Clear all filters"})})]}),r.jsx("div",{className:"space-y-4",children:b.length===0?r.jsxs("div",{className:"text-center py-12 bg-white/80 backdrop-blur rounded-xl shadow-lg border border-white/20",children:[r.jsx("p",{className:"text-gray-500",children:"No sessions found matching your criteria."}),r.jsx("button",{onClick:x,className:"mt-4 text-blue-600 hover:text-blue-800",children:"Clear filters to see all sessions"})]}):b.map(d=>{var m;return r.jsx("div",{className:"bg-white/80 backdrop-blur rounded-xl shadow-lg hover:shadow-xl transition-all transform hover:scale-[1.02] p-6 border border-white/20",children:r.jsxs("div",{className:"flex flex-col md:flex-row md:items-start md:justify-between",children:[r.jsxs("div",{className:"flex-1",children:[r.jsxs("div",{className:"flex items-center space-x-3 mb-2",children:[r.jsx("span",{className:`px-2 py-1 rounded-full text-xs font-medium ${h(d.workshopId)}`,children:((m=t[d.workshopId])==null?void 0:m.shortName)||d.workshopId}),r.jsx("span",{className:"text-gray-500 text-sm",children:d.year}),r.jsx("span",{className:"text-gray-500 text-sm",children:d.date}),r.jsx("span",{className:"text-gray-500 text-sm",children:d.time})]}),r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-2",children:r.jsx(Z,{to:`/sessions/${d.id}`,className:"hover:text-blue-600 transition-colors",children:d.topic})}),r.jsxs("div",{className:"text-sm text-gray-600 space-y-1",children:[r.jsxs("p",{children:[r.jsx("strong",{children:"Presenter:"})," ",lt(d.presenters[0])]}),d.coPresenters&&d.coPresenters.length>0&&r.jsxs("p",{children:[r.jsx("strong",{children:"Co-presenters:"})," ",d.coPresenters.map(lt).join(", ")]}),r.jsxs("p",{children:[r.jsx("strong",{children:"Type:"})," ",d.type]}),d.location&&r.jsxs("p",{children:[r.jsx("strong",{children:"Location:"})," ",d.location]})]})]}),r.jsx("div",{className:"mt-4 md:mt-0 md:ml-4",children:r.jsx(Z,{to:`/sessions/${d.id}`,className:"inline-flex items-center px-3 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 bg-white hover:bg-gray-50 transition-colors",children:"View Details"})})]})},d.id)})}),b.length>50&&r.jsx("div",{className:"text-center py-6",children:r.jsx("p",{className:"text-gray-500 text-sm",children:"Showing first 50 results. Use filters to narrow your search."})})]})};function Yd({presenter:e,showTeachingStats:t=!1,onViewProfile:i,compact:o=!1}){var u,g,b,w,v,x,h,d;const n=!!e.enrichment,a=(g=(u=e.enrichment)==null?void 0:u.professional)==null?void 0:g.affiliation,s=(w=(b=e.enrichment)==null?void 0:b.professional)==null?void 0:w.department,l=(x=(v=e.enrichment)==null?void 0:v.professional)==null?void 0:x.title,c=(d=(h=e.enrichment)==null?void 0:h.academic)==null?void 0:d.orcid,p=e.expertise.primaryAreas.slice(0,3),y=e.teaching.recentActivity;return r.jsxs("div",{className:`
      bg-white rounded-lg shadow-md border border-gray-200 overflow-hidden
      hover:shadow-lg transition-all duration-200 transform hover:-translate-y-1
      ${o?"p-3":"p-4"}
    `,children:[r.jsxs("div",{className:"flex items-start justify-between mb-3",children:[r.jsxs("div",{className:"flex-1 min-w-0",children:[r.jsx("h3",{className:`font-semibold text-gray-900 truncate ${o?"text-sm":"text-base"}`,children:e.displayName}),n&&l&&r.jsx("p",{className:`text-gray-600 truncate ${o?"text-xs":"text-sm"}`,children:l}),n&&a&&r.jsxs("p",{className:`text-gray-500 truncate ${o?"text-xs":"text-sm"}`,children:[a,s&&!o&&r.jsx("span",{className:"block text-xs text-gray-400 truncate",children:s})]})]}),r.jsxs("div",{className:"flex items-center space-x-2 ml-2",children:[y&&r.jsx("span",{className:"inline-flex items-center px-2 py-1 rounded-full text-xs font-medium bg-green-100 text-green-800",children:"Active"}),c&&r.jsx("a",{href:`https://orcid.org/${c}`,target:"_blank",rel:"noopener noreferrer",className:"text-green-600 hover:text-green-700 transition-colors",title:"ORCID Profile",children:r.jsx("svg",{className:"w-4 h-4",viewBox:"0 0 24 24",fill:"currentColor",children:r.jsx("path",{d:"M12 0C5.372 0 0 5.372 0 12s5.372 12 12 12 12-5.372 12-12S18.628 0 12 0zM7.369 4.378c.525 0 .947.431.947.947 0 .525-.422.947-.947.947-.525 0-.946-.422-.946-.947 0-.525.421-.947.946-.947zm-.722 3.038h1.444v10.041H6.647V7.416zm3.562 0h3.9c3.712 0 5.344 2.653 5.344 5.025 0 2.578-2.016 5.016-5.325 5.016h-3.919V7.416zm1.444 1.303v7.444h2.297c2.359 0 3.844-1.422 3.844-3.722 0-2.016-1.422-3.722-3.844-3.722h-2.297z"})})})]})]}),p.length>0&&r.jsx("div",{className:"mb-3",children:r.jsx("div",{className:"flex flex-wrap gap-1",children:p.map((m,f)=>r.jsx("span",{className:`
                  inline-flex items-center rounded-full bg-blue-50 text-blue-700 font-medium
                  ${o?"px-2 py-0.5 text-xs":"px-2.5 py-0.5 text-xs"}
                `,children:m},f))})}),t&&!o&&r.jsx("div",{className:"border-t border-gray-100 pt-3",children:r.jsxs("div",{className:"grid grid-cols-2 gap-4 text-sm",children:[r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500",children:"Sessions:"}),r.jsx("span",{className:"ml-1 font-medium text-gray-900",children:e.teaching.totalSessions})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500",children:"Workshops:"}),r.jsx("span",{className:"ml-1 font-medium text-gray-900",children:e.teaching.workshopsParticipated.length})]}),r.jsxs("div",{className:"col-span-2",children:[r.jsx("span",{className:"text-gray-500",children:"Active:"}),r.jsxs("span",{className:"ml-1 font-medium text-gray-900",children:[e.teaching.firstTaught,"-",e.teaching.lastTaught]})]})]})}),o&&r.jsxs("div",{className:"text-xs text-gray-500",children:[e.teaching.totalSessions," sessions  ",e.teaching.workshopsParticipated.length," workshops"]}),r.jsxs("div",{className:"mt-3 flex space-x-2",children:[i&&r.jsx("button",{onClick:()=>i(e),className:`
              flex-1 bg-primary-600 text-white rounded-md hover:bg-primary-700 
              transition-colors font-medium
              ${o?"px-3 py-1.5 text-xs":"px-4 py-2 text-sm"}
            `,children:"View Profile"}),n&&a&&r.jsx("a",{href:`https://shandley.github.io/evomics-faculty/?search=${encodeURIComponent(e.name)}`,target:"_blank",rel:"noopener noreferrer",className:`
              flex-1 bg-gray-100 text-gray-700 rounded-md hover:bg-gray-200 
              transition-colors font-medium text-center
              ${o?"px-3 py-1.5 text-xs":"px-4 py-2 text-sm"}
            `,children:"Faculty Profile"})]})]})}function Mf({presenters:e,onViewProfile:t,showTeachingStats:i=!1,compact:o=!1,emptyMessage:n="No presenters found"}){return e.length===0?r.jsx("div",{className:"text-center py-8 text-gray-500",children:r.jsx("p",{children:n})}):r.jsx("div",{className:`grid gap-4 ${o?"grid-cols-1 sm:grid-cols-2 lg:grid-cols-3":"grid-cols-1 md:grid-cols-2"}`,children:e.map(a=>r.jsx(Yd,{presenter:a,onViewProfile:t,showTeachingStats:i,compact:o},a.id))})}function Gf({presenter:e,onViewProfile:t,showExpertise:i=!0}){var s,l;const o=!!e.enrichment,n=(l=(s=e.enrichment)==null?void 0:s.professional)==null?void 0:l.affiliation,a=e.expertise.primaryAreas[0];return r.jsxs("div",{className:"inline-flex items-center space-x-2 bg-white border border-gray-200 rounded-lg px-3 py-2 shadow-sm hover:shadow-md transition-shadow",children:[r.jsxs("div",{className:"flex-1 min-w-0",children:[r.jsxs("div",{className:"flex items-center space-x-2",children:[r.jsx("span",{className:"font-medium text-gray-900 truncate",children:e.displayName}),o&&r.jsx("span",{className:"inline-flex items-center px-1.5 py-0.5 rounded text-xs font-medium bg-green-100 text-green-800",children:""})]}),n&&r.jsx("div",{className:"text-xs text-gray-500 truncate",children:n}),i&&a&&r.jsx("div",{className:"text-xs text-blue-600 truncate",children:a})]}),t&&r.jsx("button",{onClick:()=>t(e),className:"text-primary-600 hover:text-primary-700 transition-colors",title:"View full profile",children:r.jsx("svg",{className:"w-4 h-4",fill:"none",stroke:"currentColor",viewBox:"0 0 24 24",children:r.jsx("path",{strokeLinecap:"round",strokeLinejoin:"round",strokeWidth:2,d:"M13 16h-1v-4h-1m1-4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"})})})]})}function ep({presenter:e,isOpen:t,onClose:i,relatedPresenters:o=[],onViewRelated:n}){var y,u,g;if(!t||!e)return null;const a=!!e.enrichment,s=(y=e.enrichment)==null?void 0:y.professional,l=(u=e.enrichment)==null?void 0:u.academic,c=l==null?void 0:l.orcid,p={wog:"Workshop on Genomics",wpsg:"Workshop on Population and Speciation Genomics",wphylo:"Workshop on Phylogenomics",wmolevo:"Workshop on Molecular Evolution",htranscriptomics:"Harvard Workshop on Transcriptomics",hmicrobial:"Harvard Workshop on Microbial Genomics"};return r.jsx("div",{className:"fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center p-4 z-50",children:r.jsxs("div",{className:"bg-white rounded-lg shadow-xl max-w-4xl w-full max-h-[90vh] overflow-hidden",children:[r.jsxs("div",{className:"bg-gradient-to-r from-primary-600 to-primary-700 text-white p-6",children:[r.jsxs("div",{className:"flex justify-between items-start",children:[r.jsxs("div",{className:"flex-1",children:[r.jsx("h2",{className:"text-2xl font-bold mb-2",children:e.displayName}),a&&(s==null?void 0:s.title)&&r.jsx("p",{className:"text-primary-100 text-lg mb-1",children:s.title}),a&&(s==null?void 0:s.affiliation)&&r.jsxs("div",{className:"text-primary-200",children:[r.jsx("p",{className:"font-medium",children:s.affiliation}),s.department&&r.jsx("p",{className:"text-sm text-primary-300",children:s.department})]})]}),r.jsx("button",{onClick:i,className:"text-primary-200 hover:text-white transition-colors",children:r.jsx("svg",{className:"w-6 h-6",fill:"none",stroke:"currentColor",viewBox:"0 0 24 24",children:r.jsx("path",{strokeLinecap:"round",strokeLinejoin:"round",strokeWidth:2,d:"M6 18L18 6M6 6l12 12"})})})]}),r.jsxs("div",{className:"mt-4 grid grid-cols-3 gap-4 text-center",children:[r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-3",children:[r.jsx("div",{className:"text-2xl font-bold",children:e.teaching.totalSessions}),r.jsx("div",{className:"text-sm text-primary-200",children:"Total Sessions"})]}),r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-3",children:[r.jsx("div",{className:"text-2xl font-bold",children:e.teaching.workshopsParticipated.length}),r.jsx("div",{className:"text-sm text-primary-200",children:"Workshops"})]}),r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-3",children:[r.jsx("div",{className:"text-2xl font-bold",children:e.teaching.lastTaught-e.teaching.firstTaught+1}),r.jsx("div",{className:"text-sm text-primary-200",children:"Years Active"})]})]})]}),r.jsx("div",{className:"p-6 overflow-y-auto",style:{maxHeight:"calc(90vh - 200px)"},children:r.jsxs("div",{className:"grid grid-cols-1 lg:grid-cols-2 gap-6",children:[r.jsxs("div",{className:"space-y-6",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-3",children:"Research Expertise"}),e.expertise.primaryAreas.length>0&&r.jsxs("div",{className:"mb-4",children:[r.jsx("h4",{className:"text-sm font-medium text-gray-700 mb-2",children:"Primary Areas"}),r.jsx("div",{className:"flex flex-wrap gap-2",children:e.expertise.primaryAreas.map((b,w)=>r.jsx("span",{className:"inline-flex items-center px-3 py-1 rounded-full text-sm font-medium bg-blue-100 text-blue-800",children:b},w))})]}),e.expertise.techniques.length>0&&r.jsxs("div",{className:"mb-4",children:[r.jsx("h4",{className:"text-sm font-medium text-gray-700 mb-2",children:"Techniques & Tools"}),r.jsx("div",{className:"flex flex-wrap gap-2",children:e.expertise.techniques.map((b,w)=>r.jsx("span",{className:"inline-flex items-center px-3 py-1 rounded-full text-sm font-medium bg-green-100 text-green-800",children:b},w))})]}),e.expertise.workshopSpecializations.length>0&&r.jsxs("div",{children:[r.jsx("h4",{className:"text-sm font-medium text-gray-700 mb-2",children:"Workshop Specializations"}),r.jsx("div",{className:"flex flex-wrap gap-2",children:e.expertise.workshopSpecializations.map((b,w)=>r.jsx("span",{className:"inline-flex items-center px-3 py-1 rounded-full text-sm font-medium bg-purple-100 text-purple-800",children:b},w))})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-3",children:"External Profiles"}),r.jsxs("div",{className:"space-y-2",children:[c&&r.jsxs("a",{href:`https://orcid.org/${c}`,target:"_blank",rel:"noopener noreferrer",className:"inline-flex items-center space-x-2 text-green-600 hover:text-green-700 transition-colors",children:[r.jsx("svg",{className:"w-5 h-5",viewBox:"0 0 24 24",fill:"currentColor",children:r.jsx("path",{d:"M12 0C5.372 0 0 5.372 0 12s5.372 12 12 12 12-5.372 12-12S18.628 0 12 0zM7.369 4.378c.525 0 .947.431.947.947 0 .525-.422.947-.947.947-.525 0-.946-.422-.946-.947 0-.525.421-.947.946-.947zm-.722 3.038h1.444v10.041H6.647V7.416zm3.562 0h3.9c3.712 0 5.344 2.653 5.344 5.025 0 2.578-2.016 5.016-5.325 5.016h-3.919V7.416zm1.444 1.303v7.444h2.297c2.359 0 3.844-1.422 3.844-3.722 0-2.016-1.422-3.722-3.844-3.722h-2.297z"})}),r.jsx("span",{children:"ORCID Profile"})]}),a&&r.jsxs("a",{href:`https://shandley.github.io/evomics-faculty/?search=${encodeURIComponent(e.name)}`,target:"_blank",rel:"noopener noreferrer",className:"inline-flex items-center space-x-2 text-primary-600 hover:text-primary-700 transition-colors",children:[r.jsx("svg",{className:"w-5 h-5",fill:"none",stroke:"currentColor",viewBox:"0 0 24 24",children:r.jsx("path",{strokeLinecap:"round",strokeLinejoin:"round",strokeWidth:2,d:"M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z"})}),r.jsx("span",{children:"Faculty Directory Profile"})]}),(s==null?void 0:s.labWebsite)&&r.jsxs("a",{href:s.labWebsite,target:"_blank",rel:"noopener noreferrer",className:"inline-flex items-center space-x-2 text-blue-600 hover:text-blue-700 transition-colors",children:[r.jsx("svg",{className:"w-5 h-5",fill:"none",stroke:"currentColor",viewBox:"0 0 24 24",children:r.jsx("path",{strokeLinecap:"round",strokeLinejoin:"round",strokeWidth:2,d:"M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14"})}),r.jsx("span",{children:"Lab Website"})]})]})]})]}),r.jsxs("div",{className:"space-y-6",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-3",children:"Teaching History"}),r.jsx("div",{className:"bg-gray-50 rounded-lg p-4 mb-4",children:r.jsxs("div",{className:"grid grid-cols-2 gap-4 text-sm",children:[r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-600",children:"Active Period:"}),r.jsxs("div",{className:"font-medium text-gray-900",children:[e.teaching.firstTaught," - ",e.teaching.lastTaught]})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-600",children:"Current Status:"}),r.jsx("div",{className:`font-medium ${e.teaching.recentActivity?"text-green-600":"text-gray-500"}`,children:e.teaching.recentActivity?"Recently Active":"Historical"})]})]})}),r.jsxs("div",{className:"space-y-3",children:[r.jsx("h4",{className:"text-sm font-medium text-gray-700",children:"Sessions by Workshop"}),Object.entries(e.teaching.sessionsByWorkshop).map(([b,w])=>r.jsxs("div",{className:"flex justify-between items-center py-2 border-b border-gray-100",children:[r.jsx("span",{className:"text-sm text-gray-900",children:p[b]||b.toUpperCase()}),r.jsxs("span",{className:"text-sm font-medium text-gray-600",children:[w," sessions"]})]},b))]}),Object.keys(e.teaching.sessionsByYear).length>0&&r.jsxs("div",{className:"mt-4",children:[r.jsx("h4",{className:"text-sm font-medium text-gray-700 mb-2",children:"Recent Activity"}),r.jsx("div",{className:"grid grid-cols-3 gap-2",children:Object.entries(e.teaching.sessionsByYear).sort(([b],[w])=>parseInt(w)-parseInt(b)).slice(0,6).map(([b,w])=>r.jsxs("div",{className:"text-center py-1 bg-gray-100 rounded text-xs",children:[r.jsx("div",{className:"font-medium",children:b}),r.jsxs("div",{className:"text-gray-600",children:[w," sessions"]})]},b))})]})]}),o.length>0&&r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-3",children:"Related Presenters"}),r.jsx("div",{className:"space-y-2",children:o.slice(0,5).map(b=>{var w,v;return r.jsxs("div",{className:"flex items-center justify-between p-3 bg-gray-50 rounded-lg hover:bg-gray-100 transition-colors",children:[r.jsxs("div",{className:"flex-1 min-w-0",children:[r.jsx("div",{className:"font-medium text-gray-900 truncate",children:b.displayName}),((v=(w=b.enrichment)==null?void 0:w.professional)==null?void 0:v.affiliation)&&r.jsx("div",{className:"text-sm text-gray-500 truncate",children:b.enrichment.professional.affiliation}),r.jsx("div",{className:"text-xs text-blue-600",children:b.expertise.primaryAreas.slice(0,2).join(", ")})]}),n&&r.jsx("button",{onClick:()=>n(b),className:"ml-2 text-primary-600 hover:text-primary-700 transition-colors",children:r.jsx("svg",{className:"w-4 h-4",fill:"none",stroke:"currentColor",viewBox:"0 0 24 24",children:r.jsx("path",{strokeLinecap:"round",strokeLinejoin:"round",strokeWidth:2,d:"M9 5l7 7-7 7"})})})]},b.id)})})]})]})]})}),r.jsx("div",{className:"bg-gray-50 px-6 py-4 border-t border-gray-200",children:r.jsxs("div",{className:"flex justify-between items-center",children:[r.jsx("div",{className:"text-sm text-gray-500",children:a&&((g=e.enrichment)==null?void 0:g.lastUpdated)&&r.jsxs("span",{children:["Profile updated: ",new Date(e.enrichment.lastUpdated).toLocaleDateString()]})}),r.jsx("button",{onClick:i,className:"px-4 py-2 bg-primary-600 text-white rounded-md hover:bg-primary-700 transition-colors",children:"Close"})]})})]})})}const Jf=()=>{var w;const{id:e}=sh(),[t,i]=T.useState(null),o=T.useMemo(()=>jn(),[]),n=T.useMemo(()=>bi(),[]),a=T.useMemo(()=>{const v=as();return rs(v)},[]),s=T.useMemo(()=>o.find(v=>v.id===e),[o,e]),l=T.useMemo(()=>s?o.filter(v=>v.id!==s.id&&v.workshopId===s.workshopId&&v.presenters.includes(s.presenters[0])).slice(0,5):[],[o,s]),c=T.useMemo(()=>s?[...s.presenters,...s.coPresenters||[]].map(x=>Cf(x,a)).filter(Boolean):[],[s,a]),p=T.useMemo(()=>t?Xd(t,a,5):[],[t,a]),y=v=>{i(v)},u=()=>{i(null)};if(!s)return r.jsxs("div",{className:"text-center py-12",children:[r.jsx("h1",{className:"text-2xl font-bold text-gray-900 mb-4",children:"Session Not Found"}),r.jsx("p",{className:"text-gray-600 mb-6",children:"The requested session could not be found."}),r.jsx(Z,{to:"/sessions",className:"inline-flex items-center px-4 py-2 border border-transparent rounded-md shadow-sm text-sm font-medium text-white bg-blue-600 hover:bg-blue-700",children:" Back to Sessions"})]});const g=n[s.workshopId],b=v=>({wog:"bg-blue-100 text-blue-800 border-blue-300",wpsg:"bg-green-100 text-green-800 border-green-300",wphylo:"bg-purple-100 text-purple-800 border-purple-300",wmolevo:"bg-orange-100 text-orange-800 border-orange-300",htranscriptomics:"bg-pink-100 text-pink-800 border-pink-300",hmicrobial:"bg-indigo-100 text-indigo-800 border-indigo-300"})[v]||"bg-gray-100 text-gray-800 border-gray-300";return r.jsxs("div",{className:"space-y-6",children:[r.jsxs("nav",{className:"flex items-center space-x-2 text-sm text-gray-500",children:[r.jsx(Z,{to:"/sessions",className:"hover:text-gray-700",children:"Sessions"}),r.jsx("span",{children:""}),r.jsx("span",{className:"text-gray-900",children:s.topic})]}),r.jsx("div",{className:"bg-white rounded-lg shadow-lg p-6",children:r.jsxs("div",{className:"flex items-start justify-between mb-4",children:[r.jsxs("div",{className:"flex-1",children:[r.jsxs("div",{className:"flex items-center space-x-3 mb-3",children:[r.jsx("span",{className:`px-3 py-1 rounded-full text-sm font-medium border ${b(s.workshopId)}`,children:(g==null?void 0:g.shortName)||s.workshopId}),r.jsx("span",{className:"text-gray-500",children:s.year}),r.jsx("span",{className:`px-2 py-1 rounded-full text-xs font-medium ${g!=null&&g.active?"bg-green-100 text-green-800":"bg-gray-100 text-gray-600"}`,children:g!=null&&g.active?"Active Series":"Historical Series"})]}),r.jsx("h1",{className:"text-3xl font-bold text-gray-900 mb-4",children:s.topic}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4 text-sm",children:[r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Date:"}),r.jsx("div",{className:"text-gray-900",children:s.date})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Time:"}),r.jsx("div",{className:"text-gray-900",children:s.time})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Type:"}),r.jsx("div",{className:"text-gray-900 capitalize",children:s.type})]}),s.location&&r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Location:"}),r.jsx("div",{className:"text-gray-900",children:s.location})]})]})]}),r.jsx(Z,{to:"/sessions",className:"inline-flex items-center px-4 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 bg-white hover:bg-gray-50",children:" Back to Sessions"})]})}),r.jsxs("div",{className:"grid grid-cols-1 lg:grid-cols-3 gap-6",children:[r.jsxs("div",{className:"lg:col-span-2 space-y-6",children:[r.jsxs("div",{className:"bg-white rounded-lg shadow p-6",children:[r.jsx("h2",{className:"text-xl font-bold text-gray-900 mb-4",children:"Session Presenters"}),c.length>0?r.jsx("div",{className:"space-y-4",children:c.map((v,x)=>r.jsxs("div",{children:[x===0&&r.jsx("h3",{className:"text-sm font-medium text-gray-700 mb-2",children:"Primary Presenter"}),x===1&&s.coPresenters&&s.coPresenters.length>0&&r.jsx("h3",{className:"text-sm font-medium text-gray-700 mb-2 mt-6",children:"Co-presenters"}),r.jsx(Gf,{presenter:v,onViewProfile:y,showExpertise:!0})]},v.id))}):r.jsxs("div",{className:"space-y-4",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-sm font-medium text-gray-700 mb-2",children:"Primary Presenter"}),r.jsxs("div",{className:"flex items-center space-x-3 p-3 bg-gray-50 rounded-lg",children:[r.jsx("div",{className:"w-10 h-10 bg-blue-100 rounded-full flex items-center justify-center",children:r.jsx("span",{className:"text-blue-600 font-bold text-sm",children:((w=lt(s.presenters[0]))==null?void 0:w.split(" ").map(v=>v[0]).join(""))||"?"})}),r.jsxs("div",{className:"flex-1",children:[r.jsx("div",{className:"font-medium text-gray-900",children:lt(s.presenters[0])}),r.jsx("a",{href:pa(s.presenters[0]),target:"_blank",rel:"noopener noreferrer",className:"text-blue-600 hover:text-blue-800 text-sm",children:"View Faculty Profile "})]})]})]}),s.coPresenters&&s.coPresenters.length>0&&r.jsxs("div",{children:[r.jsx("h3",{className:"text-sm font-medium text-gray-700 mb-2",children:"Co-presenters"}),r.jsx("div",{className:"space-y-2",children:s.coPresenters.map((v,x)=>r.jsxs("div",{className:"flex items-center space-x-3 p-3 bg-gray-50 rounded-lg",children:[r.jsx("div",{className:"w-8 h-8 bg-gray-100 rounded-full flex items-center justify-center",children:r.jsx("span",{className:"text-gray-600 font-medium text-xs",children:lt(v).split(" ").map(h=>h[0]).join("")})}),r.jsxs("div",{className:"flex-1",children:[r.jsx("div",{className:"font-medium text-gray-900",children:lt(v)}),r.jsx("a",{href:pa(v),target:"_blank",rel:"noopener noreferrer",className:"text-blue-600 hover:text-blue-800 text-sm",children:"View Faculty Profile "})]})]},x))})]})]}),c.length>0&&r.jsx("div",{className:"mt-6 pt-4 border-t border-gray-200",children:r.jsxs("div",{className:"flex flex-wrap gap-2",children:[r.jsx(Z,{to:"/presenters",className:"inline-flex items-center px-3 py-2 border border-primary-300 rounded-md text-sm font-medium text-primary-700 bg-primary-50 hover:bg-primary-100 transition-colors",children:"Browse All Presenters"}),c[0]&&r.jsxs(Z,{to:`/sessions?presenter=${encodeURIComponent(c[0].name)}`,className:"inline-flex items-center px-3 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 bg-white hover:bg-gray-50 transition-colors",children:["More Sessions by ",c[0].displayName.split(" ")[0]]})]})})]}),g&&r.jsxs("div",{className:"bg-white rounded-lg shadow p-6",children:[r.jsx("h2",{className:"text-xl font-bold text-gray-900 mb-4",children:"Workshop Context"}),r.jsxs("div",{className:"space-y-4",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"font-semibold text-gray-900",children:g.name}),r.jsx("p",{className:"text-gray-600 mt-1",children:g.description})]}),r.jsxs("div",{className:"grid grid-cols-2 gap-4 text-sm",children:[r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Location:"}),r.jsx("div",{className:"text-gray-900",children:g.location})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Series Status:"}),r.jsx("div",{className:"text-gray-900",children:g.active?"Active":"Historical"})]}),r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"Start Year:"}),r.jsx("div",{className:"text-gray-900",children:g.startYear})]}),g.endYear&&r.jsxs("div",{children:[r.jsx("span",{className:"text-gray-500 font-medium",children:"End Year:"}),r.jsx("div",{className:"text-gray-900",children:g.endYear})]})]})]})]})]}),r.jsxs("div",{className:"space-y-6",children:[r.jsxs("div",{className:"bg-white rounded-lg shadow p-6",children:[r.jsx("h3",{className:"text-lg font-bold text-gray-900 mb-4",children:"Quick Actions"}),r.jsxs("div",{className:"space-y-3",children:[r.jsx("a",{href:pa(s.presenters[0]),target:"_blank",rel:"noopener noreferrer",className:"block w-full text-center px-4 py-2 border border-blue-300 rounded-md text-sm font-medium text-blue-700 bg-blue-50 hover:bg-blue-100",children:"View Presenter Profile"}),r.jsxs(Z,{to:`/sessions?workshop=${s.workshopId}&year=${s.year}`,className:"block w-full text-center px-4 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 bg-white hover:bg-gray-50",children:["Other ",s.year," Sessions"]}),r.jsx(Z,{to:"/timeline",className:"block w-full text-center px-4 py-2 border border-gray-300 rounded-md text-sm font-medium text-gray-700 bg-white hover:bg-gray-50",children:"View in Timeline"})]})]}),l.length>0&&r.jsxs("div",{className:"bg-white rounded-lg shadow p-6",children:[r.jsxs("h3",{className:"text-lg font-bold text-gray-900 mb-4",children:["More from ",s.presenters[0]]}),r.jsx("div",{className:"space-y-3",children:l.map(v=>r.jsxs(Z,{to:`/sessions/${v.id}`,className:"block p-3 border border-gray-200 rounded-md hover:bg-gray-50 transition-colors",children:[r.jsx("div",{className:"font-medium text-gray-900 text-sm mb-1",children:v.topic}),r.jsxs("div",{className:"text-gray-500 text-xs",children:[v.year,"  ",v.type]})]},v.id))}),r.jsx(Z,{to:`/sessions?presenter=${encodeURIComponent(s.presenters[0])}`,className:"block mt-4 text-center text-blue-600 hover:text-blue-800 text-sm font-medium",children:"View All Sessions "})]})]})]}),r.jsx(ep,{presenter:t,isOpen:!!t,onClose:u,relatedPresenters:p,onViewRelated:y})]})},Nf=[{value:"name",label:"Name (A-Z)"},{value:"sessions",label:"Most Sessions"},{value:"workshops",label:"Most Workshops"},{value:"recent",label:"Recently Active"},{value:"years",label:"Years Active"}],Ef=[{value:"all",label:"All Presenters"},{value:"enriched",label:"Faculty Members"},{value:"recent",label:"Recently Active"},{value:"experienced",label:"Experienced (10+ sessions)"}];function jf(){const[e,t]=T.useState(""),[i,o]=T.useState(null),[n,a]=T.useState("sessions"),[s,l]=T.useState("all"),[c,p]=T.useState(""),y=T.useMemo(()=>{const d=as();return rs(d)},[]),u=T.useMemo(()=>Object.values(y),[y]),g=T.useMemo(()=>{const d=new Set;return u.forEach(m=>{m.expertise.primaryAreas.forEach(f=>d.add(f)),m.expertise.techniques.forEach(f=>d.add(f))}),Array.from(d).sort()},[u]),b=T.useMemo(()=>{let d=u;if(e){const m=e.toLowerCase();d=d.filter(f=>{var S,k,A;return f.name.toLowerCase().includes(m)||((A=(k=(S=f.enrichment)==null?void 0:S.professional)==null?void 0:k.affiliation)==null?void 0:A.toLowerCase().includes(m))||f.expertise.primaryAreas.some(M=>M.toLowerCase().includes(m))||f.expertise.techniques.some(M=>M.toLowerCase().includes(m))})}switch(c&&(d=Qd(c,y)),s){case"enriched":d=d.filter(m=>!!m.enrichment);break;case"recent":d=d.filter(m=>m.teaching.recentActivity);break;case"experienced":d=d.filter(m=>m.teaching.totalSessions>=10);break}return d.sort((m,f)=>{switch(n){case"name":return m.name.localeCompare(f.name);case"sessions":return f.teaching.totalSessions-m.teaching.totalSessions;case"workshops":return f.teaching.workshopsParticipated.length-m.teaching.workshopsParticipated.length;case"recent":return(f.teaching.recentActivity?1:0)-(m.teaching.recentActivity?1:0);case"years":return f.teaching.lastTaught-f.teaching.firstTaught-(m.teaching.lastTaught-m.teaching.firstTaught);default:return 0}}),d},[u,e,c,s,n,y]),w=T.useMemo(()=>i?Xd(i,y,5):[],[i,y]),v=T.useMemo(()=>{const d=u.filter(S=>!!S.enrichment).length,m=u.filter(S=>S.teaching.recentActivity).length,f=Math.round(u.reduce((S,k)=>S+k.teaching.totalSessions,0)/u.length);return{total:u.length,enriched:d,recentlyActive:m,averageSessions:f}},[u]),x=d=>{o(d)},h=()=>{o(null)};return r.jsxs("div",{className:"min-h-screen bg-gray-50",children:[r.jsx("div",{className:"bg-gradient-to-r from-primary-600 to-primary-700 text-white",children:r.jsxs("div",{className:"max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-12",children:[r.jsx("h1",{className:"text-4xl font-bold mb-4",children:"Workshop Presenters"}),r.jsx("p",{className:"text-xl text-primary-100 mb-6",children:"Discover the experts who shape Evomics education"}),r.jsxs("div",{className:"grid grid-cols-2 md:grid-cols-4 gap-4",children:[r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-4 text-center",children:[r.jsx("div",{className:"text-2xl font-bold",children:v.total}),r.jsx("div",{className:"text-sm text-primary-200",children:"Total Presenters"})]}),r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-4 text-center",children:[r.jsx("div",{className:"text-2xl font-bold",children:v.enriched}),r.jsx("div",{className:"text-sm text-primary-200",children:"Faculty Members"})]}),r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-4 text-center",children:[r.jsx("div",{className:"text-2xl font-bold",children:v.recentlyActive}),r.jsx("div",{className:"text-sm text-primary-200",children:"Recently Active"})]}),r.jsxs("div",{className:"bg-primary-500 bg-opacity-50 rounded-lg p-4 text-center",children:[r.jsx("div",{className:"text-2xl font-bold",children:v.averageSessions}),r.jsx("div",{className:"text-sm text-primary-200",children:"Avg. Sessions"})]})]})]})}),r.jsxs("div",{className:"max-w-7xl mx-auto px-4 sm:px-6 lg:px-8 py-8",children:[r.jsxs("div",{className:"bg-white rounded-lg shadow-md p-6 mb-8",children:[r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4",children:[r.jsxs("div",{children:[r.jsx("label",{htmlFor:"search",className:"block text-sm font-medium text-gray-700 mb-2",children:"Search Presenters"}),r.jsx("input",{type:"text",id:"search",placeholder:"Search by name, institution, or expertise...",value:e,onChange:d=>t(d.target.value),className:"w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-primary-500 focus:border-transparent"})]}),r.jsxs("div",{children:[r.jsx("label",{htmlFor:"expertise",className:"block text-sm font-medium text-gray-700 mb-2",children:"Filter by Expertise"}),r.jsxs("select",{id:"expertise",value:c,onChange:d=>p(d.target.value),className:"w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-primary-500 focus:border-transparent",children:[r.jsx("option",{value:"",children:"All Areas"}),g.map(d=>r.jsx("option",{value:d,children:d},d))]})]}),r.jsxs("div",{children:[r.jsx("label",{htmlFor:"filter",className:"block text-sm font-medium text-gray-700 mb-2",children:"Filter Category"}),r.jsx("select",{id:"filter",value:s,onChange:d=>l(d.target.value),className:"w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-primary-500 focus:border-transparent",children:Ef.map(d=>r.jsx("option",{value:d.value,children:d.label},d.value))})]}),r.jsxs("div",{children:[r.jsx("label",{htmlFor:"sort",className:"block text-sm font-medium text-gray-700 mb-2",children:"Sort By"}),r.jsx("select",{id:"sort",value:n,onChange:d=>a(d.target.value),className:"w-full px-3 py-2 border border-gray-300 rounded-md focus:outline-none focus:ring-2 focus:ring-primary-500 focus:border-transparent",children:Nf.map(d=>r.jsx("option",{value:d.value,children:d.label},d.value))})]})]}),r.jsx("div",{className:"mt-4 pt-4 border-t border-gray-200",children:r.jsxs("p",{className:"text-sm text-gray-600",children:["Showing ",b.length," of ",u.length," presenters",e&&` matching "${e}"`,c&&` with expertise in "${c}"`]})})]}),r.jsx(Mf,{presenters:b,onViewProfile:x,showTeachingStats:!0,emptyMessage:e||c||s!=="all"?"No presenters match your current filters":"No presenters found"}),!e&&!c&&s==="all"&&r.jsxs("div",{className:"mt-12",children:[r.jsxs("div",{className:"text-center mb-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-4",children:"Most Active Presenters"}),r.jsx("p",{className:"text-gray-600",children:"Presenters with the most workshop sessions taught"})]}),r.jsx("div",{className:"grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6",children:u.filter(d=>d.teaching.totalSessions>=5).slice(0,6).map(d=>r.jsx(Yd,{presenter:d,onViewProfile:x,showTeachingStats:!0},d.id))})]})]}),r.jsx(ep,{presenter:i,isOpen:!!i,onClose:h,relatedPresenters:w,onViewRelated:x})]})}const zf=()=>r.jsxs("div",{className:"max-w-4xl mx-auto space-y-8",children:[r.jsxs("div",{className:"text-center",children:[r.jsx("h1",{className:"text-3xl font-bold text-gray-900 mb-4",children:"About the Workshop Archive"}),r.jsx("p",{className:"text-xl text-gray-600",children:"Preserving 15+ years of genomics education excellence"})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Our Mission"}),r.jsx("p",{className:"text-gray-700 text-lg leading-relaxed mb-6",children:"The Evomics Workshop Archive serves as a comprehensive repository of educational content, curriculum development, and faculty contributions across the world-renowned Evomics workshop series. Our goal is to preserve and share the knowledge accumulated through years of intensive genomics education."}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-3 gap-6",children:[r.jsxs("div",{className:"text-center p-4",children:[r.jsx("div",{className:"w-16 h-16 bg-blue-100 rounded-full flex items-center justify-center mx-auto mb-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"font-semibold text-gray-900 mb-2",children:"Preserve Knowledge"}),r.jsx("p",{className:"text-gray-600 text-sm",children:"Documenting educational innovations and curriculum evolution"})]}),r.jsxs("div",{className:"text-center p-4",children:[r.jsx("div",{className:"w-16 h-16 bg-green-100 rounded-full flex items-center justify-center mx-auto mb-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"font-semibold text-gray-900 mb-2",children:"Enable Discovery"}),r.jsx("p",{className:"text-gray-600 text-sm",children:"Making workshop content searchable and accessible"})]}),r.jsxs("div",{className:"text-center p-4",children:[r.jsx("div",{className:"w-16 h-16 bg-purple-100 rounded-full flex items-center justify-center mx-auto mb-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"font-semibold text-gray-900 mb-2",children:"Connect Community"}),r.jsx("p",{className:"text-gray-600 text-sm",children:"Linking faculty, students, and educational resources"})]})]})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"What's in the Archive"}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 gap-8",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Workshop Schedules"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsx("li",{children:" Complete session schedules from 2011-2025"}),r.jsx("li",{children:" Detailed timing and location information"}),r.jsx("li",{children:" Session types and learning objectives"}),r.jsx("li",{children:" Historical evolution of curriculum"})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Faculty Contributions"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsx("li",{children:" Teaching histories for 149+ faculty members"}),r.jsx("li",{children:" Presenter and co-presenter information"}),r.jsx("li",{children:" Cross-links to faculty directory profiles"}),r.jsx("li",{children:" Collaboration patterns and networks"})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Workshop Series"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsx("li",{children:" Workshop on Genomics (WoG)"}),r.jsx("li",{children:" Population & Speciation Genomics (WPSG)"}),r.jsx("li",{children:" Workshop on Phylogenomics (WPhylo)"}),r.jsx("li",{children:" Historical series (Molecular Evolution, Harvard workshops)"})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Analytics & Insights"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsx("li",{children:" Curriculum evolution tracking"}),r.jsx("li",{children:" Teaching load distribution"}),r.jsx("li",{children:" Topic popularity trends"}),r.jsx("li",{children:" Geographic and temporal patterns"})]})]})]})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Technical Implementation"}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 gap-8",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Technology Stack"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsxs("li",{children:[" ",r.jsx("strong",{children:"Frontend:"})," React 18 + TypeScript"]}),r.jsxs("li",{children:[" ",r.jsx("strong",{children:"Build Tool:"})," Vite for fast development"]}),r.jsxs("li",{children:[" ",r.jsx("strong",{children:"Styling:"})," Tailwind CSS v3"]}),r.jsxs("li",{children:[" ",r.jsx("strong",{children:"Deployment:"})," GitHub Pages"]}),r.jsxs("li",{children:[" ",r.jsx("strong",{children:"Data:"})," Static JSON with processing utilities"]})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Features"}),r.jsxs("ul",{className:"space-y-2 text-gray-700",children:[r.jsx("li",{children:" Advanced search and filtering"}),r.jsx("li",{children:" Timeline visualization"}),r.jsx("li",{children:" Responsive design"}),r.jsx("li",{children:" Cross-site navigation"}),r.jsx("li",{children:" Performance optimized"})]})]})]}),r.jsx("div",{className:"mt-6 p-4 bg-gray-50 rounded-lg",children:r.jsxs("p",{className:"text-gray-600 text-sm",children:[r.jsx("strong",{children:"Open Source:"})," This archive is part of the open-source Evomics ecosystem. View the source code and contribute on"," ",r.jsx("a",{href:"https://github.com/shandley/evomics-faculty",target:"_blank",rel:"noopener noreferrer",className:"text-blue-600 hover:text-blue-800",children:"GitHub"}),"."]})})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Evomics Ecosystem"}),r.jsx("p",{className:"text-gray-700 mb-6",children:"The Workshop Archive is part of a comprehensive three-site ecosystem documenting the Evomics community:"}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-3 gap-6",children:[r.jsxs("a",{href:"https://shandley.github.io/evomics-faculty/",target:"_blank",rel:"noopener noreferrer",className:"border border-gray-200 rounded-lg p-6 hover:shadow-md transition-shadow group",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-blue-100 rounded-lg flex items-center justify-center mr-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-lg font-bold text-gray-900 group-hover:text-blue-600",children:"Faculty Directory"})]}),r.jsx("p",{className:"text-gray-600 text-sm",children:"184 faculty profiles with teaching histories, research areas, and professional details."})]}),r.jsxs("div",{className:"border border-blue-500 bg-blue-50 rounded-lg p-6",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-blue-200 rounded-lg flex items-center justify-center mr-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-lg font-bold text-blue-900",children:"Workshop Archive"})]}),r.jsx("p",{className:"text-blue-800 text-sm",children:"You are here! Complete curriculum archive with 580+ sessions and teaching materials."})]}),r.jsxs("a",{href:"https://shandley.github.io/evomics-students/",target:"_blank",rel:"noopener noreferrer",className:"border border-gray-200 rounded-lg p-6 hover:shadow-md transition-shadow group",children:[r.jsxs("div",{className:"flex items-center mb-4",children:[r.jsx("div",{className:"w-12 h-12 bg-purple-100 rounded-lg flex items-center justify-center mr-4",children:r.jsx("span",{className:"text-2xl",children:""})}),r.jsx("h3",{className:"text-lg font-bold text-gray-900 group-hover:text-purple-600",children:"Student Alumni"})]}),r.jsx("p",{className:"text-gray-600 text-sm",children:"1,411+ student alumni with career tracking and outcomes analysis."})]})]})]}),r.jsxs("div",{className:"bg-white rounded-lg shadow-lg p-8",children:[r.jsx("h2",{className:"text-2xl font-bold text-gray-900 mb-6",children:"Contact & Feedback"}),r.jsxs("div",{className:"grid grid-cols-1 md:grid-cols-2 gap-8",children:[r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Get in Touch"}),r.jsx("p",{className:"text-gray-700 mb-4",children:"Have suggestions, corrections, or additional content to contribute? We'd love to hear from you."}),r.jsxs("div",{className:"space-y-2",children:[r.jsxs("p",{className:"text-gray-600",children:[r.jsx("strong",{children:"Email:"})," ",r.jsx("a",{href:"mailto:fourthculture@gmail.com",className:"text-blue-600 hover:text-blue-800",children:"fourthculture@gmail.com"})]}),r.jsxs("p",{className:"text-gray-600",children:[r.jsx("strong",{children:"GitHub:"})," ",r.jsx("a",{href:"https://github.com/shandley/evomics-faculty/issues",target:"_blank",rel:"noopener noreferrer",className:"text-blue-600 hover:text-blue-800",children:"Submit an Issue"})]})]})]}),r.jsxs("div",{children:[r.jsx("h3",{className:"text-lg font-semibold text-gray-900 mb-4",children:" Quick Start"}),r.jsx("p",{className:"text-gray-700 mb-4",children:"Ready to explore? Here are some great starting points:"}),r.jsxs("div",{className:"space-y-2",children:[r.jsx(Z,{to:"/dashboard",className:"block text-blue-600 hover:text-blue-800",children:" View Archive Dashboard"}),r.jsx(Z,{to:"/sessions",className:"block text-blue-600 hover:text-blue-800",children:" Browse All Sessions"}),r.jsx(Z,{to:"/timeline",className:"block text-blue-600 hover:text-blue-800",children:" Explore Timeline View"})]})]})]})]}),r.jsx("div",{className:"text-center py-8",children:r.jsxs("p",{className:"text-gray-500 text-sm",children:["Built with  for the genomics education community. Part of the broader ",r.jsx("a",{href:"https://evomics.org",target:"_blank",rel:"noopener noreferrer",className:"text-blue-600 hover:text-blue-800",children:"evomics.org"})," mission."]})})]});function Bf(){return r.jsx(Jh,{basename:"/evomics-workshops",children:r.jsx(zh,{children:r.jsxs(xh,{children:[r.jsx(et,{path:"/",element:r.jsx(Sh,{to:"/dashboard",replace:!0})}),r.jsx(et,{path:"/dashboard",element:r.jsx(Sf,{})}),r.jsx(et,{path:"/timeline",element:r.jsx(Pf,{})}),r.jsx(et,{path:"/sessions",element:r.jsx(If,{})}),r.jsx(et,{path:"/sessions/:id",element:r.jsx(Jf,{})}),r.jsx(et,{path:"/presenters",element:r.jsx(jf,{})}),r.jsx(et,{path:"/about",element:r.jsx(zf,{})})]})})})}Ld(document.getElementById("root")).render(r.jsx(T.StrictMode,{children:r.jsx(Bf,{})}));
